# Cross-Species Pathway Analysis Report for Caenorhabditis elegans

This report summarizes the BLAST search results for pathway enzymes.

## Run Parameters

- Report generated on: 2025-06-27 18:59:27
- Species: Caenorhabditis elegans (Tax ID: 6239)
- Number of threads used for BLAST: 8
- Query File: input/human_glycosylation_pathway_proteins.fasta
- BLAST program: blastp
- BLAST output format: 6 std qlen slen (Query ID, Subject ID, % Identity, Alignment Length, Mismatches, Gap Opens, Query Start, Query End, Subject Start, Subject End, E-value, Bit Score, Query Length, Subject Length)

### BLAST Parameters

- E-value cutoff: 0.001
- Max target sequences: 10
- Word size: 3
- Scoring matrix: BLOSUM62
- Gap open penalty: 11
- Gap extension penalty: 1

## BLAST Results for Caenorhabditis elegans (Tax ID: 6239)

### ABCA6 (Q8N139)

**Description:** ATP-binding cassette sub-family A member 6

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_499115.2 | 23.82 | 1616 | 5.9e-96 | 343 |
| NP_001255012.1 | 23.80 | 1618 | 7.1e-96 | 343 |
| NP_001021225.1 | 23.80 | 1618 | 1.0e-95 | 342 |
| NP_001366984.1 | 23.71 | 1632 | 1.3e-94 | 339 |
| NP_503175.1 | 29.22 | 664 | 3.4e-79 | 290 |
| NP_503175.1 | 24.34 | 645 | 3.9e-47 | 186 |
| NP_503175.1 | 26.28 | 624 | 2.0e-39 | 161 |
| NP_503175.1 | 30.27 | 327 | 1.1e-38 | 159 |
| NP_001364572.1 | 24.56 | 1189 | 1.6e-76 | 281 |
| NP_001364572.1 | 24.34 | 530 | 5.0e-40 | 162 |
| NP_001368466.1 | 30.43 | 552 | 2.7e-53 | 206 |
| NP_001368466.1 | 25.14 | 688 | 2.9e-46 | 183 |
| NP_001368466.1 | 28.52 | 263 | 5.2e-34 | 143 |
| NP_001368466.1 | 35.71 | 182 | 1.3e-30 | 132 |
| NP_490949.3 | 30.43 | 552 | 3.6e-53 | 206 |
| NP_490949.3 | 25.14 | 688 | 3.0e-46 | 183 |
| NP_490949.3 | 28.52 | 263 | 5.6e-34 | 143 |
| NP_490949.3 | 35.71 | 182 | 1.5e-30 | 132 |
| NP_502352.1 | 29.30 | 256 | 1.6e-22 | 104 |
| NP_502352.1 | 21.46 | 219 | 6.4e-08 | 57 |
| NP_001367851.1 | 27.33 | 300 | 2.3e-19 | 94 |
| NP_001367851.1 | 23.79 | 206 | 7.2e-10 | 63 |

### ABL1 (P00519)

**Description:** Tyrosine-protein kinase ABL1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_509777.2 | 53.04 | 560 | 0.0e+00 | 619 |
| NP_001368775.1 | 53.04 | 560 | 0.0e+00 | 619 |
| NP_509779.2 | 56.32 | 506 | 0.0e+00 | 618 |
| NP_509778.2 | 53.04 | 560 | 0.0e+00 | 618 |
| NP_001368776.1 | 53.04 | 560 | 0.0e+00 | 618 |
| NP_001407513.1 | 56.32 | 506 | 0.0e+00 | 618 |
| NP_001407512.1 | 56.32 | 506 | 0.0e+00 | 618 |
| NP_001293421.1 | 43.65 | 449 | 2.3e-112 | 358 |
| NP_001364774.1 | 43.65 | 449 | 3.3e-112 | 359 |
| NP_001364829.1 | 43.65 | 449 | 4.8e-112 | 359 |

### ABL2 (P42684)

**Description:** Tyrosine-protein kinase ABL2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_509777.2 | 58.75 | 463 | 0.0e+00 | 600 |
| NP_001368775.1 | 58.75 | 463 | 0.0e+00 | 599 |
| NP_509779.2 | 58.75 | 463 | 0.0e+00 | 599 |
| NP_509778.2 | 58.75 | 463 | 0.0e+00 | 599 |
| NP_001407513.1 | 58.75 | 463 | 0.0e+00 | 599 |
| NP_001407512.1 | 58.75 | 463 | 0.0e+00 | 599 |
| NP_001368776.1 | 58.75 | 463 | 0.0e+00 | 599 |
| NP_001364829.1 | 43.80 | 452 | 1.4e-110 | 356 |
| NP_001364774.1 | 43.80 | 452 | 1.6e-110 | 356 |
| NP_001293421.1 | 43.80 | 452 | 1.9e-110 | 353 |

### ADAMTS10 (Q9H324)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 10

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001359558.1 | 30.07 | 1084 | 9.9e-128 | 431 |
| NP_001359558.1 | 34.60 | 211 | 2.7e-22 | 104 |
| NP_001359558.1 | 29.07 | 258 | 4.3e-22 | 103 |
| NP_001359558.1 | 30.49 | 246 | 7.3e-22 | 102 |
| NP_001359558.1 | 29.92 | 244 | 1.6e-21 | 101 |
| NP_001359558.1 | 33.85 | 195 | 1.0e-20 | 99 |
| NP_001359558.1 | 28.41 | 257 | 2.8e-20 | 97 |
| NP_001359558.1 | 29.75 | 279 | 9.8e-18 | 89 |
| NP_001255448.1 | 29.75 | 1099 | 2.1e-124 | 421 |
| NP_001255448.1 | 34.60 | 211 | 2.9e-22 | 104 |
| NP_001255448.1 | 29.07 | 258 | 4.8e-22 | 103 |
| NP_001255448.1 | 30.49 | 246 | 8.5e-22 | 102 |
| NP_001255448.1 | 29.92 | 244 | 1.7e-21 | 101 |
| NP_001255448.1 | 33.85 | 195 | 1.1e-20 | 99 |
| NP_001255448.1 | 28.41 | 257 | 3.4e-20 | 97 |
| NP_001255448.1 | 29.75 | 279 | 1.2e-17 | 89 |
| NP_505017.1 | 35.27 | 499 | 7.2e-81 | 291 |
| NP_505017.1 | 31.43 | 245 | 3.2e-21 | 100 |
| NP_505017.1 | 26.92 | 260 | 3.2e-16 | 84 |
| NP_505017.1 | 27.78 | 90 | 2.4e-04 | 45 |
| NP_741569.1 | 35.27 | 499 | 7.6e-81 | 290 |
| NP_741569.1 | 30.86 | 243 | 4.0e-21 | 100 |
| NP_741569.1 | 26.92 | 260 | 6.5e-16 | 83 |
| NP_741569.1 | 27.17 | 92 | 2.6e-04 | 45 |
| NP_001370555.1 | 35.27 | 499 | 7.7e-81 | 290 |
| NP_001370555.1 | 30.86 | 243 | 3.8e-21 | 100 |
| NP_001370555.1 | 26.92 | 260 | 6.1e-16 | 83 |
| NP_001370555.1 | 27.17 | 92 | 2.6e-04 | 45 |
| NP_510116.2 | 25.89 | 730 | 4.8e-39 | 158 |
| NP_510116.2 | 30.67 | 225 | 6.3e-09 | 60 |
| NP_510116.2 | 35.71 | 84 | 3.2e-07 | 54 |
| NP_510116.2 | 28.41 | 257 | 3.0e-06 | 51 |
| NP_510116.2 | 23.18 | 233 | 7.7e-05 | 47 |
| NP_510116.2 | 24.19 | 248 | 1.4e-04 | 46 |
| NP_510116.2 | 24.78 | 226 | 2.5e-04 | 45 |
| NP_510116.2 | 40.00 | 55 | 8.6e-04 | 43 |
| NP_001361951.1 | 25.86 | 727 | 8.3e-39 | 157 |
| NP_001361951.1 | 30.67 | 225 | 5.9e-09 | 60 |
| NP_001361951.1 | 35.71 | 84 | 3.2e-07 | 54 |
| NP_001361951.1 | 28.41 | 257 | 2.9e-06 | 51 |
| NP_001361951.1 | 23.18 | 233 | 7.7e-05 | 47 |
| NP_001361951.1 | 24.19 | 248 | 1.5e-04 | 46 |
| NP_001361951.1 | 24.78 | 226 | 2.4e-04 | 45 |
| NP_001361951.1 | 40.00 | 55 | 8.9e-04 | 43 |
| NP_001361970.1 | 25.83 | 724 | 2.0e-36 | 149 |
| NP_001361970.1 | 26.89 | 212 | 3.3e-07 | 54 |
| NP_001361970.1 | 28.29 | 152 | 5.8e-07 | 53 |
| NP_001361970.1 | 28.44 | 211 | 8.9e-07 | 53 |
| NP_001361970.1 | 37.70 | 61 | 3.4e-04 | 44 |
| NP_001024532.2 | 25.83 | 724 | 4.4e-36 | 148 |
| NP_001024532.2 | 29.05 | 210 | 8.1e-11 | 66 |
| NP_001024532.2 | 28.44 | 211 | 1.1e-06 | 53 |
| NP_001024532.2 | 37.70 | 61 | 4.2e-04 | 44 |
| NP_508681.3 | 25.90 | 610 | 8.7e-35 | 144 |
| NP_508681.3 | 28.19 | 149 | 2.3e-05 | 48 |

### ADAMTS12 (P58397)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 12

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001359558.1 | 32.92 | 1039 | 5.4e-127 | 440 |
| NP_001359558.1 | 31.06 | 235 | 1.2e-24 | 112 |
| NP_001359558.1 | 30.59 | 255 | 7.1e-24 | 110 |
| NP_001359558.1 | 27.71 | 249 | 1.3e-23 | 109 |
| NP_001359558.1 | 32.46 | 228 | 7.2e-23 | 107 |
| NP_001359558.1 | 34.78 | 230 | 1.9e-22 | 105 |
| NP_001359558.1 | 27.71 | 267 | 1.1e-20 | 99 |
| NP_001359558.1 | 35.56 | 180 | 1.8e-20 | 99 |
| NP_001359558.1 | 35.90 | 195 | 3.1e-19 | 95 |
| NP_001359558.1 | 33.51 | 191 | 1.2e-18 | 93 |
| NP_001359558.1 | 32.39 | 176 | 5.1e-18 | 90 |
| NP_001359558.1 | 32.14 | 224 | 1.3e-17 | 89 |
| NP_001359558.1 | 30.22 | 225 | 4.4e-16 | 84 |
| NP_001359558.1 | 32.70 | 159 | 4.4e-15 | 81 |
| NP_001359558.1 | 29.66 | 209 | 8.8e-15 | 80 |
| NP_001359558.1 | 28.41 | 183 | 2.6e-14 | 79 |
| NP_001359558.1 | 33.52 | 179 | 3.1e-14 | 78 |
| NP_001359558.1 | 33.99 | 153 | 4.5e-14 | 78 |
| NP_001359558.1 | 30.94 | 181 | 5.1e-13 | 74 |
| NP_001359558.1 | 30.98 | 184 | 4.4e-12 | 71 |
| NP_001359558.1 | 26.70 | 221 | 6.3e-09 | 61 |
| NP_001359558.1 | 28.02 | 182 | 7.9e-09 | 60 |
| NP_001255448.1 | 32.45 | 1054 | 7.8e-124 | 430 |
| NP_001255448.1 | 31.06 | 235 | 1.0e-24 | 113 |
| NP_001255448.1 | 30.59 | 255 | 5.8e-24 | 110 |
| NP_001255448.1 | 27.71 | 249 | 1.1e-23 | 109 |
| NP_001255448.1 | 32.46 | 228 | 5.8e-23 | 107 |
| NP_001255448.1 | 34.78 | 230 | 1.7e-22 | 105 |
| NP_001255448.1 | 27.71 | 267 | 9.2e-21 | 100 |
| NP_001255448.1 | 35.56 | 180 | 1.6e-20 | 99 |
| NP_001255448.1 | 35.90 | 195 | 2.7e-19 | 95 |
| NP_001255448.1 | 33.51 | 191 | 1.1e-18 | 93 |
| NP_001255448.1 | 32.39 | 176 | 4.6e-18 | 91 |
| NP_001255448.1 | 32.14 | 224 | 1.1e-17 | 90 |
| NP_001255448.1 | 30.22 | 225 | 3.9e-16 | 84 |
| NP_001255448.1 | 32.70 | 159 | 4.1e-15 | 81 |
| NP_001255448.1 | 29.66 | 209 | 8.2e-15 | 80 |
| NP_001255448.1 | 28.41 | 183 | 2.5e-14 | 79 |
| NP_001255448.1 | 33.52 | 179 | 2.8e-14 | 79 |
| NP_001255448.1 | 33.99 | 153 | 4.2e-14 | 78 |
| NP_001255448.1 | 30.94 | 181 | 4.5e-13 | 74 |
| NP_001255448.1 | 30.98 | 184 | 4.0e-12 | 71 |
| NP_001255448.1 | 26.70 | 221 | 5.8e-09 | 61 |
| NP_001255448.1 | 28.02 | 182 | 6.7e-09 | 61 |
| NP_505017.1 | 36.32 | 446 | 2.3e-73 | 271 |
| NP_505017.1 | 29.84 | 258 | 8.7e-17 | 87 |
| NP_505017.1 | 30.22 | 182 | 4.3e-14 | 78 |
| NP_505017.1 | 26.48 | 287 | 7.7e-13 | 73 |
| NP_505017.1 | 26.94 | 193 | 5.0e-11 | 68 |
| NP_505017.1 | 28.39 | 243 | 8.7e-11 | 67 |
| NP_505017.1 | 28.23 | 124 | 8.9e-06 | 50 |
| NP_741569.1 | 36.32 | 446 | 3.2e-73 | 270 |
| NP_741569.1 | 29.84 | 258 | 2.2e-16 | 85 |
| NP_741569.1 | 30.22 | 182 | 1.0e-13 | 76 |
| NP_741569.1 | 26.48 | 287 | 1.9e-12 | 72 |
| NP_741569.1 | 30.00 | 140 | 7.3e-11 | 67 |
| NP_741569.1 | 29.13 | 230 | 2.9e-10 | 65 |
| NP_741569.1 | 28.23 | 124 | 1.2e-05 | 50 |
| NP_001370555.1 | 36.32 | 446 | 3.9e-73 | 270 |
| NP_001370555.1 | 29.84 | 258 | 2.6e-16 | 85 |
| NP_001370555.1 | 30.22 | 182 | 1.1e-13 | 76 |
| NP_001370555.1 | 26.60 | 297 | 2.1e-12 | 72 |
| NP_001370555.1 | 30.00 | 140 | 8.0e-11 | 67 |
| NP_001370555.1 | 29.13 | 230 | 3.3e-10 | 65 |
| NP_001370555.1 | 28.23 | 124 | 1.2e-05 | 50 |
| NP_001361951.1 | 24.52 | 942 | 2.2e-46 | 183 |
| NP_001361951.1 | 32.14 | 196 | 1.9e-09 | 62 |
| NP_001361951.1 | 30.67 | 225 | 2.7e-09 | 62 |
| NP_001361951.1 | 46.55 | 58 | 5.7e-08 | 57 |
| NP_001361951.1 | 26.96 | 204 | 1.1e-06 | 53 |
| NP_001361951.1 | 46.67 | 45 | 2.3e-05 | 49 |
| NP_001361951.1 | 27.03 | 185 | 1.5e-04 | 46 |
| NP_001361951.1 | 34.69 | 98 | 1.9e-04 | 46 |
| NP_510116.2 | 24.34 | 945 | 2.6e-46 | 183 |
| NP_510116.2 | 32.14 | 196 | 1.8e-09 | 62 |
| NP_510116.2 | 30.67 | 225 | 2.6e-09 | 62 |
| NP_510116.2 | 46.55 | 58 | 5.7e-08 | 57 |
| NP_510116.2 | 26.96 | 204 | 1.1e-06 | 53 |
| NP_510116.2 | 46.67 | 45 | 2.4e-05 | 49 |
| NP_510116.2 | 27.03 | 185 | 1.5e-04 | 46 |
| NP_510116.2 | 34.69 | 98 | 1.9e-04 | 46 |
| NP_001361970.1 | 26.92 | 624 | 2.7e-40 | 162 |
| NP_001361970.1 | 30.93 | 194 | 2.8e-09 | 62 |
| NP_001361970.1 | 30.21 | 235 | 1.1e-06 | 53 |
| NP_001361970.1 | 30.30 | 165 | 4.9e-06 | 51 |
| NP_001361970.1 | 27.98 | 193 | 3.1e-05 | 48 |
| NP_001361970.1 | 27.52 | 109 | 4.8e-04 | 45 |
| NP_001024532.2 | 26.92 | 624 | 4.8e-40 | 162 |
| NP_001024532.2 | 30.95 | 210 | 2.9e-10 | 65 |
| NP_001024532.2 | 30.93 | 194 | 5.0e-09 | 61 |
| NP_001024532.2 | 30.21 | 235 | 2.1e-06 | 52 |
| NP_001024532.2 | 27.98 | 193 | 4.3e-05 | 48 |
| NP_001024532.2 | 28.02 | 182 | 4.9e-05 | 48 |
| NP_001024532.2 | 29.63 | 81 | 6.0e-04 | 44 |
| NP_508681.3 | 25.12 | 613 | 1.9e-36 | 150 |

### ADAMTS13 (Q76LX8)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 13

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001359558.1 | 27.14 | 1190 | 1.2e-96 | 344 |
| NP_001359558.1 | 28.86 | 447 | 1.0e-28 | 125 |
| NP_001359558.1 | 25.53 | 521 | 1.2e-27 | 122 |
| NP_001359558.1 | 22.89 | 533 | 1.6e-25 | 115 |
| NP_001359558.1 | 25.05 | 519 | 8.4e-20 | 96 |
| NP_001359558.1 | 23.55 | 501 | 2.3e-17 | 88 |
| NP_001255448.1 | 26.89 | 1205 | 1.0e-93 | 335 |
| NP_001255448.1 | 28.86 | 447 | 1.2e-28 | 125 |
| NP_001255448.1 | 25.53 | 521 | 1.5e-27 | 122 |
| NP_001255448.1 | 22.89 | 533 | 2.0e-25 | 115 |
| NP_001255448.1 | 25.05 | 519 | 1.1e-19 | 96 |
| NP_001255448.1 | 23.55 | 501 | 2.8e-17 | 88 |
| NP_505017.1 | 26.23 | 709 | 2.9e-46 | 183 |
| NP_505017.1 | 22.68 | 507 | 1.6e-16 | 85 |
| NP_741569.1 | 26.23 | 709 | 6.6e-46 | 181 |
| NP_741569.1 | 22.68 | 507 | 2.9e-16 | 85 |
| NP_001370555.1 | 26.23 | 709 | 7.9e-46 | 181 |
| NP_001370555.1 | 22.68 | 507 | 3.5e-16 | 84 |
| NP_001361951.1 | 25.38 | 796 | 4.3e-37 | 152 |
| NP_001361951.1 | 24.37 | 476 | 1.0e-06 | 53 |
| NP_001361951.1 | 47.27 | 55 | 1.8e-06 | 52 |
| NP_001361951.1 | 39.22 | 51 | 6.9e-04 | 44 |
| NP_510116.2 | 25.38 | 796 | 4.5e-37 | 152 |
| NP_510116.2 | 24.37 | 476 | 1.1e-06 | 53 |
| NP_510116.2 | 47.27 | 55 | 1.8e-06 | 52 |
| NP_510116.2 | 39.22 | 51 | 7.0e-04 | 44 |
| NP_001250933.1 | 27.94 | 451 | 2.3e-28 | 124 |
| NP_001250933.1 | 28.08 | 260 | 1.5e-12 | 72 |
| NP_001250933.1 | 30.66 | 137 | 1.6e-10 | 65 |
| NP_001250933.1 | 28.72 | 195 | 3.0e-10 | 65 |
| NP_001250933.1 | 21.86 | 366 | 1.2e-07 | 56 |
| NP_001250933.1 | 27.38 | 168 | 5.6e-06 | 51 |
| NP_001024532.2 | 24.95 | 505 | 3.5e-27 | 120 |
| NP_001024532.2 | 28.22 | 163 | 9.6e-05 | 47 |
| NP_001024532.2 | 23.34 | 377 | 5.0e-04 | 44 |
| NP_001024532.2 | 27.62 | 105 | 6.2e-04 | 44 |
| NP_001361970.1 | 24.95 | 505 | 3.8e-27 | 120 |
| NP_001361970.1 | 28.22 | 163 | 8.7e-05 | 47 |
| NP_001361970.1 | 27.62 | 105 | 5.4e-04 | 44 |

### ADAMTS14 (Q8WXS8)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 14

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001359558.1 | 28.65 | 953 | 4.5e-95 | 336 |
| NP_001359558.1 | 31.46 | 178 | 3.2e-14 | 78 |
| NP_001359558.1 | 29.47 | 207 | 6.0e-14 | 77 |
| NP_001359558.1 | 30.67 | 150 | 3.4e-13 | 74 |
| NP_001359558.1 | 32.76 | 174 | 1.1e-12 | 73 |
| NP_001359558.1 | 29.77 | 178 | 1.5e-12 | 72 |
| NP_001359558.1 | 31.37 | 153 | 7.1e-12 | 70 |
| NP_001359558.1 | 28.72 | 195 | 5.3e-11 | 67 |
| NP_001359558.1 | 28.29 | 205 | 4.7e-10 | 64 |
| NP_001359558.1 | 25.63 | 199 | 1.1e-09 | 63 |
| NP_001359558.1 | 27.71 | 231 | 2.8e-09 | 62 |
| NP_001359558.1 | 28.81 | 177 | 4.1e-09 | 61 |
| NP_001255448.1 | 28.20 | 968 | 6.4e-92 | 327 |
| NP_001255448.1 | 31.46 | 178 | 2.6e-14 | 78 |
| NP_001255448.1 | 29.47 | 207 | 5.0e-14 | 77 |
| NP_001255448.1 | 30.67 | 150 | 3.1e-13 | 74 |
| NP_001255448.1 | 32.76 | 174 | 9.6e-13 | 73 |
| NP_001255448.1 | 29.77 | 178 | 1.3e-12 | 72 |
| NP_001255448.1 | 31.37 | 153 | 6.1e-12 | 70 |
| NP_001255448.1 | 28.72 | 195 | 4.6e-11 | 67 |
| NP_001255448.1 | 28.29 | 205 | 4.2e-10 | 64 |
| NP_001255448.1 | 25.63 | 199 | 9.9e-10 | 63 |
| NP_001255448.1 | 27.71 | 231 | 2.4e-09 | 62 |
| NP_001255448.1 | 28.81 | 177 | 3.5e-09 | 61 |
| NP_505017.1 | 32.58 | 488 | 4.6e-62 | 233 |
| NP_505017.1 | 27.32 | 183 | 1.1e-13 | 76 |
| NP_001370555.1 | 32.24 | 487 | 1.3e-61 | 231 |
| NP_001370555.1 | 27.32 | 183 | 1.9e-13 | 75 |
| NP_741569.1 | 32.24 | 487 | 1.4e-61 | 231 |
| NP_741569.1 | 27.32 | 183 | 1.7e-13 | 75 |
| NP_001361951.1 | 26.47 | 680 | 3.4e-42 | 169 |
| NP_001361951.1 | 30.54 | 167 | 3.4e-08 | 58 |
| NP_001361951.1 | 34.31 | 102 | 5.2e-07 | 54 |
| NP_001361951.1 | 48.00 | 50 | 6.6e-07 | 53 |
| NP_001361951.1 | 28.10 | 121 | 8.4e-06 | 50 |
| NP_001361951.1 | 27.69 | 195 | 7.2e-05 | 47 |
| NP_001361951.1 | 37.14 | 70 | 1.4e-04 | 46 |
| NP_001361951.1 | 36.84 | 57 | 8.3e-04 | 43 |
| NP_510116.2 | 26.21 | 683 | 2.4e-41 | 166 |
| NP_510116.2 | 30.54 | 167 | 3.4e-08 | 58 |
| NP_510116.2 | 34.31 | 102 | 5.2e-07 | 54 |
| NP_510116.2 | 48.00 | 50 | 6.5e-07 | 53 |
| NP_510116.2 | 28.10 | 121 | 8.3e-06 | 50 |
| NP_510116.2 | 27.69 | 195 | 6.9e-05 | 47 |
| NP_510116.2 | 37.14 | 70 | 1.3e-04 | 46 |
| NP_510116.2 | 36.84 | 57 | 8.4e-04 | 43 |
| NP_508681.3 | 26.34 | 596 | 2.8e-38 | 155 |
| NP_001361970.1 | 25.39 | 697 | 6.1e-33 | 138 |
| NP_001361970.1 | 28.90 | 173 | 6.4e-07 | 53 |
| NP_001361970.1 | 44.07 | 59 | 6.5e-06 | 50 |
| NP_001361970.1 | 31.85 | 157 | 2.9e-05 | 48 |
| NP_001361970.1 | 30.77 | 104 | 5.0e-05 | 47 |
| NP_001024532.2 | 25.39 | 697 | 1.1e-32 | 137 |
| NP_001024532.2 | 28.90 | 173 | 8.7e-07 | 53 |
| NP_001024532.2 | 44.07 | 59 | 7.6e-06 | 50 |
| NP_001024532.2 | 31.85 | 157 | 3.4e-05 | 48 |
| NP_001024532.2 | 30.77 | 104 | 6.5e-05 | 47 |
| NP_001024532.2 | 39.06 | 64 | 4.2e-04 | 44 |

### ADAMTS16 (Q8TE57)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 16

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001359558.1 | 32.21 | 950 | 2.0e-124 | 424 |
| NP_001359558.1 | 31.79 | 324 | 3.5e-28 | 124 |
| NP_001359558.1 | 26.83 | 410 | 1.9e-23 | 108 |
| NP_001359558.1 | 32.03 | 281 | 2.8e-23 | 107 |
| NP_001359558.1 | 28.76 | 306 | 4.6e-20 | 97 |
| NP_001359558.1 | 34.34 | 198 | 5.7e-19 | 93 |
| NP_001255448.1 | 31.71 | 965 | 3.7e-121 | 415 |
| NP_001255448.1 | 31.79 | 324 | 2.5e-28 | 124 |
| NP_001255448.1 | 26.83 | 410 | 1.4e-23 | 108 |
| NP_001255448.1 | 32.03 | 281 | 2.1e-23 | 108 |
| NP_001255448.1 | 28.76 | 306 | 3.2e-20 | 97 |
| NP_001255448.1 | 34.34 | 198 | 4.5e-19 | 94 |
| NP_505017.1 | 31.63 | 607 | 3.1e-71 | 262 |
| NP_505017.1 | 29.54 | 281 | 5.6e-16 | 84 |
| NP_505017.1 | 27.10 | 214 | 2.3e-11 | 68 |
| NP_505017.1 | 31.50 | 127 | 2.1e-07 | 55 |
| NP_741569.1 | 31.63 | 607 | 5.5e-71 | 261 |
| NP_741569.1 | 30.04 | 273 | 9.9e-16 | 82 |
| NP_741569.1 | 27.10 | 214 | 3.7e-11 | 67 |
| NP_741569.1 | 31.50 | 127 | 2.3e-07 | 55 |
| NP_001370555.1 | 31.63 | 607 | 7.0e-71 | 261 |
| NP_001370555.1 | 30.04 | 273 | 1.2e-15 | 82 |
| NP_001370555.1 | 27.10 | 214 | 4.2e-11 | 67 |
| NP_001370555.1 | 31.50 | 127 | 2.4e-07 | 55 |
| NP_001361951.1 | 26.07 | 721 | 3.4e-56 | 214 |
| NP_001361951.1 | 28.95 | 266 | 1.0e-08 | 60 |
| NP_001361951.1 | 25.90 | 278 | 1.3e-08 | 59 |
| NP_001361951.1 | 35.48 | 93 | 4.4e-07 | 54 |
| NP_001361951.1 | 44.83 | 58 | 7.1e-07 | 53 |
| NP_001361951.1 | 30.77 | 143 | 1.8e-06 | 52 |
| NP_001361951.1 | 38.71 | 62 | 2.5e-05 | 48 |
| NP_001361951.1 | 34.44 | 90 | 4.8e-05 | 47 |
| NP_001361951.1 | 24.34 | 304 | 5.3e-05 | 47 |
| NP_001361951.1 | 29.43 | 265 | 3.6e-04 | 45 |
| NP_510116.2 | 26.45 | 726 | 7.6e-56 | 213 |
| NP_510116.2 | 28.95 | 266 | 9.4e-09 | 60 |
| NP_510116.2 | 25.90 | 278 | 1.3e-08 | 59 |
| NP_510116.2 | 35.48 | 93 | 4.2e-07 | 54 |
| NP_510116.2 | 44.83 | 58 | 6.7e-07 | 53 |
| NP_510116.2 | 30.77 | 143 | 1.6e-06 | 52 |
| NP_510116.2 | 38.71 | 62 | 2.4e-05 | 48 |
| NP_510116.2 | 34.44 | 90 | 4.6e-05 | 48 |
| NP_510116.2 | 24.34 | 304 | 5.0e-05 | 47 |
| NP_510116.2 | 29.48 | 268 | 3.4e-04 | 45 |
| NP_508681.3 | 27.72 | 671 | 1.9e-46 | 181 |
| NP_508681.3 | 28.74 | 167 | 1.3e-06 | 52 |
| NP_508681.3 | 29.52 | 166 | 2.8e-04 | 45 |
| NP_508681.3 | 32.00 | 75 | 5.6e-04 | 44 |
| NP_001361970.1 | 27.31 | 725 | 7.0e-43 | 170 |
| NP_001361970.1 | 26.14 | 306 | 9.9e-10 | 63 |
| NP_001361970.1 | 28.85 | 104 | 7.5e-07 | 53 |
| NP_001361970.1 | 40.00 | 60 | 1.6e-04 | 46 |
| NP_001361970.1 | 36.62 | 71 | 3.6e-04 | 45 |
| NP_001024532.2 | 27.31 | 725 | 1.3e-42 | 169 |
| NP_001024532.2 | 26.39 | 341 | 9.1e-10 | 63 |
| NP_001024532.2 | 28.85 | 104 | 9.3e-07 | 53 |
| NP_001024532.2 | 40.00 | 60 | 1.8e-04 | 45 |
| NP_001024532.2 | 36.62 | 71 | 4.3e-04 | 44 |

### ADAMTS17 (Q8TE56)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 17

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001359558.1 | 29.56 | 954 | 6.6e-101 | 351 |
| NP_001359558.1 | 32.86 | 213 | 3.8e-23 | 107 |
| NP_001359558.1 | 31.36 | 271 | 1.8e-22 | 104 |
| NP_001359558.1 | 29.87 | 308 | 1.1e-21 | 102 |
| NP_001359558.1 | 24.64 | 548 | 1.0e-20 | 99 |
| NP_001359558.1 | 31.05 | 248 | 5.8e-20 | 96 |
| NP_001359558.1 | 29.88 | 251 | 4.0e-18 | 90 |
| NP_001359558.1 | 30.70 | 241 | 6.9e-16 | 83 |
| NP_001359558.1 | 29.85 | 134 | 5.1e-08 | 57 |
| NP_001255448.1 | 29.10 | 969 | 1.6e-97 | 341 |
| NP_001255448.1 | 32.86 | 213 | 3.9e-23 | 107 |
| NP_001255448.1 | 31.36 | 271 | 1.9e-22 | 104 |
| NP_001255448.1 | 29.87 | 308 | 1.2e-21 | 102 |
| NP_001255448.1 | 24.64 | 548 | 1.2e-20 | 99 |
| NP_001255448.1 | 31.05 | 248 | 6.0e-20 | 96 |
| NP_001255448.1 | 29.88 | 251 | 4.4e-18 | 90 |
| NP_001255448.1 | 30.70 | 241 | 7.1e-16 | 83 |
| NP_001255448.1 | 29.85 | 134 | 5.4e-08 | 57 |
| NP_505017.1 | 28.52 | 505 | 3.6e-46 | 181 |
| NP_505017.1 | 28.15 | 270 | 8.2e-14 | 76 |
| NP_505017.1 | 42.22 | 90 | 9.5e-10 | 63 |
| NP_505017.1 | 27.33 | 150 | 2.5e-07 | 55 |
| NP_001370555.1 | 29.52 | 437 | 9.7e-46 | 180 |
| NP_001370555.1 | 28.15 | 270 | 1.5e-13 | 75 |
| NP_001370555.1 | 42.22 | 90 | 1.6e-09 | 62 |
| NP_001370555.1 | 27.33 | 150 | 3.8e-07 | 54 |
| NP_741569.1 | 29.52 | 437 | 1.1e-45 | 179 |
| NP_741569.1 | 28.15 | 270 | 1.8e-13 | 75 |
| NP_741569.1 | 42.22 | 90 | 1.6e-09 | 62 |
| NP_741569.1 | 27.33 | 150 | 4.0e-07 | 54 |
| NP_001361951.1 | 25.00 | 660 | 1.7e-42 | 169 |
| NP_001361951.1 | 31.93 | 213 | 5.1e-10 | 63 |
| NP_001361951.1 | 28.80 | 250 | 2.3e-08 | 58 |
| NP_001361951.1 | 47.17 | 53 | 6.6e-07 | 53 |
| NP_001361951.1 | 43.55 | 62 | 2.3e-06 | 52 |
| NP_001361951.1 | 27.46 | 193 | 2.4e-06 | 52 |
| NP_001361951.1 | 27.85 | 237 | 4.9e-06 | 50 |
| NP_001361951.1 | 33.63 | 113 | 9.3e-06 | 50 |
| NP_001361951.1 | 40.00 | 55 | 7.5e-05 | 47 |
| NP_001361951.1 | 44.00 | 50 | 1.8e-04 | 45 |
| NP_001361951.1 | 27.80 | 205 | 6.1e-04 | 43 |
| NP_510116.2 | 24.66 | 657 | 8.7e-42 | 167 |
| NP_510116.2 | 31.93 | 213 | 5.6e-10 | 63 |
| NP_510116.2 | 28.80 | 250 | 2.5e-08 | 58 |
| NP_510116.2 | 47.17 | 53 | 6.8e-07 | 53 |
| NP_510116.2 | 43.55 | 62 | 2.4e-06 | 52 |
| NP_510116.2 | 27.46 | 193 | 2.4e-06 | 52 |
| NP_510116.2 | 27.85 | 237 | 5.1e-06 | 50 |
| NP_510116.2 | 33.63 | 113 | 9.6e-06 | 50 |
| NP_510116.2 | 40.00 | 55 | 7.8e-05 | 47 |
| NP_510116.2 | 44.00 | 50 | 1.9e-04 | 45 |
| NP_510116.2 | 27.80 | 205 | 6.5e-04 | 43 |
| NP_001361970.1 | 26.11 | 678 | 5.8e-37 | 150 |
| NP_001361970.1 | 32.48 | 234 | 2.4e-12 | 71 |
| NP_001361970.1 | 27.98 | 243 | 2.9e-09 | 61 |
| NP_001361970.1 | 30.84 | 107 | 1.3e-05 | 49 |
| NP_001361970.1 | 28.40 | 169 | 1.3e-05 | 49 |
| NP_001361970.1 | 45.45 | 55 | 8.2e-05 | 47 |
| NP_001361970.1 | 37.70 | 61 | 4.8e-04 | 44 |
| NP_001024532.2 | 26.11 | 678 | 8.9e-37 | 150 |
| NP_001024532.2 | 29.46 | 224 | 3.2e-11 | 67 |
| NP_001024532.2 | 27.98 | 243 | 4.3e-09 | 60 |
| NP_001024532.2 | 30.84 | 107 | 1.5e-05 | 49 |
| NP_001024532.2 | 45.45 | 55 | 1.0e-04 | 46 |
| NP_001024532.2 | 37.70 | 61 | 5.1e-04 | 44 |
| NP_001024532.2 | 32.08 | 159 | 8.8e-04 | 43 |
| NP_492405.3 | 27.32 | 538 | 1.1e-34 | 144 |
| NP_492405.3 | 27.97 | 236 | 1.8e-18 | 91 |
| NP_492405.3 | 30.51 | 236 | 3.9e-15 | 80 |
| NP_492405.3 | 26.72 | 247 | 1.5e-12 | 72 |

### ADAMTS18 (Q8TE60)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 18

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001359558.1 | 33.30 | 1000 | 2.4e-140 | 471 |
| NP_001359558.1 | 30.97 | 310 | 3.0e-29 | 127 |
| NP_001359558.1 | 29.47 | 319 | 1.3e-28 | 125 |
| NP_001359558.1 | 31.92 | 307 | 3.4e-27 | 120 |
| NP_001359558.1 | 32.71 | 214 | 1.5e-25 | 115 |
| NP_001359558.1 | 30.45 | 289 | 1.8e-25 | 115 |
| NP_001359558.1 | 30.38 | 293 | 3.5e-24 | 110 |
| NP_001359558.1 | 26.63 | 338 | 1.5e-20 | 99 |
| NP_001359558.1 | 30.43 | 207 | 3.9e-20 | 97 |
| NP_001359558.1 | 28.20 | 266 | 1.8e-17 | 88 |
| NP_001255448.1 | 32.81 | 1015 | 3.8e-137 | 461 |
| NP_001255448.1 | 30.97 | 310 | 2.4e-29 | 127 |
| NP_001255448.1 | 29.47 | 319 | 1.0e-28 | 125 |
| NP_001255448.1 | 31.92 | 307 | 2.5e-27 | 120 |
| NP_001255448.1 | 32.71 | 214 | 1.2e-25 | 115 |
| NP_001255448.1 | 30.45 | 289 | 1.4e-25 | 115 |
| NP_001255448.1 | 30.38 | 293 | 2.7e-24 | 111 |
| NP_001255448.1 | 26.63 | 338 | 1.2e-20 | 99 |
| NP_001255448.1 | 30.43 | 207 | 3.2e-20 | 97 |
| NP_001255448.1 | 28.20 | 266 | 1.5e-17 | 89 |
| NP_001370555.1 | 30.86 | 674 | 6.2e-80 | 289 |
| NP_505017.1 | 30.86 | 674 | 7.0e-80 | 289 |
| NP_741569.1 | 30.86 | 674 | 1.1e-79 | 288 |
| NP_510116.2 | 25.21 | 1174 | 2.9e-65 | 243 |
| NP_510116.2 | 26.84 | 272 | 2.2e-07 | 55 |
| NP_510116.2 | 24.92 | 297 | 8.5e-07 | 53 |
| NP_510116.2 | 26.51 | 215 | 8.8e-05 | 47 |
| NP_001361951.1 | 25.09 | 1172 | 1.2e-64 | 241 |
| NP_001361951.1 | 26.84 | 272 | 2.4e-07 | 55 |
| NP_001361951.1 | 24.92 | 297 | 8.4e-07 | 53 |
| NP_001361951.1 | 26.51 | 215 | 9.4e-05 | 47 |
| NP_508681.3 | 31.55 | 485 | 1.9e-46 | 181 |
| NP_508681.3 | 33.76 | 157 | 1.0e-08 | 59 |
| NP_508681.3 | 30.07 | 153 | 5.7e-06 | 50 |
| NP_001361970.1 | 25.80 | 717 | 2.3e-39 | 159 |
| NP_001361970.1 | 27.68 | 289 | 2.0e-14 | 78 |
| NP_001361970.1 | 30.14 | 219 | 4.2e-11 | 67 |
| NP_001361970.1 | 24.91 | 269 | 4.4e-07 | 54 |
| NP_001361970.1 | 31.03 | 145 | 2.0e-05 | 48 |
| NP_001361970.1 | 32.56 | 129 | 4.1e-05 | 48 |
| NP_001361970.1 | 38.33 | 60 | 7.3e-04 | 43 |
| NP_001024532.2 | 25.80 | 717 | 5.0e-39 | 158 |
| NP_001024532.2 | 27.34 | 289 | 3.2e-14 | 77 |
| NP_001024532.2 | 29.39 | 296 | 3.3e-14 | 77 |
| NP_001024532.2 | 32.56 | 129 | 4.8e-05 | 47 |
| NP_001024532.2 | 38.33 | 60 | 8.5e-04 | 43 |

### ADAMTS2 (O95450)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001359558.1 | 31.47 | 931 | 8.6e-110 | 380 |
| NP_001359558.1 | 31.80 | 195 | 1.8e-16 | 85 |
| NP_001359558.1 | 33.68 | 190 | 1.3e-14 | 79 |
| NP_001359558.1 | 32.27 | 189 | 1.8e-14 | 79 |
| NP_001359558.1 | 29.95 | 217 | 4.2e-14 | 77 |
| NP_001359558.1 | 29.47 | 190 | 2.0e-12 | 72 |
| NP_001359558.1 | 30.00 | 200 | 2.9e-12 | 71 |
| NP_001359558.1 | 29.25 | 188 | 8.7e-12 | 70 |
| NP_001359558.1 | 28.50 | 186 | 2.7e-11 | 68 |
| NP_001359558.1 | 31.79 | 151 | 3.3e-11 | 68 |
| NP_001359558.1 | 25.81 | 341 | 1.6e-10 | 65 |
| NP_001359558.1 | 31.32 | 182 | 3.8e-09 | 61 |
| NP_001359558.1 | 29.85 | 134 | 1.5e-06 | 52 |
| NP_001255448.1 | 30.97 | 946 | 9.3e-107 | 371 |
| NP_001255448.1 | 31.80 | 195 | 1.4e-16 | 85 |
| NP_001255448.1 | 33.68 | 190 | 1.1e-14 | 79 |
| NP_001255448.1 | 32.27 | 189 | 1.5e-14 | 79 |
| NP_001255448.1 | 29.95 | 217 | 3.8e-14 | 77 |
| NP_001255448.1 | 29.47 | 190 | 1.7e-12 | 72 |
| NP_001255448.1 | 30.00 | 200 | 2.3e-12 | 72 |
| NP_001255448.1 | 29.10 | 189 | 7.6e-12 | 70 |
| NP_001255448.1 | 28.50 | 186 | 2.2e-11 | 68 |
| NP_001255448.1 | 31.79 | 151 | 3.0e-11 | 68 |
| NP_001255448.1 | 25.81 | 341 | 1.3e-10 | 66 |
| NP_001255448.1 | 31.32 | 182 | 2.9e-09 | 61 |
| NP_001255448.1 | 29.85 | 134 | 1.3e-06 | 53 |
| NP_505017.1 | 33.47 | 487 | 1.7e-65 | 244 |
| NP_505017.1 | 27.60 | 221 | 1.0e-10 | 66 |
| NP_505017.1 | 33.33 | 123 | 1.5e-09 | 62 |
| NP_505017.1 | 31.06 | 132 | 2.4e-09 | 62 |
| NP_505017.1 | 28.23 | 124 | 2.5e-05 | 48 |
| NP_741569.1 | 33.47 | 487 | 2.3e-65 | 243 |
| NP_741569.1 | 27.60 | 221 | 1.6e-10 | 65 |
| NP_741569.1 | 33.33 | 123 | 1.7e-09 | 62 |
| NP_741569.1 | 31.06 | 132 | 3.1e-09 | 61 |
| NP_741569.1 | 28.23 | 124 | 3.1e-05 | 48 |
| NP_001370555.1 | 33.47 | 487 | 2.7e-65 | 243 |
| NP_001370555.1 | 27.60 | 221 | 1.7e-10 | 65 |
| NP_001370555.1 | 33.33 | 123 | 1.8e-09 | 62 |
| NP_001370555.1 | 31.06 | 132 | 3.2e-09 | 61 |
| NP_001370555.1 | 28.23 | 124 | 3.4e-05 | 48 |
| NP_001361951.1 | 24.65 | 929 | 8.5e-45 | 177 |
| NP_001361951.1 | 32.37 | 173 | 8.0e-12 | 70 |
| NP_001361951.1 | 48.21 | 56 | 1.1e-07 | 56 |
| NP_001361951.1 | 30.92 | 152 | 5.0e-06 | 51 |
| NP_001361951.1 | 41.94 | 62 | 1.4e-05 | 49 |
| NP_001361951.1 | 38.46 | 52 | 8.1e-05 | 47 |
| NP_001361951.1 | 25.83 | 151 | 7.0e-04 | 43 |
| NP_510116.2 | 24.38 | 923 | 1.3e-44 | 177 |
| NP_510116.2 | 32.37 | 173 | 8.3e-12 | 70 |
| NP_510116.2 | 48.21 | 56 | 1.1e-07 | 56 |
| NP_510116.2 | 30.92 | 152 | 5.0e-06 | 51 |
| NP_510116.2 | 41.94 | 62 | 1.4e-05 | 49 |
| NP_510116.2 | 38.46 | 52 | 7.9e-05 | 47 |
| NP_510116.2 | 25.83 | 151 | 6.9e-04 | 43 |
| NP_001361970.1 | 25.50 | 647 | 1.8e-37 | 152 |
| NP_001361970.1 | 28.41 | 183 | 1.2e-07 | 56 |
| NP_001361970.1 | 26.98 | 189 | 2.6e-05 | 48 |
| NP_001361970.1 | 28.97 | 107 | 5.3e-05 | 47 |
| NP_001361970.1 | 34.92 | 63 | 2.7e-04 | 45 |
| NP_001024532.2 | 25.50 | 647 | 3.7e-37 | 152 |
| NP_001024532.2 | 30.72 | 153 | 8.4e-08 | 56 |
| NP_001024532.2 | 26.98 | 189 | 3.8e-05 | 48 |
| NP_001024532.2 | 28.97 | 107 | 6.3e-05 | 47 |
| NP_001024532.2 | 35.71 | 70 | 1.2e-04 | 46 |
| NP_001024532.2 | 34.92 | 63 | 3.4e-04 | 45 |
| NP_508681.3 | 24.37 | 595 | 3.3e-34 | 142 |

### ADAMTS20 (P59510)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 20

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001359558.1 | 34.41 | 1668 | 0.0e+00 | 898 |
| NP_001359558.1 | 29.84 | 1126 | 4.4e-145 | 499 |
| NP_001255448.1 | 34.11 | 1683 | 0.0e+00 | 887 |
| NP_001255448.1 | 29.84 | 1126 | 9.3e-145 | 498 |
| NP_505017.1 | 32.10 | 676 | 3.3e-76 | 282 |
| NP_505017.1 | 28.45 | 369 | 8.1e-32 | 136 |
| NP_505017.1 | 27.06 | 377 | 3.1e-27 | 121 |
| NP_505017.1 | 24.85 | 342 | 6.5e-17 | 87 |
| NP_505017.1 | 28.35 | 328 | 1.2e-16 | 87 |
| NP_505017.1 | 26.85 | 257 | 6.5e-11 | 68 |
| NP_001370555.1 | 32.10 | 676 | 5.3e-76 | 280 |
| NP_001370555.1 | 28.61 | 367 | 1.2e-31 | 135 |
| NP_001370555.1 | 26.72 | 393 | 6.6e-27 | 120 |
| NP_001370555.1 | 25.15 | 338 | 2.4e-16 | 85 |
| NP_001370555.1 | 28.35 | 328 | 3.2e-16 | 85 |
| NP_001370555.1 | 26.85 | 257 | 1.1e-10 | 67 |
| NP_741569.1 | 32.10 | 676 | 6.8e-76 | 279 |
| NP_741569.1 | 28.45 | 369 | 1.2e-31 | 135 |
| NP_741569.1 | 26.72 | 393 | 9.0e-27 | 120 |
| NP_741569.1 | 24.85 | 342 | 3.0e-16 | 85 |
| NP_741569.1 | 28.35 | 328 | 3.3e-16 | 85 |
| NP_741569.1 | 26.85 | 257 | 1.2e-10 | 67 |
| NP_001024532.2 | 23.76 | 1052 | 3.2e-40 | 163 |
| NP_001024532.2 | 26.06 | 376 | 2.3e-11 | 69 |
| NP_001024532.2 | 30.17 | 232 | 3.6e-11 | 68 |
| NP_001024532.2 | 26.06 | 353 | 9.6e-11 | 67 |
| NP_001024532.2 | 24.38 | 525 | 4.5e-10 | 65 |
| NP_001024532.2 | 29.41 | 272 | 7.7e-10 | 64 |
| NP_001024532.2 | 25.68 | 331 | 4.6e-07 | 55 |
| NP_001024532.2 | 24.56 | 338 | 8.4e-06 | 50 |
| NP_001024532.2 | 24.76 | 206 | 3.3e-04 | 45 |
| NP_001024532.2 | 32.94 | 85 | 4.8e-04 | 45 |
| NP_492405.3 | 27.01 | 633 | 3.1e-39 | 160 |
| NP_492405.3 | 24.24 | 656 | 9.4e-37 | 152 |
| NP_492405.3 | 27.29 | 480 | 2.7e-31 | 134 |
| NP_492405.3 | 24.44 | 671 | 1.4e-30 | 132 |
| NP_492405.3 | 31.07 | 177 | 5.3e-15 | 81 |
| NP_492405.3 | 24.44 | 315 | 1.4e-14 | 79 |
| NP_492405.3 | 30.09 | 226 | 2.5e-12 | 72 |
| NP_492405.3 | 23.41 | 393 | 5.8e-12 | 71 |
| NP_492405.3 | 29.63 | 189 | 2.9e-10 | 65 |
| NP_492405.3 | 30.70 | 114 | 6.1e-06 | 51 |
| NP_492405.3 | 25.00 | 212 | 9.7e-05 | 47 |
| NP_492405.3 | 28.85 | 104 | 5.5e-04 | 45 |
| NP_510116.2 | 24.55 | 668 | 5.4e-39 | 159 |
| NP_510116.2 | 25.58 | 688 | 3.5e-19 | 95 |
| NP_510116.2 | 25.56 | 489 | 6.3e-18 | 90 |
| NP_510116.2 | 25.66 | 456 | 2.2e-14 | 79 |
| NP_510116.2 | 24.54 | 432 | 3.8e-13 | 75 |
| NP_510116.2 | 22.24 | 697 | 2.0e-09 | 63 |
| NP_001361951.1 | 24.89 | 667 | 2.3e-38 | 157 |
| NP_001361951.1 | 25.58 | 688 | 4.1e-19 | 94 |
| NP_001361951.1 | 25.56 | 489 | 7.7e-18 | 90 |
| NP_001361951.1 | 25.66 | 456 | 2.6e-14 | 79 |
| NP_001361951.1 | 24.54 | 432 | 4.3e-13 | 75 |
| NP_001361951.1 | 22.24 | 697 | 2.5e-09 | 62 |
| NP_001250933.1 | 27.01 | 633 | 5.8e-38 | 155 |
| NP_001250933.1 | 24.65 | 649 | 6.1e-37 | 152 |
| NP_001250933.1 | 27.37 | 475 | 5.1e-31 | 133 |
| NP_001250933.1 | 24.51 | 669 | 9.9e-31 | 132 |
| NP_001250933.1 | 31.07 | 177 | 5.6e-15 | 80 |
| NP_001250933.1 | 24.44 | 315 | 2.6e-13 | 75 |
| NP_001250933.1 | 30.09 | 226 | 2.8e-12 | 72 |
| NP_001250933.1 | 23.53 | 391 | 3.7e-12 | 71 |
| NP_001250933.1 | 29.63 | 189 | 3.0e-10 | 65 |
| NP_001250933.1 | 29.60 | 125 | 4.1e-06 | 52 |
| NP_001250933.1 | 25.00 | 212 | 9.8e-05 | 47 |
| NP_001250933.1 | 28.85 | 104 | 5.6e-04 | 45 |

### ADAMTS4 (O75173)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001359558.1 | 37.77 | 728 | 4.1e-146 | 475 |
| NP_001255448.1 | 37.01 | 743 | 1.2e-142 | 465 |
| NP_505017.1 | 36.24 | 287 | 9.9e-44 | 172 |
| NP_001370555.1 | 36.24 | 287 | 1.6e-43 | 171 |
| NP_741569.1 | 36.24 | 287 | 1.7e-43 | 171 |
| NP_510116.2 | 27.67 | 459 | 2.1e-29 | 126 |
| NP_510116.2 | 45.28 | 53 | 3.0e-06 | 51 |
| NP_510116.2 | 31.66 | 139 | 6.7e-05 | 46 |
| NP_510116.2 | 48.78 | 41 | 5.8e-04 | 43 |
| NP_001361951.1 | 27.67 | 459 | 2.2e-29 | 126 |
| NP_001361951.1 | 45.28 | 53 | 3.1e-06 | 51 |
| NP_001361951.1 | 31.66 | 139 | 6.8e-05 | 46 |
| NP_001361951.1 | 48.78 | 41 | 5.8e-04 | 43 |
| NP_001361970.1 | 24.52 | 681 | 1.3e-28 | 123 |
| NP_001361970.1 | 28.57 | 105 | 1.8e-05 | 48 |
| NP_001361970.1 | 31.82 | 88 | 5.4e-04 | 43 |
| NP_001024532.2 | 24.52 | 681 | 3.0e-28 | 122 |
| NP_001024532.2 | 28.57 | 105 | 2.7e-05 | 48 |
| NP_001024532.2 | 31.82 | 88 | 7.7e-04 | 43 |
| NP_508681.3 | 25.36 | 485 | 1.3e-27 | 120 |
| NP_508681.3 | 36.54 | 52 | 1.0e-03 | 42 |

### ADAMTS6 (Q9UKP5)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 6

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001359558.1 | 32.98 | 955 | 6.8e-135 | 452 |
| NP_001359558.1 | 29.14 | 278 | 5.4e-23 | 106 |
| NP_001359558.1 | 30.42 | 240 | 6.9e-21 | 99 |
| NP_001359558.1 | 29.03 | 248 | 1.1e-20 | 99 |
| NP_001359558.1 | 27.40 | 281 | 1.6e-20 | 98 |
| NP_001359558.1 | 35.10 | 208 | 1.3e-19 | 95 |
| NP_001359558.1 | 30.13 | 229 | 1.1e-18 | 92 |
| NP_001359558.1 | 31.90 | 163 | 1.2e-14 | 79 |
| NP_001255448.1 | 32.47 | 970 | 1.8e-131 | 442 |
| NP_001255448.1 | 29.14 | 278 | 5.7e-23 | 106 |
| NP_001255448.1 | 30.42 | 240 | 7.1e-21 | 99 |
| NP_001255448.1 | 29.03 | 248 | 1.1e-20 | 99 |
| NP_001255448.1 | 27.40 | 281 | 1.7e-20 | 98 |
| NP_001255448.1 | 35.10 | 208 | 1.4e-19 | 95 |
| NP_001255448.1 | 30.13 | 229 | 1.1e-18 | 92 |
| NP_001255448.1 | 31.90 | 163 | 1.2e-14 | 79 |
| NP_505017.1 | 37.48 | 507 | 1.6e-93 | 329 |
| NP_505017.1 | 30.80 | 237 | 1.1e-19 | 95 |
| NP_505017.1 | 27.87 | 122 | 2.8e-07 | 55 |
| NP_001370555.1 | 37.48 | 507 | 2.5e-93 | 328 |
| NP_001370555.1 | 30.80 | 237 | 1.5e-19 | 95 |
| NP_001370555.1 | 27.87 | 122 | 3.1e-07 | 55 |
| NP_741569.1 | 37.48 | 507 | 3.1e-93 | 327 |
| NP_741569.1 | 30.80 | 237 | 1.8e-19 | 94 |
| NP_741569.1 | 27.87 | 122 | 3.5e-07 | 54 |
| NP_001361951.1 | 26.63 | 1029 | 5.9e-63 | 234 |
| NP_001361951.1 | 28.57 | 224 | 7.3e-08 | 57 |
| NP_001361951.1 | 26.05 | 215 | 1.0e-06 | 53 |
| NP_001361951.1 | 31.21 | 157 | 8.9e-04 | 43 |
| NP_510116.2 | 26.74 | 1032 | 1.7e-62 | 233 |
| NP_510116.2 | 28.57 | 224 | 8.2e-08 | 56 |
| NP_510116.2 | 26.05 | 215 | 1.0e-06 | 53 |
| NP_510116.2 | 31.21 | 157 | 1.0e-03 | 43 |
| NP_001024532.2 | 25.32 | 1082 | 4.0e-54 | 205 |
| NP_001024532.2 | 25.71 | 210 | 3.7e-06 | 51 |
| NP_001361970.1 | 25.29 | 1024 | 2.6e-52 | 199 |
| NP_001361970.1 | 25.71 | 210 | 3.5e-06 | 51 |
| NP_508681.3 | 26.96 | 612 | 5.1e-37 | 151 |
| NP_508681.3 | 26.79 | 168 | 4.0e-04 | 44 |

### ADAMTS9 (Q9P2N4)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 9

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001359558.1 | 34.50 | 2055 | 0.0e+00 | 1088 |
| NP_001255448.1 | 34.25 | 2070 | 0.0e+00 | 1079 |
| NP_505017.1 | 29.75 | 679 | 4.0e-69 | 259 |
| NP_505017.1 | 31.61 | 367 | 1.1e-34 | 146 |
| NP_505017.1 | 29.41 | 374 | 4.3e-29 | 127 |
| NP_505017.1 | 28.03 | 421 | 5.2e-29 | 127 |
| NP_505017.1 | 27.32 | 399 | 7.7e-25 | 114 |
| NP_505017.1 | 31.03 | 174 | 2.0e-15 | 82 |
| NP_741569.1 | 29.75 | 679 | 5.0e-69 | 257 |
| NP_741569.1 | 31.61 | 367 | 3.1e-34 | 144 |
| NP_741569.1 | 28.27 | 421 | 1.3e-28 | 125 |
| NP_741569.1 | 29.41 | 374 | 1.6e-28 | 125 |
| NP_741569.1 | 27.32 | 399 | 2.5e-24 | 112 |
| NP_741569.1 | 31.03 | 174 | 2.8e-15 | 82 |
| NP_001370555.1 | 29.75 | 679 | 5.5e-69 | 258 |
| NP_001370555.1 | 31.88 | 367 | 2.9e-34 | 144 |
| NP_001370555.1 | 28.27 | 421 | 1.2e-28 | 126 |
| NP_001370555.1 | 29.41 | 374 | 1.5e-28 | 125 |
| NP_001370555.1 | 27.32 | 399 | 2.7e-24 | 112 |
| NP_001370555.1 | 31.03 | 174 | 2.5e-15 | 82 |
| NP_001024532.2 | 25.79 | 888 | 4.7e-39 | 159 |
| NP_001024532.2 | 27.66 | 452 | 5.2e-25 | 114 |
| NP_001024532.2 | 31.18 | 340 | 1.3e-16 | 86 |
| NP_001024532.2 | 28.52 | 291 | 4.9e-15 | 81 |
| NP_001024532.2 | 26.63 | 383 | 2.0e-14 | 79 |
| NP_001024532.2 | 24.67 | 381 | 1.1e-13 | 76 |
| NP_001024532.2 | 24.12 | 369 | 1.6e-08 | 60 |
| NP_001024532.2 | 24.77 | 331 | 4.5e-04 | 45 |
| NP_492405.3 | 31.03 | 419 | 1.9e-38 | 157 |
| NP_492405.3 | 26.20 | 519 | 2.2e-36 | 150 |
| NP_492405.3 | 26.83 | 697 | 6.5e-36 | 149 |
| NP_492405.3 | 24.23 | 747 | 1.2e-35 | 148 |
| NP_492405.3 | 27.95 | 254 | 4.6e-14 | 78 |
| NP_492405.3 | 32.08 | 159 | 2.3e-11 | 69 |
| NP_492405.3 | 28.36 | 208 | 1.0e-10 | 67 |
| NP_492405.3 | 28.81 | 177 | 4.1e-10 | 65 |
| NP_492405.3 | 31.25 | 192 | 9.6e-09 | 60 |
| NP_492405.3 | 28.87 | 194 | 1.0e-08 | 60 |
| NP_492405.3 | 26.94 | 193 | 1.4e-07 | 56 |
| NP_492405.3 | 29.63 | 162 | 3.0e-04 | 45 |
| NP_001250933.1 | 30.86 | 418 | 4.4e-38 | 156 |
| NP_001250933.1 | 26.31 | 517 | 1.5e-36 | 151 |
| NP_001250933.1 | 26.69 | 697 | 2.1e-35 | 147 |
| NP_001250933.1 | 31.71 | 164 | 8.1e-16 | 84 |
| NP_001250933.1 | 27.95 | 254 | 5.1e-14 | 77 |
| NP_001250933.1 | 32.08 | 159 | 2.4e-11 | 69 |
| NP_001250933.1 | 28.36 | 208 | 1.1e-10 | 67 |
| NP_001250933.1 | 28.81 | 177 | 4.3e-10 | 65 |
| NP_001250933.1 | 31.25 | 192 | 1.0e-08 | 60 |
| NP_001250933.1 | 28.87 | 194 | 1.1e-08 | 60 |
| NP_001250933.1 | 26.94 | 193 | 1.5e-07 | 56 |
| NP_001250933.1 | 29.63 | 162 | 3.2e-04 | 45 |
| NP_001361970.1 | 26.15 | 807 | 3.0e-37 | 153 |
| NP_001361970.1 | 28.93 | 363 | 5.5e-21 | 100 |
| NP_001361970.1 | 28.87 | 291 | 3.4e-15 | 81 |
| NP_001361970.1 | 26.89 | 357 | 4.4e-13 | 74 |
| NP_001361970.1 | 30.10 | 309 | 1.6e-12 | 72 |
| NP_001361970.1 | 25.91 | 328 | 6.4e-11 | 67 |
| NP_001361970.1 | 26.30 | 289 | 1.6e-10 | 66 |
| NP_001361970.1 | 24.74 | 384 | 3.9e-10 | 65 |
| NP_001361970.1 | 27.32 | 216 | 2.9e-09 | 62 |
| NP_001361970.1 | 25.00 | 308 | 1.1e-06 | 53 |
| NP_001361970.1 | 24.77 | 331 | 2.5e-04 | 46 |
| NP_510116.2 | 23.63 | 677 | 3.5e-34 | 144 |
| NP_510116.2 | 26.85 | 812 | 5.3e-25 | 114 |
| NP_510116.2 | 25.47 | 742 | 3.1e-19 | 95 |
| NP_510116.2 | 24.16 | 803 | 5.4e-18 | 91 |
| NP_510116.2 | 40.35 | 57 | 7.9e-05 | 47 |
| NP_510116.2 | 33.33 | 63 | 3.0e-04 | 45 |

### ADAMTSL1 (Q8N6G6)

**Description:** ADAMTS-like protein 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001359558.1 | 27.37 | 939 | 1.6e-79 | 292 |
| NP_001359558.1 | 26.70 | 588 | 4.2e-44 | 177 |
| NP_001359558.1 | 26.57 | 572 | 4.0e-39 | 160 |
| NP_001359558.1 | 27.17 | 622 | 7.9e-39 | 159 |
| NP_001359558.1 | 26.14 | 547 | 1.5e-26 | 119 |
| NP_001359558.1 | 30.38 | 237 | 9.1e-20 | 97 |
| NP_001359558.1 | 24.03 | 362 | 2.5e-18 | 92 |
| NP_001359558.1 | 27.95 | 254 | 1.6e-16 | 86 |
| NP_001359558.1 | 26.72 | 262 | 3.1e-14 | 79 |
| NP_001359558.1 | 26.92 | 260 | 4.9e-13 | 74 |
| NP_001359558.1 | 26.67 | 315 | 1.1e-12 | 73 |
| NP_001359558.1 | 26.18 | 191 | 2.0e-12 | 72 |
| NP_001359558.1 | 25.09 | 263 | 5.1e-12 | 71 |
| NP_001359558.1 | 23.88 | 289 | 1.6e-10 | 66 |
| NP_001359558.1 | 27.68 | 177 | 2.0e-04 | 46 |
| NP_001255448.1 | 27.04 | 954 | 3.6e-78 | 288 |
| NP_001255448.1 | 26.70 | 588 | 4.1e-44 | 177 |
| NP_001255448.1 | 26.57 | 572 | 4.5e-39 | 160 |
| NP_001255448.1 | 27.17 | 622 | 8.4e-39 | 159 |
| NP_001255448.1 | 26.14 | 547 | 1.5e-26 | 119 |
| NP_001255448.1 | 30.38 | 237 | 9.8e-20 | 97 |
| NP_001255448.1 | 24.03 | 362 | 2.4e-18 | 92 |
| NP_001255448.1 | 27.95 | 254 | 1.8e-16 | 86 |
| NP_001255448.1 | 26.72 | 262 | 2.9e-14 | 79 |
| NP_001255448.1 | 26.92 | 260 | 5.1e-13 | 74 |
| NP_001255448.1 | 26.67 | 315 | 1.0e-12 | 73 |
| NP_001255448.1 | 26.18 | 191 | 2.1e-12 | 72 |
| NP_001255448.1 | 25.09 | 263 | 5.2e-12 | 71 |
| NP_001255448.1 | 23.88 | 289 | 1.6e-10 | 66 |
| NP_001255448.1 | 27.68 | 177 | 1.8e-04 | 46 |
| NP_505017.1 | 29.39 | 735 | 4.4e-68 | 255 |
| NP_505017.1 | 28.41 | 440 | 1.4e-26 | 119 |
| NP_505017.1 | 25.45 | 389 | 7.0e-22 | 103 |
| NP_505017.1 | 31.27 | 307 | 3.9e-21 | 101 |
| NP_505017.1 | 32.14 | 168 | 4.2e-21 | 101 |
| NP_505017.1 | 31.31 | 214 | 1.5e-20 | 99 |
| NP_505017.1 | 28.30 | 265 | 4.4e-15 | 81 |
| NP_505017.1 | 29.18 | 233 | 5.7e-13 | 74 |
| NP_505017.1 | 31.77 | 192 | 1.7e-11 | 69 |
| NP_505017.1 | 31.34 | 134 | 5.5e-11 | 68 |
| NP_505017.1 | 27.39 | 157 | 1.3e-07 | 57 |
| NP_505017.1 | 35.00 | 80 | 9.2e-05 | 47 |
| NP_505017.1 | 25.16 | 155 | 1.4e-04 | 47 |
| NP_741569.1 | 29.43 | 734 | 1.1e-67 | 253 |
| NP_741569.1 | 28.41 | 440 | 2.6e-26 | 118 |
| NP_741569.1 | 25.45 | 389 | 2.0e-21 | 102 |
| NP_741569.1 | 32.14 | 168 | 7.0e-21 | 100 |
| NP_741569.1 | 31.27 | 307 | 1.3e-20 | 99 |
| NP_741569.1 | 31.31 | 214 | 4.0e-20 | 97 |
| NP_741569.1 | 28.30 | 265 | 6.4e-15 | 80 |
| NP_741569.1 | 29.18 | 233 | 7.9e-13 | 73 |
| NP_741569.1 | 31.77 | 192 | 2.6e-11 | 68 |
| NP_741569.1 | 31.34 | 134 | 6.9e-11 | 67 |
| NP_741569.1 | 28.66 | 157 | 1.5e-07 | 56 |
| NP_741569.1 | 25.16 | 155 | 2.0e-04 | 46 |
| NP_001370555.1 | 29.43 | 734 | 1.2e-67 | 253 |
| NP_001370555.1 | 28.41 | 440 | 2.3e-26 | 118 |
| NP_001370555.1 | 25.45 | 389 | 2.0e-21 | 102 |
| NP_001370555.1 | 32.14 | 168 | 7.2e-21 | 100 |
| NP_001370555.1 | 31.27 | 307 | 1.4e-20 | 99 |
| NP_001370555.1 | 31.31 | 214 | 3.9e-20 | 98 |
| NP_001370555.1 | 28.30 | 265 | 8.2e-15 | 80 |
| NP_001370555.1 | 29.18 | 233 | 9.2e-13 | 73 |
| NP_001370555.1 | 31.77 | 192 | 2.9e-11 | 68 |
| NP_001370555.1 | 31.34 | 134 | 7.3e-11 | 67 |
| NP_001370555.1 | 28.66 | 157 | 1.8e-07 | 56 |
| NP_001370555.1 | 25.16 | 155 | 2.0e-04 | 46 |
| NP_001250933.1 | 28.43 | 510 | 2.8e-53 | 204 |
| NP_001250933.1 | 25.17 | 743 | 3.9e-50 | 194 |
| NP_001250933.1 | 25.72 | 762 | 6.6e-40 | 162 |
| NP_001250933.1 | 24.55 | 391 | 9.5e-22 | 103 |
| NP_001250933.1 | 30.54 | 203 | 3.6e-13 | 75 |
| NP_001250933.1 | 27.62 | 210 | 2.3e-12 | 72 |
| NP_001250933.1 | 32.19 | 146 | 2.1e-10 | 65 |
| NP_001250933.1 | 26.07 | 234 | 2.2e-10 | 65 |
| NP_001250933.1 | 22.16 | 343 | 8.0e-10 | 63 |
| NP_001250933.1 | 25.17 | 290 | 1.2e-05 | 50 |
| NP_001250933.1 | 30.51 | 118 | 6.3e-05 | 48 |
| NP_492405.3 | 28.43 | 510 | 6.1e-53 | 203 |
| NP_492405.3 | 25.10 | 745 | 9.1e-50 | 193 |
| NP_492405.3 | 25.85 | 762 | 2.6e-40 | 163 |
| NP_492405.3 | 24.55 | 391 | 1.3e-21 | 102 |
| NP_492405.3 | 30.54 | 203 | 4.1e-13 | 74 |
| NP_492405.3 | 27.62 | 210 | 2.5e-12 | 72 |
| NP_492405.3 | 32.19 | 146 | 2.3e-10 | 65 |
| NP_492405.3 | 22.16 | 343 | 8.6e-10 | 63 |
| NP_492405.3 | 25.85 | 236 | 9.0e-09 | 60 |
| NP_492405.3 | 25.17 | 290 | 1.6e-05 | 50 |
| NP_492405.3 | 30.00 | 120 | 9.7e-05 | 47 |
| NP_492405.3 | 23.74 | 278 | 2.3e-04 | 46 |
| NP_871884.1 | 32.62 | 279 | 3.8e-38 | 154 |
| NP_871884.1 | 23.97 | 509 | 8.4e-38 | 153 |
| NP_871884.1 | 24.55 | 391 | 7.9e-23 | 105 |
| NP_871884.1 | 30.54 | 203 | 1.1e-13 | 76 |
| NP_871884.1 | 25.32 | 312 | 1.1e-13 | 76 |
| NP_871884.1 | 27.62 | 210 | 1.2e-12 | 73 |
| NP_871884.1 | 26.57 | 286 | 6.6e-12 | 70 |
| NP_871884.1 | 32.19 | 146 | 1.1e-10 | 66 |
| NP_871884.1 | 21.80 | 344 | 4.2e-10 | 64 |
| NP_871884.1 | 21.88 | 320 | 1.5e-07 | 56 |
| NP_871884.1 | 25.17 | 290 | 3.1e-06 | 52 |
| NP_001359724.1 | 28.27 | 283 | 3.6e-22 | 105 |
| NP_001359724.1 | 22.84 | 289 | 7.4e-21 | 100 |
| NP_001359724.1 | 26.90 | 290 | 1.0e-20 | 100 |
| NP_001359724.1 | 26.13 | 287 | 3.9e-18 | 91 |
| NP_001359724.1 | 25.57 | 305 | 2.3e-17 | 89 |
| NP_001359724.1 | 25.33 | 300 | 3.2e-17 | 89 |
| NP_001359724.1 | 22.33 | 318 | 7.8e-17 | 87 |
| NP_001359724.1 | 23.33 | 360 | 1.8e-15 | 83 |
| NP_001359724.1 | 22.70 | 304 | 5.9e-15 | 81 |
| NP_001359724.1 | 23.58 | 369 | 7.3e-15 | 80 |
| NP_001359724.1 | 25.00 | 292 | 9.6e-15 | 80 |
| NP_001359724.1 | 23.65 | 296 | 1.4e-14 | 80 |
| NP_001359724.1 | 22.91 | 323 | 1.5e-13 | 76 |
| NP_001359724.1 | 24.13 | 286 | 3.7e-13 | 75 |
| NP_001359724.1 | 25.42 | 295 | 7.8e-13 | 74 |
| NP_001359724.1 | 23.05 | 295 | 1.2e-12 | 73 |
| NP_001359724.1 | 22.57 | 288 | 4.8e-12 | 71 |
| NP_001359724.1 | 25.09 | 271 | 6.8e-12 | 71 |
| NP_001359724.1 | 21.48 | 284 | 7.6e-12 | 70 |
| NP_001359724.1 | 22.66 | 278 | 4.3e-08 | 58 |
| NP_001359724.1 | 21.34 | 328 | 1.1e-06 | 53 |
| NP_001359724.1 | 22.97 | 209 | 1.3e-05 | 50 |
| NP_001359724.1 | 20.76 | 289 | 1.3e-05 | 50 |
| NP_001359724.1 | 20.65 | 247 | 1.2e-04 | 47 |
| NP_001359724.1 | 37.18 | 78 | 4.9e-04 | 45 |
| NP_001360634.1 | 28.27 | 283 | 4.4e-22 | 104 |
| NP_001360634.1 | 22.84 | 289 | 9.2e-21 | 100 |
| NP_001360634.1 | 26.90 | 290 | 1.5e-20 | 99 |
| NP_001360634.1 | 24.16 | 327 | 1.9e-19 | 96 |
| NP_001360634.1 | 26.13 | 287 | 4.2e-18 | 91 |
| NP_001360634.1 | 25.57 | 305 | 2.5e-17 | 89 |
| NP_001360634.1 | 25.33 | 300 | 4.5e-17 | 88 |
| NP_001360634.1 | 22.19 | 320 | 9.1e-17 | 87 |
| NP_001360634.1 | 26.13 | 287 | 1.6e-16 | 86 |
| NP_001360634.1 | 25.50 | 298 | 3.7e-15 | 82 |
| NP_001360634.1 | 22.70 | 304 | 6.4e-15 | 81 |
| NP_001360634.1 | 23.58 | 369 | 7.9e-15 | 80 |
| NP_001360634.1 | 25.00 | 292 | 1.1e-14 | 80 |
| NP_001360634.1 | 23.65 | 296 | 1.7e-14 | 79 |
| NP_001360634.1 | 22.77 | 325 | 1.4e-13 | 76 |
| NP_001360634.1 | 24.13 | 286 | 4.4e-13 | 75 |
| NP_001360634.1 | 24.75 | 295 | 7.9e-13 | 74 |
| NP_001360634.1 | 23.05 | 295 | 1.1e-12 | 73 |
| NP_001360634.1 | 22.48 | 298 | 1.5e-12 | 73 |
| NP_001360634.1 | 22.30 | 296 | 4.4e-12 | 72 |
| NP_001360634.1 | 25.09 | 271 | 7.7e-12 | 70 |
| NP_001360634.1 | 21.48 | 284 | 8.0e-12 | 70 |
| NP_001360634.1 | 23.68 | 283 | 3.9e-09 | 62 |
| NP_001360634.1 | 21.94 | 278 | 4.6e-08 | 58 |
| NP_001360634.1 | 21.62 | 296 | 7.6e-08 | 57 |
| NP_001360634.1 | 21.34 | 328 | 1.4e-06 | 53 |
| NP_001360634.1 | 20.76 | 289 | 1.2e-05 | 50 |
| NP_001360634.1 | 22.97 | 209 | 1.7e-05 | 50 |
| NP_001360634.1 | 20.65 | 247 | 1.3e-04 | 47 |
| NP_001360634.1 | 37.18 | 78 | 3.8e-04 | 45 |

### ADAMTSL3 (P82987)

**Description:** ADAMTS-like protein 3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001359558.1 | 27.30 | 945 | 1.4e-77 | 286 |
| NP_001359558.1 | 25.70 | 642 | 6.6e-41 | 166 |
| NP_001359558.1 | 27.82 | 593 | 2.5e-39 | 161 |
| NP_001359558.1 | 27.72 | 404 | 4.8e-27 | 120 |
| NP_001359558.1 | 30.80 | 250 | 8.8e-21 | 100 |
| NP_001359558.1 | 31.86 | 204 | 2.6e-18 | 92 |
| NP_001359558.1 | 29.79 | 235 | 2.3e-16 | 85 |
| NP_001359558.1 | 28.10 | 242 | 2.2e-14 | 79 |
| NP_001359558.1 | 29.25 | 212 | 1.0e-12 | 73 |
| NP_001359558.1 | 28.78 | 278 | 4.7e-12 | 71 |
| NP_001359558.1 | 25.52 | 290 | 1.2e-10 | 67 |
| NP_001359558.1 | 27.11 | 284 | 3.5e-10 | 65 |
| NP_001359558.1 | 27.89 | 208 | 1.1e-09 | 63 |
| NP_001359558.1 | 25.70 | 249 | 2.0e-08 | 59 |
| NP_001255448.1 | 26.88 | 960 | 1.4e-75 | 279 |
| NP_001255448.1 | 25.70 | 642 | 9.1e-41 | 166 |
| NP_001255448.1 | 27.82 | 593 | 3.3e-39 | 160 |
| NP_001255448.1 | 27.72 | 404 | 5.7e-27 | 120 |
| NP_001255448.1 | 30.80 | 250 | 8.8e-21 | 100 |
| NP_001255448.1 | 31.86 | 204 | 2.4e-18 | 92 |
| NP_001255448.1 | 29.79 | 235 | 2.3e-16 | 85 |
| NP_001255448.1 | 28.10 | 242 | 2.3e-14 | 79 |
| NP_001255448.1 | 29.25 | 212 | 1.1e-12 | 73 |
| NP_001255448.1 | 28.78 | 278 | 4.9e-12 | 71 |
| NP_001255448.1 | 25.52 | 290 | 1.3e-10 | 67 |
| NP_001255448.1 | 27.11 | 284 | 3.9e-10 | 65 |
| NP_001255448.1 | 27.89 | 208 | 1.3e-09 | 63 |
| NP_001255448.1 | 25.70 | 249 | 2.2e-08 | 59 |
| NP_505017.1 | 28.98 | 728 | 2.0e-73 | 272 |
| NP_505017.1 | 28.74 | 421 | 4.9e-25 | 114 |
| NP_505017.1 | 29.92 | 254 | 1.0e-19 | 96 |
| NP_505017.1 | 28.80 | 184 | 7.8e-17 | 87 |
| NP_505017.1 | 30.24 | 291 | 4.0e-14 | 78 |
| NP_505017.1 | 24.15 | 381 | 3.2e-13 | 75 |
| NP_505017.1 | 26.67 | 345 | 6.3e-13 | 74 |
| NP_505017.1 | 35.87 | 92 | 2.2e-06 | 52 |
| NP_505017.1 | 26.45 | 155 | 1.6e-05 | 50 |
| NP_505017.1 | 25.86 | 263 | 8.7e-05 | 47 |
| NP_741569.1 | 28.98 | 728 | 4.6e-73 | 270 |
| NP_741569.1 | 28.74 | 421 | 1.4e-24 | 112 |
| NP_741569.1 | 29.92 | 254 | 1.8e-19 | 95 |
| NP_741569.1 | 28.80 | 184 | 1.4e-16 | 86 |
| NP_741569.1 | 30.24 | 291 | 1.0e-13 | 77 |
| NP_741569.1 | 24.15 | 381 | 1.0e-12 | 73 |
| NP_741569.1 | 26.67 | 345 | 1.2e-12 | 73 |
| NP_741569.1 | 35.87 | 92 | 2.9e-06 | 52 |
| NP_741569.1 | 26.45 | 155 | 1.9e-05 | 49 |
| NP_741569.1 | 25.86 | 263 | 2.3e-04 | 46 |
| NP_001370555.1 | 28.98 | 728 | 4.6e-73 | 270 |
| NP_001370555.1 | 28.74 | 421 | 9.8e-25 | 113 |
| NP_001370555.1 | 29.92 | 254 | 1.8e-19 | 95 |
| NP_001370555.1 | 28.80 | 184 | 1.3e-16 | 86 |
| NP_001370555.1 | 30.24 | 291 | 9.8e-14 | 77 |
| NP_001370555.1 | 24.15 | 381 | 9.1e-13 | 73 |
| NP_001370555.1 | 26.67 | 345 | 1.3e-12 | 73 |
| NP_001370555.1 | 35.87 | 92 | 2.8e-06 | 52 |
| NP_001370555.1 | 26.45 | 155 | 1.8e-05 | 49 |
| NP_001370555.1 | 25.86 | 263 | 2.6e-04 | 45 |
| NP_001250933.1 | 30.36 | 471 | 2.4e-54 | 207 |
| NP_001250933.1 | 25.67 | 596 | 4.2e-41 | 166 |
| NP_001250933.1 | 24.11 | 726 | 1.5e-33 | 141 |
| NP_001250933.1 | 24.17 | 451 | 1.5e-23 | 108 |
| NP_001250933.1 | 31.25 | 208 | 2.6e-15 | 82 |
| NP_001250933.1 | 26.57 | 271 | 6.9e-15 | 80 |
| NP_001250933.1 | 38.64 | 88 | 3.4e-09 | 62 |
| NP_001250933.1 | 26.59 | 267 | 3.9e-09 | 61 |
| NP_001250933.1 | 30.29 | 175 | 9.3e-09 | 60 |
| NP_001250933.1 | 27.17 | 265 | 1.7e-07 | 56 |
| NP_001250933.1 | 27.85 | 158 | 2.3e-05 | 49 |
| NP_001250933.1 | 26.56 | 192 | 5.0e-05 | 48 |
| NP_001250933.1 | 30.77 | 130 | 1.0e-04 | 47 |
| NP_001250933.1 | 29.03 | 155 | 6.1e-04 | 44 |
| NP_492405.3 | 30.36 | 471 | 5.9e-54 | 206 |
| NP_492405.3 | 25.59 | 598 | 1.0e-40 | 164 |
| NP_492405.3 | 24.24 | 726 | 6.1e-34 | 142 |
| NP_492405.3 | 24.17 | 451 | 1.9e-23 | 108 |
| NP_492405.3 | 30.95 | 210 | 1.1e-13 | 76 |
| NP_492405.3 | 26.37 | 273 | 2.8e-13 | 75 |
| NP_492405.3 | 38.64 | 88 | 3.6e-09 | 62 |
| NP_492405.3 | 26.59 | 267 | 5.0e-09 | 61 |
| NP_492405.3 | 30.29 | 175 | 9.8e-09 | 60 |
| NP_492405.3 | 27.17 | 265 | 2.0e-07 | 56 |
| NP_492405.3 | 27.85 | 158 | 2.6e-05 | 49 |
| NP_492405.3 | 31.06 | 132 | 3.9e-05 | 48 |
| NP_492405.3 | 26.56 | 192 | 5.9e-05 | 48 |
| NP_492405.3 | 29.03 | 155 | 6.9e-04 | 44 |
| NP_871884.1 | 32.96 | 267 | 1.5e-36 | 149 |
| NP_871884.1 | 30.69 | 303 | 4.6e-36 | 147 |
| NP_871884.1 | 24.37 | 476 | 2.1e-24 | 110 |
| NP_871884.1 | 28.07 | 171 | 1.4e-13 | 75 |
| NP_871884.1 | 25.09 | 291 | 6.5e-13 | 73 |
| NP_871884.1 | 26.59 | 267 | 1.0e-09 | 63 |
| NP_871884.1 | 23.95 | 309 | 3.7e-09 | 61 |
| NP_871884.1 | 30.29 | 175 | 8.7e-09 | 60 |
| NP_871884.1 | 22.33 | 318 | 4.5e-06 | 51 |
| NP_871884.1 | 27.85 | 158 | 8.6e-06 | 50 |
| NP_871884.1 | 26.56 | 192 | 2.4e-05 | 49 |
| NP_871884.1 | 28.39 | 155 | 2.0e-04 | 46 |
| NP_871884.1 | 23.81 | 147 | 2.9e-04 | 45 |
| NP_510116.2 | 21.38 | 795 | 2.2e-17 | 89 |
| NP_510116.2 | 23.74 | 615 | 7.5e-14 | 77 |
| NP_510116.2 | 27.12 | 236 | 4.3e-10 | 65 |
| NP_510116.2 | 22.55 | 581 | 4.1e-08 | 58 |
| NP_510116.2 | 23.97 | 438 | 2.6e-07 | 55 |
| NP_510116.2 | 22.59 | 487 | 9.2e-07 | 53 |
| NP_001361951.1 | 21.38 | 795 | 2.5e-17 | 89 |
| NP_001361951.1 | 23.74 | 615 | 7.5e-14 | 77 |
| NP_001361951.1 | 27.12 | 236 | 4.5e-10 | 65 |
| NP_001361951.1 | 22.55 | 581 | 3.9e-08 | 58 |
| NP_001361951.1 | 23.97 | 438 | 2.5e-07 | 55 |
| NP_001361951.1 | 22.59 | 487 | 9.4e-07 | 53 |

### ADAMTSL4 (Q6UY14)

**Description:** ADAMTS-like protein 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_505017.1 | 27.79 | 673 | 5.8e-70 | 256 |
| NP_001370555.1 | 27.79 | 673 | 7.8e-69 | 253 |
| NP_741569.1 | 27.79 | 673 | 9.8e-69 | 252 |
| NP_001359558.1 | 28.89 | 706 | 5.0e-66 | 244 |
| NP_001359558.1 | 29.02 | 386 | 8.6e-34 | 141 |
| NP_001359558.1 | 29.92 | 371 | 1.2e-33 | 141 |
| NP_001359558.1 | 31.27 | 291 | 2.7e-31 | 133 |
| NP_001359558.1 | 26.70 | 412 | 2.7e-25 | 114 |
| NP_001359558.1 | 28.12 | 377 | 6.2e-23 | 106 |
| NP_001359558.1 | 35.13 | 185 | 6.2e-22 | 103 |
| NP_001255448.1 | 28.78 | 695 | 6.6e-66 | 244 |
| NP_001255448.1 | 29.02 | 386 | 7.5e-34 | 142 |
| NP_001255448.1 | 29.92 | 371 | 1.0e-33 | 141 |
| NP_001255448.1 | 31.27 | 291 | 2.3e-31 | 134 |
| NP_001255448.1 | 26.70 | 412 | 2.4e-25 | 114 |
| NP_001255448.1 | 28.12 | 377 | 5.5e-23 | 106 |
| NP_001255448.1 | 35.13 | 185 | 5.8e-22 | 103 |
| NP_001250933.1 | 29.31 | 406 | 3.8e-32 | 135 |
| NP_001250933.1 | 28.89 | 405 | 1.4e-24 | 111 |
| NP_001250933.1 | 31.52 | 184 | 1.4e-18 | 91 |
| NP_001250933.1 | 32.47 | 194 | 2.3e-11 | 68 |
| NP_001250933.1 | 28.88 | 187 | 1.3e-09 | 62 |
| NP_001250933.1 | 34.40 | 125 | 1.5e-09 | 62 |
| NP_001250933.1 | 29.27 | 123 | 1.2e-07 | 56 |
| NP_001250933.1 | 31.69 | 142 | 9.9e-05 | 46 |
| NP_001250933.1 | 27.50 | 200 | 2.1e-04 | 45 |
| NP_492405.3 | 29.17 | 408 | 1.4e-30 | 130 |
| NP_492405.3 | 28.89 | 405 | 1.4e-24 | 111 |
| NP_492405.3 | 31.52 | 184 | 1.4e-18 | 91 |
| NP_492405.3 | 32.47 | 194 | 2.3e-11 | 68 |
| NP_492405.3 | 28.88 | 187 | 1.3e-09 | 62 |
| NP_492405.3 | 34.40 | 125 | 1.4e-09 | 62 |
| NP_492405.3 | 29.27 | 123 | 1.2e-07 | 56 |
| NP_492405.3 | 31.69 | 142 | 9.6e-05 | 46 |
| NP_492405.3 | 27.50 | 200 | 2.1e-04 | 45 |
| NP_509829.2 | 28.61 | 430 | 1.6e-19 | 94 |
| NP_509829.2 | 29.79 | 282 | 6.1e-11 | 66 |
| NP_509829.2 | 41.51 | 53 | 1.9e-04 | 45 |
| NP_871884.1 | 31.15 | 183 | 2.6e-19 | 93 |
| NP_871884.1 | 31.30 | 262 | 8.4e-18 | 88 |
| NP_871884.1 | 28.39 | 236 | 2.5e-17 | 87 |
| NP_871884.1 | 27.76 | 263 | 8.7e-13 | 72 |
| NP_871884.1 | 32.47 | 194 | 2.9e-12 | 70 |
| NP_871884.1 | 28.88 | 187 | 5.8e-10 | 63 |
| NP_871884.1 | 29.03 | 124 | 4.2e-08 | 57 |
| NP_871884.1 | 31.69 | 142 | 2.9e-05 | 48 |
| NP_871884.1 | 27.50 | 200 | 3.5e-05 | 47 |
| NP_871884.1 | 32.28 | 127 | 5.4e-04 | 43 |
| NP_001024532.2 | 29.02 | 286 | 7.8e-14 | 76 |
| NP_001024532.2 | 25.90 | 305 | 4.6e-13 | 73 |
| NP_001024532.2 | 26.79 | 377 | 1.4e-11 | 68 |
| NP_001024532.2 | 27.64 | 246 | 5.8e-10 | 63 |
| NP_001024532.2 | 30.34 | 145 | 6.5e-06 | 50 |

### ADAMTSL5 (Q6ZMM2)

**Description:** ADAMTS-like protein 5

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_505017.1 | 34.63 | 283 | 1.9e-44 | 167 |
| NP_001370555.1 | 34.63 | 283 | 4.3e-44 | 166 |
| NP_741569.1 | 34.63 | 283 | 4.6e-44 | 166 |
| NP_001359558.1 | 34.28 | 283 | 1.1e-37 | 147 |
| NP_001255448.1 | 32.55 | 298 | 5.2e-36 | 142 |
| NP_492405.3 | 30.29 | 208 | 5.4e-14 | 74 |
| NP_001250933.1 | 30.29 | 208 | 5.5e-14 | 74 |
| NP_001361970.1 | 42.86 | 56 | 1.1e-05 | 48 |
| NP_001024532.2 | 42.86 | 56 | 1.4e-05 | 47 |
| NP_001361951.1 | 45.28 | 53 | 4.0e-05 | 46 |
| NP_001361951.1 | 26.92 | 104 | 5.0e-04 | 42 |

### ALG1 (Q9BT22)

**Description:** Chitobiosyldiphosphodolichol beta-mannosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_498420.2 | 40.99 | 444 | 2.2e-106 | 324 |

### ALG10 (Q5BKT4)

**Description:** Dol-P-Glc:Glc(2)Man(9)GlcNAc(2)-PP-Dol alpha-1,2-glucosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_492634.2 | 30.25 | 476 | 1.5e-50 | 177 |

### ALG10B (Q5I7T1)

**Description:** Dol-P-Glc:Glc(2)Man(9)GlcNAc(2)-PP-Dol alpha-1,2-glucosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_492634.2 | 31.47 | 483 | 1.5e-49 | 174 |

### ALG11 (Q2TAA5)

**Description:** GDP-Man:Man(3)GlcNAc(2)-PP-Dol alpha-1,2-mannosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_498606.1 | 46.10 | 436 | 7.8e-132 | 390 |
| NP_508932.1 | 25.60 | 293 | 1.4e-10 | 63 |
| NP_001408285.1 | 25.60 | 293 | 1.8e-10 | 62 |
| NP_495010.2 | 26.92 | 234 | 5.8e-10 | 61 |

### ALG12 (Q9BV10)

**Description:** Dol-P-Man:Man(7)GlcNAc(2)-PP-Dol alpha-1,6-mannosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_505071.1 | 39.75 | 405 | 1.3e-90 | 285 |
| NP_001254097.1 | 29.27 | 205 | 6.3e-10 | 61 |

### ALG13 (Q9NP73)

**Description:** UDP-N-acetylglucosamine transferase subunit ALG13

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_506467.1 | 44.44 | 144 | 1.7e-28 | 113 |
| NP_001368281.1 | 29.52 | 105 | 1.3e-13 | 73 |
| NP_500333.3 | 29.52 | 105 | 2.4e-13 | 73 |
| NP_001255320.1 | 28.35 | 127 | 7.1e-07 | 52 |
| NP_001255319.1 | 28.35 | 127 | 7.3e-07 | 52 |
| NP_001255318.1 | 28.35 | 127 | 1.6e-06 | 51 |
| NP_505931.3 | 29.47 | 95 | 2.1e-04 | 45 |

### ALG14 (Q96F25)

**Description:** UDP-N-acetylglucosamine transferase subunit ALG14

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001294286.1 | 40.54 | 185 | 5.6e-50 | 162 |
| NP_001294287.1 | 39.06 | 192 | 6.2e-50 | 162 |

### ALG2 (Q9H553)

**Description:** Alpha-1,3/1,6-mannosyltransferase ALG2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_495010.2 | 46.90 | 403 | 7.1e-124 | 364 |
| NP_001408285.1 | 30.31 | 419 | 1.6e-47 | 168 |
| NP_508932.1 | 30.51 | 413 | 1.7e-47 | 168 |
| NP_498606.1 | 24.51 | 355 | 2.4e-10 | 62 |

### ALG3 (Q92685)

**Description:** Dol-P-Man:Man(5)GlcNAc(2)-PP-Dol alpha-1,3-mannosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001254418.1 | 48.97 | 390 | 1.1e-115 | 352 |
| NP_001254419.1 | 48.61 | 360 | 9.2e-105 | 322 |

### ALG6 (Q9Y672)

**Description:** Dolichyl pyrophosphate Man9GlcNAc2 alpha-1,3-glucosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_495685.1 | 39.17 | 485 | 2.9e-107 | 328 |
| NP_001021940.2 | 27.23 | 415 | 4.8e-24 | 106 |
| NP_001379427.1 | 32.18 | 87 | 5.8e-08 | 51 |

### ALG8 (Q9BVK2)

**Description:** Dolichyl pyrophosphate Glc1Man9GlcNAc2 alpha-1,3-glucosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001021940.2 | 38.13 | 514 | 3.6e-88 | 287 |
| NP_001379427.1 | 57.27 | 110 | 4.5e-32 | 119 |
| NP_495685.1 | 28.53 | 361 | 5.1e-24 | 105 |

### ALG9 (Q9H6U8)

**Description:** Alpha-1,2-mannosyltransferase ALG9

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_496282.2 | 37.89 | 615 | 1.3e-128 | 390 |
| NP_001254097.1 | 25.39 | 323 | 1.9e-13 | 72 |
| NP_505071.1 | 23.81 | 336 | 8.8e-06 | 48 |

### AMDHD2 (Q9Y303)

**Description:** N-acetylglucosamine-6-phosphate deacetylase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001366671.1 | 51.51 | 398 | 2.3e-143 | 414 |

### AMFR (Q9UKV5)

**Description:** E3 ubiquitin-protein ligase AMFR

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_492602.2 | 24.80 | 504 | 7.5e-42 | 160 |
| NP_505969.1 | 25.47 | 318 | 4.7e-32 | 131 |
| NP_491382.2 | 29.55 | 220 | 3.1e-16 | 82 |
| NP_001024970.1 | 29.00 | 100 | 1.0e-07 | 54 |
| NP_497129.1 | 29.70 | 101 | 3.4e-07 | 53 |
| NP_498962.1 | 43.75 | 48 | 6.6e-07 | 49 |
| NP_491091.1 | 33.96 | 53 | 7.0e-06 | 48 |
| NP_001021767.2 | 38.00 | 50 | 1.2e-05 | 46 |
| NP_001370339.1 | 31.11 | 45 | 1.6e-05 | 47 |
| NP_493231.1 | 31.03 | 58 | 3.8e-05 | 46 |

### ASGR1 (P07306)

**Description:** Asialoglycoprotein receptor 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_509919.2 | 26.27 | 118 | 7.6e-09 | 56 |
| NP_505170.2 | 28.42 | 95 | 5.5e-08 | 52 |
| NP_507830.1 | 31.54 | 130 | 6.6e-07 | 49 |
| NP_497267.2 | 24.14 | 203 | 7.8e-07 | 49 |
| NP_001338858.1 | 25.00 | 148 | 1.0e-06 | 50 |
| NP_500843.1 | 28.57 | 133 | 1.9e-06 | 47 |
| NP_001024428.1 | 28.87 | 142 | 2.3e-06 | 47 |
| NP_001024427.1 | 28.87 | 142 | 4.3e-06 | 46 |
| NP_503500.2 | 26.98 | 126 | 6.1e-06 | 47 |
| NP_001300254.1 | 26.98 | 126 | 9.1e-06 | 45 |

### ASGR2 (P07307)

**Description:** Asialoglycoprotein receptor 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_507830.1 | 35.88 | 131 | 8.3e-12 | 64 |
| NP_507830.1 | 25.40 | 126 | 4.3e-05 | 44 |
| NP_497267.2 | 33.61 | 119 | 1.0e-09 | 58 |
| NP_505170.2 | 32.99 | 97 | 4.1e-09 | 55 |
| NP_500454.2 | 30.43 | 138 | 2.3e-08 | 55 |
| NP_001379719.1 | 42.86 | 63 | 4.4e-08 | 51 |
| NP_508027.2 | 30.53 | 131 | 1.1e-07 | 52 |
| NP_001024427.1 | 42.86 | 63 | 1.7e-07 | 50 |
| NP_001024428.1 | 42.86 | 63 | 2.4e-07 | 50 |
| NP_497312.1 | 29.61 | 152 | 5.6e-07 | 50 |
| NP_507829.1 | 32.82 | 131 | 9.1e-07 | 49 |
| NP_507829.1 | 25.98 | 127 | 6.3e-06 | 47 |

### ATXN3 (P54252)

**Description:** Ataxin-3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001379190.1 | 33.95 | 327 | 9.7e-54 | 179 |
| NP_871685.1 | 22.76 | 145 | 1.2e-07 | 51 |

### B3GALNT2 (Q8NCR0)

**Description:** UDP-GalNAc:beta-1,3-N-acetylgalactosaminyltransferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_494394.1 | 26.53 | 196 | 1.3e-14 | 74 |
| NP_001025003.2 | 28.29 | 152 | 6.3e-08 | 54 |
| NP_493113.1 | 27.56 | 156 | 5.3e-07 | 51 |
| NP_001255188.1 | 23.27 | 159 | 5.1e-06 | 48 |
| NP_001255189.1 | 22.98 | 161 | 4.1e-05 | 45 |
| NP_001367175.1 | 22.98 | 161 | 5.8e-05 | 45 |
| NP_493136.2 | 24.03 | 154 | 4.2e-04 | 42 |
| NP_499715.2 | 23.02 | 126 | 5.4e-04 | 42 |
| NP_493115.1 | 25.15 | 167 | 1.0e-03 | 41 |

### B3GNT2 (Q9NY97)

**Description:** N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_741615.2 | 26.91 | 223 | 5.6e-15 | 75 |
| NP_001255613.1 | 26.00 | 200 | 5.8e-14 | 70 |
| NP_001255612.1 | 26.00 | 200 | 1.7e-13 | 70 |
| NP_001361857.1 | 24.41 | 254 | 3.5e-13 | 70 |
| NP_493088.2 | 23.11 | 212 | 3.6e-13 | 70 |
| NP_493115.1 | 22.93 | 253 | 7.7e-13 | 68 |
| NP_493113.1 | 27.36 | 212 | 8.9e-13 | 68 |
| NP_001025003.2 | 26.48 | 219 | 3.1e-11 | 63 |
| NP_493116.2 | 23.41 | 252 | 1.6e-10 | 62 |
| NP_001255233.1 | 24.73 | 275 | 6.1e-10 | 60 |

### B3GNT4 (Q9C0J1)

**Description:** N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_741615.2 | 23.84 | 302 | 1.9e-18 | 85 |
| NP_001255613.1 | 25.77 | 194 | 1.1e-17 | 81 |
| NP_001255612.1 | 25.62 | 203 | 2.0e-17 | 82 |
| NP_001025003.2 | 25.00 | 208 | 4.5e-14 | 72 |
| NP_494394.1 | 28.30 | 258 | 1.2e-12 | 68 |
| NP_493115.1 | 24.67 | 227 | 2.0e-11 | 64 |
| NP_001255234.1 | 25.33 | 225 | 2.1e-11 | 64 |
| NP_001255233.1 | 25.33 | 225 | 2.9e-11 | 63 |
| NP_493116.2 | 22.93 | 205 | 7.9e-11 | 62 |
| NP_493136.2 | 24.88 | 205 | 8.1e-11 | 62 |

### B3GNT6 (Q6ZMB0)

**Description:** Acetylgalactosaminyl-O-glycosyl-glycoprotein beta-1,3-N-acetylglucosaminyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001255613.1 | 28.00 | 200 | 2.1e-22 | 94 |
| NP_001255612.1 | 27.75 | 209 | 7.0e-22 | 95 |
| NP_001025003.2 | 25.70 | 214 | 2.4e-19 | 88 |
| NP_499715.2 | 24.88 | 205 | 8.6e-17 | 80 |
| NP_001255188.1 | 24.38 | 201 | 1.2e-16 | 80 |
| NP_493088.2 | 25.12 | 215 | 2.4e-16 | 79 |
| NP_001367175.1 | 24.87 | 197 | 5.9e-16 | 78 |
| NP_001255189.1 | 24.87 | 197 | 8.7e-16 | 78 |
| NP_001361857.1 | 24.19 | 215 | 1.1e-14 | 74 |
| NP_493115.1 | 21.75 | 285 | 2.9e-14 | 73 |

### B4GALNT2 (Q8NHY0)

**Description:** Beta-1,4 N-acetylgalactosaminyltransferase 2

No significant BLAST hits found for this protein.

### B4GALT1 (P15291)

**Description:** Beta-1,4-galactosyltransferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_490872.1 | 44.24 | 269 | 4.7e-84 | 261 |
| NP_001370927.1 | 46.00 | 200 | 1.7e-64 | 205 |
| NP_496449.1 | 36.40 | 239 | 4.4e-52 | 178 |
| NP_499164.1 | 31.84 | 201 | 2.4e-23 | 98 |

### B4GALT2 (O60909)

**Description:** Beta-1,4-galactosyltransferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_490872.1 | 43.07 | 274 | 6.8e-82 | 254 |
| NP_001370927.1 | 45.77 | 201 | 4.7e-66 | 208 |
| NP_496449.1 | 34.46 | 296 | 2.2e-54 | 183 |
| NP_499164.1 | 29.88 | 251 | 1.6e-19 | 87 |

### B4GALT3 (O60512)

**Description:** Beta-1,4-galactosyltransferase 3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_490872.1 | 48.91 | 274 | 7.0e-94 | 286 |
| NP_001370927.1 | 51.48 | 202 | 5.5e-76 | 234 |
| NP_496449.1 | 34.98 | 283 | 1.2e-48 | 169 |
| NP_499164.1 | 37.18 | 156 | 4.5e-21 | 92 |

### B4GALT4 (O60513)

**Description:** Beta-1,4-galactosyltransferase 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_490872.1 | 44.74 | 266 | 7.2e-81 | 251 |
| NP_001370927.1 | 47.26 | 201 | 2.4e-68 | 213 |
| NP_496449.1 | 36.23 | 276 | 3.3e-51 | 174 |
| NP_499164.1 | 32.00 | 175 | 1.1e-22 | 95 |

### B4GALT5 (O43286)

**Description:** Beta-1,4-galactosyltransferase 5

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_490872.1 | 41.95 | 267 | 7.5e-72 | 229 |
| NP_496449.1 | 33.59 | 396 | 3.9e-65 | 212 |
| NP_001370927.1 | 44.78 | 201 | 2.0e-55 | 181 |
| NP_499164.1 | 32.50 | 160 | 6.2e-22 | 94 |

### B4GALT6 (Q9UBX8)

**Description:** Beta-1,4-galactosyltransferase 6

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_490872.1 | 42.16 | 268 | 2.1e-70 | 225 |
| NP_496449.1 | 41.37 | 249 | 5.8e-60 | 198 |
| NP_001370927.1 | 44.78 | 201 | 2.0e-54 | 179 |
| NP_499164.1 | 36.25 | 160 | 1.0e-23 | 99 |
| NP_490872.1 | 42.16 | 268 | 2.1e-70 | 225 |
| NP_496449.1 | 41.37 | 249 | 5.8e-60 | 198 |
| NP_001370927.1 | 44.78 | 201 | 2.0e-54 | 179 |
| NP_499164.1 | 36.25 | 160 | 1.0e-23 | 99 |

### B4GALT6 (Q9UBX8)

**Description:** Beta-1,4-galactosyltransferase 6

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_490872.1 | 42.16 | 268 | 2.1e-70 | 225 |
| NP_496449.1 | 41.37 | 249 | 5.8e-60 | 198 |
| NP_001370927.1 | 44.78 | 201 | 2.0e-54 | 179 |
| NP_499164.1 | 36.25 | 160 | 1.0e-23 | 99 |
| NP_490872.1 | 42.16 | 268 | 2.1e-70 | 225 |
| NP_496449.1 | 41.37 | 249 | 5.8e-60 | 198 |
| NP_001370927.1 | 44.78 | 201 | 2.0e-54 | 179 |
| NP_499164.1 | 36.25 | 160 | 1.0e-23 | 99 |

### BCL2L11 (O43521)

**Description:** Bcl-2-like protein 11

No significant BLAST hits found for this protein.

### BCL6 (P41182)

**Description:** B-cell lymphoma 6 protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001251370.1 | 40.14 | 142 | 3.9e-32 | 133 |
| NP_001251370.1 | 34.40 | 157 | 7.4e-30 | 126 |
| NP_001251371.1 | 40.14 | 142 | 4.5e-32 | 133 |
| NP_001251371.1 | 33.75 | 160 | 8.4e-30 | 125 |
| NP_510480.1 | 38.61 | 158 | 1.1e-31 | 126 |
| NP_510480.1 | 37.67 | 146 | 5.7e-29 | 118 |
| NP_510480.1 | 22.88 | 236 | 3.2e-06 | 49 |
| NP_502594.2 | 35.79 | 190 | 5.7e-30 | 117 |
| NP_502594.2 | 40.96 | 83 | 5.2e-13 | 68 |
| NP_001076597.1 | 46.73 | 107 | 1.9e-26 | 107 |
| NP_001076597.1 | 41.28 | 109 | 4.0e-25 | 103 |
| NP_001076597.1 | 48.19 | 83 | 4.4e-18 | 83 |
| NP_001076597.1 | 36.45 | 107 | 3.5e-17 | 80 |
| NP_001076598.1 | 46.73 | 107 | 2.6e-26 | 108 |
| NP_001076598.1 | 41.28 | 109 | 7.4e-25 | 104 |
| NP_001076598.1 | 48.19 | 83 | 5.5e-18 | 84 |
| NP_001076598.1 | 36.45 | 107 | 8.7e-17 | 81 |
| NP_500033.1 | 30.05 | 193 | 1.3e-22 | 102 |
| NP_500033.1 | 37.86 | 140 | 1.1e-20 | 96 |
| NP_500033.1 | 32.14 | 168 | 2.7e-20 | 95 |
| NP_500033.1 | 38.79 | 116 | 2.0e-15 | 80 |
| NP_500033.1 | 40.00 | 90 | 2.9e-12 | 70 |
| NP_500033.1 | 27.91 | 86 | 1.0e-03 | 42 |
| NP_491096.1 | 39.67 | 121 | 3.2e-22 | 99 |
| NP_491096.1 | 34.06 | 138 | 1.3e-16 | 82 |
| NP_491096.1 | 34.86 | 109 | 2.6e-16 | 81 |
| NP_001024478.1 | 40.71 | 113 | 5.1e-22 | 97 |
| NP_001024478.1 | 30.00 | 160 | 5.9e-13 | 70 |
| NP_001024477.1 | 40.71 | 113 | 6.0e-22 | 97 |
| NP_001024477.1 | 30.00 | 160 | 7.9e-13 | 69 |

### C1GALT1 (Q9NS00)

**Description:** Glycoprotein-N-acetylgalactosamine 3-beta-galactosyltransferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_499293.2 | 49.25 | 333 | 1.7e-113 | 335 |
| NP_505126.2 | 30.71 | 267 | 5.9e-36 | 133 |
| NP_493101.1 | 34.03 | 238 | 1.5e-35 | 132 |
| NP_500615.2 | 30.89 | 259 | 9.4e-34 | 128 |
| NP_504891.1 | 29.06 | 265 | 3.8e-33 | 125 |
| NP_499857.3 | 30.61 | 245 | 1.1e-32 | 122 |
| NP_499851.3 | 30.61 | 245 | 1.1e-32 | 122 |
| NP_493146.2 | 35.23 | 193 | 6.9e-32 | 122 |
| NP_494669.1 | 32.83 | 198 | 2.3e-31 | 121 |
| NP_001021640.2 | 28.65 | 178 | 1.4e-10 | 62 |

### CAP1 (Q01518)

**Description:** Adenylyl cyclase-associated protein 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_510713.1 | 35.08 | 516 | 1.8e-101 | 312 |
| NP_001360035.1 | 37.42 | 465 | 1.5e-94 | 310 |
| NP_491324.1 | 34.54 | 469 | 3.2e-86 | 271 |
| NP_001293294.1 | 39.02 | 305 | 1.9e-58 | 194 |

### CAP2 (P40123)

**Description:** Adenylyl cyclase-associated protein 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_510713.1 | 37.28 | 515 | 2.5e-101 | 312 |
| NP_491324.1 | 35.39 | 438 | 1.8e-89 | 280 |
| NP_001293294.1 | 38.59 | 311 | 1.8e-62 | 205 |
| NP_001360035.1 | 34.25 | 181 | 3.4e-32 | 130 |

### CAV1 (Q03135)

**Description:** Caveolin-1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001255421.1 | 35.48 | 124 | 8.3e-24 | 93 |
| NP_001255422.1 | 33.04 | 112 | 7.5e-20 | 80 |
| NP_001256524.1 | 28.30 | 159 | 1.2e-18 | 80 |
| NP_001256525.1 | 28.30 | 159 | 2.4e-18 | 80 |
| NP_001256523.1 | 28.30 | 159 | 7.2e-18 | 79 |
| NP_001256526.1 | 27.40 | 146 | 1.1e-16 | 73 |

### CCNG2 (Q16589)

**Description:** Cyclin-G2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_501987.1 | 27.93 | 111 | 3.9e-04 | 42 |

### CGA (P01215)

**Description:** Glycoprotein hormones alpha chain

No significant BLAST hits found for this protein.

### CHST10 (O43529)

**Description:** Carbohydrate sulfotransferase 10

No significant BLAST hits found for this protein.

### CHST8 (Q9H2A9)

**Description:** Carbohydrate sulfotransferase 8

No significant BLAST hits found for this protein.

### CITED2 (Q99967)

**Description:** Cbp/p300-interacting transactivator 2

No significant BLAST hits found for this protein.

### CLASP1 (Q7Z460)

**Description:** CLIP-associating protein 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_499005.2 | 34.41 | 247 | 2.1e-37 | 153 |
| NP_499005.2 | 26.32 | 285 | 3.7e-21 | 100 |
| NP_499005.2 | 24.36 | 472 | 5.6e-18 | 90 |
| NP_001263723.1 | 34.41 | 247 | 2.3e-37 | 153 |
| NP_001263723.1 | 26.04 | 288 | 1.0e-20 | 99 |
| NP_001263723.1 | 24.36 | 472 | 6.6e-18 | 90 |
| NP_001379049.1 | 35.34 | 249 | 3.9e-36 | 149 |
| NP_001379049.1 | 27.95 | 229 | 1.7e-20 | 98 |
| NP_001379049.1 | 27.00 | 263 | 6.4e-18 | 90 |
| NP_498649.1 | 27.49 | 251 | 1.7e-26 | 118 |
| NP_498649.1 | 27.00 | 237 | 1.4e-19 | 95 |
| NP_498649.1 | 29.55 | 220 | 3.7e-18 | 91 |
| NP_498649.1 | 29.91 | 214 | 6.0e-15 | 80 |
| NP_495784.1 | 24.35 | 115 | 6.7e-05 | 47 |
| NP_495784.1 | 17.81 | 466 | 5.2e-04 | 44 |
| NP_001256204.1 | 25.95 | 131 | 2.2e-04 | 46 |
| NP_494219.4 | 23.94 | 213 | 3.5e-04 | 45 |

### CLASP2 (O75122)

**Description:** CLIP-associating protein 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_499005.2 | 25.96 | 732 | 9.7e-41 | 164 |
| NP_499005.2 | 27.07 | 229 | 9.5e-21 | 99 |
| NP_499005.2 | 25.99 | 277 | 7.3e-19 | 93 |
| NP_001263723.1 | 25.96 | 732 | 9.8e-41 | 163 |
| NP_001263723.1 | 27.07 | 229 | 9.9e-21 | 99 |
| NP_001263723.1 | 25.71 | 280 | 1.7e-18 | 91 |
| NP_001379049.1 | 31.21 | 282 | 1.6e-33 | 140 |
| NP_001379049.1 | 25.50 | 251 | 9.4e-19 | 92 |
| NP_001379049.1 | 26.67 | 240 | 1.8e-17 | 88 |
| NP_498649.1 | 31.05 | 248 | 7.2e-30 | 129 |
| NP_498649.1 | 24.75 | 295 | 1.3e-21 | 102 |
| NP_498649.1 | 26.78 | 239 | 5.8e-18 | 90 |
| NP_510464.1 | 33.87 | 62 | 3.8e-04 | 43 |

### CREBBP (Q92793)

**Description:** CREB-binding protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_499160.2 | 48.71 | 893 | 0.0e+00 | 863 |
| NP_499160.2 | 58.18 | 110 | 1.7e-32 | 139 |
| NP_499160.2 | 53.90 | 141 | 2.0e-26 | 119 |
| NP_499161.2 | 48.71 | 893 | 0.0e+00 | 863 |
| NP_499161.2 | 58.18 | 110 | 1.9e-32 | 139 |
| NP_499161.2 | 56.92 | 130 | 2.4e-28 | 125 |
| NP_001122711.2 | 48.71 | 893 | 0.0e+00 | 862 |
| NP_001122711.2 | 58.18 | 110 | 2.1e-32 | 139 |
| NP_001122711.2 | 56.92 | 130 | 2.6e-28 | 125 |
| NP_001366682.1 | 63.37 | 101 | 3.1e-37 | 140 |
| NP_499201.1 | 60.00 | 85 | 3.4e-27 | 114 |
| NP_499201.1 | 53.27 | 107 | 3.7e-25 | 108 |
| NP_001355379.1 | 47.78 | 90 | 1.2e-19 | 91 |
| NP_001355379.1 | 27.78 | 72 | 2.1e-05 | 48 |
| NP_001359599.1 | 36.08 | 97 | 1.8e-17 | 80 |
| NP_497144.3 | 41.30 | 92 | 9.0e-16 | 79 |
| NP_497144.3 | 27.36 | 106 | 4.6e-04 | 44 |
| NP_001364495.1 | 30.18 | 169 | 2.4e-15 | 82 |
| NP_001364495.1 | 33.33 | 81 | 1.3e-07 | 57 |
| NP_001367704.1 | 30.18 | 169 | 2.5e-15 | 82 |
| NP_001367704.1 | 33.33 | 81 | 1.2e-07 | 57 |

### CYP1A2 (P05177)

**Description:** Cytochrome P450 1A2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_510205.1 | 30.18 | 487 | 1.2e-60 | 207 |
| NP_510204.1 | 30.15 | 481 | 5.9e-59 | 202 |
| NP_504435.1 | 29.31 | 481 | 2.7e-58 | 201 |
| NP_503846.1 | 29.44 | 445 | 1.4e-56 | 196 |
| NP_503615.1 | 28.48 | 481 | 1.4e-56 | 196 |
| NP_503616.1 | 27.82 | 496 | 5.9e-56 | 194 |
| NP_503594.1 | 28.41 | 454 | 7.5e-56 | 194 |
| NP_001303728.1 | 27.66 | 488 | 1.1e-55 | 194 |
| NP_510203.1 | 28.77 | 504 | 2.0e-55 | 193 |
| NP_503592.2 | 28.26 | 453 | 2.5e-55 | 193 |

### DAD1 (P61803)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit DAD1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_491889.1 | 61.95 | 113 | 6.4e-50 | 154 |

### DDIT3 (P35638)

**Description:** DNA damage-inducible transcript 3 protein

No significant BLAST hits found for this protein.

### DDOST (P39656)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase 48 kDa subunit

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_495655.1 | 53.27 | 428 | 2.7e-160 | 460 |

### DERL1 (Q9BUN8)

**Description:** Derlin-1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_492721.1 | 47.79 | 249 | 3.3e-74 | 226 |
| NP_001379935.1 | 30.84 | 214 | 2.4e-27 | 105 |

### DPAGT1 (Q9H3H5)

**Description:** UDP-N-acetylglucosamine--dolichyl-phosphate N-acetylglucosaminephosphotransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001379258.1 | 50.61 | 411 | 2.1e-133 | 389 |

### ENGASE (Q8NFI3)

**Description:** Cytosolic endo-beta-N-acetylglucosaminidase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001021238.1 | 40.62 | 352 | 3.0e-76 | 252 |
| NP_498267.3 | 44.21 | 285 | 3.1e-71 | 234 |
| NP_001359901.1 | 41.23 | 308 | 3.9e-69 | 232 |

### FASLG (P48023)

**Description:** Tumor necrosis factor ligand superfamily member 6

No significant BLAST hits found for this protein.

### FBXO32 (Q969P5)

**Description:** F-box only protein 32

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_492376.1 | 26.65 | 394 | 1.4e-38 | 141 |

### FOXO3 (O43524)

**Description:** Forkhead box protein O3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001364784.1 | 40.47 | 341 | 7.2e-57 | 201 |
| NP_001364785.1 | 39.88 | 341 | 1.6e-56 | 200 |
| NP_001368502.1 | 50.73 | 205 | 7.2e-54 | 190 |
| NP_001368501.1 | 50.73 | 205 | 9.8e-54 | 189 |
| NP_001368499.1 | 50.73 | 205 | 1.9e-53 | 191 |
| NP_001021596.1 | 50.73 | 205 | 2.1e-53 | 191 |
| NP_001368498.1 | 50.73 | 205 | 4.9e-53 | 190 |
| NP_001021598.2 | 50.73 | 205 | 6.4e-53 | 190 |
| NP_001367893.1 | 50.73 | 205 | 1.0e-52 | 190 |
| NP_001021593.1 | 42.42 | 198 | 2.6e-38 | 149 |

### FOXO4 (P98177)

**Description:** Forkhead box protein O4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001364784.1 | 39.53 | 296 | 8.1e-50 | 178 |
| NP_001364785.1 | 40.00 | 290 | 3.0e-49 | 176 |
| NP_001368502.1 | 48.20 | 195 | 5.8e-49 | 173 |
| NP_001368501.1 | 48.20 | 195 | 6.6e-49 | 173 |
| NP_001368499.1 | 48.20 | 195 | 2.1e-48 | 174 |
| NP_001021596.1 | 48.20 | 195 | 2.3e-48 | 174 |
| NP_001368498.1 | 48.20 | 195 | 4.7e-48 | 173 |
| NP_001021598.2 | 48.20 | 195 | 4.8e-48 | 173 |
| NP_001367893.1 | 48.20 | 195 | 7.2e-48 | 173 |
| NP_001021593.1 | 38.30 | 188 | 2.2e-32 | 130 |

### FUCA1 (P04066)

**Description:** Tissue alpha-L-fucosidase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_510020.2 | 44.03 | 452 | 3.2e-135 | 398 |

### FUT3 (P21217)

**Description:** 3-galactosyl-N-acetylglucosaminide 4-alpha-L-fucosyltransferase FUT3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001022310.1 | 27.87 | 348 | 1.7e-35 | 133 |
| NP_001369885.1 | 32.80 | 250 | 1.2e-34 | 132 |
| NP_494823.2 | 26.50 | 366 | 1.3e-28 | 114 |
| NP_494372.2 | 30.43 | 299 | 1.5e-28 | 114 |
| NP_495106.1 | 28.35 | 388 | 2.2e-28 | 114 |

### FUT8 (Q9BYC5)

**Description:** Alpha-(1,6)-fucosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_504555.2 | 36.35 | 575 | 8.5e-126 | 380 |

### G6PC1 (P35575)

**Description:** Glucose-6-phosphatase catalytic subunit 1

No significant BLAST hits found for this protein.

### GADD45A (P24522)

**Description:** Growth arrest and DNA damage-inducible protein GADD45 alpha

No significant BLAST hits found for this protein.

### GALNT1 (Q10472)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_498722.1 | 47.70 | 608 | 0.0e+00 | 581 |
| NP_001022852.1 | 52.20 | 477 | 5.0e-171 | 498 |
| NP_001379253.1 | 52.40 | 479 | 8.7e-171 | 498 |
| NP_001370120.1 | 50.81 | 492 | 6.3e-168 | 490 |
| NP_001370116.1 | 45.49 | 543 | 3.5e-162 | 475 |
| NP_001022644.1 | 45.39 | 542 | 1.6e-161 | 474 |
| NP_001022646.1 | 54.06 | 394 | 1.4e-154 | 454 |
| NP_001024216.1 | 48.28 | 437 | 1.4e-140 | 419 |
| NP_001022876.1 | 41.50 | 506 | 5.8e-135 | 404 |
| NP_001041037.1 | 41.41 | 524 | 1.5e-131 | 397 |

### GALNT10 (Q86SR1)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 10

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001041037.1 | 49.36 | 549 | 0.0e+00 | 529 |
| NP_001041036.2 | 49.36 | 549 | 0.0e+00 | 529 |
| NP_001300252.1 | 42.83 | 516 | 1.4e-140 | 419 |
| NP_503512.1 | 42.83 | 516 | 3.6e-140 | 420 |
| NP_001022852.1 | 45.47 | 497 | 2.3e-137 | 413 |
| NP_001370120.1 | 45.49 | 499 | 6.5e-137 | 412 |
| NP_001379253.1 | 44.40 | 509 | 9.6e-136 | 409 |
| NP_498722.1 | 40.31 | 521 | 1.9e-124 | 380 |
| NP_001022646.1 | 48.91 | 368 | 1.1e-119 | 366 |
| NP_001370116.1 | 46.13 | 414 | 1.0e-118 | 365 |

### GALNT11 (Q8NCW6)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 11

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001022948.1 | 45.81 | 489 | 6.5e-144 | 430 |
| NP_001022852.1 | 45.45 | 473 | 2.7e-137 | 414 |
| NP_001379253.1 | 45.88 | 473 | 2.0e-135 | 409 |
| NP_001370120.1 | 45.45 | 473 | 5.2e-134 | 405 |
| NP_001022644.1 | 43.03 | 488 | 1.9e-122 | 375 |
| NP_498722.1 | 39.23 | 520 | 5.2e-122 | 374 |
| NP_001370116.1 | 43.35 | 489 | 6.2e-122 | 374 |
| NP_001022646.1 | 51.85 | 351 | 9.5e-121 | 369 |
| NP_001022949.1 | 52.74 | 328 | 1.1e-119 | 362 |
| NP_001022876.1 | 41.18 | 493 | 2.1e-119 | 366 |

### GALNT12 (Q8IXK2)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 12

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001370120.1 | 46.97 | 528 | 1.6e-152 | 451 |
| NP_001022852.1 | 46.78 | 528 | 1.9e-148 | 441 |
| NP_001379253.1 | 46.70 | 531 | 4.6e-148 | 440 |
| NP_498722.1 | 44.47 | 470 | 1.2e-127 | 387 |
| NP_001024216.1 | 42.48 | 492 | 1.9e-125 | 381 |
| NP_001022876.1 | 41.33 | 525 | 4.8e-123 | 374 |
| NP_001370116.1 | 47.24 | 453 | 1.2e-122 | 374 |
| NP_001022646.1 | 49.26 | 404 | 4.3e-120 | 366 |
| NP_001022644.1 | 46.14 | 453 | 4.4e-120 | 368 |
| NP_001379271.1 | 51.15 | 348 | 4.7e-116 | 352 |

### GALNT13 (Q8IUC8)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 13

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_498722.1 | 47.59 | 603 | 0.0e+00 | 575 |
| NP_001379253.1 | 52.51 | 478 | 2.1e-169 | 494 |
| NP_001022852.1 | 51.89 | 476 | 3.4e-169 | 493 |
| NP_001370120.1 | 51.26 | 476 | 6.4e-166 | 485 |
| NP_001370116.1 | 47.32 | 541 | 5.6e-162 | 474 |
| NP_001022644.1 | 47.04 | 540 | 4.1e-160 | 470 |
| NP_001022646.1 | 55.67 | 397 | 2.4e-153 | 451 |
| NP_001024216.1 | 48.16 | 434 | 9.2e-139 | 414 |
| NP_001022876.1 | 43.54 | 480 | 7.1e-131 | 394 |
| NP_001041037.1 | 41.62 | 519 | 2.5e-129 | 391 |

### GALNT14 (Q96FL9)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 14

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001024216.1 | 48.56 | 451 | 4.1e-155 | 456 |
| NP_001379271.1 | 54.12 | 364 | 6.3e-149 | 435 |
| NP_001256888.1 | 54.12 | 364 | 1.2e-148 | 436 |
| NP_001022852.1 | 44.40 | 482 | 3.5e-132 | 398 |
| NP_001379253.1 | 43.59 | 484 | 3.0e-129 | 391 |
| NP_001370120.1 | 43.18 | 484 | 3.8e-128 | 388 |
| NP_498722.1 | 47.20 | 411 | 2.9e-122 | 372 |
| NP_001022644.1 | 41.68 | 511 | 4.6e-117 | 359 |
| NP_001370116.1 | 42.16 | 510 | 1.4e-116 | 358 |
| NP_001022646.1 | 48.35 | 364 | 1.7e-112 | 345 |

### GALNT16 (Q8N428)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 16

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001024216.1 | 47.40 | 500 | 3.5e-148 | 438 |
| NP_001379271.1 | 56.67 | 360 | 2.1e-145 | 426 |
| NP_001256888.1 | 56.67 | 360 | 4.4e-145 | 427 |
| NP_001379253.1 | 44.02 | 502 | 1.3e-131 | 397 |
| NP_001370120.1 | 44.22 | 502 | 5.3e-131 | 395 |
| NP_001022852.1 | 44.00 | 500 | 2.7e-130 | 394 |
| NP_498722.1 | 41.19 | 488 | 4.3e-120 | 367 |
| NP_001022876.1 | 37.62 | 513 | 1.7e-112 | 346 |
| NP_001022644.1 | 39.30 | 514 | 5.4e-111 | 343 |
| NP_001370116.1 | 39.49 | 509 | 7.1e-111 | 343 |

### GALNT18 (Q6P9A2)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 18

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001370120.1 | 34.58 | 509 | 5.9e-92 | 296 |
| NP_498722.1 | 36.42 | 486 | 7.1e-92 | 295 |
| NP_001022852.1 | 35.23 | 511 | 8.9e-92 | 295 |
| NP_001379253.1 | 34.96 | 512 | 2.6e-91 | 294 |
| NP_001022644.1 | 31.37 | 612 | 2.0e-90 | 291 |
| NP_001370116.1 | 32.04 | 618 | 4.8e-89 | 288 |
| NP_001022646.1 | 34.45 | 479 | 2.6e-88 | 285 |
| NP_001022876.1 | 32.74 | 510 | 1.8e-84 | 275 |
| NP_001024216.1 | 39.44 | 393 | 9.8e-83 | 270 |
| NP_001379271.1 | 41.08 | 353 | 6.5e-82 | 264 |

### GALNT2 (Q10471)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001024216.1 | 55.43 | 442 | 5.5e-177 | 513 |
| NP_001379271.1 | 61.71 | 363 | 1.0e-165 | 479 |
| NP_001256888.1 | 61.71 | 363 | 1.0e-165 | 479 |
| NP_498722.1 | 45.90 | 488 | 2.7e-140 | 419 |
| NP_001379253.1 | 44.77 | 478 | 1.2e-133 | 403 |
| NP_001370120.1 | 44.96 | 476 | 5.5e-133 | 401 |
| NP_001022852.1 | 44.54 | 476 | 1.6e-132 | 400 |
| NP_001022644.1 | 42.73 | 461 | 6.6e-119 | 365 |
| NP_001022646.1 | 47.95 | 365 | 1.7e-117 | 359 |
| NP_001370116.1 | 42.37 | 465 | 1.8e-117 | 361 |

### GALNT3 (Q14435)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001370120.1 | 45.66 | 519 | 4.5e-143 | 429 |
| NP_001379253.1 | 45.77 | 520 | 7.4e-143 | 429 |
| NP_001022852.1 | 45.66 | 519 | 4.3e-142 | 427 |
| NP_498722.1 | 42.38 | 538 | 1.5e-125 | 384 |
| NP_001024216.1 | 41.38 | 563 | 1.7e-122 | 375 |
| NP_001022644.1 | 41.12 | 552 | 6.4e-117 | 362 |
| NP_001022876.1 | 39.20 | 523 | 4.2e-115 | 355 |
| NP_001370116.1 | 41.04 | 558 | 4.4e-115 | 357 |
| NP_001022646.1 | 48.51 | 402 | 1.4e-114 | 353 |
| NP_001379271.1 | 47.57 | 412 | 1.8e-113 | 347 |

### GALNT4 (Q8N4A0)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001370120.1 | 47.18 | 532 | 1.4e-152 | 451 |
| NP_001379253.1 | 47.48 | 535 | 2.1e-151 | 449 |
| NP_001022852.1 | 47.46 | 531 | 3.3e-150 | 446 |
| NP_498722.1 | 41.73 | 544 | 3.6e-136 | 409 |
| NP_001024216.1 | 42.63 | 502 | 8.6e-124 | 376 |
| NP_001022876.1 | 38.08 | 583 | 5.5e-120 | 366 |
| NP_001022644.1 | 40.83 | 578 | 4.2e-117 | 360 |
| NP_001370116.1 | 40.87 | 575 | 9.0e-117 | 359 |
| NP_001022646.1 | 44.47 | 479 | 1.2e-116 | 357 |
| NP_001379271.1 | 50.14 | 357 | 1.8e-113 | 345 |

### GALNT9 (Q9HCQ5)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 9

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_498722.1 | 36.27 | 590 | 9.7e-107 | 334 |
| NP_001022852.1 | 41.24 | 451 | 8.3e-104 | 327 |
| NP_001379253.1 | 40.88 | 455 | 7.8e-103 | 324 |
| NP_001370120.1 | 39.82 | 452 | 5.8e-102 | 322 |
| NP_001024216.1 | 38.24 | 455 | 1.4e-94 | 301 |
| NP_001379271.1 | 42.53 | 355 | 7.7e-92 | 290 |
| NP_001256888.1 | 42.53 | 355 | 1.8e-91 | 290 |
| NP_001022646.1 | 39.03 | 392 | 7.8e-91 | 291 |
| NP_001300252.1 | 37.58 | 463 | 2.8e-90 | 289 |
| NP_001022644.1 | 39.03 | 392 | 2.8e-90 | 291 |

### GALNTL5 (Q7Z4T8)

**Description:** Inactive polypeptide N-acetylgalactosaminyltransferase-like protein 5

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001022852.1 | 46.82 | 346 | 8.5e-107 | 329 |
| NP_001370120.1 | 46.82 | 346 | 1.2e-106 | 328 |
| NP_001379253.1 | 46.82 | 346 | 1.5e-106 | 328 |
| NP_001022948.1 | 45.93 | 344 | 6.3e-104 | 321 |
| NP_001022646.1 | 44.44 | 369 | 6.3e-100 | 309 |
| NP_001022644.1 | 43.94 | 371 | 5.4e-99 | 308 |
| NP_001370116.1 | 44.44 | 369 | 1.1e-98 | 308 |
| NP_498722.1 | 44.85 | 359 | 5.6e-98 | 306 |
| NP_001022876.1 | 44.06 | 345 | 1.4e-96 | 301 |
| NP_001256888.1 | 43.35 | 346 | 1.6e-92 | 288 |

### GALNTL6 (Q49A17)

**Description:** Polypeptide N-acetylgalactosaminyltransferase-like 6

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001041037.1 | 46.25 | 560 | 5.1e-173 | 504 |
| NP_001041036.2 | 46.25 | 560 | 3.6e-172 | 505 |
| NP_001300252.1 | 41.71 | 537 | 2.0e-139 | 416 |
| NP_503512.1 | 41.71 | 537 | 7.9e-139 | 416 |
| NP_001022852.1 | 42.88 | 534 | 7.1e-129 | 392 |
| NP_001370120.1 | 42.75 | 531 | 1.4e-127 | 388 |
| NP_001379253.1 | 42.59 | 533 | 7.0e-126 | 384 |
| NP_001022646.1 | 46.61 | 369 | 9.8e-112 | 345 |
| NP_001370116.1 | 44.42 | 403 | 1.2e-111 | 347 |
| NP_498722.1 | 37.07 | 518 | 3.9e-111 | 345 |

### GANAB (Q14697)

**Description:** Neutral alpha-glucosidase AB

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_505508.1 | 45.25 | 937 | 0.0e+00 | 828 |
| NP_505507.1 | 45.15 | 948 | 0.0e+00 | 824 |
| NP_508105.2 | 43.50 | 938 | 0.0e+00 | 802 |
| NP_001263844.1 | 51.13 | 661 | 0.0e+00 | 710 |
| NP_494897.3 | 28.55 | 571 | 3.8e-55 | 207 |
| NP_501419.2 | 28.77 | 563 | 7.1e-53 | 200 |

### GCK (P35557)

**Description:** Hexokinase-4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001370182.1 | 45.28 | 466 | 7.7e-135 | 397 |
| NP_001021106.1 | 45.99 | 461 | 9.8e-135 | 397 |
| NP_492905.1 | 40.00 | 455 | 2.5e-108 | 331 |
| NP_001370784.1 | 32.30 | 452 | 5.6e-79 | 252 |
| NP_001294488.1 | 32.32 | 263 | 3.8e-38 | 139 |
| NP_001294489.1 | 34.45 | 209 | 1.3e-30 | 117 |

### GFPT1 (Q06210)

**Description:** Glutamine--fructose-6-phosphate aminotransferase [isomerizing] 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_502156.1 | 58.88 | 715 | 0.0e+00 | 870 |
| NP_001379278.1 | 58.02 | 717 | 0.0e+00 | 861 |
| NP_001369886.1 | 56.77 | 738 | 0.0e+00 | 853 |

### GFPT2 (O94808)

**Description:** Glutamine--fructose-6-phosphate aminotransferase [isomerizing] 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001379278.1 | 58.24 | 716 | 0.0e+00 | 847 |
| NP_001369886.1 | 57.20 | 729 | 0.0e+00 | 842 |
| NP_502156.1 | 57.70 | 714 | 0.0e+00 | 835 |
| NP_497958.1 | 29.11 | 158 | 4.1e-04 | 43 |

### GNPNAT1 (Q96EK6)

**Description:** Glucosamine 6-phosphate N-acetyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_505654.1 | 41.98 | 162 | 9.9e-41 | 135 |
| NP_492144.1 | 38.98 | 177 | 1.1e-33 | 122 |
| NP_492144.1 | 40.37 | 161 | 1.5e-33 | 121 |

### HDAC2 (Q92769)

**Description:** Histone deacetylase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_506599.1 | 64.60 | 452 | 0.0e+00 | 636 |
| NP_493026.1 | 60.74 | 461 | 0.0e+00 | 605 |
| NP_495678.1 | 50.40 | 379 | 6.9e-141 | 414 |
| NP_001257278.1 | 27.41 | 270 | 1.3e-20 | 95 |
| NP_001257279.1 | 27.41 | 270 | 1.3e-20 | 95 |
| NP_500788.1 | 25.98 | 331 | 2.9e-20 | 94 |
| NP_500788.1 | 31.15 | 183 | 8.5e-19 | 89 |
| NP_500787.1 | 25.83 | 333 | 6.2e-20 | 93 |
| NP_500787.1 | 31.15 | 183 | 8.2e-19 | 89 |
| NP_001122780.1 | 25.83 | 333 | 6.9e-20 | 93 |
| NP_001122780.1 | 31.15 | 183 | 8.3e-19 | 89 |
| NP_496910.1 | 24.00 | 275 | 1.0e-17 | 85 |
| NP_505699.3 | 23.49 | 315 | 9.9e-15 | 75 |

### INS (P01308)

**Description:** Insulin

No significant BLAST hits found for this protein.

### KLF4 (O43474)

**Description:** Krueppel-like factor 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_497632.1 | 79.76 | 84 | 3.2e-44 | 162 |
| NP_507995.2 | 65.39 | 104 | 6.1e-40 | 146 |
| NP_001022204.1 | 65.26 | 95 | 4.0e-39 | 144 |
| NP_001022205.1 | 65.26 | 95 | 4.8e-39 | 144 |
| NP_495833.1 | 59.26 | 81 | 2.1e-26 | 105 |
| NP_493353.3 | 52.87 | 87 | 2.5e-24 | 105 |
| NP_493353.3 | 50.88 | 57 | 2.8e-11 | 65 |
| NP_001368248.1 | 57.38 | 61 | 3.6e-18 | 80 |
| NP_001368248.1 | 40.96 | 83 | 1.9e-16 | 75 |
| NP_001379380.1 | 57.38 | 61 | 3.9e-18 | 80 |
| NP_001379380.1 | 40.96 | 83 | 2.1e-16 | 75 |
| NP_001379380.1 | 50.00 | 64 | 3.0e-12 | 63 |
| NP_001368249.1 | 57.38 | 61 | 7.4e-18 | 79 |
| NP_001368249.1 | 40.96 | 83 | 4.1e-16 | 74 |
| NP_001368249.1 | 54.72 | 53 | 2.3e-11 | 61 |
| NP_510462.2 | 62.50 | 56 | 8.6e-18 | 85 |
| NP_510462.2 | 44.58 | 83 | 4.1e-17 | 83 |
| NP_510462.2 | 53.85 | 52 | 6.0e-11 | 64 |

### LARGE1 (O95461)

**Description:** Xylosyl- and glucuronyltransferase LARGE1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_509833.3 | 33.92 | 625 | 2.0e-109 | 346 |
| NP_001256048.1 | 23.08 | 208 | 2.4e-09 | 60 |
| NP_001024774.1 | 22.49 | 209 | 6.2e-07 | 52 |
| NP_001300086.1 | 21.14 | 246 | 1.2e-06 | 51 |
| NP_493157.2 | 20.39 | 309 | 1.5e-06 | 51 |
| NP_001364608.1 | 21.01 | 276 | 2.2e-05 | 47 |
| NP_507102.1 | 21.26 | 207 | 2.3e-05 | 47 |
| NP_001023113.1 | 19.13 | 230 | 2.6e-04 | 44 |

### LHB (P01229)

**Description:** Lutropin subunit beta

No significant BLAST hits found for this protein.

### MAGT1 (Q9H0U3)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit MAGT1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_498691.1 | 41.77 | 328 | 6.1e-97 | 290 |

### MAN2A1 (Q16706)

**Description:** Alpha-mannosidase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_505995.2 | 36.75 | 1140 | 0.0e+00 | 736 |
| NP_001362173.1 | 30.77 | 949 | 1.2e-133 | 432 |
| NP_001361919.1 | 30.76 | 933 | 3.4e-132 | 429 |
| NP_508811.1 | 25.88 | 935 | 3.6e-75 | 268 |

### MAN2A2 (P49641)

**Description:** Alpha-mannosidase 2x

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_505995.2 | 40.02 | 1032 | 0.0e+00 | 732 |
| NP_001361919.1 | 30.80 | 974 | 2.2e-138 | 446 |
| NP_001362173.1 | 30.83 | 973 | 2.2e-138 | 445 |
| NP_508811.1 | 26.42 | 931 | 1.1e-77 | 276 |

### MANEA (Q5SRI9)

**Description:** Glycoprotein endo-alpha-1,2-mannosidase

No significant BLAST hits found for this protein.

### MGAT1 (P26572)

**Description:** Alpha-1,3-mannosyl-glycoprotein 2-beta-N-acetylglucosaminyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_509566.1 | 44.29 | 350 | 1.5e-108 | 328 |
| NP_497719.1 | 45.24 | 347 | 9.3e-107 | 323 |
| NP_741838.1 | 36.52 | 356 | 3.2e-74 | 240 |

### MGAT2 (Q10469)

**Description:** Alpha-1,6-mannosyl-glycoprotein 2-beta-N-acetylglucosaminyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_505864.1 | 35.10 | 453 | 1.1e-100 | 309 |

### MGAT3 (Q09327)

**Description:** Beta-1,4-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase

No significant BLAST hits found for this protein.

### MGAT4A (Q9UM21)

**Description:** Alpha-1,3-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase A

No significant BLAST hits found for this protein.

### MGAT4B (Q9UQ53)

**Description:** Alpha-1,3-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase B

No significant BLAST hits found for this protein.

### MGAT4C (Q9UBM8)

**Description:** Alpha-1,3-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase C

No significant BLAST hits found for this protein.

### MGAT5 (Q09328)

**Description:** Alpha-1,6-mannosylglycoprotein 6-beta-N-acetylglucosaminyltransferase A

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_491874.1 | 35.92 | 643 | 2.9e-124 | 385 |

### MLEC (Q14165)

**Description:** Malectin

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_498951.1 | 34.21 | 266 | 5.3e-49 | 164 |

### MOGS (Q13724)

**Description:** Mannosyl-oligosaccharide glucosidase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001122771.1 | 36.77 | 775 | 6.8e-160 | 485 |
| NP_001076682.1 | 36.65 | 775 | 8.6e-160 | 485 |
| NP_001076681.1 | 36.76 | 778 | 1.1e-159 | 485 |

### MPDU1 (O75352)

**Description:** Mannose-P-dolichol utilization defect 1 protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_505155.2 | 39.91 | 233 | 4.1e-59 | 187 |

### MUC1 (P15941)

**Description:** Mucin-1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_493835.1 | 30.11 | 797 | 4.0e-28 | 123 |
| NP_493835.1 | 30.24 | 797 | 1.3e-27 | 121 |
| NP_493835.1 | 30.24 | 797 | 1.3e-27 | 121 |
| NP_493835.1 | 30.24 | 797 | 1.3e-27 | 121 |
| NP_493835.1 | 30.24 | 797 | 1.3e-27 | 121 |
| NP_493835.1 | 30.41 | 753 | 1.4e-26 | 118 |
| NP_493835.1 | 30.02 | 786 | 3.2e-26 | 117 |
| NP_494447.1 | 41.03 | 273 | 5.1e-07 | 53 |

### MUC12 (Q9UKN1)

**Description:** Mucin-12

No significant BLAST hits found for this protein.

### MUC15 (Q8N387)

**Description:** Mucin-15

No significant BLAST hits found for this protein.

### MUC17 (Q685J3)

**Description:** Mucin-17

No significant BLAST hits found for this protein.

### MUC19 (Q7Z5P9)

**Description:** Mucin-19

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001361993.1 | 30.98 | 297 | 1.4e-09 | 65 |
| NP_001361993.1 | 32.03 | 231 | 1.8e-08 | 62 |
| NP_001361993.1 | 30.40 | 250 | 2.6e-07 | 58 |
| NP_001361993.1 | 27.54 | 305 | 3.6e-07 | 57 |
| NP_001368773.1 | 30.98 | 297 | 2.4e-09 | 65 |
| NP_001368773.1 | 32.03 | 231 | 3.6e-08 | 61 |
| NP_001362103.1 | 30.98 | 297 | 8.8e-09 | 63 |
| NP_001362103.1 | 32.03 | 231 | 1.5e-07 | 58 |
| NP_001362103.1 | 29.52 | 271 | 1.2e-06 | 56 |
| NP_001362103.1 | 27.54 | 305 | 2.0e-06 | 55 |
| NP_001362104.1 | 30.98 | 297 | 2.0e-08 | 62 |
| NP_001362104.1 | 32.03 | 231 | 3.4e-07 | 58 |
| NP_001362104.1 | 26.52 | 313 | 6.9e-04 | 47 |
| NP_001362105.1 | 30.98 | 297 | 3.0e-08 | 61 |
| NP_001362105.1 | 32.03 | 231 | 5.4e-07 | 57 |
| NP_506395.1 | 21.60 | 375 | 2.0e-07 | 58 |
| NP_001361994.1 | 30.98 | 297 | 1.3e-06 | 56 |
| NP_001361994.1 | 29.79 | 329 | 4.0e-06 | 54 |
| NP_001361994.1 | 27.54 | 305 | 5.8e-04 | 47 |
| NP_001263928.1 | 25.13 | 191 | 4.3e-06 | 53 |
| NP_001123062.1 | 25.13 | 191 | 6.0e-06 | 53 |
| NP_506394.1 | 20.97 | 391 | 8.5e-06 | 53 |

### MUC20 (Q8N307)

**Description:** Mucin-20

No significant BLAST hits found for this protein.

### MUC21 (Q5SSG8)

**Description:** Mucin-21

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001337278.1 | 41.90 | 432 | 3.1e-11 | 66 |
| NP_001337278.1 | 42.42 | 356 | 1.4e-09 | 61 |
| NP_001337278.1 | 42.66 | 443 | 4.6e-08 | 56 |
| NP_494447.1 | 34.38 | 349 | 8.1e-04 | 42 |

### MUC3A (Q02505)

**Description:** Mucin-3A

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_501964.2 | 40.91 | 66 | 3.3e-09 | 63 |
| NP_001338858.1 | 44.62 | 65 | 7.5e-08 | 58 |
| NP_001338858.1 | 49.06 | 53 | 1.3e-06 | 54 |
| NP_001024741.1 | 41.54 | 65 | 9.8e-05 | 48 |
| NP_001338800.1 | 46.67 | 45 | 1.0e-04 | 48 |
| NP_001367835.1 | 46.67 | 45 | 1.1e-04 | 48 |
| NP_508552.2 | 41.67 | 60 | 1.1e-04 | 48 |
| NP_001362066.1 | 41.67 | 60 | 1.1e-04 | 48 |
| NP_001369791.1 | 41.54 | 65 | 1.1e-04 | 48 |
| NP_500454.2 | 34.43 | 61 | 1.2e-04 | 48 |
| NP_001367788.1 | 40.00 | 60 | 4.0e-04 | 46 |

### MUC4 (Q99102)

**Description:** Mucin-4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_499205.1 | 23.21 | 504 | 7.1e-18 | 92 |
| NP_510822.1 | 27.82 | 133 | 2.9e-05 | 50 |
| NP_001293936.1 | 29.29 | 140 | 8.3e-04 | 46 |

### NAGK (Q9UJ70)

**Description:** N-acetyl-D-glucosamine kinase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_494789.1 | 31.29 | 326 | 9.6e-49 | 171 |
| NP_494790.2 | 22.42 | 339 | 4.4e-07 | 51 |

### NFYA (P23511)

**Description:** Nuclear transcription factor Y subunit alpha

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_509999.1 | 43.56 | 101 | 6.1e-18 | 84 |
| NP_001251587.1 | 55.17 | 58 | 4.1e-15 | 74 |
| NP_001251586.1 | 55.17 | 58 | 4.6e-15 | 74 |
| NP_001251588.1 | 57.69 | 52 | 2.1e-13 | 66 |

### NFYC (Q13952)

**Description:** Nuclear transcription factor Y subunit gamma

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001300537.1 | 40.00 | 110 | 1.9e-22 | 94 |
| NP_493645.1 | 40.00 | 110 | 2.3e-22 | 95 |
| NP_497087.1 | 43.33 | 90 | 5.2e-11 | 61 |

### NGLY1 (Q96IV0)

**Description:** Peptide-N(4)-(N-acetyl-beta-glucosaminyl)asparagine amidase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_492913.1 | 37.94 | 427 | 1.4e-90 | 293 |
| NP_500648.2 | 32.00 | 125 | 1.5e-06 | 51 |

### NR3C1 (P04150)

**Description:** Glucocorticoid receptor

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_492615.2 | 25.34 | 292 | 3.5e-27 | 114 |
| NP_001379703.1 | 24.49 | 343 | 8.3e-24 | 106 |
| NP_741456.1 | 24.49 | 343 | 1.0e-23 | 106 |
| NP_510115.1 | 32.65 | 196 | 3.8e-22 | 99 |
| NP_001122482.2 | 28.64 | 213 | 1.0e-21 | 97 |
| NP_001021042.2 | 25.77 | 291 | 4.0e-21 | 95 |
| NP_001343721.1 | 25.77 | 291 | 5.9e-21 | 95 |
| NP_001024953.1 | 29.60 | 223 | 8.7e-21 | 96 |
| NP_001024550.1 | 46.91 | 81 | 1.5e-20 | 96 |
| NP_494368.2 | 24.43 | 262 | 3.5e-20 | 95 |

### PCBP4 (P57723)

**Description:** Poly(rC)-binding protein 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_497351.3 | 58.82 | 170 | 1.2e-67 | 220 |
| NP_001379658.1 | 27.48 | 353 | 1.7e-34 | 131 |
| NP_740869.1 | 27.48 | 353 | 2.4e-34 | 131 |
| NP_001256769.1 | 31.75 | 189 | 1.7e-14 | 74 |
| NP_001360614.1 | 31.75 | 189 | 1.9e-14 | 74 |
| NP_491627.2 | 29.55 | 176 | 2.5e-13 | 71 |
| NP_491627.2 | 25.71 | 140 | 1.2e-08 | 56 |
| NP_491627.2 | 33.33 | 84 | 5.9e-04 | 42 |
| NP_505633.1 | 32.37 | 139 | 3.0e-13 | 71 |
| NP_505633.1 | 24.48 | 290 | 1.8e-08 | 56 |
| NP_505633.1 | 27.89 | 147 | 1.3e-07 | 53 |
| NP_505633.1 | 22.50 | 320 | 1.3e-04 | 43 |
| NP_505633.1 | 26.13 | 111 | 1.7e-04 | 43 |
| NP_001367132.1 | 32.37 | 139 | 4.3e-13 | 70 |
| NP_001367132.1 | 24.48 | 290 | 2.9e-08 | 55 |
| NP_001367132.1 | 27.89 | 147 | 1.7e-07 | 53 |
| NP_001367132.1 | 33.82 | 68 | 1.0e-03 | 41 |
| NP_505632.1 | 32.37 | 139 | 4.9e-13 | 70 |
| NP_505632.1 | 24.48 | 290 | 2.2e-08 | 55 |
| NP_505632.1 | 27.89 | 147 | 1.3e-07 | 53 |
| NP_505632.1 | 25.77 | 97 | 6.2e-04 | 42 |
| NP_510825.1 | 27.66 | 141 | 1.1e-10 | 63 |
| NP_510825.1 | 23.67 | 169 | 5.2e-09 | 58 |
| NP_510825.1 | 23.12 | 333 | 7.5e-09 | 57 |

### PCK1 (P35558)

**Description:** Phosphoenolpyruvate carboxykinase, cytosolic [GTP]

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001366912.1 | 57.77 | 611 | 0.0e+00 | 756 |
| NP_001379420.1 | 58.13 | 609 | 0.0e+00 | 755 |
| NP_001367091.1 | 57.77 | 611 | 0.0e+00 | 755 |
| NP_001367090.1 | 58.77 | 587 | 0.0e+00 | 739 |
| NP_497134.1 | 57.01 | 621 | 0.0e+00 | 738 |
| NP_500887.1 | 24.45 | 634 | 6.6e-42 | 160 |

### PGM3 (O95394)

**Description:** Phosphoacetylglucosamine mutase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_501500.1 | 42.02 | 545 | 3.2e-133 | 398 |

### PINK1 (Q9BXM7)

**Description:** Serine/threonine-protein kinase PINK1, mitochondrial

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_495017.1 | 31.74 | 482 | 2.3e-61 | 214 |
| NP_491914.1 | 30.13 | 156 | 5.5e-11 | 64 |
| NP_499922.1 | 27.85 | 237 | 3.7e-10 | 62 |
| NP_001368037.1 | 26.36 | 239 | 5.4e-10 | 61 |
| NP_494947.2 | 30.99 | 213 | 7.2e-09 | 58 |
| NP_001359980.1 | 28.23 | 170 | 1.1e-08 | 57 |
| NP_001022563.1 | 29.27 | 164 | 1.4e-08 | 57 |
| NP_001021603.1 | 31.61 | 155 | 1.6e-08 | 57 |
| NP_001021603.1 | 31.71 | 123 | 6.8e-05 | 45 |
| NP_500139.1 | 30.89 | 123 | 1.7e-08 | 56 |
| NP_001359981.1 | 28.23 | 170 | 2.0e-08 | 57 |

### PLXNA4 (Q9HCM2)

**Description:** Plexin-A4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001294554.1 | 34.90 | 1931 | 0.0e+00 | 1080 |
| NP_500018.3 | 34.90 | 1920 | 0.0e+00 | 1073 |
| NP_001294555.1 | 35.81 | 1737 | 0.0e+00 | 1025 |
| NP_001294556.1 | 35.80 | 1726 | 0.0e+00 | 1020 |
| NP_001294558.1 | 39.84 | 1142 | 0.0e+00 | 795 |
| NP_001294557.1 | 39.88 | 1131 | 0.0e+00 | 787 |
| NP_001364609.1 | 25.95 | 1942 | 1.5e-158 | 533 |
| NP_493582.3 | 23.42 | 316 | 3.1e-11 | 68 |
| NP_001391076.1 | 23.42 | 316 | 3.3e-11 | 68 |
| NP_491197.3 | 29.36 | 109 | 2.4e-08 | 58 |

### POFUT2 (Q9Y2G5)

**Description:** GDP-fucose protein O-fucosyltransferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001255069.1 | 45.39 | 401 | 6.6e-119 | 353 |
| NP_001255070.1 | 45.39 | 401 | 2.1e-117 | 349 |

### POMGNT1 (Q8WZA1)

**Description:** Protein O-linked-mannose beta-1,2-N-acetylglucosaminyltransferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_500183.2 | 26.11 | 383 | 7.1e-36 | 144 |
| NP_497719.1 | 31.20 | 343 | 2.4e-32 | 130 |
| NP_741838.1 | 34.63 | 231 | 2.2e-27 | 115 |
| NP_509566.1 | 37.82 | 156 | 2.4e-26 | 112 |

### POMT1 (Q9Y6A1)

**Description:** Protein O-mannosyl-transferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_491320.1 | 26.32 | 190 | 2.2e-05 | 46 |

### POMT2 (Q9UKY4)

**Description:** Protein O-mannosyl-transferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_491320.1 | 32.16 | 171 | 4.0e-11 | 63 |

### PPARGC1A (Q9UBK2)

**Description:** Peroxisome proliferator-activated receptor gamma coactivator 1-alpha

No significant BLAST hits found for this protein.

### PRKCSH (P14314)

**Description:** Glucosidase 2 subunit beta

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_496073.1 | 35.03 | 531 | 1.1e-101 | 315 |
| NP_491270.2 | 35.16 | 91 | 2.2e-06 | 50 |
| NP_491270.2 | 35.16 | 91 | 1.9e-04 | 44 |

### PSMC1 (P62191)

**Description:** 26S proteasome regulatory subunit 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_504558.1 | 83.07 | 443 | 0.0e+00 | 719 |
| NP_001379656.1 | 56.84 | 424 | 7.4e-164 | 468 |
| NP_498429.1 | 52.16 | 370 | 6.1e-136 | 396 |
| NP_001379988.1 | 50.14 | 353 | 1.2e-120 | 357 |
| NP_001367906.1 | 50.14 | 353 | 2.7e-120 | 357 |
| NP_499609.1 | 49.58 | 353 | 3.3e-120 | 356 |
| NP_491672.1 | 47.32 | 355 | 7.2e-113 | 338 |
| NP_506005.1 | 47.20 | 339 | 3.8e-106 | 321 |
| NP_001022114.1 | 43.75 | 352 | 2.4e-102 | 310 |
| NP_001022113.1 | 43.75 | 352 | 3.2e-102 | 310 |

### RAD23B (P54727)

**Description:** UV excision repair protein RAD23 homolog B

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_496488.2 | 39.21 | 227 | 1.0e-37 | 139 |
| NP_496488.2 | 42.05 | 88 | 3.3e-12 | 67 |
| NP_001379170.1 | 39.06 | 64 | 8.7e-09 | 52 |
| NP_499695.1 | 33.82 | 68 | 2.6e-06 | 46 |
| NP_001368176.1 | 24.68 | 158 | 1.1e-05 | 47 |
| NP_001368179.1 | 24.68 | 158 | 1.1e-05 | 47 |
| NP_495932.4 | 24.68 | 158 | 1.6e-05 | 47 |
| NP_495931.4 | 24.68 | 158 | 1.6e-05 | 47 |
| NP_001293495.1 | 25.71 | 140 | 4.2e-05 | 45 |
| NP_001293493.1 | 25.71 | 140 | 4.3e-05 | 45 |
| NP_741157.1 | 35.94 | 64 | 6.3e-04 | 42 |
| NP_741157.1 | 35.94 | 64 | 6.3e-04 | 42 |
| NP_741157.1 | 35.94 | 64 | 6.3e-04 | 42 |
| NP_741157.1 | 35.94 | 64 | 6.3e-04 | 42 |
| NP_741157.1 | 35.94 | 64 | 6.3e-04 | 42 |
| NP_741157.1 | 35.94 | 64 | 6.3e-04 | 42 |
| NP_741157.1 | 35.94 | 64 | 6.3e-04 | 42 |
| NP_741157.1 | 35.94 | 64 | 6.3e-04 | 42 |
| NP_741157.1 | 35.94 | 64 | 6.3e-04 | 42 |
| NP_741157.1 | 35.94 | 64 | 6.3e-04 | 42 |
| NP_741157.1 | 35.94 | 64 | 8.7e-04 | 41 |

### RBL2 (Q08999)

**Description:** Retinoblastoma-like protein 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_491686.1 | 25.87 | 603 | 6.0e-36 | 147 |
| NP_491686.1 | 33.67 | 98 | 2.8e-13 | 74 |

### RENBP (P51606)

**Description:** N-acylglucosamine 2-epimerase

No significant BLAST hits found for this protein.

### RETN (Q9HD89)

**Description:** Resistin

No significant BLAST hits found for this protein.

### RFT1 (Q96AA3)

**Description:** Man(5)GlcNAc(2)-PP-dolichol translocation protein RFT1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001023610.1 | 29.32 | 498 | 2.7e-61 | 210 |
| NP_001367982.1 | 29.55 | 335 | 3.5e-40 | 149 |
| NP_001368266.1 | 27.97 | 118 | 1.3e-04 | 42 |

### ROBO1 (Q9Y6N7)

**Description:** Roundabout homolog 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001024990.1 | 37.79 | 860 | 4.6e-166 | 538 |
| NP_741748.2 | 37.79 | 860 | 5.9e-166 | 537 |
| NP_001024582.1 | 29.49 | 529 | 3.2e-53 | 207 |
| NP_001024582.1 | 29.55 | 511 | 2.2e-42 | 171 |
| NP_001024582.1 | 26.95 | 538 | 7.1e-42 | 169 |
| NP_001024582.1 | 27.69 | 502 | 1.5e-41 | 168 |
| NP_001024582.1 | 26.79 | 530 | 1.9e-38 | 158 |
| NP_001024582.1 | 28.63 | 496 | 2.3e-38 | 158 |
| NP_001024582.1 | 28.97 | 497 | 1.1e-37 | 156 |
| NP_001024582.1 | 29.28 | 461 | 5.4e-37 | 153 |
| NP_001024582.1 | 27.83 | 503 | 1.4e-36 | 152 |
| NP_001024582.1 | 28.38 | 458 | 6.4e-35 | 147 |
| NP_001024582.1 | 29.54 | 501 | 6.6e-34 | 143 |
| NP_001024582.1 | 29.25 | 441 | 3.0e-33 | 141 |
| NP_001024582.1 | 24.85 | 499 | 4.5e-33 | 140 |
| NP_001024582.1 | 27.29 | 535 | 7.2e-32 | 137 |
| NP_001024582.1 | 25.90 | 471 | 1.4e-30 | 132 |
| NP_001024582.1 | 26.23 | 469 | 5.2e-28 | 124 |
| NP_001024582.1 | 27.43 | 452 | 1.3e-26 | 119 |
| NP_001024582.1 | 27.60 | 500 | 1.5e-26 | 119 |
| NP_001024582.1 | 26.48 | 423 | 3.0e-23 | 108 |
| NP_001024582.1 | 26.93 | 453 | 6.6e-23 | 107 |
| NP_001024582.1 | 24.95 | 505 | 4.6e-22 | 104 |
| NP_001024582.1 | 25.55 | 497 | 3.7e-21 | 101 |
| NP_001024582.1 | 25.76 | 528 | 1.2e-18 | 93 |
| NP_001024582.1 | 24.29 | 490 | 1.7e-17 | 89 |
| NP_001024582.1 | 27.00 | 337 | 6.0e-13 | 74 |
| NP_001360633.1 | 29.49 | 529 | 3.3e-53 | 207 |
| NP_001360633.1 | 29.55 | 511 | 2.2e-42 | 171 |
| NP_001360633.1 | 26.95 | 538 | 6.7e-42 | 169 |
| NP_001360633.1 | 27.69 | 502 | 1.5e-41 | 168 |
| NP_001360633.1 | 26.79 | 530 | 1.8e-38 | 158 |
| NP_001360633.1 | 28.63 | 496 | 2.2e-38 | 158 |
| NP_001360633.1 | 28.97 | 497 | 9.0e-38 | 156 |
| NP_001360633.1 | 29.28 | 461 | 5.2e-37 | 154 |
| NP_001360633.1 | 27.83 | 503 | 1.4e-36 | 152 |
| NP_001360633.1 | 28.38 | 458 | 6.1e-35 | 147 |
| NP_001360633.1 | 29.54 | 501 | 6.2e-34 | 143 |
| NP_001360633.1 | 29.25 | 441 | 2.9e-33 | 141 |
| NP_001360633.1 | 24.85 | 499 | 4.5e-33 | 140 |
| NP_001360633.1 | 27.29 | 535 | 7.4e-32 | 137 |
| NP_001360633.1 | 25.90 | 471 | 1.5e-30 | 132 |
| NP_001360633.1 | 26.23 | 469 | 5.3e-28 | 124 |
| NP_001360633.1 | 27.43 | 452 | 1.2e-26 | 119 |
| NP_001360633.1 | 27.60 | 500 | 1.5e-26 | 119 |
| NP_001360633.1 | 26.48 | 423 | 2.8e-23 | 108 |
| NP_001360633.1 | 26.93 | 453 | 6.8e-23 | 107 |
| NP_001360633.1 | 24.95 | 505 | 4.5e-22 | 104 |
| NP_001360633.1 | 25.55 | 497 | 3.3e-21 | 101 |
| NP_001360633.1 | 25.76 | 528 | 1.1e-18 | 93 |
| NP_001360633.1 | 24.29 | 490 | 1.6e-17 | 89 |
| NP_001360633.1 | 27.00 | 337 | 6.5e-13 | 74 |
| NP_001360634.1 | 29.49 | 529 | 3.3e-53 | 207 |
| NP_001360634.1 | 29.55 | 511 | 2.2e-42 | 171 |
| NP_001360634.1 | 26.95 | 538 | 7.1e-42 | 169 |
| NP_001360634.1 | 27.69 | 502 | 1.5e-41 | 168 |
| NP_001360634.1 | 26.79 | 530 | 1.8e-38 | 158 |
| NP_001360634.1 | 28.63 | 496 | 2.2e-38 | 158 |
| NP_001360634.1 | 28.97 | 497 | 9.5e-38 | 156 |
| NP_001360634.1 | 29.28 | 461 | 5.6e-37 | 153 |
| NP_001360634.1 | 27.83 | 503 | 1.3e-36 | 152 |
| NP_001360634.1 | 28.38 | 458 | 6.0e-35 | 147 |
| NP_001360634.1 | 29.54 | 501 | 6.5e-34 | 143 |
| NP_001360634.1 | 29.25 | 441 | 3.3e-33 | 141 |
| NP_001360634.1 | 24.85 | 499 | 4.4e-33 | 140 |
| NP_001360634.1 | 27.29 | 535 | 7.7e-32 | 136 |
| NP_001360634.1 | 25.90 | 471 | 1.4e-30 | 132 |
| NP_001360634.1 | 26.23 | 469 | 5.4e-28 | 124 |
| NP_001360634.1 | 27.43 | 452 | 1.2e-26 | 119 |
| NP_001360634.1 | 27.60 | 500 | 1.6e-26 | 119 |
| NP_001360634.1 | 26.48 | 423 | 3.1e-23 | 108 |
| NP_001360634.1 | 26.93 | 453 | 7.5e-23 | 107 |
| NP_001360634.1 | 24.95 | 505 | 5.0e-22 | 104 |
| NP_001360634.1 | 25.55 | 497 | 3.5e-21 | 101 |
| NP_001360634.1 | 25.76 | 528 | 1.3e-18 | 93 |
| NP_001360634.1 | 24.38 | 480 | 2.2e-17 | 89 |
| NP_001360634.1 | 26.87 | 335 | 1.8e-12 | 73 |
| NP_001360634.1 | 25.26 | 285 | 9.5e-11 | 67 |
| NP_509635.2 | 29.49 | 529 | 3.6e-53 | 207 |
| NP_509635.2 | 29.55 | 511 | 2.3e-42 | 171 |
| NP_509635.2 | 26.95 | 538 | 7.5e-42 | 169 |
| NP_509635.2 | 27.69 | 502 | 1.5e-41 | 168 |
| NP_509635.2 | 26.79 | 530 | 2.0e-38 | 158 |
| NP_509635.2 | 28.63 | 496 | 2.4e-38 | 158 |
| NP_509635.2 | 28.97 | 497 | 1.1e-37 | 155 |
| NP_509635.2 | 29.28 | 461 | 5.9e-37 | 153 |
| NP_509635.2 | 27.83 | 503 | 1.4e-36 | 152 |
| NP_509635.2 | 28.38 | 458 | 6.4e-35 | 147 |
| NP_509635.2 | 29.54 | 501 | 6.9e-34 | 143 |
| NP_509635.2 | 29.25 | 441 | 3.2e-33 | 141 |
| NP_509635.2 | 24.85 | 499 | 4.6e-33 | 140 |
| NP_509635.2 | 27.29 | 535 | 7.3e-32 | 137 |
| NP_509635.2 | 25.90 | 471 | 1.5e-30 | 132 |
| NP_509635.2 | 26.23 | 469 | 5.4e-28 | 124 |
| NP_509635.2 | 27.43 | 452 | 1.3e-26 | 119 |
| NP_509635.2 | 27.60 | 500 | 1.6e-26 | 119 |
| NP_509635.2 | 26.48 | 423 | 3.3e-23 | 108 |
| NP_509635.2 | 26.93 | 453 | 7.1e-23 | 107 |
| NP_509635.2 | 24.95 | 505 | 4.8e-22 | 104 |
| NP_509635.2 | 25.55 | 497 | 3.9e-21 | 101 |
| NP_509635.2 | 25.76 | 528 | 1.3e-18 | 93 |
| NP_509635.2 | 24.38 | 480 | 2.3e-17 | 89 |
| NP_509635.2 | 26.87 | 335 | 1.7e-12 | 73 |
| NP_509635.2 | 25.26 | 285 | 9.9e-11 | 67 |
| NP_001359724.1 | 29.49 | 529 | 4.5e-53 | 206 |
| NP_001359724.1 | 29.77 | 514 | 1.9e-42 | 171 |
| NP_001359724.1 | 26.95 | 538 | 7.8e-42 | 169 |
| NP_001359724.1 | 27.69 | 502 | 1.4e-41 | 169 |
| NP_001359724.1 | 28.63 | 496 | 1.9e-38 | 158 |
| NP_001359724.1 | 26.69 | 532 | 2.1e-38 | 158 |
| NP_001359724.1 | 28.97 | 497 | 1.1e-37 | 155 |
| NP_001359724.1 | 28.02 | 514 | 1.6e-36 | 152 |
| NP_001359724.1 | 29.94 | 491 | 6.8e-34 | 143 |
| NP_001359724.1 | 24.85 | 499 | 6.7e-33 | 140 |
| NP_001359724.1 | 27.29 | 535 | 8.2e-32 | 136 |
| NP_001359724.1 | 25.90 | 471 | 1.5e-30 | 132 |
| NP_001359724.1 | 27.60 | 500 | 1.3e-26 | 119 |
| NP_001359724.1 | 26.48 | 423 | 3.5e-23 | 108 |
| NP_001359724.1 | 26.93 | 453 | 7.3e-23 | 107 |
| NP_001359724.1 | 25.55 | 497 | 4.3e-21 | 101 |
| NP_001359724.1 | 25.76 | 528 | 1.4e-18 | 93 |
| NP_001343716.1 | 30.13 | 531 | 2.3e-44 | 178 |
| NP_001343716.1 | 27.55 | 490 | 3.0e-38 | 157 |
| NP_001343716.1 | 28.52 | 505 | 4.5e-38 | 157 |
| NP_001343716.1 | 26.97 | 545 | 1.7e-35 | 149 |
| NP_001343716.1 | 27.82 | 550 | 4.4e-35 | 147 |
| NP_001343716.1 | 28.75 | 560 | 1.1e-34 | 146 |
| NP_001343716.1 | 27.75 | 537 | 1.1e-34 | 146 |
| NP_001343716.1 | 27.66 | 517 | 2.1e-33 | 142 |
| NP_001343716.1 | 25.93 | 729 | 1.7e-31 | 135 |
| NP_001343716.1 | 27.75 | 501 | 2.1e-31 | 135 |
| NP_001343716.1 | 24.39 | 578 | 8.6e-29 | 126 |
| NP_001343716.1 | 27.63 | 503 | 1.4e-28 | 126 |
| NP_001343716.1 | 25.68 | 553 | 2.1e-27 | 122 |
| NP_001343716.1 | 26.76 | 441 | 1.1e-26 | 119 |
| NP_001343716.1 | 26.69 | 502 | 6.6e-25 | 114 |
| NP_001343716.1 | 26.10 | 544 | 6.6e-24 | 110 |
| NP_001343716.1 | 23.30 | 485 | 2.3e-22 | 105 |
| NP_001343716.1 | 23.43 | 572 | 8.3e-22 | 103 |
| NP_001343716.1 | 30.90 | 301 | 7.3e-21 | 100 |
| NP_001343716.1 | 29.51 | 349 | 1.2e-19 | 96 |
| NP_001343716.1 | 24.76 | 517 | 5.4e-18 | 91 |
| NP_001343716.1 | 27.94 | 451 | 6.1e-18 | 91 |
| NP_001343716.1 | 24.19 | 554 | 6.9e-18 | 90 |
| NP_001343716.1 | 24.44 | 491 | 1.8e-17 | 89 |
| NP_001343716.1 | 25.16 | 465 | 2.1e-15 | 82 |
| NP_001343716.1 | 25.62 | 324 | 1.2e-11 | 70 |
| NP_001343716.1 | 26.98 | 189 | 3.7e-10 | 65 |
| NP_001343716.1 | 27.81 | 187 | 1.1e-07 | 57 |
| NP_001343716.1 | 27.13 | 188 | 5.3e-06 | 51 |
| NP_001343716.1 | 24.41 | 340 | 6.8e-06 | 51 |
| NP_001343716.1 | 23.08 | 182 | 1.0e-05 | 50 |
| NP_001343719.1 | 30.13 | 531 | 2.5e-44 | 177 |
| NP_001343719.1 | 28.75 | 560 | 1.2e-34 | 145 |
| NP_001343719.1 | 27.75 | 537 | 1.2e-34 | 145 |
| NP_001343719.1 | 25.93 | 729 | 1.8e-31 | 135 |
| NP_001343719.1 | 25.89 | 533 | 2.8e-31 | 135 |
| NP_001343719.1 | 27.75 | 501 | 3.2e-31 | 134 |
| NP_001343719.1 | 25.54 | 552 | 6.2e-29 | 127 |
| NP_001343719.1 | 27.63 | 503 | 2.0e-28 | 125 |
| NP_001343719.1 | 25.19 | 532 | 1.6e-27 | 122 |
| NP_001343719.1 | 26.69 | 502 | 1.1e-24 | 113 |
| NP_001343719.1 | 25.68 | 518 | 3.7e-24 | 111 |
| NP_001343719.1 | 26.10 | 544 | 9.3e-24 | 110 |
| NP_001343719.1 | 24.91 | 546 | 1.3e-23 | 109 |
| NP_001343719.1 | 30.90 | 301 | 8.9e-21 | 100 |
| NP_001343719.1 | 24.72 | 449 | 1.3e-19 | 96 |
| NP_001343719.1 | 29.51 | 349 | 1.6e-19 | 96 |
| NP_001343719.1 | 24.76 | 517 | 7.4e-18 | 90 |
| NP_001343719.1 | 24.19 | 554 | 8.0e-18 | 90 |
| NP_001343719.1 | 27.94 | 451 | 8.5e-18 | 90 |
| NP_001343719.1 | 24.44 | 491 | 2.7e-17 | 89 |
| NP_001343719.1 | 25.16 | 465 | 2.7e-15 | 82 |
| NP_001343719.1 | 25.62 | 324 | 1.6e-11 | 69 |
| NP_001343719.1 | 26.98 | 189 | 4.1e-10 | 65 |
| NP_001343719.1 | 27.81 | 187 | 1.2e-07 | 57 |
| NP_001343719.1 | 27.13 | 188 | 5.2e-06 | 51 |
| NP_001343719.1 | 24.41 | 340 | 7.9e-06 | 51 |
| NP_001343719.1 | 23.08 | 182 | 8.7e-06 | 50 |
| NP_001343617.1 | 30.13 | 531 | 2.6e-44 | 177 |
| NP_001343617.1 | 28.52 | 505 | 5.7e-38 | 157 |
| NP_001343617.1 | 26.97 | 545 | 2.4e-35 | 148 |
| NP_001343617.1 | 27.82 | 550 | 5.7e-35 | 147 |
| NP_001343617.1 | 28.75 | 560 | 1.1e-34 | 146 |
| NP_001343617.1 | 27.75 | 537 | 1.2e-34 | 146 |
| NP_001343617.1 | 27.61 | 518 | 2.6e-33 | 141 |
| NP_001343617.1 | 25.93 | 729 | 2.1e-31 | 135 |
| NP_001343617.1 | 27.75 | 501 | 3.1e-31 | 134 |
| NP_001343617.1 | 25.05 | 531 | 2.3e-29 | 128 |
| NP_001343617.1 | 24.39 | 578 | 1.1e-28 | 126 |
| NP_001343617.1 | 27.63 | 503 | 2.0e-28 | 125 |
| NP_001343617.1 | 25.40 | 563 | 2.8e-27 | 122 |
| NP_001343617.1 | 25.24 | 519 | 3.0e-27 | 121 |
| NP_001343617.1 | 24.77 | 549 | 9.2e-25 | 113 |
| NP_001343617.1 | 26.69 | 502 | 1.0e-24 | 113 |
| NP_001343617.1 | 25.68 | 518 | 3.5e-24 | 111 |
| NP_001343617.1 | 26.10 | 544 | 9.1e-24 | 110 |
| NP_001343617.1 | 30.90 | 301 | 9.3e-21 | 100 |
| NP_001343617.1 | 24.72 | 449 | 1.3e-19 | 96 |
| NP_001343617.1 | 29.51 | 349 | 1.7e-19 | 96 |
| NP_001343617.1 | 27.94 | 451 | 7.2e-18 | 90 |
| NP_001343617.1 | 24.76 | 517 | 7.6e-18 | 90 |
| NP_001343617.1 | 24.19 | 554 | 8.5e-18 | 90 |
| NP_001343617.1 | 24.44 | 491 | 2.8e-17 | 89 |
| NP_001343617.1 | 25.16 | 465 | 2.7e-15 | 82 |
| NP_001343617.1 | 25.62 | 324 | 1.3e-11 | 70 |
| NP_001343617.1 | 26.98 | 189 | 4.6e-10 | 65 |
| NP_001343617.1 | 27.81 | 187 | 1.3e-07 | 57 |
| NP_001343617.1 | 27.13 | 188 | 6.1e-06 | 51 |
| NP_001343617.1 | 24.41 | 340 | 8.4e-06 | 51 |
| NP_001343617.1 | 23.08 | 182 | 9.5e-06 | 50 |

### RPN1 (P04843)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_501065.2 | 41.85 | 540 | 3.0e-127 | 386 |

### RPN2 (P04844)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_491624.1 | 38.34 | 253 | 1.4e-51 | 179 |

### RPS27A (P62979)

**Description:** Ubiquitin-ribosomal protein eS31 fusion protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_741102.1 | 61.27 | 142 | 5.1e-50 | 157 |
| NP_499695.1 | 98.68 | 76 | 2.2e-48 | 152 |
| NP_741157.1 | 96.15 | 78 | 3.0e-44 | 154 |
| NP_741157.1 | 96.15 | 78 | 3.0e-44 | 154 |
| NP_741157.1 | 96.15 | 78 | 3.0e-44 | 154 |
| NP_741157.1 | 96.15 | 78 | 3.0e-44 | 154 |
| NP_741157.1 | 96.15 | 78 | 3.0e-44 | 154 |
| NP_741157.1 | 96.15 | 78 | 3.0e-44 | 154 |
| NP_741157.1 | 96.15 | 78 | 3.0e-44 | 154 |
| NP_741157.1 | 96.15 | 78 | 3.0e-44 | 154 |
| NP_741157.1 | 96.15 | 78 | 3.0e-44 | 154 |
| NP_741157.1 | 98.68 | 76 | 3.6e-44 | 154 |
| NP_741157.1 | 94.87 | 78 | 1.6e-43 | 152 |
| NP_001379170.1 | 61.84 | 76 | 3.6e-28 | 99 |
| NP_494128.3 | 45.07 | 71 | 3.5e-13 | 64 |
| NP_494542.1 | 40.28 | 72 | 4.0e-10 | 56 |
| NP_001022637.1 | 43.48 | 46 | 4.2e-06 | 42 |
| NP_494511.2 | 38.89 | 72 | 2.5e-05 | 42 |
| NP_494127.2 | 38.89 | 72 | 2.5e-05 | 42 |
| NP_493746.1 | 26.39 | 72 | 4.2e-05 | 42 |

### SEMA5B (Q9P283)

**Description:** Semaphorin-5B

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_493582.3 | 32.37 | 482 | 5.7e-66 | 237 |
| NP_001391076.1 | 32.37 | 482 | 7.3e-66 | 237 |
| NP_490875.1 | 28.07 | 513 | 1.5e-48 | 184 |
| NP_001293443.1 | 29.90 | 398 | 1.1e-42 | 164 |
| NP_001293444.1 | 30.47 | 361 | 9.0e-40 | 154 |
| NP_491197.3 | 25.94 | 478 | 6.4e-38 | 152 |
| NP_001361951.1 | 31.25 | 432 | 4.0e-33 | 139 |
| NP_001361951.1 | 29.23 | 366 | 5.0e-25 | 113 |
| NP_001361951.1 | 30.48 | 315 | 1.7e-23 | 108 |
| NP_001361951.1 | 30.07 | 296 | 9.5e-20 | 95 |
| NP_001361951.1 | 26.65 | 454 | 7.0e-19 | 93 |
| NP_001361951.1 | 25.39 | 386 | 9.8e-16 | 82 |
| NP_001361951.1 | 31.39 | 172 | 1.5e-11 | 68 |
| NP_001361951.1 | 30.60 | 183 | 9.3e-08 | 56 |
| NP_510116.2 | 31.25 | 432 | 4.0e-33 | 139 |
| NP_510116.2 | 29.12 | 364 | 4.7e-25 | 113 |
| NP_510116.2 | 30.48 | 315 | 1.8e-23 | 108 |
| NP_510116.2 | 30.07 | 296 | 1.0e-19 | 95 |
| NP_510116.2 | 26.65 | 454 | 6.7e-19 | 93 |
| NP_510116.2 | 25.39 | 386 | 9.7e-16 | 82 |
| NP_510116.2 | 31.39 | 172 | 1.6e-11 | 68 |
| NP_510116.2 | 30.60 | 183 | 8.8e-08 | 56 |
| NP_001370129.1 | 39.58 | 192 | 4.9e-33 | 129 |
| NP_001361970.1 | 33.11 | 296 | 1.4e-32 | 137 |
| NP_001361970.1 | 30.41 | 319 | 1.2e-27 | 121 |
| NP_001361970.1 | 35.68 | 213 | 4.7e-19 | 93 |
| NP_001361970.1 | 32.71 | 107 | 3.6e-10 | 64 |

### SIRT3 (Q9NTG7)

**Description:** NAD-dependent protein deacetylase sirtuin-3, mitochondrial

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001255485.1 | 45.13 | 226 | 3.9e-60 | 204 |
| NP_001255484.1 | 45.13 | 226 | 3.9e-60 | 205 |
| NP_001024672.1 | 29.30 | 256 | 7.7e-17 | 80 |
| NP_001367072.1 | 29.07 | 258 | 5.1e-15 | 74 |
| NP_001367227.1 | 29.07 | 258 | 5.2e-15 | 75 |
| NP_510220.1 | 26.48 | 253 | 2.6e-14 | 72 |
| NP_491733.2 | 26.82 | 261 | 1.8e-13 | 70 |

### SLIT2 (O94813)

**Description:** Slit homolog 2 protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_510437.2 | 38.95 | 1376 | 0.0e+00 | 930 |
| NP_510437.2 | 29.57 | 115 | 1.8e-09 | 62 |
| NP_510437.2 | 30.77 | 117 | 6.8e-07 | 54 |
| NP_510437.2 | 30.70 | 114 | 9.8e-06 | 50 |
| NP_510437.2 | 30.39 | 102 | 2.5e-04 | 45 |
| NP_510437.2 | 23.83 | 256 | 1.0e-03 | 43 |
| NP_500454.2 | 36.36 | 264 | 1.9e-37 | 155 |
| NP_500454.2 | 38.89 | 234 | 5.0e-35 | 147 |
| NP_500454.2 | 33.61 | 238 | 9.8e-34 | 142 |
| NP_500454.2 | 33.70 | 276 | 2.2e-32 | 138 |
| NP_500454.2 | 33.33 | 237 | 5.4e-30 | 130 |
| NP_500454.2 | 33.50 | 197 | 1.2e-26 | 119 |
| NP_500454.2 | 31.39 | 172 | 3.3e-15 | 82 |
| NP_500454.2 | 28.81 | 118 | 3.8e-11 | 68 |
| NP_500454.2 | 36.36 | 99 | 1.9e-07 | 56 |
| NP_499007.1 | 37.14 | 245 | 1.4e-34 | 145 |
| NP_499007.1 | 29.64 | 361 | 2.2e-23 | 108 |
| NP_499007.1 | 27.01 | 311 | 3.0e-17 | 88 |
| NP_499007.1 | 32.62 | 187 | 3.8e-13 | 74 |
| NP_499007.1 | 37.39 | 115 | 3.1e-12 | 72 |
| NP_499007.1 | 33.88 | 121 | 1.5e-06 | 53 |
| NP_499007.1 | 33.04 | 112 | 1.7e-05 | 49 |
| NP_499896.1 | 26.77 | 493 | 6.5e-31 | 132 |
| NP_499896.1 | 25.24 | 515 | 3.4e-21 | 100 |
| NP_001367941.1 | 26.77 | 493 | 7.1e-31 | 131 |
| NP_001367941.1 | 25.24 | 515 | 4.9e-21 | 100 |
| NP_499014.1 | 35.08 | 248 | 1.5e-28 | 125 |
| NP_499014.1 | 27.98 | 336 | 9.0e-23 | 106 |
| NP_499014.1 | 28.12 | 256 | 3.0e-15 | 81 |
| NP_499014.1 | 32.49 | 197 | 1.1e-10 | 66 |
| NP_499014.1 | 29.92 | 127 | 2.4e-07 | 55 |
| NP_499014.1 | 30.43 | 115 | 4.2e-07 | 55 |
| NP_499014.1 | 31.15 | 122 | 5.2e-04 | 44 |
| NP_001020983.1 | 21.08 | 873 | 1.4e-26 | 118 |
| NP_001020983.1 | 25.55 | 454 | 1.9e-19 | 95 |
| NP_001020983.1 | 22.92 | 589 | 5.1e-14 | 77 |
| NP_001020983.1 | 25.00 | 160 | 8.1e-08 | 57 |
| NP_492348.1 | 23.53 | 510 | 2.3e-23 | 107 |
| NP_492348.1 | 26.43 | 367 | 1.5e-18 | 92 |
| NP_492348.1 | 25.62 | 406 | 5.0e-17 | 87 |
| NP_492348.1 | 26.53 | 245 | 2.6e-13 | 75 |
| NP_492348.1 | 22.14 | 384 | 1.4e-09 | 62 |
| NP_492348.1 | 21.99 | 382 | 5.9e-07 | 54 |
| NP_497925.1 | 25.81 | 403 | 2.6e-23 | 106 |
| NP_497925.1 | 25.34 | 367 | 3.7e-20 | 96 |
| NP_497925.1 | 22.71 | 458 | 1.7e-14 | 78 |
| NP_497925.1 | 29.67 | 182 | 2.7e-11 | 68 |
| NP_497925.1 | 23.43 | 303 | 1.9e-04 | 45 |
| NP_495322.3 | 21.59 | 815 | 3.8e-23 | 107 |
| NP_495322.3 | 23.75 | 598 | 1.3e-19 | 95 |
| NP_495322.3 | 22.46 | 886 | 2.1e-18 | 92 |
| NP_495322.3 | 21.54 | 938 | 6.9e-17 | 87 |

### SMAD2 (Q15796)

**Description:** Mothers against decapentaplegic homolog 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001370321.1 | 45.37 | 454 | 1.7e-124 | 368 |
| NP_498493.1 | 38.25 | 468 | 2.1e-81 | 257 |
| NP_001360602.1 | 53.33 | 225 | 1.1e-75 | 238 |
| NP_001040864.1 | 32.34 | 470 | 1.4e-51 | 183 |
| NP_492321.1 | 41.18 | 204 | 1.9e-46 | 169 |
| NP_492321.1 | 36.84 | 76 | 1.3e-06 | 50 |
| NP_001255476.1 | 33.17 | 202 | 6.4e-27 | 109 |
| NP_001255475.1 | 33.17 | 202 | 2.6e-26 | 108 |
| NP_001040865.1 | 32.61 | 184 | 2.3e-17 | 80 |
| NP_508161.3 | 40.00 | 115 | 2.5e-16 | 81 |
| NP_001300343.1 | 40.00 | 115 | 2.6e-16 | 81 |

### SMAD3 (P84022)

**Description:** Mothers against decapentaplegic homolog 3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001370321.1 | 48.71 | 425 | 3.1e-130 | 381 |
| NP_498493.1 | 42.42 | 422 | 6.8e-97 | 295 |
| NP_001360602.1 | 57.71 | 201 | 1.1e-75 | 236 |
| NP_001040864.1 | 35.07 | 442 | 5.3e-58 | 199 |
| NP_492321.1 | 41.18 | 204 | 4.1e-46 | 166 |
| NP_492321.1 | 26.19 | 126 | 2.6e-06 | 49 |
| NP_001255476.1 | 33.67 | 199 | 5.1e-26 | 106 |
| NP_001255475.1 | 33.67 | 199 | 2.5e-25 | 105 |
| NP_001368578.1 | 35.62 | 219 | 9.5e-23 | 100 |
| NP_001368578.1 | 26.29 | 213 | 1.8e-09 | 59 |
| NP_001024605.1 | 35.62 | 219 | 1.2e-22 | 100 |
| NP_001024605.1 | 26.29 | 213 | 2.0e-09 | 59 |
| NP_001300343.1 | 35.62 | 219 | 1.2e-22 | 100 |
| NP_001300343.1 | 26.29 | 213 | 2.1e-09 | 59 |

### SOD2 (P04179)

**Description:** Superoxide dismutase [Mn], mitochondrial

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_492290.1 | 63.51 | 222 | 6.0e-106 | 304 |
| NP_510764.1 | 60.36 | 222 | 4.3e-101 | 291 |

### SREBF1 (P36956)

**Description:** Sterol regulatory element-binding protein 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_499472.1 | 24.84 | 777 | 1.6e-41 | 166 |
| NP_001368701.1 | 35.87 | 92 | 2.6e-09 | 60 |
| NP_001350991.1 | 35.87 | 92 | 2.8e-09 | 60 |
| NP_001023413.1 | 35.87 | 92 | 3.0e-09 | 60 |
| NP_001379926.1 | 35.87 | 92 | 3.0e-09 | 60 |
| NP_500461.1 | 35.87 | 92 | 3.4e-09 | 60 |
| NP_001023411.1 | 35.87 | 92 | 3.8e-09 | 60 |
| NP_500462.1 | 35.87 | 92 | 5.3e-09 | 60 |
| NP_001023414.1 | 35.87 | 92 | 5.7e-09 | 60 |
| NP_001360026.1 | 35.87 | 92 | 5.8e-09 | 59 |

### SSPOP (A2VEC9)

**Description:** SCO-spondin

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001361951.1 | 26.39 | 1061 | 5.7e-46 | 184 |
| NP_001361951.1 | 25.20 | 897 | 5.6e-34 | 145 |
| NP_001361951.1 | 25.64 | 936 | 6.9e-29 | 128 |
| NP_001361951.1 | 28.30 | 682 | 4.7e-26 | 119 |
| NP_001361951.1 | 26.73 | 793 | 1.7e-24 | 114 |
| NP_001361951.1 | 23.29 | 1026 | 8.3e-24 | 112 |
| NP_001361951.1 | 24.75 | 703 | 4.3e-22 | 106 |
| NP_001361951.1 | 26.71 | 700 | 5.4e-22 | 105 |
| NP_001361951.1 | 37.43 | 179 | 1.4e-18 | 94 |
| NP_001361951.1 | 25.62 | 648 | 5.7e-17 | 89 |
| NP_001361951.1 | 30.00 | 340 | 3.3e-15 | 83 |
| NP_001361951.1 | 24.53 | 742 | 2.7e-13 | 77 |
| NP_001361951.1 | 23.56 | 641 | 4.2e-12 | 73 |
| NP_001361951.1 | 24.29 | 597 | 4.5e-11 | 70 |
| NP_001361951.1 | 41.82 | 110 | 9.7e-10 | 65 |
| NP_001361951.1 | 36.04 | 111 | 3.4e-09 | 63 |
| NP_001361951.1 | 35.25 | 122 | 1.5e-07 | 58 |
| NP_001361951.1 | 29.51 | 288 | 1.3e-06 | 55 |
| NP_001361951.1 | 27.91 | 301 | 3.0e-06 | 53 |
| NP_001361951.1 | 25.15 | 330 | 1.0e-04 | 48 |
| NP_510116.2 | 26.39 | 1061 | 6.6e-46 | 184 |
| NP_510116.2 | 25.20 | 897 | 6.4e-34 | 145 |
| NP_510116.2 | 25.64 | 936 | 7.1e-29 | 128 |
| NP_510116.2 | 28.30 | 682 | 5.1e-26 | 119 |
| NP_510116.2 | 26.73 | 793 | 1.8e-24 | 114 |
| NP_510116.2 | 23.29 | 1026 | 8.6e-24 | 112 |
| NP_510116.2 | 24.75 | 703 | 4.8e-22 | 106 |
| NP_510116.2 | 25.90 | 695 | 5.4e-22 | 105 |
| NP_510116.2 | 37.43 | 179 | 1.6e-18 | 94 |
| NP_510116.2 | 25.62 | 648 | 5.8e-17 | 89 |
| NP_510116.2 | 30.00 | 340 | 3.3e-15 | 83 |
| NP_510116.2 | 24.53 | 742 | 2.8e-13 | 77 |
| NP_510116.2 | 23.56 | 641 | 4.8e-12 | 73 |
| NP_510116.2 | 24.29 | 597 | 4.8e-11 | 69 |
| NP_510116.2 | 41.82 | 110 | 1.0e-09 | 65 |
| NP_510116.2 | 36.04 | 111 | 3.4e-09 | 63 |
| NP_510116.2 | 35.25 | 122 | 1.5e-07 | 58 |
| NP_510116.2 | 29.51 | 288 | 1.3e-06 | 55 |
| NP_510116.2 | 27.43 | 299 | 3.1e-06 | 53 |
| NP_510116.2 | 25.15 | 330 | 1.1e-04 | 48 |
| NP_001379135.1 | 36.33 | 311 | 3.6e-37 | 156 |
| NP_001379135.1 | 30.81 | 357 | 6.2e-32 | 139 |
| NP_001379135.1 | 33.13 | 332 | 4.3e-29 | 129 |
| NP_001379135.1 | 29.82 | 379 | 3.9e-27 | 123 |
| NP_001379135.1 | 32.84 | 271 | 2.7e-26 | 120 |
| NP_001379135.1 | 32.28 | 347 | 3.3e-25 | 117 |
| NP_001379135.1 | 30.61 | 281 | 1.9e-22 | 107 |
| NP_001379135.1 | 33.33 | 237 | 8.8e-22 | 105 |
| NP_001379135.1 | 28.96 | 404 | 7.3e-21 | 102 |
| NP_001379135.1 | 31.15 | 244 | 2.1e-18 | 94 |
| NP_001379135.1 | 35.51 | 138 | 2.2e-15 | 84 |
| NP_001379135.1 | 30.66 | 212 | 1.2e-14 | 82 |
| NP_001379135.1 | 35.81 | 148 | 3.2e-13 | 77 |
| NP_001379135.1 | 29.90 | 194 | 1.2e-08 | 62 |
| NP_001379135.1 | 28.86 | 149 | 7.1e-07 | 56 |
| NP_001379135.1 | 33.33 | 105 | 1.6e-04 | 48 |
| NP_001379135.1 | 35.54 | 121 | 3.9e-04 | 47 |
| NP_491270.2 | 31.25 | 400 | 2.9e-31 | 137 |
| NP_491270.2 | 30.60 | 317 | 1.2e-27 | 125 |
| NP_491270.2 | 30.25 | 367 | 2.3e-23 | 110 |
| NP_491270.2 | 32.63 | 334 | 2.0e-22 | 107 |
| NP_491270.2 | 30.04 | 283 | 1.2e-20 | 102 |
| NP_491270.2 | 30.53 | 262 | 3.8e-13 | 77 |
| NP_491270.2 | 34.38 | 192 | 6.2e-11 | 69 |
| NP_491270.2 | 35.71 | 140 | 8.3e-10 | 65 |
| NP_491270.2 | 38.94 | 113 | 1.8e-06 | 55 |
| NP_491270.2 | 40.24 | 82 | 6.0e-06 | 53 |
| NP_491270.2 | 28.89 | 135 | 2.0e-05 | 51 |
| NP_495387.2 | 30.18 | 328 | 1.6e-26 | 120 |
| NP_495387.2 | 30.22 | 278 | 3.2e-17 | 89 |
| NP_495387.2 | 31.79 | 151 | 1.7e-06 | 54 |
| NP_872023.2 | 30.00 | 330 | 2.4e-26 | 119 |
| NP_872023.2 | 30.22 | 278 | 2.6e-17 | 90 |
| NP_872023.2 | 31.37 | 153 | 2.4e-05 | 50 |
| NP_872023.2 | 32.43 | 111 | 1.0e-03 | 45 |
| NP_001024532.2 | 26.13 | 551 | 2.8e-24 | 113 |
| NP_001024532.2 | 32.87 | 286 | 2.2e-21 | 103 |
| NP_001024532.2 | 28.30 | 364 | 1.1e-19 | 98 |
| NP_001024532.2 | 25.54 | 556 | 3.1e-18 | 93 |
| NP_001024532.2 | 30.04 | 253 | 3.6e-17 | 89 |
| NP_001024532.2 | 27.23 | 426 | 1.8e-16 | 87 |
| NP_001024532.2 | 27.83 | 345 | 2.4e-16 | 87 |
| NP_001024532.2 | 31.84 | 179 | 1.1e-14 | 81 |
| NP_001024532.2 | 24.25 | 400 | 4.0e-14 | 79 |
| NP_001024532.2 | 26.09 | 345 | 5.7e-13 | 76 |
| NP_001024532.2 | 27.59 | 348 | 2.0e-12 | 74 |
| NP_001024532.2 | 26.81 | 276 | 9.2e-12 | 72 |
| NP_001024532.2 | 29.12 | 182 | 1.5e-10 | 68 |
| NP_001024532.2 | 41.76 | 91 | 2.5e-10 | 67 |
| NP_001024532.2 | 29.91 | 224 | 1.9e-09 | 64 |
| NP_001024532.2 | 39.76 | 83 | 1.1e-07 | 58 |
| NP_001024532.2 | 23.72 | 527 | 7.1e-06 | 52 |
| NP_001024532.2 | 27.24 | 268 | 6.4e-05 | 49 |
| NP_001024532.2 | 25.22 | 230 | 6.1e-04 | 46 |
| NP_498237.2 | 33.45 | 284 | 5.6e-22 | 103 |
| NP_498237.2 | 29.68 | 283 | 4.3e-17 | 88 |
| NP_498237.2 | 32.31 | 229 | 1.4e-15 | 84 |
| NP_498237.2 | 29.39 | 313 | 1.6e-14 | 80 |
| NP_509829.2 | 29.97 | 347 | 1.5e-21 | 103 |
| NP_509829.2 | 30.48 | 292 | 1.5e-17 | 90 |
| NP_509829.2 | 28.61 | 374 | 2.4e-17 | 90 |
| NP_509829.2 | 30.64 | 248 | 2.8e-15 | 83 |
| NP_509829.2 | 26.74 | 460 | 9.1e-14 | 78 |
| NP_509829.2 | 28.19 | 376 | 2.6e-13 | 77 |
| NP_509829.2 | 28.76 | 452 | 3.7e-13 | 76 |
| NP_509829.2 | 26.76 | 411 | 1.4e-12 | 74 |
| NP_509829.2 | 27.67 | 412 | 2.0e-11 | 70 |
| NP_509829.2 | 24.80 | 375 | 2.9e-09 | 63 |
| NP_509829.2 | 27.75 | 227 | 1.8e-08 | 61 |
| NP_509829.2 | 37.10 | 124 | 5.8e-08 | 59 |
| NP_509829.2 | 34.67 | 150 | 1.4e-06 | 55 |
| NP_509829.2 | 37.50 | 88 | 3.5e-05 | 50 |
| NP_509829.2 | 26.68 | 371 | 1.7e-04 | 48 |
| NP_001361970.1 | 32.87 | 286 | 2.1e-21 | 103 |
| NP_001361970.1 | 25.09 | 530 | 2.0e-19 | 97 |
| NP_001361970.1 | 25.74 | 540 | 6.8e-18 | 92 |
| NP_001361970.1 | 24.25 | 400 | 4.0e-14 | 79 |
| NP_001361970.1 | 28.87 | 336 | 8.8e-14 | 78 |
| NP_001361970.1 | 26.09 | 345 | 6.5e-13 | 75 |
| NP_001361970.1 | 29.12 | 182 | 1.5e-10 | 68 |
| NP_001361970.1 | 41.76 | 91 | 2.3e-10 | 67 |
| NP_001361970.1 | 27.84 | 255 | 2.9e-09 | 63 |
| NP_001361970.1 | 38.37 | 86 | 1.0e-08 | 62 |
| NP_001361970.1 | 33.65 | 107 | 4.8e-08 | 59 |
| NP_001361970.1 | 27.99 | 293 | 7.5e-08 | 59 |
| NP_001361970.1 | 39.76 | 83 | 9.9e-08 | 58 |
| NP_001361970.1 | 45.45 | 55 | 8.1e-07 | 55 |
| NP_001361970.1 | 40.00 | 55 | 1.1e-04 | 48 |
| NP_001361970.1 | 25.22 | 230 | 6.8e-04 | 46 |

### ST3GAL3 (Q11203)

**Description:** CMP-N-acetylneuraminate-beta-1,4-galactoside alpha-2,3-sialyltransferase

No significant BLAST hits found for this protein.

### ST3GAL4 (Q11206)

**Description:** CMP-N-acetylneuraminate-beta-galactosamide-alpha-2,3-sialyltransferase 4

No significant BLAST hits found for this protein.

### ST3GAL4 (Q11206)

**Description:** CMP-N-acetylneuraminate-beta-galactosamide-alpha-2,3-sialyltransferase 4

No significant BLAST hits found for this protein.

### ST6GAL1 (P15907)

**Description:** Beta-galactoside alpha-2,6-sialyltransferase 1

No significant BLAST hits found for this protein.

### ST6GAL1 (P15907)

**Description:** Beta-galactoside alpha-2,6-sialyltransferase 1

No significant BLAST hits found for this protein.

### ST6GALNAC3 (Q8NDV1)

**Description:** Alpha-N-acetylgalactosaminide alpha-2,6-sialyltransferase 3

No significant BLAST hits found for this protein.

### ST8SIA2 (Q92186)

**Description:** Alpha-2,8-sialyltransferase 8B

No significant BLAST hits found for this protein.

### ST8SIA3 (O43173)

**Description:** Alpha-N-acetylneuraminate alpha-2,8-sialyltransferase ST8SIA3

No significant BLAST hits found for this protein.

### ST8SIA6 (P61647)

**Description:** Alpha-2,8-sialyltransferase 8F

No significant BLAST hits found for this protein.

### STK11 (Q15831)

**Description:** Serine/threonine-protein kinase STK11

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001367284.1 | 38.21 | 403 | 3.0e-81 | 258 |
| NP_001367285.1 | 42.25 | 329 | 8.9e-80 | 258 |
| NP_001366977.1 | 42.46 | 325 | 1.0e-79 | 258 |
| NP_001024868.1 | 35.19 | 270 | 4.6e-48 | 172 |
| NP_510710.2 | 33.66 | 306 | 1.2e-47 | 172 |
| NP_510711.2 | 35.19 | 270 | 1.5e-47 | 172 |
| NP_001023421.2 | 36.36 | 275 | 1.5e-45 | 167 |
| NP_001023420.2 | 36.36 | 275 | 1.5e-45 | 167 |
| NP_001294354.1 | 36.36 | 275 | 1.5e-45 | 167 |
| NP_001294353.1 | 36.36 | 275 | 1.6e-45 | 167 |

### STT3A (P46977)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit STT3A

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_498362.1 | 56.60 | 705 | 0.0e+00 | 840 |

### THBS1 (P07996)

**Description:** Thrombospondin-1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001361951.1 | 36.10 | 241 | 1.2e-26 | 118 |
| NP_001361951.1 | 37.28 | 169 | 4.9e-18 | 90 |
| NP_001361951.1 | 35.46 | 141 | 7.7e-18 | 89 |
| NP_001361951.1 | 34.53 | 139 | 2.4e-13 | 75 |
| NP_001361951.1 | 32.77 | 177 | 1.1e-11 | 69 |
| NP_001361951.1 | 30.64 | 173 | 6.5e-08 | 57 |
| NP_510116.2 | 36.10 | 241 | 1.2e-26 | 118 |
| NP_510116.2 | 37.28 | 169 | 4.9e-18 | 90 |
| NP_510116.2 | 35.46 | 141 | 8.0e-18 | 89 |
| NP_510116.2 | 34.53 | 139 | 2.4e-13 | 75 |
| NP_510116.2 | 32.77 | 177 | 1.1e-11 | 69 |
| NP_510116.2 | 30.64 | 173 | 6.6e-08 | 57 |
| NP_001361970.1 | 33.79 | 219 | 1.4e-26 | 117 |
| NP_001361970.1 | 38.32 | 167 | 9.4e-26 | 115 |
| NP_001361970.1 | 34.84 | 155 | 6.6e-14 | 76 |
| NP_001024532.2 | 33.79 | 219 | 2.6e-26 | 117 |
| NP_001024532.2 | 36.59 | 205 | 2.5e-24 | 110 |
| NP_001024532.2 | 30.23 | 86 | 9.8e-05 | 46 |
| NP_509829.2 | 38.79 | 165 | 3.8e-24 | 109 |
| NP_509829.2 | 35.93 | 167 | 3.7e-21 | 100 |
| NP_509829.2 | 36.09 | 169 | 1.0e-19 | 95 |
| NP_509829.2 | 35.40 | 113 | 3.7e-11 | 67 |
| NP_508745.1 | 38.64 | 176 | 2.2e-21 | 95 |
| NP_001359920.1 | 38.30 | 141 | 2.4e-19 | 94 |
| NP_001359920.1 | 38.41 | 164 | 3.4e-19 | 94 |
| NP_001359920.1 | 31.77 | 192 | 1.3e-15 | 82 |
| NP_001379783.1 | 38.30 | 141 | 3.2e-19 | 94 |
| NP_001379783.1 | 38.41 | 164 | 4.8e-19 | 93 |
| NP_001379783.1 | 31.77 | 192 | 1.7e-15 | 82 |
| NP_001293749.1 | 35.09 | 171 | 6.7e-19 | 92 |
| NP_001293749.1 | 35.34 | 133 | 2.3e-10 | 65 |
| NP_500822.2 | 35.09 | 171 | 8.6e-19 | 91 |
| NP_500822.2 | 34.59 | 133 | 3.4e-10 | 63 |

### THSD1 (Q9NS62)

**Description:** Thrombospondin type-1 domain-containing protein 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001367660.1 | 36.99 | 73 | 2.9e-10 | 58 |
| NP_001367660.1 | 41.79 | 67 | 5.8e-08 | 52 |
| NP_508701.2 | 32.74 | 113 | 6.7e-08 | 56 |
| NP_001361970.1 | 35.71 | 84 | 1.4e-07 | 55 |
| NP_001361970.1 | 26.24 | 141 | 1.8e-05 | 48 |
| NP_001361970.1 | 40.35 | 57 | 3.0e-05 | 47 |
| NP_001024532.2 | 35.71 | 84 | 1.9e-07 | 55 |
| NP_001024532.2 | 41.79 | 67 | 4.2e-06 | 50 |
| NP_001024532.2 | 26.24 | 141 | 2.4e-05 | 48 |
| NP_001024532.2 | 40.35 | 57 | 3.8e-05 | 47 |
| NP_872065.1 | 43.64 | 55 | 2.6e-05 | 44 |
| NP_508745.1 | 41.51 | 53 | 1.4e-04 | 44 |
| NP_508681.3 | 35.09 | 57 | 1.6e-04 | 45 |
| NP_001024413.1 | 37.50 | 56 | 2.3e-04 | 44 |
| NP_001022731.1 | 38.18 | 55 | 2.5e-04 | 44 |
| NP_001041240.1 | 33.90 | 59 | 2.5e-04 | 42 |

### THSD4 (Q6ZMP0)

**Description:** Thrombospondin type-1 domain-containing protein 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_505017.1 | 32.47 | 653 | 7.4e-76 | 274 |
| NP_505017.1 | 32.89 | 377 | 1.2e-43 | 173 |
| NP_001370555.1 | 32.47 | 653 | 1.4e-75 | 273 |
| NP_001370555.1 | 31.04 | 422 | 1.8e-43 | 172 |
| NP_741569.1 | 32.47 | 653 | 1.9e-75 | 272 |
| NP_741569.1 | 31.04 | 422 | 2.6e-43 | 172 |
| NP_001359558.1 | 30.08 | 728 | 1.7e-70 | 257 |
| NP_001359558.1 | 30.18 | 401 | 2.8e-37 | 152 |
| NP_001359558.1 | 35.67 | 314 | 3.8e-37 | 152 |
| NP_001359558.1 | 28.50 | 414 | 4.8e-35 | 145 |
| NP_001359558.1 | 30.36 | 359 | 7.6e-31 | 132 |
| NP_001359558.1 | 30.87 | 311 | 7.9e-31 | 132 |
| NP_001359558.1 | 29.25 | 376 | 5.2e-30 | 129 |
| NP_001359558.1 | 28.26 | 407 | 1.6e-26 | 117 |
| NP_001359558.1 | 32.00 | 125 | 5.7e-06 | 50 |
| NP_001255448.1 | 30.43 | 700 | 3.1e-70 | 256 |
| NP_001255448.1 | 30.18 | 401 | 2.6e-37 | 152 |
| NP_001255448.1 | 35.67 | 314 | 3.5e-37 | 152 |
| NP_001255448.1 | 28.50 | 414 | 4.8e-35 | 145 |
| NP_001255448.1 | 30.36 | 359 | 7.7e-31 | 132 |
| NP_001255448.1 | 30.87 | 311 | 7.9e-31 | 132 |
| NP_001255448.1 | 29.25 | 376 | 5.4e-30 | 129 |
| NP_001255448.1 | 28.26 | 407 | 1.6e-26 | 117 |
| NP_001255448.1 | 32.00 | 125 | 5.8e-06 | 50 |
| NP_001250933.1 | 27.20 | 386 | 1.1e-30 | 130 |
| NP_001250933.1 | 27.50 | 440 | 2.5e-28 | 123 |
| NP_001250933.1 | 30.09 | 236 | 3.9e-24 | 109 |
| NP_001250933.1 | 33.12 | 154 | 4.1e-12 | 70 |
| NP_001250933.1 | 25.94 | 239 | 2.3e-10 | 64 |
| NP_001250933.1 | 28.65 | 192 | 5.8e-10 | 63 |
| NP_001250933.1 | 29.08 | 141 | 2.6e-07 | 55 |
| NP_492405.3 | 27.06 | 388 | 4.8e-29 | 125 |
| NP_492405.3 | 27.60 | 442 | 1.8e-28 | 123 |
| NP_492405.3 | 30.09 | 236 | 4.1e-24 | 109 |
| NP_492405.3 | 33.12 | 154 | 4.4e-12 | 70 |
| NP_492405.3 | 28.65 | 192 | 2.6e-11 | 67 |
| NP_492405.3 | 25.94 | 239 | 2.6e-10 | 64 |
| NP_492405.3 | 29.08 | 141 | 2.9e-07 | 54 |
| NP_509829.2 | 30.61 | 379 | 2.1e-25 | 113 |
| NP_871884.1 | 30.09 | 236 | 2.6e-24 | 109 |
| NP_871884.1 | 29.09 | 275 | 4.4e-20 | 95 |
| NP_871884.1 | 28.68 | 258 | 3.6e-19 | 93 |
| NP_871884.1 | 27.56 | 254 | 9.8e-16 | 82 |
| NP_871884.1 | 33.12 | 154 | 1.5e-12 | 71 |
| NP_871884.1 | 25.11 | 239 | 8.5e-11 | 65 |
| NP_001024532.2 | 30.23 | 258 | 1.2e-14 | 78 |
| NP_001024532.2 | 30.25 | 238 | 4.3e-13 | 73 |
| NP_001024532.2 | 27.89 | 294 | 1.2e-12 | 72 |
| NP_001024532.2 | 26.65 | 319 | 6.2e-12 | 70 |
| NP_001024532.2 | 27.87 | 244 | 5.9e-08 | 57 |

### TRIM63 (Q969Q1)

**Description:** E3 ubiquitin-protein ligase TRIM63

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001122877.1 | 24.40 | 209 | 7.1e-14 | 72 |
| NP_503395.1 | 24.40 | 209 | 7.5e-14 | 72 |
| NP_499028.1 | 31.25 | 96 | 5.8e-08 | 54 |
| NP_001309582.1 | 31.25 | 96 | 6.6e-08 | 54 |
| NP_001123114.1 | 21.16 | 293 | 1.9e-06 | 48 |
| NP_001040817.2 | 24.32 | 148 | 3.5e-06 | 48 |
| NP_491164.2 | 26.89 | 119 | 3.7e-05 | 45 |
| NP_001364681.1 | 23.27 | 245 | 4.0e-05 | 44 |
| NP_494241.2 | 24.22 | 256 | 8.2e-05 | 43 |
| NP_741866.1 | 23.41 | 205 | 5.8e-04 | 41 |

### TUSC3 (Q13454)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit TUSC3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_498691.1 | 45.51 | 323 | 1.5e-105 | 312 |

### UAP1 (Q16222)

**Description:** UDP-N-acetylhexosamine pyrophosphorylase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_497777.1 | 40.75 | 427 | 9.6e-92 | 288 |
| NP_508277.2 | 23.18 | 233 | 1.9e-08 | 56 |

### UBA52 (P62987)

**Description:** Ubiquitin-ribosomal protein eL40 fusion protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_499695.1 | 92.97 | 128 | 2.1e-84 | 242 |
| NP_741157.1 | 97.40 | 77 | 7.9e-45 | 155 |
| NP_741157.1 | 97.40 | 77 | 7.9e-45 | 155 |
| NP_741157.1 | 97.40 | 77 | 7.9e-45 | 155 |
| NP_741157.1 | 97.40 | 77 | 7.9e-45 | 155 |
| NP_741157.1 | 97.40 | 77 | 7.9e-45 | 155 |
| NP_741157.1 | 97.40 | 77 | 7.9e-45 | 155 |
| NP_741157.1 | 97.40 | 77 | 7.9e-45 | 155 |
| NP_741157.1 | 97.40 | 77 | 7.9e-45 | 155 |
| NP_741157.1 | 97.40 | 77 | 7.9e-45 | 155 |
| NP_741157.1 | 97.44 | 78 | 9.1e-45 | 154 |
| NP_741157.1 | 96.10 | 77 | 4.8e-44 | 152 |
| NP_001379170.1 | 61.04 | 77 | 1.4e-28 | 99 |
| NP_741102.1 | 45.95 | 74 | 4.5e-14 | 64 |
| NP_494128.3 | 45.07 | 71 | 1.1e-13 | 64 |
| NP_494542.1 | 40.28 | 72 | 4.0e-11 | 58 |
| NP_001022637.1 | 43.48 | 46 | 1.1e-06 | 42 |
| NP_494511.2 | 38.36 | 73 | 5.9e-06 | 43 |
| NP_494127.2 | 38.89 | 72 | 1.3e-05 | 42 |
| NP_001257042.1 | 30.51 | 59 | 7.7e-05 | 38 |

### UBB (P0CG47)

**Description:** Polyubiquitin-B

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_741157.1 | 98.25 | 229 | 3.5e-153 | 446 |
| NP_741157.1 | 98.25 | 229 | 3.5e-153 | 446 |
| NP_741157.1 | 98.25 | 229 | 3.5e-153 | 446 |
| NP_741157.1 | 98.25 | 229 | 3.5e-153 | 446 |
| NP_741157.1 | 98.25 | 229 | 3.5e-153 | 446 |
| NP_741157.1 | 98.68 | 228 | 4.3e-153 | 445 |
| NP_741157.1 | 97.82 | 229 | 2.4e-152 | 444 |
| NP_741157.1 | 97.82 | 229 | 2.4e-152 | 444 |
| NP_741157.1 | 97.82 | 229 | 2.4e-152 | 444 |
| NP_499695.1 | 97.40 | 77 | 9.2e-48 | 153 |
| NP_499695.1 | 97.40 | 77 | 9.2e-48 | 153 |
| NP_499695.1 | 98.68 | 76 | 1.5e-47 | 153 |
| NP_001379170.1 | 61.04 | 77 | 4.8e-27 | 99 |
| NP_001379170.1 | 61.04 | 77 | 4.8e-27 | 99 |
| NP_001379170.1 | 61.84 | 76 | 5.8e-27 | 99 |
| NP_494128.3 | 31.97 | 244 | 7.9e-17 | 76 |
| NP_494127.2 | 29.76 | 205 | 1.5e-14 | 69 |
| NP_741102.1 | 44.74 | 76 | 3.8e-13 | 65 |
| NP_741102.1 | 44.74 | 76 | 3.8e-13 | 65 |
| NP_741102.1 | 45.95 | 74 | 4.7e-13 | 64 |
| NP_494542.1 | 40.28 | 72 | 1.5e-10 | 58 |
| NP_494542.1 | 40.28 | 72 | 1.5e-10 | 58 |
| NP_494542.1 | 40.28 | 72 | 1.5e-10 | 58 |
| NP_001379809.1 | 38.74 | 111 | 3.3e-08 | 50 |
| NP_001379809.1 | 38.74 | 111 | 3.3e-08 | 50 |
| NP_001022637.1 | 43.48 | 46 | 1.5e-05 | 41 |
| NP_001022637.1 | 43.48 | 46 | 1.5e-05 | 41 |
| NP_001022637.1 | 43.48 | 46 | 1.5e-05 | 41 |
| NP_494511.2 | 38.36 | 73 | 1.6e-05 | 44 |
| NP_494511.2 | 38.36 | 73 | 1.6e-05 | 44 |
| NP_494511.2 | 38.36 | 73 | 1.6e-05 | 44 |

### UBC (P0CG48)

**Description:** Polyubiquitin-C

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_741157.1 | 98.39 | 685 | 0.0e+00 | 1326 |
| NP_741157.1 | 98.39 | 685 | 0.0e+00 | 1326 |
| NP_741157.1 | 98.54 | 684 | 0.0e+00 | 1326 |
| NP_499695.1 | 97.40 | 77 | 2.5e-44 | 154 |
| NP_499695.1 | 97.40 | 77 | 4.2e-44 | 154 |
| NP_499695.1 | 97.40 | 77 | 4.2e-44 | 154 |
| NP_499695.1 | 97.40 | 77 | 4.2e-44 | 154 |
| NP_499695.1 | 97.40 | 77 | 4.2e-44 | 154 |
| NP_499695.1 | 97.40 | 77 | 4.2e-44 | 154 |
| NP_499695.1 | 97.40 | 77 | 4.2e-44 | 154 |
| NP_499695.1 | 97.40 | 77 | 4.2e-44 | 154 |
| NP_499695.1 | 97.40 | 77 | 4.2e-44 | 154 |
| NP_001379170.1 | 61.04 | 77 | 4.2e-25 | 99 |
| NP_001379170.1 | 61.04 | 77 | 4.2e-25 | 99 |
| NP_001379170.1 | 61.04 | 77 | 4.2e-25 | 99 |
| NP_001379170.1 | 61.04 | 77 | 4.2e-25 | 99 |
| NP_001379170.1 | 61.04 | 77 | 4.2e-25 | 99 |
| NP_001379170.1 | 61.04 | 77 | 4.2e-25 | 99 |
| NP_001379170.1 | 61.04 | 77 | 4.2e-25 | 99 |
| NP_001379170.1 | 61.04 | 77 | 4.2e-25 | 99 |
| NP_001379170.1 | 61.84 | 76 | 5.2e-25 | 99 |
| NP_494128.3 | 31.97 | 244 | 1.3e-15 | 76 |
| NP_494128.3 | 31.97 | 244 | 1.3e-15 | 76 |
| NP_494128.3 | 31.97 | 244 | 1.3e-15 | 76 |
| NP_494128.3 | 31.97 | 244 | 1.3e-15 | 76 |
| NP_494128.3 | 31.97 | 244 | 1.3e-15 | 76 |
| NP_494128.3 | 31.97 | 244 | 1.3e-15 | 76 |
| NP_494128.3 | 31.97 | 244 | 1.3e-15 | 76 |
| NP_494127.2 | 29.76 | 252 | 9.8e-14 | 70 |
| NP_494127.2 | 29.76 | 252 | 9.8e-14 | 70 |
| NP_494127.2 | 29.76 | 252 | 9.8e-14 | 70 |
| NP_494127.2 | 29.76 | 252 | 9.8e-14 | 70 |
| NP_494127.2 | 29.76 | 252 | 9.8e-14 | 70 |
| NP_494127.2 | 29.76 | 252 | 9.8e-14 | 70 |
| NP_494127.2 | 28.51 | 228 | 5.4e-07 | 50 |
| NP_741102.1 | 44.74 | 76 | 3.5e-11 | 62 |
| NP_741102.1 | 44.74 | 76 | 3.5e-11 | 62 |
| NP_741102.1 | 44.74 | 76 | 3.5e-11 | 62 |
| NP_741102.1 | 44.74 | 76 | 3.5e-11 | 62 |
| NP_741102.1 | 44.74 | 76 | 3.5e-11 | 62 |
| NP_741102.1 | 44.74 | 76 | 3.5e-11 | 62 |
| NP_741102.1 | 44.74 | 76 | 3.5e-11 | 62 |
| NP_741102.1 | 44.74 | 76 | 3.5e-11 | 62 |
| NP_741102.1 | 45.95 | 74 | 3.9e-11 | 62 |
| NP_494542.1 | 40.28 | 72 | 9.1e-10 | 59 |
| NP_494542.1 | 40.28 | 72 | 9.1e-10 | 59 |
| NP_494542.1 | 40.28 | 72 | 9.1e-10 | 59 |
| NP_494542.1 | 40.28 | 72 | 9.1e-10 | 59 |
| NP_494542.1 | 40.28 | 72 | 9.1e-10 | 59 |
| NP_494542.1 | 40.28 | 72 | 9.1e-10 | 59 |
| NP_494542.1 | 40.28 | 72 | 9.1e-10 | 59 |
| NP_494542.1 | 40.28 | 72 | 9.1e-10 | 59 |
| NP_494542.1 | 40.28 | 72 | 9.1e-10 | 59 |
| NP_001379809.1 | 38.74 | 111 | 1.1e-07 | 50 |
| NP_001379809.1 | 38.74 | 111 | 1.1e-07 | 50 |
| NP_001379809.1 | 38.74 | 111 | 1.1e-07 | 50 |
| NP_001379809.1 | 38.74 | 111 | 1.1e-07 | 50 |
| NP_001379809.1 | 38.74 | 111 | 1.1e-07 | 50 |
| NP_001379809.1 | 38.74 | 111 | 1.1e-07 | 50 |
| NP_001379809.1 | 38.74 | 111 | 1.1e-07 | 50 |
| NP_001379809.1 | 38.74 | 111 | 1.1e-07 | 50 |
| NP_001022637.1 | 43.48 | 46 | 4.8e-05 | 41 |
| NP_001022637.1 | 43.48 | 46 | 4.8e-05 | 41 |
| NP_001022637.1 | 43.48 | 46 | 4.8e-05 | 41 |
| NP_001022637.1 | 43.48 | 46 | 4.8e-05 | 41 |
| NP_001022637.1 | 43.48 | 46 | 4.8e-05 | 41 |
| NP_001022637.1 | 43.48 | 46 | 4.8e-05 | 41 |
| NP_001022637.1 | 43.48 | 46 | 4.8e-05 | 41 |
| NP_001022637.1 | 43.48 | 46 | 4.8e-05 | 41 |
| NP_001022637.1 | 43.48 | 46 | 4.8e-05 | 41 |
| NP_494511.2 | 38.36 | 73 | 6.8e-05 | 44 |
| NP_494511.2 | 38.36 | 73 | 6.8e-05 | 44 |
| NP_494511.2 | 38.36 | 73 | 6.8e-05 | 44 |
| NP_494511.2 | 38.36 | 73 | 6.8e-05 | 44 |
| NP_494511.2 | 38.36 | 73 | 6.8e-05 | 44 |
| NP_494511.2 | 38.36 | 73 | 6.8e-05 | 44 |
| NP_494511.2 | 38.36 | 73 | 6.8e-05 | 44 |
| NP_494511.2 | 38.36 | 73 | 6.8e-05 | 44 |
| NP_494511.2 | 38.36 | 73 | 6.8e-05 | 44 |

### UBXN1 (Q04323)

**Description:** UBX domain-containing protein 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_490978.1 | 29.11 | 316 | 2.1e-28 | 110 |
| NP_001370812.1 | 26.85 | 216 | 1.8e-14 | 73 |

### UMOD (P07911)

**Description:** Uromodulin

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_497982.2 | 43.88 | 139 | 1.8e-20 | 96 |
| NP_497982.2 | 39.67 | 121 | 1.4e-16 | 84 |
| NP_497982.2 | 35.53 | 76 | 7.6e-05 | 45 |
| NP_001263718.1 | 43.59 | 117 | 3.3e-18 | 89 |
| NP_001263718.1 | 40.98 | 122 | 1.2e-14 | 78 |
| NP_001263718.1 | 40.62 | 128 | 2.4e-12 | 70 |
| NP_001263718.1 | 36.07 | 122 | 2.5e-12 | 70 |
| NP_001263718.1 | 33.84 | 133 | 1.6e-10 | 64 |
| NP_001263718.1 | 29.21 | 178 | 1.6e-10 | 64 |
| NP_001263718.1 | 28.18 | 252 | 2.2e-08 | 57 |
| NP_001263718.1 | 52.00 | 50 | 5.9e-07 | 53 |
| NP_001263718.1 | 26.63 | 169 | 5.3e-06 | 50 |
| NP_001263718.1 | 30.25 | 119 | 5.6e-06 | 49 |
| NP_001263715.1 | 43.59 | 117 | 3.4e-18 | 89 |
| NP_001263715.1 | 40.98 | 122 | 1.1e-14 | 78 |
| NP_001263715.1 | 36.07 | 122 | 1.9e-12 | 70 |
| NP_001263715.1 | 40.62 | 128 | 2.6e-12 | 70 |
| NP_001263715.1 | 33.84 | 133 | 1.1e-10 | 65 |
| NP_001263715.1 | 29.21 | 178 | 1.6e-10 | 64 |
| NP_001263715.1 | 40.00 | 130 | 2.2e-09 | 60 |
| NP_001263715.1 | 28.18 | 252 | 1.3e-08 | 58 |
| NP_001263715.1 | 52.00 | 50 | 5.7e-07 | 53 |
| NP_001263715.1 | 26.63 | 169 | 4.3e-06 | 50 |
| NP_001263715.1 | 30.25 | 119 | 5.3e-06 | 50 |
| NP_001263714.1 | 43.59 | 117 | 3.5e-18 | 89 |
| NP_001263714.1 | 40.98 | 122 | 1.1e-14 | 78 |
| NP_001263714.1 | 36.07 | 122 | 2.0e-12 | 70 |
| NP_001263714.1 | 40.62 | 128 | 2.6e-12 | 70 |
| NP_001263714.1 | 33.84 | 133 | 1.1e-10 | 65 |
| NP_001263714.1 | 29.21 | 178 | 1.5e-10 | 64 |
| NP_001263714.1 | 40.00 | 130 | 2.1e-09 | 60 |
| NP_001263714.1 | 28.18 | 252 | 1.3e-08 | 58 |
| NP_001263714.1 | 52.00 | 50 | 5.7e-07 | 53 |
| NP_001263714.1 | 26.63 | 169 | 4.8e-06 | 50 |
| NP_001263714.1 | 30.25 | 119 | 5.6e-06 | 49 |
| NP_001263712.1 | 43.59 | 117 | 3.5e-18 | 89 |
| NP_001263712.1 | 40.98 | 122 | 1.1e-14 | 78 |
| NP_001263712.1 | 36.07 | 122 | 1.9e-12 | 70 |
| NP_001263712.1 | 40.62 | 128 | 2.7e-12 | 70 |
| NP_001263712.1 | 33.84 | 133 | 1.1e-10 | 65 |
| NP_001263712.1 | 29.21 | 178 | 1.5e-10 | 64 |
| NP_001263712.1 | 40.00 | 130 | 2.1e-09 | 60 |
| NP_001263712.1 | 28.18 | 252 | 1.3e-08 | 58 |
| NP_001263712.1 | 52.00 | 50 | 5.7e-07 | 53 |
| NP_001263712.1 | 26.63 | 169 | 4.7e-06 | 50 |
| NP_001263712.1 | 30.25 | 119 | 5.7e-06 | 49 |
| NP_001263717.1 | 43.59 | 117 | 3.5e-18 | 89 |
| NP_001263717.1 | 40.98 | 122 | 1.1e-14 | 78 |
| NP_001263717.1 | 36.07 | 122 | 2.0e-12 | 70 |
| NP_001263717.1 | 40.62 | 128 | 2.7e-12 | 70 |
| NP_001263717.1 | 33.84 | 133 | 1.1e-10 | 65 |
| NP_001263717.1 | 29.21 | 178 | 1.5e-10 | 64 |
| NP_001263717.1 | 40.00 | 130 | 2.2e-09 | 60 |
| NP_001263717.1 | 28.18 | 252 | 1.4e-08 | 58 |
| NP_001263717.1 | 52.00 | 50 | 5.7e-07 | 53 |
| NP_001263717.1 | 26.63 | 169 | 4.4e-06 | 50 |
| NP_001263717.1 | 30.25 | 119 | 5.4e-06 | 50 |
| NP_001263711.1 | 43.59 | 117 | 3.7e-18 | 89 |
| NP_001263711.1 | 40.98 | 122 | 1.2e-14 | 78 |
| NP_001263711.1 | 36.07 | 122 | 2.0e-12 | 70 |
| NP_001263711.1 | 40.62 | 128 | 2.8e-12 | 70 |
| NP_001263711.1 | 33.84 | 133 | 1.2e-10 | 65 |
| NP_001263711.1 | 29.21 | 178 | 1.6e-10 | 64 |
| NP_001263711.1 | 40.00 | 130 | 2.3e-09 | 60 |
| NP_001263711.1 | 28.18 | 252 | 1.4e-08 | 58 |
| NP_001263711.1 | 52.00 | 50 | 5.8e-07 | 53 |
| NP_001263711.1 | 26.63 | 169 | 5.0e-06 | 50 |
| NP_001263711.1 | 30.25 | 119 | 5.7e-06 | 49 |
| NP_001263709.1 | 43.59 | 117 | 3.7e-18 | 89 |
| NP_001263709.1 | 40.98 | 122 | 1.2e-14 | 78 |
| NP_001263709.1 | 36.07 | 122 | 2.0e-12 | 70 |
| NP_001263709.1 | 40.62 | 128 | 2.8e-12 | 70 |
| NP_001263709.1 | 33.84 | 133 | 1.2e-10 | 65 |
| NP_001263709.1 | 29.21 | 178 | 1.6e-10 | 64 |
| NP_001263709.1 | 40.00 | 130 | 2.3e-09 | 60 |
| NP_001263709.1 | 28.18 | 252 | 1.4e-08 | 58 |
| NP_001263709.1 | 52.00 | 50 | 5.8e-07 | 53 |
| NP_001263709.1 | 26.63 | 169 | 4.8e-06 | 50 |
| NP_001263709.1 | 30.25 | 119 | 5.7e-06 | 49 |
| NP_001263716.1 | 43.59 | 117 | 3.8e-18 | 89 |
| NP_001263716.1 | 40.98 | 122 | 1.1e-14 | 78 |
| NP_001263716.1 | 36.07 | 122 | 2.0e-12 | 70 |
| NP_001263716.1 | 40.62 | 128 | 2.7e-12 | 70 |
| NP_001263716.1 | 33.84 | 133 | 1.2e-10 | 65 |
| NP_001263716.1 | 29.21 | 178 | 1.4e-10 | 64 |
| NP_001263716.1 | 40.00 | 130 | 2.3e-09 | 60 |
| NP_001263716.1 | 28.18 | 252 | 1.5e-08 | 58 |
| NP_001263716.1 | 52.00 | 50 | 5.3e-07 | 53 |
| NP_001263716.1 | 26.63 | 169 | 4.1e-06 | 50 |
| NP_001263716.1 | 30.25 | 119 | 5.4e-06 | 50 |
| NP_498670.2 | 43.59 | 117 | 3.8e-18 | 89 |
| NP_498670.2 | 40.98 | 122 | 1.1e-14 | 78 |
| NP_498670.2 | 36.07 | 122 | 2.1e-12 | 70 |
| NP_498670.2 | 40.62 | 128 | 2.9e-12 | 70 |
| NP_498670.2 | 33.84 | 133 | 1.2e-10 | 65 |
| NP_498670.2 | 29.21 | 178 | 1.6e-10 | 64 |
| NP_498670.2 | 40.00 | 130 | 2.3e-09 | 60 |
| NP_498670.2 | 28.18 | 252 | 1.4e-08 | 58 |
| NP_498670.2 | 52.00 | 50 | 5.5e-07 | 53 |
| NP_498670.2 | 26.63 | 169 | 4.5e-06 | 50 |
| NP_498670.2 | 30.25 | 119 | 5.6e-06 | 49 |

### VCP (P55072)

**Description:** Transitional endoplasmic reticulum ATPase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_495705.1 | 79.64 | 786 | 0.0e+00 | 1288 |
| NP_496273.1 | 77.26 | 809 | 0.0e+00 | 1270 |
| NP_496814.1 | 38.39 | 620 | 1.5e-136 | 424 |
| NP_496814.1 | 39.65 | 285 | 2.2e-48 | 184 |
| NP_492211.1 | 33.92 | 507 | 3.0e-86 | 289 |
| NP_492211.1 | 45.02 | 231 | 6.0e-56 | 205 |
| NP_504268.1 | 33.00 | 506 | 5.1e-66 | 233 |
| NP_504268.1 | 40.34 | 238 | 1.1e-38 | 154 |
| NP_504558.1 | 45.11 | 235 | 6.1e-61 | 212 |
| NP_504558.1 | 39.22 | 232 | 2.2e-53 | 191 |
| NP_499609.1 | 36.74 | 313 | 2.3e-60 | 210 |
| NP_499609.1 | 40.74 | 243 | 5.7e-56 | 198 |
| NP_506005.1 | 36.46 | 277 | 8.7e-60 | 209 |
| NP_506005.1 | 42.42 | 231 | 5.4e-57 | 201 |
| NP_001367171.1 | 41.83 | 263 | 3.0e-59 | 202 |
| NP_001367171.1 | 41.70 | 223 | 2.0e-47 | 169 |
| NP_001022114.1 | 37.67 | 292 | 3.1e-59 | 206 |
| NP_001022114.1 | 41.50 | 253 | 3.2e-58 | 203 |

