# Cross-Species Pathway Analysis Report for Rattus norvegicus

This report summarizes the BLAST search results for pathway enzymes.

## Run Parameters

- Report generated on: 2025-06-27 18:59:28
- Species: Rattus norvegicus (Tax ID: 10116)
- Number of threads used for BLAST: 8
- Query File: input/human_glycosylation_pathway_proteins.fasta
- BLAST program: blastp
- BLAST output format: 6 std qlen slen (Query ID, Subject ID, % Identity, Alignment Length, Mismatches, Gap Opens, Query Start, Query End, Subject Start, Subject End, E-value, Bit Score, Query Length, Subject Length)

### BLAST Parameters

- E-value cutoff: 0.001
- Max target sequences: 10
- Word size: 3
- Scoring matrix: BLOSUM62
- Gap open penalty: 11
- Gap extension penalty: 1

## BLAST Results for Rattus norvegicus (Tax ID: 10116)

### ABCA6 (Q8N139)

**Description:** ATP-binding cassette sub-family A member 6

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| XP_017459658.1 | 68.17 | 1624 | 0.0e+00 | 2331 |
| XP_008773685.1 | 68.17 | 1624 | 0.0e+00 | 2331 |
| XP_006220948.1 | 68.17 | 1624 | 0.0e+00 | 2331 |
| XP_006220947.1 | 68.17 | 1624 | 0.0e+00 | 2331 |
| XP_001081607.3 | 68.17 | 1624 | 0.0e+00 | 2331 |
| XP_006220946.1 | 68.17 | 1624 | 0.0e+00 | 2330 |
| XP_017459656.1 | 67.69 | 1603 | 0.0e+00 | 2286 |
| XP_017459655.1 | 56.81 | 1630 | 0.0e+00 | 1785 |
| NP_001268753.1 | 53.71 | 1629 | 0.0e+00 | 1672 |
| XP_008774047.1 | 53.09 | 1669 | 0.0e+00 | 1652 |

### ABL1 (P00519)

**Description:** Tyrosine-protein kinase ABL1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| XP_008773541.1 | 87.20 | 1109 | 0.0e+00 | 1915 |
| NP_001094320.1 | 87.20 | 1109 | 0.0e+00 | 1915 |
| NP_001100656.1 | 68.21 | 736 | 0.0e+00 | 974 |
| NP_077344.1 | 44.49 | 454 | 1.4e-124 | 392 |
| NP_036887.1 | 44.24 | 443 | 1.5e-116 | 371 |
| NP_114183.1 | 42.98 | 456 | 3.8e-115 | 368 |
| NP_001094179.1 | 44.59 | 444 | 6.4e-115 | 366 |
| NP_077059.2 | 43.21 | 442 | 2.4e-114 | 365 |
| NP_037317.2 | 42.95 | 461 | 3.2e-114 | 365 |
| NP_001104568.1 | 41.65 | 449 | 3.7e-111 | 355 |

### ABL2 (P42684)

**Description:** Tyrosine-protein kinase ABL2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001100656.1 | 91.29 | 1182 | 0.0e+00 | 2118 |
| XP_008773541.1 | 50.99 | 1263 | 0.0e+00 | 1074 |
| NP_001094320.1 | 50.99 | 1263 | 0.0e+00 | 1074 |
| NP_077344.1 | 43.94 | 462 | 7.8e-122 | 385 |
| NP_036887.1 | 45.37 | 443 | 6.2e-118 | 376 |
| NP_077059.2 | 42.60 | 453 | 1.7e-114 | 366 |
| NP_114183.1 | 44.80 | 442 | 3.8e-113 | 363 |
| NP_037317.2 | 43.30 | 448 | 6.5e-113 | 362 |
| NP_001094179.1 | 44.27 | 445 | 1.5e-112 | 361 |
| NP_001020922.1 | 43.71 | 453 | 2.8e-111 | 357 |

### ADAMTS10 (Q9H324)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 10

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001258374.1 | 94.38 | 1103 | 0.0e+00 | 2129 |
| XP_017449174.1 | 59.35 | 861 | 0.0e+00 | 1069 |
| XP_017449174.1 | 62.96 | 216 | 1.4e-86 | 305 |
| XP_017449174.1 | 25.68 | 222 | 3.4e-08 | 58 |
| NP_001178873.1 | 37.45 | 1052 | 0.0e+00 | 677 |
| NP_001178873.1 | 31.85 | 292 | 5.4e-32 | 135 |
| XP_003753205.1 | 37.29 | 1054 | 0.0e+00 | 671 |
| XP_003753205.1 | 31.10 | 299 | 3.5e-29 | 126 |
| XP_003753205.1 | 33.08 | 133 | 1.3e-07 | 56 |
| NP_001040566.1 | 38.97 | 988 | 0.0e+00 | 633 |
| NP_001040566.1 | 30.77 | 351 | 1.0e-24 | 112 |
| NP_001040566.1 | 35.13 | 185 | 4.0e-19 | 94 |
| XP_006222944.1 | 40.88 | 817 | 0.0e+00 | 627 |
| XP_006222944.1 | 30.91 | 317 | 6.6e-29 | 125 |
| XP_006222944.1 | 33.06 | 124 | 1.6e-07 | 55 |
| NP_001099890.2 | 37.14 | 1007 | 0.0e+00 | 624 |
| NP_001099890.2 | 25.89 | 448 | 2.4e-24 | 111 |
| XP_008772532.1 | 35.87 | 934 | 0.0e+00 | 558 |
| XP_017458840.1 | 33.45 | 1130 | 1.1e-169 | 543 |
| XP_017458840.1 | 37.12 | 229 | 2.8e-31 | 133 |
| XP_017458840.1 | 28.02 | 232 | 1.3e-20 | 99 |
| XP_017458840.1 | 29.07 | 227 | 1.4e-19 | 95 |
| XP_017458840.1 | 29.29 | 239 | 1.6e-19 | 95 |
| XP_017458840.1 | 23.76 | 202 | 2.4e-07 | 55 |
| XP_001059825.2 | 32.92 | 1121 | 7.1e-167 | 522 |

### ADAMTS12 (P58397)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 12

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001099890.2 | 80.75 | 1605 | 0.0e+00 | 2666 |
| NP_001040566.1 | 45.59 | 1588 | 0.0e+00 | 1349 |
| NP_001178873.1 | 37.57 | 1014 | 0.0e+00 | 641 |
| NP_001178873.1 | 31.62 | 234 | 5.7e-18 | 90 |
| NP_001178873.1 | 33.15 | 184 | 2.5e-17 | 89 |
| NP_001178873.1 | 31.41 | 191 | 6.5e-16 | 84 |
| NP_001178873.1 | 26.78 | 295 | 3.6e-15 | 81 |
| XP_017449174.1 | 37.72 | 1026 | 0.0e+00 | 635 |
| XP_017449174.1 | 29.52 | 227 | 1.5e-19 | 95 |
| XP_017449174.1 | 33.78 | 148 | 5.1e-17 | 87 |
| XP_017449174.1 | 36.80 | 125 | 6.7e-11 | 67 |
| XP_003753205.1 | 36.81 | 1016 | 0.0e+00 | 633 |
| XP_003753205.1 | 29.00 | 300 | 1.8e-18 | 92 |
| XP_003753205.1 | 27.06 | 303 | 1.4e-17 | 89 |
| XP_003753205.1 | 31.66 | 199 | 3.2e-14 | 78 |
| XP_003753205.1 | 33.88 | 121 | 4.2e-09 | 62 |
| NP_001258374.1 | 36.97 | 1009 | 0.0e+00 | 601 |
| NP_001258374.1 | 30.85 | 282 | 2.0e-25 | 115 |
| NP_001258374.1 | 31.35 | 185 | 1.1e-16 | 86 |
| XP_006222944.1 | 40.12 | 825 | 0.0e+00 | 591 |
| XP_006222944.1 | 29.05 | 296 | 4.3e-18 | 91 |
| XP_006222944.1 | 27.06 | 303 | 5.4e-17 | 87 |
| XP_006222944.1 | 31.66 | 199 | 5.3e-14 | 77 |
| XP_006222944.1 | 33.88 | 121 | 9.2e-09 | 60 |
| XP_008772532.1 | 37.46 | 889 | 0.0e+00 | 574 |
| XP_001059825.2 | 37.05 | 934 | 8.5e-174 | 553 |
| XP_001059825.2 | 30.53 | 285 | 4.2e-30 | 130 |
| XP_001059825.2 | 28.57 | 175 | 1.7e-13 | 76 |
| XP_006223406.1 | 36.16 | 957 | 1.4e-167 | 537 |
| XP_006223406.1 | 30.53 | 285 | 6.9e-30 | 129 |
| XP_006223406.1 | 28.57 | 175 | 2.0e-13 | 75 |

### ADAMTS13 (Q76LX8)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 13

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001178873.1 | 30.22 | 953 | 4.3e-110 | 377 |
| NP_001178873.1 | 29.76 | 252 | 6.1e-19 | 94 |
| NP_001178873.1 | 29.38 | 177 | 2.1e-12 | 72 |
| XP_017458840.1 | 28.64 | 1128 | 9.9e-104 | 365 |
| XP_017458840.1 | 24.89 | 454 | 7.0e-22 | 103 |
| XP_017458840.1 | 24.68 | 474 | 7.2e-20 | 97 |
| XP_006222944.1 | 29.11 | 962 | 5.1e-103 | 353 |
| XP_006222944.1 | 30.50 | 259 | 8.5e-20 | 96 |
| XP_006222944.1 | 27.36 | 296 | 1.9e-13 | 75 |
| NP_001099890.2 | 31.23 | 778 | 1.1e-102 | 361 |
| NP_001099890.2 | 34.90 | 149 | 6.4e-16 | 84 |
| NP_001099890.2 | 30.46 | 174 | 1.4e-12 | 73 |
| NP_001099890.2 | 30.92 | 152 | 1.4e-10 | 66 |
| NP_001099890.2 | 28.27 | 237 | 1.6e-10 | 66 |
| NP_001099890.2 | 33.04 | 115 | 8.8e-08 | 57 |
| NP_001099890.2 | 32.06 | 131 | 2.3e-07 | 56 |
| NP_001099890.2 | 24.91 | 277 | 1.4e-04 | 47 |
| XP_003753205.1 | 29.11 | 962 | 1.5e-101 | 353 |
| XP_003753205.1 | 30.50 | 259 | 7.1e-20 | 97 |
| XP_003753205.1 | 27.36 | 296 | 1.2e-13 | 76 |
| XP_001059825.2 | 30.50 | 941 | 5.3e-95 | 332 |
| XP_001059825.2 | 28.10 | 242 | 9.8e-18 | 90 |
| NP_077376.2 | 30.00 | 740 | 7.7e-95 | 328 |
| NP_077376.2 | 28.36 | 134 | 8.6e-08 | 57 |
| NP_077376.2 | 33.95 | 109 | 1.1e-07 | 57 |
| NP_001131094.1 | 32.77 | 769 | 9.1e-95 | 332 |
| NP_001101347.1 | 25.61 | 1156 | 6.8e-92 | 329 |
| NP_001101347.1 | 30.29 | 449 | 5.0e-33 | 140 |
| NP_001101347.1 | 29.28 | 461 | 5.9e-32 | 136 |
| NP_001101347.1 | 26.79 | 448 | 1.8e-24 | 112 |
| NP_001101347.1 | 30.30 | 165 | 1.3e-10 | 66 |
| NP_001040566.1 | 31.63 | 784 | 4.9e-91 | 327 |
| NP_001040566.1 | 37.70 | 191 | 2.1e-20 | 99 |
| NP_001040566.1 | 29.14 | 302 | 3.9e-16 | 85 |
| NP_001040566.1 | 33.15 | 178 | 6.6e-16 | 84 |
| NP_001040566.1 | 33.33 | 138 | 5.7e-06 | 51 |

### ADAMTS14 (Q8WXS8)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 14

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001101106.1 | 76.61 | 1180 | 0.0e+00 | 1792 |
| NP_001131094.1 | 51.72 | 1162 | 0.0e+00 | 1162 |
| NP_001100682.1 | 60.11 | 905 | 0.0e+00 | 1152 |
| NP_001178873.1 | 31.07 | 1059 | 7.9e-136 | 445 |
| NP_001178873.1 | 28.19 | 188 | 2.0e-11 | 68 |
| NP_001040566.1 | 32.31 | 975 | 3.6e-135 | 453 |
| NP_001040566.1 | 33.54 | 158 | 1.4e-14 | 79 |
| NP_001040566.1 | 32.52 | 123 | 3.4e-05 | 48 |
| XP_003753205.1 | 31.37 | 966 | 5.2e-135 | 443 |
| XP_003753205.1 | 25.48 | 208 | 4.5e-12 | 71 |
| NP_001258374.1 | 31.52 | 1104 | 3.4e-130 | 427 |
| XP_001059825.2 | 32.91 | 957 | 4.3e-129 | 424 |
| XP_006223406.1 | 32.28 | 979 | 1.1e-124 | 413 |
| NP_001099890.2 | 29.64 | 1029 | 1.5e-124 | 421 |
| NP_001099890.2 | 27.72 | 303 | 1.2e-15 | 82 |

### ADAMTS16 (Q8TE57)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 16

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| XP_003753205.1 | 81.12 | 1218 | 0.0e+00 | 2071 |
| XP_006222944.1 | 85.61 | 973 | 0.0e+00 | 1780 |
| XP_008772532.1 | 81.17 | 924 | 0.0e+00 | 1570 |
| NP_001178873.1 | 56.46 | 1176 | 0.0e+00 | 1387 |
| XP_017449174.1 | 34.47 | 1204 | 0.0e+00 | 674 |
| NP_001258374.1 | 36.35 | 1070 | 0.0e+00 | 623 |
| NP_001258374.1 | 31.31 | 99 | 2.3e-05 | 49 |
| NP_001099890.2 | 36.23 | 1024 | 0.0e+00 | 604 |
| NP_001099890.2 | 27.96 | 304 | 3.5e-18 | 91 |
| NP_001099890.2 | 28.43 | 197 | 6.2e-13 | 73 |
| NP_001099890.2 | 34.17 | 120 | 6.6e-07 | 54 |
| NP_001040566.1 | 36.61 | 1008 | 0.0e+00 | 596 |
| NP_001040566.1 | 31.73 | 312 | 4.9e-24 | 110 |
| NP_001040566.1 | 33.33 | 243 | 1.0e-19 | 96 |
| NP_001040566.1 | 30.53 | 190 | 1.9e-13 | 75 |
| XP_001059825.2 | 34.57 | 1053 | 2.2e-173 | 543 |
| XP_001059825.2 | 28.44 | 225 | 2.1e-12 | 72 |
| XP_006223406.1 | 33.86 | 1075 | 1.5e-169 | 533 |
| XP_006223406.1 | 28.44 | 225 | 2.5e-12 | 72 |

### ADAMTS17 (Q8TE56)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 17

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| XP_001059825.2 | 88.00 | 1100 | 0.0e+00 | 1994 |
| XP_006223406.1 | 86.20 | 1123 | 0.0e+00 | 1986 |
| NP_001101903.1 | 52.99 | 651 | 0.0e+00 | 691 |
| NP_001040566.1 | 35.14 | 942 | 1.1e-167 | 540 |
| NP_001040566.1 | 33.92 | 283 | 8.3e-29 | 125 |
| NP_001178873.1 | 34.24 | 1034 | 3.8e-162 | 512 |
| NP_001178873.1 | 29.80 | 302 | 2.0e-25 | 114 |
| XP_003753205.1 | 33.65 | 1070 | 1.4e-160 | 508 |
| XP_003753205.1 | 28.11 | 306 | 7.5e-22 | 102 |
| XP_003753205.1 | 28.91 | 211 | 1.3e-15 | 82 |
| XP_006222944.1 | 36.05 | 885 | 1.3e-155 | 488 |
| XP_006222944.1 | 28.11 | 306 | 2.9e-21 | 100 |
| XP_006222944.1 | 28.91 | 211 | 3.0e-15 | 81 |
| NP_001258374.1 | 32.92 | 1118 | 1.7e-155 | 491 |
| NP_001099890.2 | 34.38 | 928 | 1.9e-152 | 494 |
| NP_001099890.2 | 31.93 | 285 | 4.2e-29 | 126 |
| NP_001099890.2 | 32.03 | 153 | 4.0e-12 | 71 |
| XP_017449174.1 | 32.04 | 1080 | 2.2e-144 | 464 |
| XP_017449174.1 | 30.40 | 227 | 4.9e-23 | 106 |
| XP_017449174.1 | 34.48 | 174 | 9.8e-14 | 76 |

### ADAMTS18 (Q8TE60)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 18

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001178873.1 | 87.89 | 1222 | 0.0e+00 | 2231 |
| XP_003753205.1 | 56.24 | 1186 | 0.0e+00 | 1397 |
| XP_006222944.1 | 59.86 | 974 | 0.0e+00 | 1256 |
| XP_008772532.1 | 59.15 | 891 | 0.0e+00 | 1103 |
| XP_017449174.1 | 36.40 | 1206 | 0.0e+00 | 723 |
| NP_001258374.1 | 37.01 | 1070 | 0.0e+00 | 652 |
| NP_001258374.1 | 32.77 | 296 | 4.1e-34 | 142 |
| NP_001258374.1 | 28.57 | 196 | 4.9e-14 | 77 |
| NP_001099890.2 | 37.52 | 1018 | 0.0e+00 | 641 |
| NP_001099890.2 | 27.43 | 299 | 1.3e-19 | 95 |
| NP_001099890.2 | 33.68 | 190 | 2.9e-16 | 85 |
| NP_001099890.2 | 30.30 | 198 | 4.3e-15 | 81 |
| NP_001099890.2 | 29.59 | 196 | 4.6e-14 | 77 |
| NP_001099890.2 | 31.34 | 134 | 4.6e-07 | 54 |
| NP_001040566.1 | 35.26 | 1055 | 0.0e+00 | 591 |
| NP_001040566.1 | 31.85 | 292 | 8.5e-22 | 103 |
| NP_001040566.1 | 30.00 | 250 | 7.5e-17 | 87 |
| NP_001040566.1 | 31.46 | 178 | 1.9e-14 | 79 |
| NP_001040566.1 | 33.55 | 152 | 1.3e-13 | 76 |
| XP_001059825.2 | 35.20 | 1020 | 4.5e-174 | 545 |
| XP_006223406.1 | 34.45 | 1042 | 3.2e-170 | 535 |

### ADAMTS2 (O95450)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001131094.1 | 85.34 | 1214 | 0.0e+00 | 2048 |
| NP_001100682.1 | 63.47 | 887 | 0.0e+00 | 1180 |
| NP_001101106.1 | 50.62 | 1049 | 0.0e+00 | 1041 |
| NP_001178873.1 | 33.27 | 1034 | 2.9e-154 | 494 |
| NP_001178873.1 | 27.20 | 250 | 3.2e-14 | 78 |
| XP_003753205.1 | 32.76 | 983 | 2.1e-146 | 473 |
| XP_003753205.1 | 30.96 | 197 | 5.7e-15 | 80 |
| XP_003753205.1 | 28.15 | 135 | 9.8e-05 | 47 |
| XP_001059825.2 | 33.44 | 978 | 7.2e-141 | 456 |
| NP_001258374.1 | 31.85 | 1014 | 3.4e-138 | 448 |
| NP_001040566.1 | 32.56 | 989 | 5.4e-138 | 461 |
| NP_001040566.1 | 28.98 | 245 | 6.1e-15 | 80 |
| NP_001040566.1 | 30.17 | 179 | 1.4e-12 | 72 |
| NP_001099890.2 | 31.10 | 1013 | 1.1e-137 | 457 |
| NP_001099890.2 | 31.29 | 163 | 2.0e-15 | 82 |
| NP_001099890.2 | 28.57 | 245 | 1.5e-14 | 79 |
| XP_006223406.1 | 32.80 | 1000 | 2.3e-136 | 444 |

### ADAMTS20 (P59510)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 20

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001101347.1 | 49.13 | 1895 | 0.0e+00 | 1857 |
| XP_017458840.1 | 50.51 | 1566 | 0.0e+00 | 1455 |
| XP_017458840.1 | 68.49 | 749 | 0.0e+00 | 1075 |
| XP_017458840.1 | 30.06 | 499 | 3.3e-50 | 197 |
| XP_017458840.1 | 23.36 | 428 | 3.6e-21 | 102 |
| NP_001100280.1 | 38.92 | 984 | 0.0e+00 | 677 |
| NP_001100280.1 | 38.26 | 149 | 2.2e-20 | 99 |
| NP_001100280.1 | 35.62 | 146 | 1.5e-14 | 79 |
| NP_001100280.1 | 31.82 | 110 | 2.3e-12 | 72 |
| NP_001100280.1 | 33.33 | 114 | 2.7e-09 | 62 |
| NP_001100280.1 | 31.82 | 132 | 5.3e-09 | 61 |
| NP_001100280.1 | 32.46 | 114 | 1.3e-07 | 57 |
| NP_001100280.1 | 30.00 | 110 | 2.7e-07 | 56 |
| NP_077376.2 | 40.36 | 882 | 0.0e+00 | 635 |
| NP_077376.2 | 39.81 | 103 | 2.2e-17 | 89 |
| NP_077376.2 | 38.39 | 112 | 1.8e-14 | 79 |
| NP_077376.2 | 39.45 | 109 | 5.6e-13 | 74 |
| NP_077376.2 | 32.41 | 108 | 2.0e-10 | 66 |
| NP_077376.2 | 36.19 | 105 | 3.8e-10 | 65 |
| NP_077376.2 | 26.21 | 145 | 2.2e-08 | 59 |
| NP_077376.2 | 29.73 | 111 | 5.1e-05 | 48 |
| NP_077376.2 | 22.55 | 204 | 8.8e-04 | 44 |
| XP_003754443.1 | 39.46 | 854 | 0.0e+00 | 613 |
| XP_003754443.1 | 45.28 | 53 | 2.7e-07 | 56 |
| XP_003754443.1 | 40.98 | 61 | 4.6e-05 | 48 |
| XP_003754443.1 | 42.10 | 57 | 7.3e-04 | 45 |
| XP_017449174.1 | 32.40 | 1176 | 0.0e+00 | 586 |
| XP_017449174.1 | 32.97 | 182 | 5.0e-19 | 94 |
| XP_017449174.1 | 30.19 | 212 | 7.5e-18 | 90 |
| XP_017449174.1 | 25.24 | 317 | 3.5e-13 | 75 |
| XP_017449174.1 | 26.83 | 205 | 1.7e-11 | 70 |
| XP_017449174.1 | 24.48 | 241 | 1.4e-10 | 67 |
| XP_017449174.1 | 28.88 | 187 | 6.5e-10 | 64 |
| XP_017449174.1 | 28.09 | 178 | 5.9e-09 | 61 |
| NP_942056.1 | 36.77 | 854 | 3.6e-165 | 530 |
| NP_942056.1 | 43.75 | 48 | 1.6e-04 | 47 |
| NP_001258374.1 | 33.55 | 1100 | 1.0e-164 | 534 |
| NP_001258374.1 | 37.83 | 230 | 1.5e-31 | 135 |
| NP_001258374.1 | 29.12 | 285 | 3.7e-24 | 111 |
| NP_001258374.1 | 29.33 | 208 | 3.2e-17 | 88 |
| NP_001258374.1 | 24.32 | 292 | 8.1e-13 | 74 |
| NP_001178873.1 | 30.90 | 1191 | 7.5e-163 | 532 |
| NP_001178873.1 | 28.94 | 311 | 5.8e-29 | 127 |
| NP_001178873.1 | 26.87 | 361 | 7.8e-28 | 123 |
| NP_001178873.1 | 24.93 | 349 | 1.6e-21 | 102 |
| NP_001178873.1 | 27.51 | 349 | 3.1e-21 | 102 |
| NP_001178873.1 | 25.00 | 208 | 3.1e-07 | 56 |
| NP_076449.1 | 36.66 | 832 | 1.3e-160 | 514 |

### ADAMTS4 (O75173)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_076449.1 | 89.15 | 839 | 0.0e+00 | 1440 |
| NP_077376.2 | 49.52 | 834 | 0.0e+00 | 798 |
| NP_001100280.1 | 45.85 | 820 | 0.0e+00 | 701 |
| XP_003754443.1 | 43.70 | 810 | 0.0e+00 | 655 |
| NP_942056.1 | 41.53 | 773 | 0.0e+00 | 572 |
| XP_017458840.1 | 39.48 | 851 | 0.0e+00 | 565 |
| NP_001101347.1 | 41.76 | 601 | 9.9e-147 | 474 |
| XP_017449174.1 | 36.07 | 779 | 4.1e-132 | 423 |
| XP_008772532.1 | 32.92 | 802 | 2.2e-111 | 363 |
| NP_001040566.1 | 35.34 | 696 | 3.4e-109 | 368 |

### ADAMTS6 (Q9UKP5)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 6

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| XP_017449174.1 | 80.22 | 1188 | 0.0e+00 | 1910 |
| NP_001258374.1 | 59.82 | 1115 | 0.0e+00 | 1382 |
| NP_001178873.1 | 39.60 | 1063 | 0.0e+00 | 767 |
| NP_001178873.1 | 34.11 | 129 | 1.4e-09 | 62 |
| XP_003753205.1 | 38.38 | 1084 | 0.0e+00 | 731 |
| XP_003753205.1 | 32.66 | 297 | 4.7e-33 | 139 |
| XP_003753205.1 | 34.71 | 121 | 1.7e-05 | 49 |
| NP_001099890.2 | 40.10 | 1015 | 0.0e+00 | 696 |
| NP_001099890.2 | 28.83 | 274 | 1.6e-21 | 102 |
| NP_001040566.1 | 41.44 | 958 | 0.0e+00 | 694 |
| NP_001040566.1 | 30.96 | 281 | 1.0e-22 | 105 |
| NP_001040566.1 | 34.73 | 167 | 7.0e-18 | 90 |
| XP_006222944.1 | 41.27 | 848 | 0.0e+00 | 665 |
| XP_006222944.1 | 32.66 | 297 | 1.1e-32 | 137 |
| XP_006222944.1 | 34.71 | 121 | 2.1e-05 | 49 |
| XP_008772532.1 | 39.04 | 899 | 0.0e+00 | 629 |
| XP_001059825.2 | 35.10 | 1094 | 0.0e+00 | 588 |
| NP_001101347.1 | 35.17 | 1106 | 0.0e+00 | 582 |
| NP_001101347.1 | 39.69 | 257 | 7.0e-31 | 132 |
| NP_001101347.1 | 31.64 | 256 | 1.8e-30 | 131 |
| NP_001101347.1 | 33.87 | 248 | 2.1e-27 | 121 |
| NP_001101347.1 | 31.39 | 274 | 1.8e-26 | 118 |
| NP_001101347.1 | 31.15 | 244 | 3.7e-21 | 100 |
| NP_001101347.1 | 33.52 | 176 | 4.3e-21 | 100 |
| NP_001101347.1 | 29.22 | 243 | 1.0e-18 | 92 |

### ADAMTS9 (Q9P2N4)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 9

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001101347.1 | 86.05 | 1907 | 0.0e+00 | 3380 |
| XP_017458840.1 | 44.20 | 1656 | 0.0e+00 | 1312 |
| XP_017458840.1 | 47.46 | 767 | 0.0e+00 | 717 |
| XP_017458840.1 | 28.43 | 394 | 3.2e-32 | 138 |
| XP_017458840.1 | 28.35 | 261 | 1.8e-18 | 93 |
| NP_001100280.1 | 39.98 | 1018 | 0.0e+00 | 710 |
| NP_001100280.1 | 38.60 | 114 | 7.0e-16 | 84 |
| NP_001100280.1 | 37.39 | 115 | 2.3e-13 | 76 |
| NP_001100280.1 | 33.73 | 166 | 4.0e-13 | 75 |
| NP_001100280.1 | 28.99 | 138 | 6.7e-12 | 71 |
| NP_001100280.1 | 36.45 | 107 | 1.1e-09 | 63 |
| NP_001100280.1 | 40.00 | 85 | 1.2e-09 | 63 |
| NP_001100280.1 | 33.03 | 109 | 8.8e-08 | 57 |
| NP_001100280.1 | 32.20 | 118 | 8.6e-07 | 54 |
| NP_001100280.1 | 44.00 | 50 | 1.8e-04 | 47 |
| NP_077376.2 | 40.72 | 894 | 0.0e+00 | 660 |
| NP_077376.2 | 38.33 | 120 | 2.5e-19 | 95 |
| NP_077376.2 | 37.17 | 113 | 3.3e-14 | 79 |
| NP_077376.2 | 40.40 | 99 | 4.2e-12 | 72 |
| NP_077376.2 | 44.44 | 81 | 2.4e-11 | 69 |
| NP_077376.2 | 33.04 | 112 | 1.7e-10 | 66 |
| NP_077376.2 | 33.60 | 125 | 1.7e-10 | 66 |
| NP_077376.2 | 36.56 | 93 | 3.0e-10 | 65 |
| NP_077376.2 | 34.82 | 112 | 7.2e-10 | 64 |
| NP_077376.2 | 33.98 | 103 | 2.4e-09 | 62 |
| NP_077376.2 | 25.83 | 120 | 5.2e-05 | 48 |
| XP_003754443.1 | 39.80 | 887 | 0.0e+00 | 635 |
| XP_003754443.1 | 41.54 | 65 | 2.4e-07 | 56 |
| XP_003754443.1 | 36.67 | 60 | 3.9e-04 | 45 |
| XP_017449174.1 | 35.12 | 1213 | 0.0e+00 | 629 |
| XP_017449174.1 | 38.20 | 178 | 5.6e-24 | 110 |
| XP_017449174.1 | 35.43 | 175 | 4.9e-22 | 104 |
| XP_017449174.1 | 35.09 | 171 | 2.6e-21 | 102 |
| XP_017449174.1 | 32.67 | 202 | 3.0e-18 | 92 |
| XP_017449174.1 | 29.61 | 206 | 8.0e-15 | 80 |
| XP_017449174.1 | 32.64 | 144 | 1.2e-12 | 73 |
| NP_001178873.1 | 31.86 | 1246 | 0.0e+00 | 586 |
| NP_001178873.1 | 33.00 | 303 | 5.2e-34 | 143 |
| NP_001178873.1 | 31.29 | 310 | 2.0e-30 | 132 |
| NP_001178873.1 | 31.72 | 268 | 9.2e-30 | 129 |
| NP_001178873.1 | 30.15 | 325 | 5.0e-27 | 120 |
| NP_001178873.1 | 29.64 | 334 | 5.2e-27 | 120 |
| NP_001178873.1 | 30.99 | 242 | 5.4e-19 | 94 |
| XP_003753205.1 | 31.89 | 1226 | 1.2e-171 | 557 |
| XP_003753205.1 | 30.42 | 309 | 2.9e-31 | 134 |
| XP_003753205.1 | 30.82 | 305 | 8.0e-29 | 126 |
| XP_003753205.1 | 30.61 | 343 | 3.3e-28 | 124 |
| XP_003753205.1 | 27.76 | 353 | 3.5e-24 | 111 |
| NP_001258374.1 | 34.85 | 1099 | 7.1e-169 | 546 |
| NP_001258374.1 | 37.75 | 249 | 3.0e-33 | 140 |
| NP_001258374.1 | 35.22 | 247 | 3.1e-29 | 128 |
| NP_001258374.1 | 31.91 | 257 | 1.5e-25 | 115 |
| NP_001258374.1 | 27.80 | 277 | 3.2e-21 | 102 |
| NP_001258374.1 | 28.62 | 276 | 9.7e-21 | 100 |
| NP_942056.1 | 37.78 | 839 | 1.3e-165 | 531 |
| NP_942056.1 | 43.40 | 53 | 3.5e-04 | 45 |

### ADAMTSL1 (Q8N6G6)

**Description:** ADAMTS-like protein 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| XP_017458386.1 | 91.56 | 1742 | 0.0e+00 | 3284 |
| XP_017458390.1 | 90.58 | 1763 | 0.0e+00 | 3253 |
| XP_017458389.1 | 91.09 | 1739 | 0.0e+00 | 3240 |
| XP_017458389.1 | 27.59 | 232 | 2.8e-08 | 59 |
| XP_017458388.1 | 91.09 | 1739 | 0.0e+00 | 3240 |
| XP_017458388.1 | 27.59 | 232 | 2.8e-08 | 59 |
| XP_017458391.1 | 91.09 | 1718 | 0.0e+00 | 3221 |
| XP_017458391.1 | 27.59 | 232 | 2.1e-08 | 60 |
| XP_017458387.1 | 90.11 | 1718 | 0.0e+00 | 3171 |
| XP_017458387.1 | 27.59 | 232 | 2.4e-08 | 59 |
| NP_001101003.2 | 41.44 | 1694 | 0.0e+00 | 1248 |
| NP_001101003.2 | 29.59 | 517 | 2.9e-53 | 206 |
| NP_001101003.2 | 32.20 | 236 | 1.0e-16 | 87 |
| NP_001101003.2 | 28.02 | 257 | 6.0e-13 | 74 |
| NP_001101003.2 | 29.67 | 182 | 5.5e-11 | 68 |
| NP_001101003.2 | 26.04 | 169 | 3.8e-08 | 58 |
| NP_001101003.2 | 26.59 | 173 | 1.0e-05 | 50 |
| XP_017458840.1 | 29.35 | 879 | 7.0e-81 | 296 |
| XP_017458840.1 | 25.82 | 550 | 5.9e-29 | 127 |
| XP_017458840.1 | 24.85 | 503 | 1.2e-27 | 123 |
| XP_017458840.1 | 26.14 | 241 | 5.9e-16 | 84 |
| XP_017458840.1 | 25.99 | 277 | 7.3e-16 | 84 |
| XP_017458840.1 | 26.67 | 240 | 5.4e-12 | 71 |
| XP_017458840.1 | 29.13 | 254 | 2.4e-11 | 69 |
| XP_017458840.1 | 29.92 | 264 | 4.6e-11 | 68 |
| NP_001101347.1 | 27.49 | 902 | 9.9e-72 | 267 |
| NP_001101347.1 | 28.74 | 602 | 1.9e-46 | 184 |
| NP_001101347.1 | 26.05 | 622 | 6.8e-39 | 160 |
| NP_001101347.1 | 31.51 | 238 | 3.0e-22 | 105 |
| NP_001101347.1 | 25.27 | 463 | 3.9e-21 | 101 |
| NP_001101347.1 | 31.50 | 254 | 6.7e-18 | 91 |
| NP_001101347.1 | 30.05 | 183 | 3.4e-15 | 82 |
| NP_001101347.1 | 29.60 | 223 | 1.2e-14 | 80 |
| NP_001101347.1 | 27.44 | 266 | 6.2e-14 | 78 |
| NP_001101347.1 | 24.66 | 296 | 3.5e-12 | 72 |
| NP_001101347.1 | 24.88 | 209 | 3.9e-10 | 65 |
| NP_001101347.1 | 23.02 | 252 | 7.2e-06 | 51 |
| NP_001101347.1 | 26.05 | 119 | 3.0e-04 | 46 |
| NP_001101347.1 | 32.04 | 103 | 3.1e-04 | 46 |
| XP_006225903.1 | 31.57 | 586 | 2.0e-69 | 258 |
| XP_006225903.1 | 33.19 | 235 | 6.8e-28 | 123 |
| XP_006225903.1 | 32.79 | 247 | 2.9e-25 | 115 |
| XP_006225903.1 | 29.21 | 267 | 1.5e-21 | 102 |
| XP_006225903.1 | 38.16 | 152 | 1.1e-19 | 96 |
| XP_006225903.1 | 25.55 | 321 | 5.1e-13 | 75 |
| XP_006225903.1 | 33.33 | 189 | 1.4e-12 | 73 |
| XP_006225903.1 | 33.06 | 124 | 4.4e-08 | 58 |

### ADAMTSL3 (P82987)

**Description:** ADAMTS-like protein 3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001101003.2 | 84.96 | 1695 | 0.0e+00 | 2954 |
| XP_017458391.1 | 42.29 | 1726 | 0.0e+00 | 1311 |
| XP_017458391.1 | 28.57 | 525 | 7.9e-50 | 195 |
| XP_017458391.1 | 34.11 | 214 | 1.1e-16 | 87 |
| XP_017458391.1 | 29.49 | 217 | 3.6e-14 | 79 |
| XP_017458391.1 | 27.09 | 251 | 7.5e-13 | 74 |
| XP_017458391.1 | 28.65 | 192 | 6.4e-10 | 64 |
| XP_017458386.1 | 42.29 | 1726 | 0.0e+00 | 1310 |
| XP_017458386.1 | 29.91 | 545 | 4.2e-59 | 226 |
| XP_017458386.1 | 30.32 | 277 | 8.8e-17 | 87 |
| XP_017458386.1 | 29.49 | 217 | 3.5e-14 | 79 |
| XP_017458386.1 | 27.09 | 251 | 9.1e-13 | 74 |
| XP_017458386.1 | 28.65 | 192 | 6.8e-10 | 64 |
| XP_017458389.1 | 42.55 | 1685 | 0.0e+00 | 1288 |
| XP_017458389.1 | 28.57 | 525 | 1.0e-49 | 195 |
| XP_017458389.1 | 34.11 | 214 | 1.5e-16 | 86 |
| XP_017458389.1 | 29.49 | 217 | 3.8e-14 | 78 |
| XP_017458389.1 | 27.09 | 251 | 9.5e-13 | 73 |
| XP_017458389.1 | 28.65 | 192 | 7.5e-10 | 64 |
| XP_017458388.1 | 42.55 | 1685 | 0.0e+00 | 1288 |
| XP_017458388.1 | 28.57 | 525 | 1.0e-49 | 195 |
| XP_017458388.1 | 34.11 | 214 | 1.5e-16 | 86 |
| XP_017458388.1 | 29.49 | 217 | 3.8e-14 | 78 |
| XP_017458388.1 | 27.09 | 251 | 9.5e-13 | 73 |
| XP_017458388.1 | 28.65 | 192 | 7.5e-10 | 64 |
| XP_017458387.1 | 41.60 | 1726 | 0.0e+00 | 1271 |
| XP_017458387.1 | 28.57 | 525 | 8.4e-50 | 195 |
| XP_017458387.1 | 34.11 | 214 | 1.3e-16 | 87 |
| XP_017458387.1 | 29.49 | 217 | 3.8e-14 | 78 |
| XP_017458387.1 | 27.09 | 251 | 8.9e-13 | 74 |
| XP_017458387.1 | 28.65 | 192 | 6.6e-10 | 64 |
| XP_017458390.1 | 41.84 | 1685 | 0.0e+00 | 1248 |
| XP_017458390.1 | 29.91 | 545 | 5.0e-59 | 225 |
| XP_017458390.1 | 30.32 | 277 | 1.0e-16 | 87 |
| XP_017458390.1 | 29.49 | 217 | 4.0e-14 | 78 |
| XP_017458390.1 | 27.09 | 251 | 1.3e-12 | 73 |
| XP_017458390.1 | 28.65 | 192 | 7.1e-10 | 64 |
| XP_008774516.1 | 30.55 | 599 | 2.3e-73 | 270 |
| XP_008774516.1 | 35.15 | 293 | 5.9e-30 | 130 |
| XP_008774516.1 | 29.74 | 232 | 8.2e-16 | 84 |
| XP_008774516.1 | 26.92 | 286 | 1.1e-15 | 83 |
| XP_008774516.1 | 22.10 | 276 | 1.7e-07 | 56 |
| XP_006225904.1 | 30.55 | 599 | 2.3e-73 | 270 |
| XP_006225904.1 | 35.15 | 293 | 5.9e-30 | 130 |
| XP_006225904.1 | 29.74 | 232 | 8.2e-16 | 84 |
| XP_006225904.1 | 26.92 | 286 | 1.1e-15 | 83 |
| XP_006225904.1 | 22.10 | 276 | 1.7e-07 | 56 |
| XP_006225903.1 | 30.55 | 599 | 3.3e-73 | 270 |
| XP_006225903.1 | 35.15 | 293 | 6.2e-30 | 130 |
| XP_006225903.1 | 29.74 | 232 | 6.5e-16 | 84 |
| XP_006225903.1 | 26.92 | 286 | 1.2e-15 | 83 |
| XP_006225903.1 | 22.10 | 276 | 2.0e-07 | 56 |

### ADAMTSL4 (Q6UY14)

**Description:** ADAMTS-like protein 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001029184.1 | 76.29 | 1046 | 0.0e+00 | 1478 |
| XP_017459138.1 | 44.00 | 750 | 0.0e+00 | 639 |
| XP_017459138.1 | 31.88 | 207 | 1.1e-11 | 69 |
| XP_017459137.1 | 44.00 | 750 | 0.0e+00 | 639 |
| XP_017459137.1 | 31.88 | 207 | 1.1e-11 | 69 |
| XP_017459136.1 | 44.00 | 750 | 0.0e+00 | 639 |
| XP_017459136.1 | 31.88 | 207 | 1.1e-11 | 69 |
| XP_006226486.1 | 43.96 | 637 | 1.0e-179 | 540 |
| NP_001178873.1 | 29.17 | 713 | 2.0e-82 | 292 |
| XP_006222944.1 | 32.61 | 552 | 1.4e-74 | 266 |
| XP_006222944.1 | 30.96 | 365 | 1.8e-27 | 120 |
| XP_003753205.1 | 32.61 | 552 | 3.9e-74 | 268 |
| XP_003753205.1 | 30.96 | 365 | 3.8e-28 | 123 |
| XP_001059825.2 | 32.43 | 552 | 5.0e-69 | 252 |
| XP_001059825.2 | 26.07 | 303 | 1.6e-21 | 101 |
| XP_001059825.2 | 34.44 | 180 | 2.3e-18 | 91 |
| XP_001059825.2 | 33.67 | 98 | 2.4e-10 | 65 |
| NP_001040566.1 | 33.33 | 504 | 2.9e-67 | 248 |
| NP_001040566.1 | 34.55 | 246 | 2.9e-27 | 120 |
| NP_001040566.1 | 34.62 | 286 | 4.8e-25 | 113 |
| NP_001040566.1 | 36.56 | 186 | 4.0e-18 | 90 |
| NP_001040566.1 | 34.72 | 193 | 2.3e-17 | 88 |
| NP_001040566.1 | 30.74 | 270 | 5.2e-15 | 80 |
| NP_001040566.1 | 29.79 | 188 | 7.8e-10 | 63 |

### ADAMTSL5 (Q6ZMM2)

**Description:** ADAMTS-like protein 5

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001101541.1 | 86.44 | 177 | 9.0e-107 | 318 |
| XP_017449174.1 | 40.22 | 276 | 9.2e-55 | 197 |
| XP_006225903.1 | 39.56 | 273 | 1.6e-54 | 197 |
| XP_008774516.1 | 39.56 | 273 | 1.6e-54 | 197 |
| XP_006225904.1 | 39.56 | 273 | 1.6e-54 | 197 |
| XP_006225902.1 | 39.56 | 273 | 1.7e-54 | 197 |
| XP_008774518.1 | 39.56 | 273 | 1.8e-54 | 197 |
| XP_008772532.1 | 38.55 | 275 | 7.9e-54 | 194 |
| XP_003753205.1 | 38.41 | 276 | 1.8e-53 | 194 |
| XP_006222944.1 | 38.41 | 276 | 2.4e-53 | 193 |

### ALG1 (Q9BT22)

**Description:** Chitobiosyldiphosphodolichol beta-mannosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001101734.1 | 80.22 | 278 | 6.6e-160 | 454 |

### ALG10 (Q5BKT4)

**Description:** Dol-P-Glc:Glc(2)Man(9)GlcNAc(2)-PP-Dol alpha-1,2-glucosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_620801.1 | 86.71 | 474 | 0.0e+00 | 808 |

### ALG10B (Q5I7T1)

**Description:** Dol-P-Glc:Glc(2)Man(9)GlcNAc(2)-PP-Dol alpha-1,2-glucosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_620801.1 | 85.86 | 474 | 0.0e+00 | 794 |

### ALG11 (Q2TAA5)

**Description:** GDP-Man:Man(3)GlcNAc(2)-PP-Dol alpha-1,2-mannosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001101871.1 | 86.97 | 491 | 0.0e+00 | 892 |
| NP_001094180.1 | 24.10 | 278 | 5.0e-12 | 67 |

### ALG12 (Q9BV10)

**Description:** Dol-P-Man:Man(7)GlcNAc(2)-PP-Dol alpha-1,6-mannosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001101574.2 | 79.18 | 490 | 0.0e+00 | 763 |
| NP_001101636.3 | 27.83 | 230 | 2.9e-10 | 62 |
| NP_001102470.1 | 22.63 | 358 | 1.0e-03 | 42 |

### ALG13 (Q9NP73)

**Description:** UDP-N-acetylglucosamine transferase subunit ALG13

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| XP_017443787.1 | 70.85 | 693 | 0.0e+00 | 1004 |
| XP_017443786.1 | 67.68 | 693 | 0.0e+00 | 937 |
| XP_017443786.1 | 63.49 | 126 | 1.0e-37 | 153 |
| NP_001178629.1 | 31.05 | 802 | 1.4e-85 | 301 |
| NP_001013973.1 | 77.40 | 146 | 1.3e-70 | 233 |
| NP_001032585.1 | 31.15 | 122 | 2.7e-12 | 70 |
| NP_001178912.1 | 28.83 | 163 | 1.1e-09 | 62 |
| XP_017443154.1 | 26.06 | 142 | 7.9e-08 | 56 |

### ALG14 (Q96F25)

**Description:** UDP-N-acetylglucosamine transferase subunit ALG14

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001014198.1 | 80.09 | 216 | 1.5e-129 | 363 |

### ALG2 (Q9H553)

**Description:** Alpha-1,3/1,6-mannosyltransferase ALG2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001094180.1 | 81.45 | 415 | 0.0e+00 | 717 |
| NP_001101871.1 | 23.94 | 284 | 4.1e-12 | 67 |
| XP_017459968.1 | 25.42 | 118 | 3.5e-06 | 48 |
| XP_001073476.1 | 25.42 | 118 | 3.5e-06 | 48 |

### ALG3 (Q92685)

**Description:** Dol-P-Man:Man(5)GlcNAc(2)-PP-Dol alpha-1,3-mannosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001135835.1 | 87.67 | 438 | 0.0e+00 | 738 |

### ALG6 (Q9Y672)

**Description:** Dolichyl pyrophosphate Man9GlcNAc2 alpha-1,3-glucosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001028881.1 | 86.39 | 507 | 0.0e+00 | 888 |
| NP_001029299.1 | 29.72 | 424 | 5.3e-33 | 132 |

### ALG8 (Q9BVK2)

**Description:** Dolichyl pyrophosphate Glc1Man9GlcNAc2 alpha-1,3-glucosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001029299.1 | 87.26 | 526 | 0.0e+00 | 938 |
| NP_001028881.1 | 29.61 | 385 | 9.9e-32 | 128 |

### ALG9 (Q9H6U8)

**Description:** Alpha-1,2-mannosyltransferase ALG9

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001102470.1 | 91.27 | 401 | 0.0e+00 | 705 |
| NP_001101636.3 | 23.11 | 238 | 1.2e-06 | 51 |

### AMDHD2 (Q9Y303)

**Description:** N-acetylglucosamine-6-phosphate deacetylase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001020161.2 | 90.91 | 407 | 0.0e+00 | 748 |

### AMFR (Q9UKV5)

**Description:** E3 ubiquitin-protein ligase AMFR

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| XP_006222727.2 | 94.89 | 548 | 0.0e+00 | 1046 |
| NP_001094209.1 | 27.04 | 318 | 2.1e-29 | 123 |
| NP_001099248.1 | 36.79 | 106 | 3.6e-12 | 69 |
| NP_001178011.1 | 27.13 | 129 | 1.3e-09 | 60 |
| NP_001028874.1 | 41.67 | 72 | 9.8e-09 | 57 |
| NP_001178022.1 | 28.71 | 101 | 1.1e-08 | 58 |
| XP_017459348.1 | 27.00 | 100 | 2.1e-08 | 57 |
| NP_001032747.1 | 26.36 | 110 | 2.8e-08 | 56 |
| NP_001121017.1 | 33.72 | 86 | 5.3e-08 | 56 |
| NP_001037743.1 | 25.42 | 118 | 4.5e-07 | 52 |

### ASGR1 (P07306)

**Description:** Asialoglycoprotein receptor 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_036635.1 | 80.00 | 285 | 3.9e-156 | 437 |
| NP_058885.1 | 51.00 | 300 | 6.1e-107 | 313 |
| NP_071788.2 | 53.72 | 309 | 3.5e-103 | 303 |
| NP_446205.1 | 35.13 | 222 | 1.5e-29 | 117 |
| NP_001020892.1 | 39.71 | 136 | 3.4e-25 | 105 |
| NP_598234.2 | 29.46 | 224 | 3.2e-23 | 97 |
| NP_001029096.1 | 29.46 | 224 | 3.3e-23 | 97 |
| NP_001005897.1 | 34.64 | 153 | 1.0e-21 | 90 |
| NP_001163868.1 | 29.73 | 259 | 1.5e-20 | 89 |
| NP_001099374.1 | 38.69 | 137 | 8.4e-20 | 86 |

### ASGR2 (P07307)

**Description:** Asialoglycoprotein receptor 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_058885.1 | 62.22 | 307 | 3.1e-139 | 395 |
| NP_036635.1 | 54.15 | 301 | 1.5e-103 | 304 |
| NP_071788.2 | 44.08 | 245 | 5.7e-75 | 232 |
| NP_446205.1 | 35.27 | 224 | 8.9e-32 | 124 |
| NP_001020892.1 | 42.14 | 140 | 3.0e-26 | 108 |
| NP_001005897.1 | 39.16 | 143 | 1.0e-25 | 102 |
| NP_598234.2 | 32.29 | 223 | 1.5e-23 | 98 |
| NP_001029096.1 | 32.29 | 223 | 1.8e-23 | 98 |
| NP_001163868.1 | 31.56 | 225 | 3.5e-21 | 92 |
| XP_002724815.3 | 40.77 | 130 | 1.4e-19 | 87 |

### ATXN3 (P54252)

**Description:** Ataxin-3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_067734.1 | 85.08 | 362 | 0.0e+00 | 626 |
| NP_001020180.1 | 28.02 | 157 | 1.3e-07 | 51 |
| NP_001099726.2 | 24.72 | 178 | 1.4e-07 | 51 |

### B3GALNT2 (Q8NCR0)

**Description:** UDP-GalNAc:beta-1,3-N-acetylgalactosaminyltransferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001138323.1 | 87.65 | 502 | 0.0e+00 | 886 |
| NP_001100169.1 | 29.19 | 185 | 1.1e-17 | 84 |
| NP_001102424.1 | 24.34 | 189 | 2.0e-14 | 74 |
| NP_001102962.1 | 27.18 | 206 | 6.8e-14 | 73 |
| NP_001013176.1 | 28.17 | 142 | 6.0e-11 | 63 |
| NP_598237.2 | 23.01 | 226 | 2.7e-08 | 56 |
| NP_001012134.1 | 24.36 | 197 | 2.8e-07 | 53 |
| NP_001099357.1 | 26.90 | 145 | 5.4e-07 | 51 |
| NP_001099681.1 | 24.16 | 149 | 5.1e-06 | 48 |
| NP_446384.1 | 26.53 | 147 | 6.4e-06 | 48 |

### B3GNT2 (Q9NY97)

**Description:** N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001100710.1 | 87.91 | 397 | 0.0e+00 | 736 |
| NP_001012134.1 | 43.10 | 297 | 3.0e-82 | 257 |
| NP_001099681.1 | 40.40 | 302 | 3.9e-78 | 246 |
| NP_001099408.1 | 41.42 | 309 | 8.6e-78 | 244 |
| NP_001099538.1 | 39.62 | 313 | 7.6e-74 | 235 |
| XP_006222857.1 | 42.81 | 285 | 1.0e-72 | 229 |
| NP_001100962.1 | 35.53 | 349 | 1.6e-60 | 201 |
| NP_001102424.1 | 35.60 | 309 | 1.4e-54 | 183 |
| NP_001099357.1 | 32.06 | 287 | 8.0e-47 | 162 |
| NP_001102962.1 | 32.48 | 274 | 1.6e-45 | 162 |

### B3GNT4 (Q9C0J1)

**Description:** N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001099408.1 | 78.66 | 342 | 0.0e+00 | 553 |
| NP_001012134.1 | 48.14 | 295 | 4.5e-86 | 266 |
| NP_001100710.1 | 42.39 | 335 | 6.7e-82 | 255 |
| NP_001099681.1 | 44.59 | 296 | 3.2e-81 | 253 |
| XP_006222857.1 | 49.83 | 291 | 5.7e-81 | 250 |
| NP_001099538.1 | 44.41 | 322 | 1.8e-80 | 251 |
| NP_001100962.1 | 40.74 | 297 | 4.8e-46 | 162 |
| NP_001102424.1 | 32.46 | 268 | 3.0e-45 | 158 |
| NP_446384.1 | 34.47 | 264 | 1.0e-44 | 158 |
| XP_008773576.1 | 32.59 | 270 | 3.9e-42 | 151 |

### B3GNT6 (Q6ZMB0)

**Description:** Acetylgalactosaminyl-O-glycosyl-glycoprotein beta-1,3-N-acetylglucosaminyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001099681.1 | 70.64 | 378 | 0.0e+00 | 528 |
| NP_001099538.1 | 50.59 | 338 | 1.1e-110 | 329 |
| NP_001012134.1 | 40.62 | 352 | 5.5e-91 | 279 |
| NP_001100710.1 | 42.05 | 302 | 9.6e-84 | 260 |
| NP_001099408.1 | 42.43 | 337 | 3.8e-83 | 257 |
| XP_006222857.1 | 44.37 | 284 | 1.4e-77 | 241 |
| NP_446384.1 | 34.89 | 278 | 1.7e-51 | 176 |
| NP_001100962.1 | 38.49 | 317 | 1.2e-49 | 172 |
| NP_001102424.1 | 31.39 | 274 | 1.7e-47 | 164 |
| NP_001102962.1 | 29.79 | 339 | 1.2e-46 | 164 |

### B4GALNT2 (Q8NHY0)

**Description:** Beta-1,4 N-acetylgalactosaminyltransferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| XP_006220879.1 | 75.90 | 502 | 0.0e+00 | 786 |
| NP_074051.1 | 43.67 | 458 | 2.8e-119 | 363 |

### B4GALT1 (P15291)

**Description:** Beta-1,4-galactosyltransferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_445739.1 | 87.94 | 398 | 0.0e+00 | 711 |
| NP_001101435.1 | 63.10 | 290 | 1.0e-133 | 387 |
| NP_001009539.1 | 57.20 | 271 | 1.4e-102 | 309 |
| NP_001012018.1 | 51.28 | 273 | 2.2e-95 | 289 |
| NP_113928.1 | 42.25 | 284 | 7.7e-71 | 227 |
| NP_001026831.1 | 35.58 | 208 | 5.8e-34 | 129 |
| NP_001102078.1 | 40.62 | 64 | 4.5e-08 | 52 |

### B4GALT2 (O60909)

**Description:** Beta-1,4-galactosyltransferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001101435.1 | 91.40 | 372 | 0.0e+00 | 688 |
| NP_445739.1 | 52.99 | 385 | 1.5e-139 | 402 |
| NP_001012018.1 | 53.90 | 269 | 8.2e-99 | 297 |
| NP_001009539.1 | 54.48 | 279 | 5.1e-93 | 284 |
| NP_113928.1 | 38.84 | 327 | 8.2e-73 | 231 |
| NP_001026831.1 | 32.35 | 204 | 4.1e-27 | 109 |
| NP_001099738.1 | 28.36 | 134 | 3.7e-05 | 46 |
| NP_001102078.1 | 29.70 | 101 | 6.8e-05 | 42 |

### B4GALT3 (O60512)

**Description:** Beta-1,4-galactosyltransferase 3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001009539.1 | 96.20 | 395 | 0.0e+00 | 766 |
| NP_445739.1 | 53.40 | 309 | 4.2e-110 | 328 |
| NP_001012018.1 | 54.48 | 279 | 8.8e-105 | 313 |
| NP_001101435.1 | 54.83 | 290 | 2.1e-103 | 310 |
| NP_113928.1 | 42.55 | 282 | 5.8e-77 | 243 |
| NP_001026831.1 | 33.97 | 209 | 2.2e-29 | 116 |
| NP_001102078.1 | 35.82 | 67 | 1.3e-08 | 53 |

### B4GALT4 (O60513)

**Description:** Beta-1,4-galactosyltransferase 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001012018.1 | 84.59 | 318 | 0.0e+00 | 575 |
| NP_001009539.1 | 56.88 | 269 | 5.1e-99 | 298 |
| NP_001101435.1 | 50.56 | 269 | 3.8e-95 | 287 |
| NP_445739.1 | 48.21 | 280 | 3.5e-93 | 283 |
| NP_113928.1 | 43.07 | 267 | 1.3e-65 | 212 |
| NP_001026831.1 | 34.62 | 208 | 2.6e-29 | 115 |

### B4GALT5 (O43286)

**Description:** Beta-1,4-galactosyltransferase 5

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_113928.1 | 69.07 | 388 | 0.0e+00 | 584 |
| NP_001102078.1 | 88.55 | 131 | 6.9e-84 | 252 |
| NP_445739.1 | 45.23 | 283 | 1.7e-79 | 249 |
| NP_001101435.1 | 43.38 | 272 | 3.7e-74 | 235 |
| NP_001009539.1 | 44.03 | 268 | 1.5e-73 | 234 |
| NP_001012018.1 | 42.75 | 269 | 2.9e-69 | 221 |
| NP_001026831.1 | 34.48 | 203 | 3.1e-28 | 113 |

### B4GALT6 (Q9UBX8)

**Description:** Beta-1,4-galactosyltransferase 6

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_113928.1 | 94.24 | 382 | 0.0e+00 | 770 |
| NP_001009539.1 | 43.91 | 271 | 8.8e-73 | 232 |
| NP_001101435.1 | 40.97 | 310 | 1.6e-72 | 231 |
| NP_445739.1 | 41.20 | 284 | 1.8e-71 | 229 |
| NP_001012018.1 | 44.20 | 267 | 6.2e-68 | 218 |
| NP_001102078.1 | 53.52 | 142 | 3.8e-47 | 157 |
| NP_001026831.1 | 35.29 | 204 | 5.7e-29 | 115 |
| NP_113928.1 | 94.24 | 382 | 0.0e+00 | 770 |
| NP_001009539.1 | 43.91 | 271 | 8.8e-73 | 232 |
| NP_001101435.1 | 40.97 | 310 | 1.6e-72 | 231 |
| NP_445739.1 | 41.20 | 284 | 1.8e-71 | 229 |
| NP_001012018.1 | 44.20 | 267 | 6.2e-68 | 218 |
| NP_001102078.1 | 53.52 | 142 | 3.8e-47 | 157 |
| NP_001026831.1 | 35.29 | 204 | 5.7e-29 | 115 |

### B4GALT6 (Q9UBX8)

**Description:** Beta-1,4-galactosyltransferase 6

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_113928.1 | 94.24 | 382 | 0.0e+00 | 770 |
| NP_001009539.1 | 43.91 | 271 | 8.8e-73 | 232 |
| NP_001101435.1 | 40.97 | 310 | 1.6e-72 | 231 |
| NP_445739.1 | 41.20 | 284 | 1.8e-71 | 229 |
| NP_001012018.1 | 44.20 | 267 | 6.2e-68 | 218 |
| NP_001102078.1 | 53.52 | 142 | 3.8e-47 | 157 |
| NP_001026831.1 | 35.29 | 204 | 5.7e-29 | 115 |
| NP_113928.1 | 94.24 | 382 | 0.0e+00 | 770 |
| NP_001009539.1 | 43.91 | 271 | 8.8e-73 | 232 |
| NP_001101435.1 | 40.97 | 310 | 1.6e-72 | 231 |
| NP_445739.1 | 41.20 | 284 | 1.8e-71 | 229 |
| NP_001012018.1 | 44.20 | 267 | 6.2e-68 | 218 |
| NP_001102078.1 | 53.52 | 142 | 3.8e-47 | 157 |
| NP_001026831.1 | 35.29 | 204 | 5.7e-29 | 115 |

### BCL2L11 (O43521)

**Description:** Bcl-2-like protein 11

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_741985.1 | 85.00 | 200 | 3.3e-112 | 318 |
| NP_072134.1 | 55.00 | 200 | 6.5e-64 | 194 |
| NP_741986.1 | 66.67 | 72 | 3.6e-28 | 102 |
| NP_741986.1 | 88.37 | 43 | 9.9e-19 | 77 |

### BCL6 (P41182)

**Description:** B-cell lymphoma 6 protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001100554.1 | 95.05 | 707 | 0.0e+00 | 1372 |
| NP_001101749.1 | 86.09 | 151 | 1.8e-91 | 293 |
| NP_001101749.1 | 51.88 | 133 | 1.4e-32 | 132 |
| NP_001101749.1 | 40.74 | 81 | 1.4e-12 | 70 |
| XP_017460043.1 | 40.35 | 228 | 5.5e-44 | 164 |
| XP_017460043.1 | 42.10 | 114 | 2.4e-19 | 91 |
| XP_008757503.1 | 40.35 | 228 | 5.5e-44 | 164 |
| XP_008757503.1 | 42.10 | 114 | 2.4e-19 | 91 |
| XP_008756456.2 | 44.39 | 196 | 1.3e-43 | 161 |
| XP_008756456.2 | 42.35 | 196 | 1.5e-42 | 158 |
| XP_008756456.2 | 40.37 | 218 | 4.6e-42 | 157 |
| XP_008756456.2 | 48.45 | 161 | 1.3e-41 | 155 |
| XP_008756456.2 | 40.40 | 198 | 9.7e-37 | 142 |
| XP_008756456.2 | 50.37 | 137 | 1.0e-36 | 142 |
| XP_008756456.2 | 52.68 | 112 | 1.6e-30 | 124 |
| XP_008756456.2 | 38.46 | 143 | 5.3e-19 | 89 |
| XP_017458632.1 | 48.10 | 158 | 1.6e-43 | 162 |
| XP_017458632.1 | 45.34 | 161 | 1.7e-38 | 147 |
| XP_017458632.1 | 43.48 | 161 | 3.2e-36 | 141 |
| XP_017458632.1 | 42.77 | 159 | 1.1e-34 | 136 |
| XP_017458632.1 | 46.62 | 133 | 8.0e-32 | 128 |
| XP_017458632.1 | 43.48 | 115 | 1.7e-23 | 103 |
| XP_017458632.1 | 30.57 | 157 | 2.1e-06 | 50 |
| XP_017460172.1 | 48.43 | 159 | 2.4e-43 | 156 |
| XP_017460172.1 | 45.83 | 168 | 1.6e-40 | 148 |
| XP_017460172.1 | 44.78 | 163 | 2.2e-40 | 148 |
| XP_017460172.1 | 39.46 | 147 | 7.8e-28 | 112 |
| XP_017460172.1 | 37.42 | 155 | 5.9e-23 | 98 |
| XP_017460172.1 | 45.88 | 85 | 4.0e-18 | 84 |
| XP_017460171.1 | 41.28 | 218 | 3.4e-43 | 162 |
| XP_017460171.1 | 37.19 | 242 | 5.4e-41 | 156 |
| XP_017460171.1 | 40.54 | 222 | 2.3e-38 | 149 |
| XP_017460171.1 | 44.24 | 165 | 6.4e-37 | 144 |
| XP_017460171.1 | 37.44 | 211 | 1.3e-36 | 144 |
| XP_017460171.1 | 35.88 | 262 | 3.1e-36 | 142 |
| XP_017460171.1 | 43.67 | 158 | 2.1e-33 | 134 |
| XP_017460171.1 | 39.18 | 171 | 5.6e-31 | 127 |
| XP_017460171.1 | 35.36 | 181 | 2.4e-25 | 110 |
| XP_017460171.1 | 44.14 | 111 | 2.3e-21 | 98 |
| XP_017460171.1 | 41.18 | 85 | 1.3e-11 | 67 |
| XP_017443493.1 | 39.56 | 225 | 3.6e-43 | 159 |
| XP_017443493.1 | 44.44 | 162 | 1.7e-34 | 134 |
| XP_017443493.1 | 37.09 | 213 | 1.6e-33 | 131 |
| XP_017443493.1 | 44.83 | 145 | 4.5e-33 | 130 |
| XP_017443493.1 | 41.03 | 156 | 1.0e-28 | 117 |
| XP_017443493.1 | 41.67 | 84 | 1.3e-14 | 75 |
| XP_017458339.1 | 47.20 | 161 | 4.8e-43 | 154 |
| XP_017458339.1 | 45.34 | 161 | 9.5e-41 | 148 |
| XP_017458339.1 | 48.53 | 136 | 2.3e-38 | 141 |
| XP_017458339.1 | 38.12 | 181 | 2.3e-31 | 121 |

### C1GALT1 (Q9NS00)

**Description:** Glycoprotein-N-acetylgalactosamine 3-beta-galactosyltransferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_075239.1 | 89.81 | 363 | 0.0e+00 | 707 |
| NP_001025204.1 | 28.57 | 280 | 1.2e-28 | 113 |
| NP_001099738.1 | 29.55 | 176 | 8.9e-16 | 79 |
| XP_003753099.1 | 28.00 | 175 | 9.6e-14 | 72 |
| NP_001100044.1 | 27.17 | 173 | 2.6e-08 | 55 |
| XP_017460024.1 | 29.67 | 91 | 9.8e-07 | 50 |

### CAP1 (Q01518)

**Description:** Adenylyl cyclase-associated protein 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_071778.2 | 94.32 | 475 | 0.0e+00 | 867 |
| NP_446326.1 | 61.73 | 486 | 0.0e+00 | 572 |

### CAP2 (P40123)

**Description:** Adenylyl cyclase-associated protein 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_446326.1 | 87.24 | 478 | 0.0e+00 | 819 |
| NP_071778.2 | 62.21 | 479 | 0.0e+00 | 579 |

### CAV1 (Q03135)

**Description:** Caveolin-1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_113744.2 | 94.94 | 178 | 1.9e-128 | 357 |
| NP_598412.2 | 93.88 | 147 | 8.2e-103 | 291 |
| NP_062028.1 | 64.34 | 143 | 1.5e-72 | 215 |
| NP_571989.2 | 41.60 | 125 | 6.1e-30 | 107 |

### CCNG2 (Q16589)

**Description:** Cyclin-G2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001099195.1 | 90.09 | 343 | 0.0e+00 | 590 |
| NP_037055.1 | 54.09 | 281 | 2.8e-98 | 292 |
| NP_001099468.1 | 36.56 | 227 | 1.2e-31 | 122 |
| NP_001011949.1 | 32.14 | 112 | 4.2e-08 | 54 |
| XP_008772511.2 | 31.19 | 109 | 1.8e-07 | 52 |
| NP_001009470.1 | 31.19 | 109 | 2.3e-07 | 52 |
| NP_446154.3 | 28.08 | 146 | 3.8e-07 | 51 |
| NP_001102126.1 | 28.45 | 116 | 8.0e-07 | 50 |
| NP_071603.1 | 31.62 | 136 | 3.7e-06 | 48 |
| XP_017443667.1 | 26.73 | 202 | 1.0e-05 | 47 |

### CGA (P01215)

**Description:** Glycoprotein hormones alpha chain

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_446370.2 | 70.83 | 120 | 2.8e-60 | 181 |

### CHST10 (O43529)

**Description:** Carbohydrate sulfotransferase 10

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_536322.1 | 90.17 | 356 | 0.0e+00 | 665 |
| NP_001101549.1 | 32.34 | 303 | 2.2e-47 | 164 |
| NP_001100974.1 | 35.38 | 277 | 2.6e-42 | 152 |
| NP_001163941.1 | 35.34 | 283 | 2.4e-41 | 149 |
| NP_001178903.1 | 32.46 | 345 | 4.4e-39 | 142 |
| NP_001032864.1 | 32.67 | 303 | 1.5e-34 | 131 |
| NP_001103109.1 | 28.73 | 275 | 4.5e-29 | 115 |

### CHST8 (Q9H2A9)

**Description:** Carbohydrate sulfotransferase 8

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001100974.1 | 85.65 | 425 | 0.0e+00 | 690 |
| NP_001163941.1 | 49.77 | 430 | 1.0e-140 | 408 |
| NP_001101549.1 | 42.28 | 272 | 1.9e-68 | 221 |
| NP_001178903.1 | 40.00 | 270 | 1.9e-55 | 187 |
| NP_001032864.1 | 35.59 | 295 | 7.7e-49 | 171 |
| NP_001103109.1 | 36.36 | 286 | 2.9e-44 | 158 |
| NP_536322.1 | 35.25 | 278 | 7.1e-44 | 157 |

### CITED2 (Q99967)

**Description:** Cbp/p300-interacting transactivator 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_446150.1 | 95.56 | 270 | 0.0e+00 | 520 |
| NP_742052.2 | 41.91 | 136 | 6.2e-21 | 88 |
| NP_446151.1 | 50.00 | 54 | 3.0e-12 | 63 |

### CLASP1 (Q7Z460)

**Description:** CLIP-associating protein 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| XP_006221506.1 | 98.05 | 1539 | 0.0e+00 | 3095 |
| XP_008759594.1 | 97.04 | 1555 | 0.0e+00 | 3081 |
| XP_017460066.1 | 92.80 | 1626 | 0.0e+00 | 3044 |
| XP_017460067.1 | 92.74 | 1626 | 0.0e+00 | 3038 |
| XP_017460065.1 | 91.50 | 1648 | 0.0e+00 | 3026 |
| XP_017460070.1 | 92.42 | 1584 | 0.0e+00 | 2949 |
| XP_017460068.1 | 91.83 | 1592 | 0.0e+00 | 2939 |
| XP_017460069.1 | 91.77 | 1592 | 0.0e+00 | 2932 |
| XP_006221510.1 | 92.79 | 1539 | 0.0e+00 | 2883 |
| XP_001053715.3 | 92.31 | 1547 | 0.0e+00 | 2877 |

### CLASP2 (O75122)

**Description:** CLIP-associating protein 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_446174.1 | 95.00 | 1300 | 0.0e+00 | 2523 |
| XP_017460072.1 | 71.56 | 1294 | 0.0e+00 | 1764 |
| XP_017460071.1 | 71.12 | 1302 | 0.0e+00 | 1756 |
| XP_017460070.1 | 70.99 | 1303 | 0.0e+00 | 1752 |
| XP_017460073.1 | 71.05 | 1302 | 0.0e+00 | 1749 |
| XP_017460068.1 | 70.56 | 1311 | 0.0e+00 | 1746 |
| XP_017460069.1 | 70.48 | 1311 | 0.0e+00 | 1740 |
| XP_017460066.1 | 66.83 | 1384 | 0.0e+00 | 1705 |
| XP_017460067.1 | 66.76 | 1384 | 0.0e+00 | 1698 |
| XP_008759594.1 | 67.94 | 1338 | 0.0e+00 | 1696 |

### CREBBP (Q92793)

**Description:** CREB-binding protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_596872.3 | 95.75 | 2447 | 0.0e+00 | 4789 |
| XP_001076610.1 | 58.61 | 2532 | 0.0e+00 | 2408 |
| XP_006226228.1 | 58.33 | 2532 | 0.0e+00 | 2384 |
| XP_003754519.1 | 35.21 | 142 | 1.2e-16 | 87 |
| XP_017443520.1 | 46.75 | 77 | 1.5e-15 | 83 |
| XP_017443518.1 | 45.57 | 79 | 2.6e-15 | 82 |
| XP_017443518.1 | 37.63 | 93 | 3.1e-14 | 79 |
| XP_017443519.1 | 45.57 | 79 | 2.9e-15 | 82 |
| XP_017443519.1 | 37.63 | 93 | 3.7e-14 | 79 |
| NP_001094373.1 | 45.57 | 79 | 5.4e-15 | 82 |
| NP_001094373.1 | 29.05 | 148 | 1.1e-14 | 80 |
| NP_001102045.1 | 44.30 | 79 | 1.6e-14 | 80 |
| NP_001102045.1 | 32.17 | 115 | 8.2e-13 | 74 |
| XP_008774525.1 | 36.70 | 109 | 2.9e-14 | 79 |

### CYP1A2 (P05177)

**Description:** Cytochrome P450 1A2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_036673.3 | 75.10 | 514 | 0.0e+00 | 816 |
| NP_036672.2 | 68.25 | 507 | 0.0e+00 | 732 |
| NP_037072.1 | 40.24 | 497 | 1.4e-116 | 354 |
| NP_001019950.1 | 31.70 | 448 | 8.6e-80 | 259 |
| NP_036885.1 | 31.84 | 468 | 5.7e-77 | 251 |
| NP_075414.2 | 31.10 | 492 | 3.0e-75 | 246 |
| NP_775116.1 | 32.01 | 478 | 1.7e-74 | 244 |
| NP_786942.1 | 31.82 | 484 | 3.7e-74 | 243 |
| NP_612524.1 | 31.52 | 479 | 1.6e-72 | 239 |
| NP_001128452.1 | 29.27 | 492 | 2.2e-69 | 231 |

### DAD1 (P61803)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit DAD1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_620265.1 | 100.00 | 113 | 2.1e-78 | 226 |

### DDIT3 (P35638)

**Description:** DNA damage-inducible transcript 3 protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_077048.2 | 90.53 | 169 | 6.5e-106 | 300 |
| NP_001103456.1 | 90.53 | 169 | 6.5e-106 | 300 |
| NP_037286.1 | 33.33 | 54 | 4.2e-04 | 39 |

### DDOST (P39656)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase 48 kDa subunit

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001012104.1 | 95.45 | 440 | 0.0e+00 | 852 |

### DERL1 (Q9BUN8)

**Description:** Derlin-1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001014224.1 | 97.61 | 251 | 1.3e-180 | 496 |
| XP_006220770.1 | 35.83 | 240 | 1.2e-37 | 132 |
| NP_001103047.2 | 37.19 | 199 | 5.8e-35 | 125 |
| XP_008773284.1 | 34.48 | 174 | 1.4e-24 | 96 |

### DPAGT1 (Q9H3H5)

**Description:** UDP-N-acetylglucosamine--dolichyl-phosphate N-acetylglucosaminephosphotransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_955420.1 | 92.93 | 410 | 0.0e+00 | 741 |

### ENGASE (Q8NFI3)

**Description:** Cytosolic endo-beta-N-acetylglucosaminidase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001100540.1 | 78.17 | 742 | 0.0e+00 | 1178 |

### FASLG (P48023)

**Description:** Tumor necrosis factor ligand superfamily member 6

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_037040.1 | 74.38 | 281 | 2.6e-129 | 368 |
| NP_001178732.1 | 34.01 | 147 | 3.7e-20 | 87 |
| XP_017443496.1 | 33.55 | 155 | 4.5e-16 | 75 |
| NP_665708.1 | 35.08 | 134 | 2.2e-15 | 74 |
| NP_663714.2 | 26.92 | 208 | 6.0e-11 | 62 |
| NP_036807.1 | 31.51 | 146 | 1.9e-08 | 53 |
| NP_997672.1 | 26.35 | 148 | 5.1e-08 | 53 |
| NP_476490.1 | 25.81 | 155 | 2.8e-05 | 45 |
| NP_445805.1 | 26.06 | 165 | 3.6e-04 | 41 |

### FBXO32 (Q969P5)

**Description:** F-box only protein 32

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_598205.1 | 95.21 | 355 | 0.0e+00 | 701 |
| NP_001014261.1 | 60.72 | 359 | 1.9e-157 | 446 |

### FOXO3 (O43524)

**Description:** Forkhead box protein O3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001099865.1 | 94.50 | 673 | 0.0e+00 | 1137 |
| NP_001178775.1 | 47.76 | 536 | 9.4e-139 | 421 |
| XP_001053458.1 | 54.55 | 319 | 1.8e-87 | 284 |
| XP_001053458.1 | 49.27 | 69 | 5.1e-13 | 72 |
| XP_001053458.1 | 62.50 | 32 | 9.5e-06 | 48 |
| NP_001100413.1 | 46.90 | 371 | 5.3e-86 | 279 |
| NP_001100413.1 | 62.50 | 40 | 1.8e-09 | 60 |
| NP_001100413.1 | 84.00 | 25 | 2.6e-07 | 53 |
| NP_001101441.1 | 46.00 | 100 | 5.2e-19 | 91 |
| NP_036692.1 | 38.89 | 126 | 3.6e-17 | 85 |
| NP_001102822.1 | 46.00 | 100 | 6.4e-17 | 84 |
| XP_001056035.1 | 40.15 | 132 | 2.1e-15 | 78 |
| NP_001099246.1 | 34.46 | 148 | 2.4e-15 | 78 |
| XP_003754496.1 | 38.68 | 106 | 2.5e-15 | 78 |

### FOXO4 (P98177)

**Description:** Forkhead box protein O4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001100413.1 | 88.14 | 506 | 0.0e+00 | 845 |
| NP_001099865.1 | 42.36 | 491 | 4.6e-95 | 302 |
| NP_001099865.1 | 60.98 | 41 | 9.8e-08 | 55 |
| NP_001178775.1 | 42.36 | 491 | 2.1e-92 | 295 |
| NP_001178775.1 | 66.67 | 30 | 2.6e-06 | 50 |
| NP_036692.1 | 43.24 | 111 | 3.4e-19 | 90 |
| NP_001101441.1 | 43.14 | 102 | 4.9e-18 | 87 |
| NP_001099246.1 | 42.72 | 103 | 8.3e-17 | 82 |
| NP_001102819.1 | 33.33 | 192 | 8.5e-17 | 82 |
| XP_001056035.1 | 48.15 | 81 | 1.6e-16 | 80 |
| NP_001100545.1 | 48.75 | 80 | 5.6e-16 | 80 |
| NP_001095150.1 | 28.51 | 228 | 6.9e-16 | 80 |

### FUCA1 (P04066)

**Description:** Tissue alpha-L-fucosidase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_036694.2 | 82.45 | 433 | 0.0e+00 | 769 |
| NP_001004218.1 | 56.36 | 440 | 0.0e+00 | 518 |

### FUT3 (P21217)

**Description:** 3-galactosyl-N-acetylglucosaminide 4-alpha-L-fucosyltransferase FUT3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| XP_017459247.1 | 51.77 | 367 | 9.3e-110 | 327 |
| NP_955785.1 | 46.71 | 304 | 2.2e-82 | 255 |
| NP_445917.1 | 44.00 | 300 | 9.2e-80 | 248 |
| NP_071555.3 | 42.99 | 314 | 4.7e-71 | 228 |
| NP_001012360.1 | 31.70 | 224 | 4.8e-18 | 85 |
| NP_775430.2 | 33.62 | 229 | 7.6e-18 | 84 |

### FUT8 (Q9BYC5)

**Description:** Alpha-(1,6)-fucosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001002289.1 | 96.70 | 575 | 0.0e+00 | 1167 |
| NP_683690.2 | 38.81 | 67 | 9.3e-06 | 47 |

### G6PC1 (P35575)

**Description:** Glucose-6-phosphatase catalytic subunit 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_037230.2 | 86.83 | 357 | 0.0e+00 | 608 |
| NP_788266.2 | 37.24 | 341 | 1.5e-63 | 206 |
| XP_017456231.1 | 36.18 | 246 | 1.5e-37 | 135 |
| XP_017457961.1 | 51.16 | 43 | 9.8e-10 | 54 |

### GADD45A (P24522)

**Description:** Growth arrest and DNA damage-inducible protein GADD45 alpha

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_077041.2 | 92.73 | 165 | 4.3e-111 | 313 |
| NP_001008322.1 | 54.55 | 165 | 2.6e-59 | 181 |
| NP_001071108.1 | 53.89 | 167 | 1.4e-58 | 180 |
| XP_003753152.1 | 31.71 | 123 | 6.0e-07 | 46 |
| NP_113897.2 | 31.71 | 123 | 1.1e-06 | 45 |
| NP_001100247.1 | 33.72 | 86 | 2.7e-06 | 44 |

### GALNT1 (Q10472)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_077349.1 | 98.93 | 559 | 0.0e+00 | 1159 |
| NP_954537.1 | 84.17 | 556 | 0.0e+00 | 1003 |
| NP_570098.1 | 43.73 | 526 | 1.7e-144 | 430 |
| NP_001015032.2 | 47.48 | 516 | 2.2e-144 | 431 |
| NP_001020224.2 | 44.36 | 532 | 1.1e-143 | 427 |
| NP_001165534.1 | 46.43 | 532 | 1.2e-140 | 421 |
| NP_001129228.1 | 41.62 | 519 | 1.1e-139 | 417 |
| NP_955425.2 | 40.13 | 608 | 1.9e-138 | 414 |
| NP_001258248.1 | 40.13 | 608 | 1.9e-138 | 414 |
| XP_001066416.1 | 43.99 | 516 | 1.3e-137 | 411 |

### GALNT10 (Q86SR1)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 10

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_570098.1 | 96.02 | 603 | 0.0e+00 | 1219 |
| NP_001129228.1 | 70.60 | 602 | 0.0e+00 | 922 |
| NP_077349.1 | 43.24 | 525 | 2.4e-143 | 427 |
| NP_954537.1 | 43.52 | 517 | 7.0e-143 | 426 |
| NP_075215.1 | 41.41 | 524 | 5.7e-130 | 395 |
| NP_113984.1 | 42.69 | 520 | 4.2e-129 | 401 |
| NP_001020224.2 | 40.96 | 520 | 3.7e-127 | 386 |
| XP_001066416.1 | 41.27 | 504 | 7.5e-122 | 372 |
| NP_001094333.1 | 42.12 | 482 | 1.0e-119 | 366 |
| NP_001012109.1 | 39.28 | 499 | 2.5e-118 | 362 |

### GALNT11 (Q8NCW6)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 11

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_955425.2 | 88.65 | 608 | 0.0e+00 | 1150 |
| NP_001258248.1 | 88.65 | 608 | 0.0e+00 | 1150 |
| NP_001020319.1 | 56.92 | 383 | 1.5e-152 | 446 |
| NP_077349.1 | 44.63 | 531 | 2.0e-145 | 432 |
| NP_954537.1 | 46.05 | 506 | 3.8e-144 | 429 |
| NP_001094333.1 | 45.70 | 477 | 6.5e-133 | 400 |
| NP_001012109.1 | 42.05 | 516 | 7.9e-130 | 392 |
| NP_001020224.2 | 41.55 | 503 | 1.5e-123 | 377 |
| NP_001015032.2 | 41.34 | 508 | 4.7e-120 | 369 |
| NP_001129228.1 | 39.05 | 525 | 2.3e-119 | 367 |

### GALNT12 (Q8IXK2)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 12

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| XP_001066416.1 | 84.95 | 545 | 0.0e+00 | 980 |
| NP_001020224.2 | 57.87 | 553 | 0.0e+00 | 635 |
| NP_001165534.1 | 45.27 | 539 | 7.3e-150 | 445 |
| NP_001015032.2 | 45.25 | 537 | 1.3e-146 | 437 |
| NP_954537.1 | 46.20 | 513 | 1.3e-144 | 429 |
| NP_077349.1 | 43.74 | 519 | 9.2e-139 | 414 |
| XP_003752912.1 | 43.18 | 484 | 5.0e-121 | 372 |
| NP_570098.1 | 41.18 | 493 | 7.8e-120 | 367 |
| NP_001129228.1 | 39.56 | 498 | 5.5e-115 | 354 |
| NP_001012109.1 | 47.95 | 365 | 1.8e-110 | 341 |

### GALNT13 (Q8IUC8)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 13

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_954537.1 | 99.28 | 556 | 0.0e+00 | 1151 |
| NP_077349.1 | 84.35 | 556 | 0.0e+00 | 1008 |
| NP_001165534.1 | 48.02 | 529 | 1.1e-145 | 433 |
| NP_001020224.2 | 46.03 | 504 | 2.2e-145 | 431 |
| NP_001015032.2 | 47.59 | 519 | 6.3e-144 | 429 |
| NP_570098.1 | 43.71 | 517 | 1.5e-142 | 424 |
| XP_001066416.1 | 45.29 | 510 | 1.1e-139 | 416 |
| NP_001129228.1 | 41.38 | 522 | 2.2e-139 | 416 |
| NP_955425.2 | 44.07 | 506 | 2.7e-136 | 409 |
| NP_001258248.1 | 44.07 | 506 | 2.7e-136 | 409 |

### GALNT14 (Q96FL9)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 14

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001012109.1 | 92.21 | 552 | 0.0e+00 | 1081 |
| NP_001094333.1 | 61.33 | 468 | 0.0e+00 | 619 |
| NP_001099666.1 | 47.79 | 498 | 7.4e-166 | 479 |
| NP_954537.1 | 44.42 | 457 | 1.6e-131 | 394 |
| NP_077349.1 | 43.00 | 486 | 6.2e-131 | 393 |
| NP_955425.2 | 45.42 | 469 | 3.8e-130 | 393 |
| NP_001258248.1 | 45.42 | 469 | 3.8e-130 | 393 |
| NP_570098.1 | 40.64 | 497 | 4.0e-119 | 364 |
| NP_001165534.1 | 42.18 | 505 | 7.5e-117 | 359 |
| NP_001020224.2 | 45.43 | 416 | 7.8e-115 | 352 |

### GALNT16 (Q8N428)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 16

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001094333.1 | 96.06 | 558 | 0.0e+00 | 1127 |
| NP_001012109.1 | 52.61 | 574 | 0.0e+00 | 622 |
| NP_001099666.1 | 51.31 | 495 | 2.5e-171 | 493 |
| NP_955425.2 | 44.16 | 505 | 5.1e-133 | 400 |
| NP_001258248.1 | 44.16 | 505 | 5.1e-133 | 400 |
| NP_954537.1 | 42.59 | 486 | 4.1e-125 | 378 |
| NP_001129228.1 | 40.86 | 487 | 5.0e-124 | 377 |
| NP_570098.1 | 41.45 | 497 | 9.9e-122 | 371 |
| NP_077349.1 | 40.58 | 483 | 4.9e-121 | 368 |
| NP_001165534.1 | 40.54 | 518 | 7.4e-116 | 357 |

### GALNT18 (Q6P9A2)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 18

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001073353.1 | 98.19 | 607 | 0.0e+00 | 1249 |
| NP_001116116.1 | 49.92 | 609 | 0.0e+00 | 610 |
| NP_001020283.1 | 57.26 | 503 | 0.0e+00 | 606 |
| NP_077349.1 | 35.53 | 501 | 2.6e-101 | 318 |
| NP_954537.1 | 36.33 | 512 | 2.3e-100 | 316 |
| NP_955425.2 | 30.56 | 638 | 4.6e-93 | 298 |
| NP_001258248.1 | 30.56 | 638 | 4.6e-93 | 298 |
| XP_001066416.1 | 36.61 | 508 | 2.7e-89 | 288 |
| NP_001165534.1 | 36.19 | 478 | 2.4e-86 | 281 |
| NP_001015032.2 | 38.41 | 427 | 6.5e-86 | 280 |

### GALNT2 (Q10471)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001099666.1 | 91.86 | 504 | 0.0e+00 | 957 |
| NP_001094333.1 | 49.46 | 558 | 0.0e+00 | 544 |
| NP_001012109.1 | 52.21 | 498 | 0.0e+00 | 536 |
| NP_077349.1 | 46.96 | 477 | 1.7e-153 | 452 |
| NP_954537.1 | 46.85 | 476 | 7.1e-153 | 450 |
| NP_001165534.1 | 45.36 | 496 | 3.2e-132 | 399 |
| NP_955425.2 | 42.66 | 497 | 6.5e-123 | 375 |
| NP_001258248.1 | 42.66 | 497 | 6.5e-123 | 375 |
| XP_001066416.1 | 42.17 | 498 | 1.3e-122 | 373 |
| NP_001020224.2 | 38.70 | 571 | 4.0e-121 | 369 |

### GALNT3 (Q14435)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001015032.2 | 91.94 | 633 | 0.0e+00 | 1243 |
| NP_001165534.1 | 65.27 | 619 | 0.0e+00 | 872 |
| NP_001020224.2 | 47.92 | 530 | 1.2e-161 | 476 |
| XP_001066416.1 | 45.27 | 539 | 2.4e-148 | 441 |
| NP_954537.1 | 46.55 | 537 | 5.8e-141 | 422 |
| NP_077349.1 | 45.01 | 531 | 6.2e-140 | 419 |
| NP_955425.2 | 40.47 | 509 | 3.1e-116 | 360 |
| NP_001258248.1 | 40.47 | 509 | 3.1e-116 | 360 |
| XP_003752912.1 | 40.81 | 468 | 1.7e-113 | 355 |
| NP_001094333.1 | 43.29 | 499 | 2.4e-113 | 350 |

### GALNT4 (Q8N4A0)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001020224.2 | 91.18 | 578 | 0.0e+00 | 1104 |
| XP_001066416.1 | 59.81 | 520 | 0.0e+00 | 630 |
| NP_001165534.1 | 48.87 | 532 | 4.9e-163 | 479 |
| NP_001015032.2 | 47.31 | 539 | 4.2e-159 | 469 |
| NP_954537.1 | 47.92 | 482 | 2.2e-146 | 434 |
| NP_077349.1 | 45.47 | 508 | 4.0e-143 | 425 |
| NP_570098.1 | 41.54 | 520 | 5.6e-128 | 388 |
| XP_003752912.1 | 43.18 | 484 | 1.0e-125 | 384 |
| NP_955425.2 | 40.07 | 569 | 1.3e-121 | 372 |
| NP_001258248.1 | 40.07 | 569 | 1.3e-121 | 372 |

### GALNT9 (Q9HCQ5)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 9

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001116116.1 | 91.21 | 603 | 0.0e+00 | 1142 |
| NP_001020283.1 | 61.02 | 608 | 0.0e+00 | 777 |
| NP_001073353.1 | 49.76 | 619 | 0.0e+00 | 610 |
| NP_001100621.1 | 96.19 | 236 | 5.2e-171 | 485 |
| NP_954537.1 | 39.27 | 522 | 2.1e-115 | 355 |
| NP_077349.1 | 36.33 | 523 | 7.5e-111 | 343 |
| NP_001020224.2 | 39.76 | 498 | 6.0e-102 | 321 |
| NP_955425.2 | 36.75 | 517 | 6.7e-102 | 322 |
| NP_001258248.1 | 36.75 | 517 | 6.7e-102 | 322 |
| NP_001165534.1 | 37.82 | 513 | 4.3e-99 | 315 |

### GALNTL5 (Q7Z4T8)

**Description:** Inactive polypeptide N-acetylgalactosaminyltransferase-like protein 5

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001020319.1 | 65.45 | 437 | 0.0e+00 | 592 |
| NP_955425.2 | 52.28 | 461 | 1.5e-160 | 466 |
| NP_001258248.1 | 52.28 | 461 | 1.5e-160 | 466 |
| NP_954537.1 | 45.79 | 380 | 8.1e-110 | 335 |
| NP_077349.1 | 42.08 | 423 | 1.1e-108 | 332 |
| NP_001020224.2 | 46.80 | 344 | 3.6e-102 | 316 |
| NP_001012109.1 | 44.67 | 347 | 6.7e-98 | 304 |
| XP_003752912.1 | 42.69 | 342 | 3.9e-97 | 305 |
| NP_001094333.1 | 44.54 | 348 | 4.3e-97 | 302 |
| NP_001129228.1 | 44.61 | 334 | 8.7e-96 | 300 |

### GALNTL6 (Q49A17)

**Description:** Polypeptide N-acetylgalactosaminyltransferase-like 6

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001129228.1 | 95.51 | 601 | 0.0e+00 | 1212 |
| NP_570098.1 | 69.98 | 613 | 0.0e+00 | 905 |
| NP_954537.1 | 41.38 | 522 | 1.5e-138 | 414 |
| NP_077349.1 | 41.23 | 519 | 1.6e-138 | 414 |
| NP_075215.1 | 41.22 | 524 | 2.3e-128 | 392 |
| NP_113984.1 | 41.00 | 522 | 4.1e-123 | 385 |
| NP_001094333.1 | 42.09 | 468 | 6.4e-121 | 369 |
| NP_001020224.2 | 40.57 | 493 | 1.1e-120 | 369 |
| NP_955425.2 | 37.90 | 533 | 3.6e-120 | 369 |
| NP_001258248.1 | 37.90 | 533 | 3.6e-120 | 369 |

### GANAB (Q14697)

**Description:** Neutral alpha-glucosidase AB

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001099804.1 | 91.29 | 735 | 0.0e+00 | 1427 |
| NP_001139312.1 | 50.38 | 913 | 0.0e+00 | 946 |
| NP_954549.1 | 32.62 | 745 | 5.7e-98 | 329 |
| XP_017458271.1 | 31.44 | 633 | 2.1e-77 | 277 |
| XP_017458271.1 | 29.05 | 444 | 1.5e-48 | 188 |
| XP_017458272.1 | 31.44 | 633 | 3.7e-77 | 277 |
| XP_017458272.1 | 28.68 | 659 | 2.3e-71 | 259 |
| XP_017458272.1 | 29.34 | 651 | 9.4e-70 | 254 |
| XP_017458272.1 | 29.62 | 628 | 4.7e-69 | 252 |
| NP_037193.1 | 30.11 | 621 | 3.3e-68 | 249 |
| NP_037193.1 | 26.84 | 585 | 3.1e-48 | 187 |
| XP_008773944.1 | 30.43 | 575 | 9.7e-68 | 248 |
| XP_008773944.1 | 27.79 | 716 | 9.1e-62 | 229 |
| NP_001102441.1 | 24.23 | 425 | 6.3e-15 | 79 |

### GCK (P35557)

**Description:** Hexokinase-4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_036697.1 | 94.84 | 465 | 0.0e+00 | 891 |
| NP_001257779.1 | 93.33 | 465 | 0.0e+00 | 891 |
| NP_001257778.1 | 96.50 | 400 | 0.0e+00 | 803 |
| NP_036867.1 | 55.03 | 447 | 0.0e+00 | 532 |
| NP_036867.1 | 56.34 | 442 | 7.7e-178 | 522 |
| NP_036866.1 | 52.56 | 449 | 3.8e-174 | 512 |
| NP_036866.1 | 50.11 | 447 | 5.2e-162 | 481 |
| NP_071515.1 | 52.10 | 453 | 8.4e-165 | 488 |
| NP_071515.1 | 42.54 | 449 | 2.0e-127 | 392 |
| XP_017443500.1 | 50.34 | 445 | 7.6e-154 | 459 |
| XP_017443500.1 | 45.74 | 446 | 4.9e-131 | 400 |

### GFPT1 (Q06210)

**Description:** Glutamine--fructose-6-phosphate aminotransferase [isomerizing] 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001005879.1 | 96.42 | 699 | 0.0e+00 | 1388 |
| NP_001002819.2 | 77.32 | 701 | 0.0e+00 | 1142 |
| NP_476546.1 | 24.87 | 189 | 1.1e-09 | 61 |

### GFPT2 (O94808)

**Description:** Glutamine--fructose-6-phosphate aminotransferase [isomerizing] 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001002819.2 | 97.65 | 682 | 0.0e+00 | 1390 |
| NP_001005879.1 | 78.98 | 685 | 0.0e+00 | 1146 |
| NP_476546.1 | 27.01 | 211 | 3.7e-10 | 63 |

### GNPNAT1 (Q96EK6)

**Description:** Glucosamine 6-phosphate N-acetyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001128229.1 | 97.83 | 184 | 6.0e-135 | 374 |
| NP_001128228.1 | 97.83 | 184 | 6.0e-135 | 374 |

### HDAC2 (Q92769)

**Description:** Histone deacetylase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_445899.1 | 98.97 | 488 | 0.0e+00 | 1013 |
| NP_001020580.1 | 90.29 | 443 | 0.0e+00 | 813 |
| NP_445898.1 | 88.09 | 445 | 0.0e+00 | 803 |
| NP_445900.1 | 62.31 | 390 | 0.0e+00 | 551 |
| NP_001119845.2 | 43.70 | 341 | 1.9e-107 | 324 |
| NP_445901.1 | 28.73 | 275 | 9.1e-25 | 108 |
| XP_006226311.1 | 26.10 | 318 | 1.4e-24 | 108 |
| XP_006226310.1 | 26.10 | 318 | 1.4e-24 | 108 |
| XP_006226308.1 | 26.10 | 318 | 2.0e-24 | 107 |
| XP_006226309.1 | 26.37 | 292 | 2.3e-24 | 107 |

### INS (P01308)

**Description:** Insulin

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_062003.1 | 82.73 | 110 | 3.1e-57 | 172 |
| NP_062002.1 | 82.73 | 110 | 9.8e-57 | 171 |
| NP_001177092.1 | 30.68 | 88 | 3.2e-07 | 46 |
| NP_001177091.1 | 30.68 | 88 | 3.2e-07 | 46 |
| NP_113699.2 | 30.68 | 88 | 3.2e-07 | 46 |
| NP_849197.3 | 35.37 | 82 | 1.1e-06 | 44 |
| NP_001075946.2 | 36.25 | 80 | 1.5e-06 | 44 |
| NP_001075948.1 | 35.37 | 82 | 2.1e-06 | 43 |
| NP_001075947.1 | 35.37 | 82 | 2.7e-06 | 43 |

### KLF4 (O43474)

**Description:** Krueppel-like factor 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_446165.1 | 84.42 | 520 | 0.0e+00 | 745 |
| NP_001007685.1 | 84.43 | 122 | 4.0e-67 | 220 |
| NP_001100634.1 | 79.80 | 99 | 2.9e-50 | 176 |
| NP_445846.2 | 73.47 | 98 | 2.6e-45 | 165 |
| XP_017443682.1 | 69.79 | 96 | 9.9e-45 | 161 |
| XP_017443681.1 | 69.79 | 96 | 9.9e-45 | 161 |
| XP_017443680.1 | 69.79 | 96 | 9.9e-45 | 161 |
| XP_017443679.1 | 69.79 | 96 | 9.9e-45 | 161 |
| XP_017443678.1 | 69.79 | 96 | 9.9e-45 | 161 |
| XP_017443677.1 | 69.79 | 96 | 9.9e-45 | 161 |

### LARGE1 (O95461)

**Description:** Xylosyl- and glucuronyltransferase LARGE1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_954538.1 | 66.16 | 656 | 0.0e+00 | 927 |
| NP_001101909.1 | 97.09 | 378 | 0.0e+00 | 774 |
| NP_001099794.1 | 28.96 | 328 | 9.5e-30 | 122 |
| XP_002724719.3 | 23.68 | 266 | 3.3e-12 | 69 |
| NP_001094357.1 | 25.00 | 244 | 9.2e-12 | 68 |
| XP_003753976.1 | 21.37 | 262 | 2.4e-08 | 57 |

### LHB (P01229)

**Description:** Lutropin subunit beta

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001029147.1 | 71.22 | 139 | 1.6e-63 | 190 |
| NP_036990.1 | 71.63 | 141 | 3.3e-62 | 187 |
| NP_037248.2 | 40.95 | 127 | 5.4e-27 | 98 |
| NP_001007598.1 | 39.83 | 118 | 3.0e-25 | 93 |
| NP_001007014.1 | 33.72 | 86 | 3.2e-10 | 54 |

### MAGT1 (Q9H0U3)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit MAGT1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_446398.1 | 92.81 | 334 | 0.0e+00 | 647 |
| NP_001004212.1 | 71.75 | 308 | 7.1e-167 | 468 |

### MAN2A1 (Q16706)

**Description:** Alpha-mannosidase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_037111.2 | 79.03 | 1149 | 0.0e+00 | 1896 |
| NP_001100997.1 | 54.28 | 1168 | 0.0e+00 | 1284 |
| NP_955436.1 | 26.57 | 1035 | 5.9e-86 | 301 |
| NP_001128443.1 | 26.46 | 1058 | 1.4e-79 | 282 |
| NP_640349.1 | 26.20 | 187 | 4.0e-06 | 51 |

### MAN2A2 (P49641)

**Description:** Alpha-mannosidase 2x

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001100997.1 | 89.09 | 1164 | 0.0e+00 | 2157 |
| NP_037111.2 | 53.25 | 1155 | 0.0e+00 | 1261 |
| NP_955436.1 | 26.81 | 1048 | 4.1e-81 | 287 |
| NP_001128443.1 | 27.36 | 837 | 4.8e-68 | 248 |
| NP_640349.1 | 20.11 | 189 | 2.7e-05 | 48 |

### MANEA (Q5SRI9)

**Description:** Glycoprotein endo-alpha-1,2-mannosidase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_542963.2 | 85.50 | 462 | 0.0e+00 | 839 |
| XP_001056614.1 | 56.96 | 467 | 0.0e+00 | 550 |
| XP_017458577.1 | 68.55 | 213 | 1.7e-110 | 326 |

### MGAT1 (P26572)

**Description:** Alpha-1,3-mannosyl-glycoprotein 2-beta-N-acetylglucosaminyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_110488.1 | 91.05 | 447 | 0.0e+00 | 830 |
| NP_001007748.1 | 34.12 | 340 | 2.3e-36 | 141 |

### MGAT2 (Q10469)

**Description:** Alpha-1,6-mannosyl-glycoprotein 2-beta-N-acetylglucosaminyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_446056.1 | 89.04 | 447 | 0.0e+00 | 837 |

### MGAT3 (Q09327)

**Description:** Beta-1,4-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_062112.2 | 92.18 | 537 | 0.0e+00 | 1021 |

### MGAT4A (Q9UM21)

**Description:** Alpha-1,3-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase A

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001153627.1 | 94.58 | 535 | 0.0e+00 | 1057 |
| NP_001012225.1 | 93.46 | 535 | 0.0e+00 | 1039 |
| NP_001121005.1 | 59.96 | 547 | 0.0e+00 | 658 |
| NP_001020837.1 | 42.60 | 277 | 2.6e-74 | 240 |
| NP_001129286.1 | 33.42 | 398 | 3.5e-69 | 230 |
| XP_017460107.1 | 24.71 | 340 | 1.1e-29 | 121 |
| XP_008760512.1 | 24.71 | 340 | 1.1e-29 | 121 |
| XP_008760506.1 | 24.71 | 340 | 1.1e-29 | 121 |
| XP_006221537.1 | 24.71 | 340 | 1.1e-29 | 121 |
| XP_017460108.1 | 25.00 | 300 | 2.0e-27 | 113 |

### MGAT4B (Q9UQ53)

**Description:** Alpha-1,3-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase B

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001121005.1 | 98.54 | 548 | 0.0e+00 | 1119 |
| NP_001012225.1 | 60.84 | 549 | 0.0e+00 | 669 |
| NP_001153627.1 | 60.29 | 549 | 0.0e+00 | 667 |
| NP_001020837.1 | 46.62 | 266 | 1.7e-76 | 246 |
| NP_001129286.1 | 31.59 | 402 | 4.8e-62 | 211 |
| XP_017460107.1 | 26.41 | 318 | 1.8e-30 | 124 |
| XP_008760512.1 | 26.41 | 318 | 1.8e-30 | 124 |
| XP_008760506.1 | 26.41 | 318 | 1.8e-30 | 124 |
| XP_006221537.1 | 26.41 | 318 | 1.8e-30 | 124 |
| XP_017460108.1 | 25.80 | 283 | 4.6e-27 | 112 |

### MGAT4C (Q9UBM8)

**Description:** Alpha-1,3-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase C

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001129286.1 | 95.82 | 478 | 0.0e+00 | 960 |
| XP_017460107.1 | 31.77 | 406 | 1.5e-69 | 228 |
| XP_008760512.1 | 31.77 | 406 | 1.5e-69 | 228 |
| XP_008760506.1 | 31.77 | 406 | 1.5e-69 | 228 |
| XP_006221537.1 | 31.77 | 406 | 1.5e-69 | 228 |
| NP_001012225.1 | 32.75 | 397 | 2.0e-68 | 228 |
| NP_001153627.1 | 32.75 | 397 | 2.3e-68 | 228 |
| NP_001013894.2 | 30.70 | 469 | 8.8e-63 | 211 |
| NP_001121005.1 | 30.85 | 402 | 5.4e-61 | 208 |
| XP_017460108.1 | 32.16 | 342 | 3.0e-58 | 196 |

### MGAT5 (Q09328)

**Description:** Alpha-1,6-mannosylglycoprotein 6-beta-N-acetylglucosaminyltransferase A

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_075583.1 | 97.17 | 741 | 0.0e+00 | 1511 |
| NP_001100538.1 | 48.63 | 619 | 0.0e+00 | 610 |
| XP_017460077.1 | 96.30 | 135 | 1.0e-83 | 262 |
| XP_017460076.1 | 96.30 | 135 | 1.0e-83 | 262 |
| XP_017460075.1 | 96.30 | 135 | 1.0e-83 | 262 |
| NP_001102702.1 | 64.42 | 104 | 9.8e-36 | 132 |

### MLEC (Q14165)

**Description:** Malectin

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001014005.1 | 98.79 | 248 | 3.2e-176 | 488 |

### MOGS (Q13724)

**Description:** Mannosyl-oligosaccharide glucosidase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_113937.1 | 85.49 | 813 | 0.0e+00 | 1341 |

### MPDU1 (O75352)

**Description:** Mannose-P-dolichol utilization defect 1 protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001100481.1 | 88.66 | 247 | 2.9e-162 | 449 |
| NP_001030124.1 | 26.45 | 189 | 3.4e-08 | 52 |

### MUC1 (P15941)

**Description:** Mucin-1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_036734.1 | 46.34 | 628 | 2.4e-122 | 394 |
| XP_006226319.3 | 34.45 | 714 | 5.3e-28 | 123 |
| XP_006226319.3 | 34.36 | 713 | 3.3e-27 | 121 |
| XP_006226319.3 | 34.94 | 561 | 7.4e-23 | 107 |
| XP_008774874.1 | 23.83 | 193 | 3.4e-06 | 52 |

### MUC12 (Q9UKN1)

**Description:** Mucin-12

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| XP_008759070.1 | 37.15 | 358 | 6.7e-61 | 216 |
| XP_008758464.2 | 55.77 | 156 | 1.3e-55 | 216 |
| XP_017459958.1 | 53.30 | 182 | 4.7e-54 | 189 |
| XP_017459957.1 | 28.99 | 307 | 8.3e-16 | 82 |
| NP_620788.2 | 25.00 | 264 | 3.2e-06 | 54 |

### MUC15 (Q8N387)

**Description:** Mucin-15

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001037766.1 | 60.18 | 334 | 1.2e-120 | 350 |

### MUC17 (Q685J3)

**Description:** Mucin-17

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| XP_008759070.1 | 52.49 | 362 | 2.3e-128 | 410 |
| XP_008758464.2 | 42.10 | 152 | 1.2e-35 | 150 |
| XP_017459958.1 | 39.46 | 185 | 3.0e-21 | 94 |
| XP_017459957.1 | 31.50 | 200 | 8.2e-21 | 97 |
| NP_620788.2 | 25.00 | 268 | 8.1e-11 | 69 |

### MUC19 (Q7Z5P9)

**Description:** Mucin-19

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001296418.1 | 75.47 | 1227 | 0.0e+00 | 2346 |
| NP_001296418.1 | 66.79 | 265 | 1.1e-130 | 467 |
| XP_017459733.1 | 36.42 | 1200 | 0.0e+00 | 763 |
| XP_017459733.1 | 25.93 | 401 | 5.8e-20 | 100 |
| XP_017459733.1 | 26.58 | 316 | 1.2e-15 | 86 |
| XP_017459733.1 | 22.86 | 363 | 2.5e-08 | 62 |
| XP_017459733.1 | 21.56 | 320 | 6.7e-08 | 60 |
| XP_006223636.1 | 36.86 | 1210 | 0.0e+00 | 756 |
| XP_006223636.1 | 25.66 | 343 | 1.8e-15 | 85 |
| XP_006223636.1 | 25.58 | 387 | 1.1e-14 | 83 |
| XP_006223636.1 | 25.71 | 385 | 1.3e-11 | 73 |
| XP_006223636.1 | 29.55 | 132 | 2.1e-04 | 49 |
| XP_006223763.2 | 32.93 | 1230 | 0.0e+00 | 696 |
| XP_006223763.2 | 27.72 | 505 | 5.3e-46 | 186 |
| XP_006223763.2 | 24.53 | 424 | 1.6e-24 | 115 |
| XP_006223763.2 | 36.36 | 55 | 2.3e-05 | 52 |
| XP_001063331.3 | 36.24 | 1192 | 0.0e+00 | 690 |
| XP_001063331.3 | 29.32 | 382 | 6.7e-19 | 97 |
| XP_001063331.3 | 27.59 | 406 | 2.2e-16 | 89 |
| XP_001063331.3 | 27.95 | 229 | 2.2e-09 | 65 |
| XP_001063331.3 | 28.09 | 178 | 9.4e-05 | 50 |
| XP_017458829.1 | 27.74 | 1139 | 2.5e-138 | 488 |
| XP_017458829.1 | 28.48 | 488 | 2.0e-52 | 207 |
| XP_017458829.1 | 22.13 | 497 | 1.2e-07 | 60 |
| XP_017458829.1 | 24.27 | 206 | 2.0e-06 | 55 |
| XP_017458829.1 | 20.20 | 510 | 2.3e-05 | 52 |
| XP_017458829.1 | 43.86 | 57 | 2.7e-05 | 52 |
| NP_001099732.1 | 28.93 | 1151 | 4.1e-114 | 411 |
| NP_001099732.1 | 29.81 | 463 | 1.0e-40 | 169 |
| NP_001099732.1 | 27.39 | 387 | 2.0e-18 | 95 |
| NP_001099732.1 | 30.85 | 94 | 5.1e-05 | 51 |
| NP_001007017.1 | 26.24 | 1292 | 2.4e-86 | 320 |
| NP_001007017.1 | 36.45 | 107 | 1.4e-04 | 49 |
| NP_001007017.1 | 34.48 | 87 | 2.0e-04 | 49 |
| NP_001007017.1 | 35.00 | 100 | 6.7e-04 | 47 |
| NP_001158128.1 | 23.43 | 1255 | 7.9e-56 | 219 |
| NP_001158128.1 | 23.30 | 1249 | 3.3e-39 | 164 |
| NP_001158128.1 | 24.90 | 755 | 9.5e-28 | 126 |
| NP_001158128.1 | 26.52 | 411 | 2.1e-24 | 115 |
| NP_001158128.1 | 25.94 | 478 | 6.1e-24 | 113 |
| NP_001158128.1 | 26.80 | 388 | 2.6e-22 | 108 |
| NP_001158128.1 | 23.32 | 746 | 2.9e-22 | 108 |
| NP_001158128.1 | 24.48 | 478 | 5.2e-17 | 90 |
| NP_001158128.1 | 25.09 | 570 | 6.5e-15 | 84 |
| NP_001100284.1 | 23.34 | 1264 | 3.9e-49 | 196 |
| NP_001100284.1 | 23.36 | 745 | 2.5e-31 | 138 |
| NP_001100284.1 | 23.38 | 663 | 2.0e-23 | 112 |
| NP_001100284.1 | 25.53 | 427 | 3.1e-19 | 98 |
| NP_001100284.1 | 25.14 | 545 | 2.1e-17 | 92 |
| NP_001100284.1 | 23.03 | 356 | 1.7e-16 | 89 |

### MUC20 (Q8N307)

**Description:** Mucin-20

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001100559.1 | 46.30 | 270 | 2.8e-41 | 155 |
| NP_001100559.1 | 59.26 | 81 | 1.9e-10 | 63 |
| NP_001100559.1 | 29.05 | 148 | 6.3e-05 | 46 |

### MUC21 (Q5SSG8)

**Description:** Mucin-21

No significant BLAST hits found for this protein.

### MUC3A (Q02505)

**Description:** Mucin-3A

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| XP_017459957.1 | 64.19 | 296 | 1.8e-123 | 393 |
| XP_008759070.1 | 36.02 | 397 | 7.4e-58 | 206 |
| XP_017459958.1 | 31.82 | 88 | 2.4e-08 | 57 |
| NP_445997.1 | 47.27 | 55 | 1.3e-06 | 55 |
| NP_620788.2 | 26.34 | 224 | 1.7e-06 | 54 |
| NP_001164029.1 | 36.99 | 73 | 3.1e-05 | 50 |
| NP_446115.1 | 36.99 | 73 | 1.7e-04 | 48 |
| NP_001164031.1 | 36.99 | 73 | 4.9e-04 | 46 |
| XP_017459251.1 | 30.11 | 93 | 6.3e-04 | 45 |

### MUC4 (Q99102)

**Description:** Mucin-4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| XP_006221229.1 | 65.25 | 1154 | 0.0e+00 | 1693 |
| NP_001099851.1 | 30.50 | 341 | 1.0e-31 | 137 |
| XP_017459281.1 | 28.08 | 292 | 7.0e-15 | 80 |
| XP_017459281.1 | 27.49 | 291 | 1.1e-14 | 79 |
| XP_017459281.1 | 28.11 | 306 | 1.2e-14 | 79 |
| XP_017459281.1 | 28.28 | 290 | 1.7e-14 | 79 |
| XP_017459281.1 | 27.95 | 297 | 2.9e-14 | 78 |
| XP_017459281.1 | 27.89 | 294 | 5.2e-14 | 77 |
| XP_017459281.1 | 28.18 | 291 | 5.2e-14 | 77 |
| XP_017459281.1 | 28.18 | 291 | 5.3e-14 | 77 |
| XP_017459281.1 | 27.49 | 291 | 7.7e-14 | 77 |
| XP_017459281.1 | 28.08 | 292 | 7.8e-14 | 77 |
| XP_017459281.1 | 27.84 | 291 | 9.2e-14 | 76 |
| XP_017459281.1 | 28.18 | 291 | 9.7e-14 | 76 |
| XP_017459281.1 | 27.78 | 306 | 1.1e-13 | 76 |
| XP_017459281.1 | 27.55 | 294 | 1.2e-13 | 76 |
| XP_017459281.1 | 27.70 | 296 | 1.4e-13 | 76 |
| XP_017459281.1 | 27.89 | 294 | 1.7e-13 | 75 |
| XP_017459281.1 | 27.49 | 291 | 1.8e-13 | 75 |
| XP_017459281.1 | 27.93 | 290 | 1.8e-13 | 75 |
| XP_017459281.1 | 27.27 | 297 | 2.2e-13 | 75 |
| XP_017459281.1 | 28.28 | 290 | 2.3e-13 | 75 |
| XP_017459281.1 | 27.66 | 282 | 2.3e-13 | 75 |
| XP_017459281.1 | 28.26 | 276 | 2.4e-13 | 75 |
| XP_017459281.1 | 27.64 | 275 | 2.9e-13 | 75 |
| XP_017459281.1 | 27.59 | 290 | 4.0e-13 | 74 |
| XP_017459281.1 | 27.70 | 296 | 4.7e-13 | 74 |
| XP_017459281.1 | 28.28 | 290 | 5.0e-13 | 74 |
| XP_017459281.1 | 26.58 | 301 | 5.2e-13 | 74 |
| XP_017459281.1 | 28.37 | 282 | 7.4e-13 | 73 |
| XP_017459281.1 | 27.40 | 292 | 9.5e-13 | 73 |
| XP_017459281.1 | 27.27 | 297 | 1.1e-12 | 73 |
| XP_017459281.1 | 27.59 | 290 | 1.4e-12 | 73 |
| XP_017459281.1 | 27.49 | 291 | 1.6e-12 | 72 |
| XP_017459281.1 | 28.28 | 290 | 1.8e-12 | 72 |
| XP_017459281.1 | 27.15 | 291 | 2.0e-12 | 72 |
| XP_017459281.1 | 27.84 | 291 | 2.0e-12 | 72 |
| XP_017459281.1 | 25.91 | 301 | 2.6e-12 | 72 |
| XP_017459281.1 | 27.24 | 301 | 2.8e-12 | 72 |
| XP_017459281.1 | 26.95 | 282 | 3.1e-12 | 72 |
| XP_017459281.1 | 26.90 | 290 | 4.7e-12 | 71 |
| XP_017459281.1 | 27.49 | 291 | 6.1e-12 | 70 |
| XP_017459281.1 | 27.84 | 291 | 6.4e-12 | 70 |
| XP_017459281.1 | 27.40 | 292 | 7.3e-12 | 70 |
| XP_017459281.1 | 27.12 | 295 | 7.9e-12 | 70 |
| XP_017459281.1 | 27.30 | 282 | 8.3e-12 | 70 |
| XP_017459281.1 | 27.21 | 294 | 8.7e-12 | 70 |
| XP_017459281.1 | 27.15 | 291 | 8.7e-12 | 70 |
| XP_017459281.1 | 27.46 | 295 | 9.2e-12 | 70 |
| XP_017459281.1 | 26.90 | 290 | 9.7e-12 | 70 |
| XP_017459281.1 | 27.55 | 294 | 1.0e-11 | 70 |
| XP_017459281.1 | 27.66 | 282 | 1.1e-11 | 70 |
| XP_017459281.1 | 27.72 | 285 | 1.1e-11 | 70 |
| XP_017459281.1 | 27.30 | 282 | 1.1e-11 | 70 |
| XP_017459281.1 | 27.27 | 297 | 1.3e-11 | 70 |
| XP_017459281.1 | 27.49 | 291 | 1.7e-11 | 69 |
| XP_017459281.1 | 27.15 | 291 | 2.0e-11 | 69 |
| XP_017459281.1 | 26.80 | 291 | 2.2e-11 | 69 |
| XP_017459281.1 | 27.59 | 290 | 3.4e-11 | 68 |
| XP_017459281.1 | 25.86 | 290 | 3.8e-11 | 68 |
| XP_017459281.1 | 26.80 | 306 | 3.9e-11 | 68 |
| XP_017459281.1 | 26.55 | 290 | 4.3e-11 | 68 |
| XP_017459281.1 | 26.87 | 294 | 5.0e-11 | 68 |
| XP_017459281.1 | 26.09 | 299 | 8.3e-11 | 67 |
| XP_017459281.1 | 26.69 | 296 | 9.5e-11 | 67 |
| XP_017459281.1 | 26.46 | 291 | 1.1e-10 | 67 |
| XP_017459281.1 | 26.19 | 294 | 1.2e-10 | 67 |
| XP_017459281.1 | 26.80 | 291 | 1.2e-10 | 67 |
| XP_017459281.1 | 26.46 | 291 | 1.2e-10 | 67 |
| XP_017459281.1 | 26.47 | 306 | 1.3e-10 | 67 |
| XP_017459281.1 | 26.00 | 300 | 1.5e-10 | 66 |
| XP_017459281.1 | 27.15 | 291 | 1.6e-10 | 66 |
| XP_017459281.1 | 25.93 | 297 | 1.9e-10 | 66 |
| XP_017459281.1 | 27.15 | 291 | 1.9e-10 | 66 |
| XP_017459281.1 | 26.55 | 290 | 2.3e-10 | 66 |
| XP_017459281.1 | 26.26 | 297 | 2.3e-10 | 66 |
| XP_017459281.1 | 27.24 | 290 | 2.4e-10 | 66 |
| XP_017459281.1 | 26.55 | 290 | 2.5e-10 | 66 |
| XP_017459281.1 | 26.37 | 292 | 3.1e-10 | 65 |
| XP_017459281.1 | 26.00 | 300 | 3.5e-10 | 65 |
| XP_017459281.1 | 27.15 | 291 | 3.7e-10 | 65 |
| XP_017459281.1 | 26.55 | 290 | 3.7e-10 | 65 |
| XP_017459281.1 | 26.12 | 291 | 3.8e-10 | 65 |
| XP_017459281.1 | 26.46 | 291 | 4.0e-10 | 65 |
| XP_017459281.1 | 27.07 | 266 | 4.7e-10 | 65 |
| XP_017459281.1 | 27.24 | 290 | 4.8e-10 | 65 |
| XP_017459281.1 | 26.12 | 291 | 4.9e-10 | 65 |
| XP_017459281.1 | 26.03 | 292 | 5.1e-10 | 65 |
| XP_017459281.1 | 26.12 | 291 | 5.5e-10 | 65 |
| XP_017459281.1 | 26.90 | 290 | 5.6e-10 | 65 |
| XP_017459281.1 | 26.46 | 291 | 5.7e-10 | 65 |
| XP_017459281.1 | 26.96 | 293 | 6.0e-10 | 65 |
| XP_017459281.1 | 26.80 | 291 | 6.3e-10 | 64 |
| XP_017459281.1 | 26.46 | 291 | 7.1e-10 | 64 |
| XP_017459281.1 | 26.90 | 290 | 7.1e-10 | 64 |
| XP_017459281.1 | 27.24 | 290 | 7.2e-10 | 64 |
| XP_017459281.1 | 26.90 | 290 | 9.3e-10 | 64 |
| XP_017459281.1 | 26.46 | 291 | 9.7e-10 | 64 |
| XP_017459281.1 | 26.21 | 290 | 1.0e-09 | 64 |
| XP_017459281.1 | 26.12 | 291 | 1.0e-09 | 64 |
| XP_017459281.1 | 27.05 | 292 | 1.0e-09 | 64 |
| XP_017459281.1 | 26.24 | 282 | 1.1e-09 | 64 |
| XP_017459281.1 | 26.21 | 290 | 1.1e-09 | 63 |
| XP_017459281.1 | 26.46 | 291 | 1.4e-09 | 63 |
| XP_017459281.1 | 27.30 | 282 | 1.4e-09 | 63 |
| XP_017459281.1 | 26.69 | 296 | 1.4e-09 | 63 |
| XP_017459281.1 | 25.76 | 295 | 1.5e-09 | 63 |
| XP_017459281.1 | 26.80 | 291 | 1.7e-09 | 63 |
| XP_017459281.1 | 26.17 | 298 | 1.8e-09 | 63 |
| XP_017459281.1 | 27.27 | 275 | 2.0e-09 | 63 |
| XP_017459281.1 | 26.78 | 295 | 2.0e-09 | 63 |
| XP_017459281.1 | 26.71 | 292 | 2.1e-09 | 63 |
| XP_017459281.1 | 26.26 | 297 | 2.2e-09 | 63 |
| XP_017459281.1 | 26.44 | 295 | 2.3e-09 | 63 |
| XP_017459281.1 | 26.80 | 291 | 2.3e-09 | 63 |
| XP_017459281.1 | 26.46 | 291 | 2.3e-09 | 63 |
| XP_017459281.1 | 26.80 | 291 | 2.8e-09 | 62 |
| XP_017459281.1 | 26.19 | 294 | 3.0e-09 | 62 |
| XP_017459281.1 | 26.55 | 275 | 3.1e-09 | 62 |
| XP_017459281.1 | 26.80 | 291 | 3.2e-09 | 62 |
| XP_017459281.1 | 26.12 | 291 | 3.8e-09 | 62 |
| XP_017459281.1 | 26.12 | 291 | 3.9e-09 | 62 |
| XP_017459281.1 | 27.05 | 292 | 4.3e-09 | 62 |
| XP_017459281.1 | 26.19 | 294 | 4.3e-09 | 62 |
| XP_017459281.1 | 26.95 | 282 | 4.4e-09 | 62 |
| XP_017459281.1 | 26.80 | 291 | 4.7e-09 | 62 |
| XP_017459281.1 | 25.77 | 291 | 4.9e-09 | 62 |
| XP_017459281.1 | 26.46 | 291 | 5.2e-09 | 62 |
| XP_017459281.1 | 25.77 | 291 | 5.2e-09 | 62 |
| XP_017459281.1 | 26.18 | 275 | 6.4e-09 | 61 |
| XP_017459281.1 | 26.80 | 291 | 6.6e-09 | 61 |
| XP_017459281.1 | 26.55 | 290 | 6.8e-09 | 61 |
| XP_017459281.1 | 26.62 | 293 | 7.1e-09 | 61 |
| XP_017459281.1 | 26.53 | 294 | 7.1e-09 | 61 |
| XP_017459281.1 | 26.48 | 287 | 7.4e-09 | 61 |
| XP_017459281.1 | 27.05 | 292 | 7.6e-09 | 61 |
| XP_017459281.1 | 26.46 | 291 | 7.7e-09 | 61 |
| XP_017459281.1 | 26.09 | 276 | 8.0e-09 | 61 |
| XP_017459281.1 | 25.77 | 291 | 1.2e-08 | 60 |
| XP_017459281.1 | 26.46 | 291 | 1.6e-08 | 60 |
| XP_017459281.1 | 26.77 | 269 | 1.6e-08 | 60 |
| XP_017459281.1 | 25.33 | 300 | 1.6e-08 | 60 |
| XP_017459281.1 | 25.50 | 298 | 2.1e-08 | 60 |
| XP_017459281.1 | 25.95 | 289 | 2.2e-08 | 60 |
| XP_017459281.1 | 25.86 | 290 | 2.4e-08 | 60 |
| XP_017459281.1 | 25.68 | 296 | 2.7e-08 | 59 |
| XP_017459281.1 | 26.28 | 293 | 2.8e-08 | 59 |
| XP_017459281.1 | 25.68 | 292 | 3.3e-08 | 59 |
| XP_017459281.1 | 26.12 | 291 | 4.0e-08 | 59 |
| XP_017459281.1 | 26.90 | 290 | 4.5e-08 | 58 |
| XP_017459281.1 | 25.86 | 290 | 5.1e-08 | 58 |
| XP_017459281.1 | 25.77 | 291 | 5.7e-08 | 58 |
| XP_017459281.1 | 25.43 | 291 | 6.8e-08 | 58 |
| XP_017459281.1 | 25.59 | 297 | 7.1e-08 | 58 |
| XP_017459281.1 | 25.95 | 262 | 7.2e-08 | 58 |
| XP_017459281.1 | 26.12 | 268 | 8.9e-08 | 58 |
| XP_017459281.1 | 25.00 | 296 | 1.0e-07 | 58 |
| XP_017459281.1 | 25.00 | 284 | 1.1e-07 | 57 |
| XP_017459281.1 | 25.09 | 291 | 1.4e-07 | 57 |
| XP_017459281.1 | 25.43 | 291 | 1.5e-07 | 57 |
| XP_017459281.1 | 25.73 | 276 | 1.8e-07 | 57 |
| XP_017459281.1 | 25.53 | 282 | 1.8e-07 | 57 |
| XP_017459281.1 | 25.25 | 297 | 2.3e-07 | 57 |
| XP_017459281.1 | 26.45 | 276 | 2.4e-07 | 57 |
| XP_017459281.1 | 25.68 | 292 | 3.1e-07 | 56 |
| XP_017459281.1 | 25.86 | 290 | 3.4e-07 | 56 |
| XP_017459281.1 | 25.52 | 290 | 4.4e-07 | 55 |
| XP_017459281.1 | 25.00 | 292 | 5.5e-07 | 55 |
| XP_017459281.1 | 25.52 | 290 | 6.3e-07 | 55 |
| XP_017459281.1 | 25.25 | 297 | 1.8e-06 | 53 |
| XP_017459281.1 | 25.17 | 290 | 3.1e-06 | 53 |
| XP_017459281.1 | 25.17 | 290 | 3.4e-06 | 53 |
| XP_017459281.1 | 24.74 | 291 | 1.4e-05 | 51 |
| XP_017459281.1 | 25.51 | 294 | 1.9e-05 | 50 |
| XP_017459281.1 | 25.17 | 290 | 2.4e-05 | 50 |
| NP_598199.2 | 20.59 | 340 | 5.6e-10 | 66 |
| NP_598199.2 | 23.43 | 303 | 1.6e-09 | 65 |
| NP_598199.2 | 21.99 | 332 | 1.8e-09 | 64 |
| NP_598199.2 | 21.78 | 303 | 3.6e-09 | 63 |
| NP_598199.2 | 21.48 | 391 | 1.0e-08 | 62 |
| NP_598199.2 | 21.49 | 242 | 1.2e-08 | 62 |
| NP_598199.2 | 22.12 | 226 | 1.8e-08 | 61 |
| NP_598199.2 | 21.18 | 321 | 2.1e-08 | 61 |
| NP_598199.2 | 21.82 | 252 | 3.2e-08 | 60 |
| NP_598199.2 | 22.63 | 380 | 4.8e-08 | 60 |
| NP_598199.2 | 21.18 | 321 | 5.0e-08 | 60 |
| NP_598199.2 | 20.51 | 356 | 5.4e-08 | 60 |
| NP_598199.2 | 21.27 | 315 | 5.7e-08 | 59 |
| NP_598199.2 | 20.84 | 355 | 2.0e-07 | 58 |
| NP_598199.2 | 22.77 | 202 | 2.5e-07 | 57 |
| NP_598199.2 | 22.19 | 347 | 4.1e-07 | 57 |
| NP_598199.2 | 20.59 | 340 | 1.0e-06 | 55 |
| NP_598199.2 | 20.79 | 279 | 2.2e-06 | 54 |
| NP_598199.2 | 20.31 | 261 | 2.6e-06 | 54 |
| NP_598199.2 | 21.24 | 226 | 7.3e-06 | 52 |
| NP_598199.2 | 20.29 | 340 | 9.1e-06 | 52 |
| NP_598199.2 | 20.80 | 226 | 1.5e-05 | 52 |
| NP_598199.2 | 20.48 | 249 | 1.6e-05 | 52 |
| NP_598199.2 | 21.64 | 416 | 1.9e-05 | 51 |
| NP_598199.2 | 20.31 | 320 | 2.3e-05 | 51 |
| NP_598199.2 | 21.24 | 306 | 2.8e-05 | 50 |
| NP_598199.2 | 19.77 | 344 | 1.2e-04 | 48 |
| NP_598199.2 | 19.91 | 226 | 1.9e-04 | 48 |
| NP_598199.2 | 20.08 | 259 | 2.9e-04 | 47 |
| NP_598199.2 | 20.19 | 208 | 3.5e-04 | 47 |
| NP_598199.2 | 21.36 | 337 | 6.2e-04 | 46 |
| XP_017460029.1 | 23.23 | 198 | 1.8e-08 | 62 |
| NP_001161314.1 | 26.67 | 180 | 8.2e-07 | 56 |
| XP_008772277.1 | 28.35 | 254 | 8.5e-07 | 56 |
| NP_001158129.2 | 27.37 | 190 | 2.7e-06 | 54 |
| NP_001158129.2 | 24.63 | 203 | 9.4e-05 | 49 |
| XP_017443728.1 | 20.11 | 348 | 1.6e-05 | 51 |
| XP_017443728.1 | 19.83 | 358 | 1.8e-05 | 51 |
| XP_017443728.1 | 19.44 | 360 | 8.6e-05 | 49 |
| XP_017443728.1 | 18.99 | 358 | 1.6e-04 | 48 |
| XP_017443728.1 | 19.26 | 353 | 5.3e-04 | 46 |
| XP_017443728.1 | 20.43 | 279 | 8.8e-04 | 46 |
| NP_001100284.1 | 31.21 | 157 | 2.0e-05 | 52 |

### NAGK (Q9UJ70)

**Description:** N-acetyl-D-glucosamine kinase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001032857.1 | 92.42 | 343 | 0.0e+00 | 659 |

### NFYA (P23511)

**Description:** Nuclear transcription factor Y subunit alpha

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_036997.1 | 98.27 | 347 | 0.0e+00 | 647 |

### NFYC (Q13952)

**Description:** Nuclear transcription factor Y subunit gamma

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_036998.1 | 93.35 | 316 | 1.2e-170 | 483 |
| NP_036998.1 | 97.50 | 40 | 1.1e-16 | 81 |
| NP_001102104.1 | 40.79 | 76 | 9.7e-14 | 67 |
| NP_001128352.1 | 36.59 | 82 | 4.5e-09 | 54 |
| NP_001071136.1 | 35.53 | 76 | 9.0e-09 | 55 |

### NGLY1 (Q96IV0)

**Description:** Peptide-N(4)-(N-acetyl-beta-glucosaminyl)asparagine amidase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001014158.1 | 83.62 | 586 | 0.0e+00 | 1010 |

### NR3C1 (P04150)

**Description:** Glucocorticoid receptor

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_036708.2 | 88.33 | 797 | 0.0e+00 | 1408 |
| NP_037263.1 | 49.71 | 517 | 2.5e-152 | 469 |
| NP_074038.1 | 56.45 | 372 | 6.3e-144 | 446 |
| NP_036634.1 | 52.73 | 366 | 6.7e-127 | 401 |
| NP_976081.1 | 37.36 | 273 | 2.4e-51 | 185 |
| NP_001008516.2 | 39.15 | 258 | 7.2e-49 | 178 |
| NP_001008511.2 | 37.36 | 265 | 2.3e-46 | 171 |
| NP_036821.1 | 31.91 | 351 | 4.6e-46 | 174 |
| NP_036886.1 | 32.67 | 352 | 8.4e-44 | 165 |
| NP_036937.2 | 28.57 | 322 | 7.3e-28 | 118 |

### PCBP4 (P57723)

**Description:** Poly(rC)-binding protein 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001178812.1 | 97.77 | 403 | 0.0e+00 | 785 |
| NP_001011945.2 | 64.33 | 300 | 5.2e-137 | 395 |
| NP_001013241.1 | 53.61 | 360 | 3.0e-119 | 351 |
| NP_001013241.1 | 23.88 | 289 | 3.8e-09 | 58 |
| XP_008774006.1 | 52.99 | 334 | 7.6e-107 | 318 |
| XP_008774006.1 | 32.29 | 96 | 2.3e-04 | 43 |
| XP_017459237.1 | 62.24 | 98 | 1.1e-36 | 129 |
| XP_017459237.1 | 33.90 | 59 | 8.1e-05 | 42 |
| NP_476482.1 | 32.09 | 187 | 2.9e-22 | 98 |
| NP_476482.1 | 48.28 | 58 | 8.2e-09 | 57 |
| NP_476482.1 | 36.92 | 65 | 1.8e-06 | 50 |
| NP_598286.1 | 25.59 | 297 | 1.9e-13 | 72 |
| NP_598286.1 | 27.27 | 154 | 4.3e-08 | 55 |
| NP_598286.1 | 33.33 | 78 | 4.1e-07 | 52 |
| XP_017458283.1 | 30.27 | 185 | 4.3e-13 | 70 |
| XP_017458283.1 | 23.81 | 294 | 4.2e-09 | 58 |
| XP_017458283.1 | 26.88 | 160 | 7.2e-08 | 54 |
| NP_001034426.1 | 24.79 | 355 | 5.7e-13 | 70 |
| NP_001034426.1 | 28.75 | 160 | 5.5e-11 | 64 |
| NP_001034426.1 | 36.00 | 75 | 4.8e-08 | 55 |
| XP_006223372.1 | 31.98 | 172 | 1.1e-12 | 69 |
| XP_006223372.1 | 24.70 | 247 | 3.1e-10 | 62 |
| XP_006223372.1 | 41.54 | 65 | 1.8e-08 | 56 |
| XP_006223372.1 | 35.09 | 57 | 9.5e-05 | 44 |

### PCK1 (P35558)

**Description:** Phosphoenolpyruvate carboxykinase, cytosolic [GTP]

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_942075.1 | 90.84 | 622 | 0.0e+00 | 1189 |
| NP_001101847.2 | 71.24 | 612 | 0.0e+00 | 944 |

### PGM3 (O95394)

**Description:** Phosphoacetylglucosamine mutase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001102242.1 | 85.71 | 518 | 0.0e+00 | 929 |

### PINK1 (Q9BXM7)

**Description:** Serine/threonine-protein kinase PINK1, mitochondrial

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001100164.1 | 78.42 | 241 | 1.3e-120 | 357 |
| XP_006223167.1 | 31.35 | 185 | 9.9e-11 | 64 |
| XP_008772662.1 | 31.35 | 185 | 1.1e-10 | 64 |
| NP_604463.1 | 29.52 | 166 | 1.1e-10 | 63 |
| XP_006223168.1 | 31.35 | 185 | 1.1e-10 | 64 |
| XP_006223166.1 | 31.35 | 185 | 1.2e-10 | 64 |
| XP_006223170.1 | 31.35 | 185 | 1.2e-10 | 64 |
| XP_008772663.1 | 31.35 | 185 | 1.3e-10 | 64 |
| XP_017443541.1 | 31.35 | 185 | 1.3e-10 | 64 |
| XP_006223174.1 | 31.35 | 185 | 1.3e-10 | 64 |

### PLXNA4 (Q9HCM2)

**Description:** Plexin-A4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| XP_017458260.1 | 97.62 | 1894 | 0.0e+00 | 3870 |
| NP_001099458.2 | 68.03 | 1861 | 0.0e+00 | 2647 |
| XP_001072622.3 | 64.08 | 1879 | 0.0e+00 | 2465 |
| NP_001101051.2 | 60.66 | 1848 | 0.0e+00 | 2345 |
| NP_001129350.1 | 33.06 | 1966 | 0.0e+00 | 952 |
| NP_001101576.1 | 32.94 | 1931 | 0.0e+00 | 934 |
| NP_001101351.1 | 34.30 | 1647 | 0.0e+00 | 838 |
| XP_006226635.1 | 39.02 | 1248 | 0.0e+00 | 825 |
| XP_006226635.1 | 27.65 | 669 | 3.7e-65 | 246 |
| XP_006226634.1 | 39.02 | 1248 | 0.0e+00 | 825 |
| XP_006226634.1 | 27.65 | 669 | 3.7e-65 | 246 |
| XP_006226633.1 | 39.02 | 1248 | 0.0e+00 | 825 |
| XP_006226633.1 | 27.65 | 669 | 3.7e-65 | 246 |

### POFUT2 (Q9Y2G5)

**Description:** GDP-fucose protein O-fucosyltransferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001101091.2 | 91.84 | 429 | 0.0e+00 | 830 |

### POMGNT1 (Q8WZA1)

**Description:** Protein O-linked-mannose beta-1,2-N-acetylglucosaminyltransferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001007748.1 | 98.64 | 660 | 0.0e+00 | 1352 |
| NP_110488.1 | 38.70 | 261 | 8.1e-39 | 149 |
| XP_001063314.1 | 43.48 | 92 | 3.5e-12 | 70 |
| XP_017443795.1 | 43.48 | 92 | 3.6e-12 | 70 |
| XP_008772823.1 | 43.48 | 92 | 3.6e-12 | 70 |
| NP_942066.1 | 27.40 | 146 | 6.9e-12 | 65 |
| NP_001099497.1 | 34.31 | 102 | 9.2e-11 | 62 |
| NP_001100572.1 | 27.59 | 116 | 1.1e-10 | 62 |
| NP_001102794.1 | 26.67 | 90 | 2.0e-07 | 52 |
| NP_001101066.1 | 35.23 | 88 | 3.8e-06 | 50 |

### POMT1 (Q9Y6A1)

**Description:** Protein O-mannosyl-transferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_445858.1 | 83.00 | 747 | 0.0e+00 | 1272 |
| NP_001040579.1 | 32.57 | 746 | 3.7e-98 | 322 |
| NP_001102903.1 | 29.70 | 165 | 3.2e-07 | 52 |
| NP_001099273.2 | 27.38 | 168 | 1.0e-04 | 44 |

### POMT2 (Q9UKY4)

**Description:** Protein O-mannosyl-transferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001040579.1 | 89.73 | 750 | 0.0e+00 | 1353 |
| NP_445858.1 | 33.93 | 722 | 1.4e-100 | 326 |
| NP_001102903.1 | 29.68 | 155 | 1.4e-08 | 56 |
| NP_001099273.2 | 31.61 | 155 | 1.7e-08 | 55 |

### PPARGC1A (Q9UBK2)

**Description:** Peroxisome proliferator-activated receptor gamma coactivator 1-alpha

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_112637.1 | 95.11 | 798 | 0.0e+00 | 1561 |
| NP_001099833.1 | 61.03 | 136 | 1.2e-50 | 193 |
| NP_788264.1 | 47.06 | 136 | 2.3e-29 | 125 |
| NP_788264.1 | 35.87 | 223 | 5.1e-20 | 95 |
| NP_788264.1 | 33.67 | 196 | 5.7e-10 | 63 |
| NP_001020592.1 | 32.81 | 64 | 7.6e-04 | 43 |

### PRKCSH (P14314)

**Description:** Glucosidase 2 subunit beta

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001100276.1 | 85.33 | 518 | 0.0e+00 | 836 |
| NP_001093963.1 | 30.48 | 105 | 4.2e-08 | 55 |
| NP_110454.1 | 38.96 | 77 | 3.6e-05 | 47 |
| NP_110454.1 | 43.42 | 76 | 3.5e-04 | 43 |
| NP_110454.1 | 36.59 | 82 | 1.0e-03 | 42 |
| NP_001101313.1 | 36.36 | 77 | 3.6e-04 | 43 |

### PSMC1 (P62191)

**Description:** 26S proteasome regulatory subunit 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_476464.1 | 100.00 | 440 | 0.0e+00 | 891 |
| NP_476463.1 | 50.68 | 369 | 3.3e-137 | 400 |
| NP_112411.1 | 52.16 | 324 | 2.0e-117 | 349 |
| NP_113783.1 | 45.12 | 379 | 3.3e-116 | 347 |
| NP_150239.1 | 47.02 | 336 | 1.4e-104 | 317 |
| NP_001093979.1 | 43.77 | 345 | 9.0e-101 | 306 |
| NP_446316.1 | 46.38 | 235 | 6.0e-62 | 214 |
| NP_446316.1 | 35.53 | 304 | 1.3e-55 | 197 |
| NP_001101926.3 | 43.15 | 248 | 2.6e-58 | 204 |
| NP_001128336.1 | 42.34 | 248 | 9.7e-57 | 200 |
| NP_001128351.1 | 41.47 | 258 | 1.9e-54 | 196 |

### RAD23B (P54727)

**Description:** UV excision repair protein RAD23 homolog B

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001020446.1 | 89.66 | 416 | 0.0e+00 | 633 |
| NP_001013208.1 | 56.06 | 371 | 7.2e-107 | 319 |
| NP_620233.1 | 39.06 | 64 | 1.3e-07 | 48 |
| NP_113875.1 | 35.29 | 68 | 1.4e-07 | 50 |
| XP_006221493.1 | 38.23 | 68 | 2.5e-07 | 48 |
| XP_003752767.1 | 36.77 | 68 | 3.5e-07 | 47 |
| XP_003754135.1 | 37.50 | 64 | 1.6e-06 | 45 |
| XP_002724552.1 | 37.50 | 64 | 1.7e-06 | 45 |
| NP_446061.2 | 30.97 | 113 | 1.8e-06 | 50 |
| NP_001029140.1 | 30.97 | 113 | 1.9e-06 | 50 |

### RBL2 (Q08999)

**Description:** Retinoblastoma-like protein 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_112356.1 | 90.39 | 1114 | 0.0e+00 | 2064 |
| NP_001177995.1 | 49.05 | 1105 | 0.0e+00 | 1040 |
| NP_058741.1 | 25.08 | 594 | 1.9e-44 | 175 |
| NP_058741.1 | 44.04 | 109 | 1.8e-19 | 95 |

### RENBP (P51606)

**Description:** N-acylglucosamine 2-epimerase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_112357.1 | 85.68 | 405 | 0.0e+00 | 739 |

### RETN (Q9HD89)

**Description:** Resistin

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001019452.1 | 47.83 | 92 | 1.1e-25 | 92 |
| NP_853656.1 | 46.08 | 102 | 6.7e-25 | 90 |
| NP_653342.1 | 54.44 | 90 | 1.0e-23 | 87 |
| NP_445785.1 | 36.45 | 107 | 5.1e-16 | 68 |

### RFT1 (Q96AA3)

**Description:** Man(5)GlcNAc(2)-PP-dolichol translocation protein RFT1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001129338.1 | 87.25 | 541 | 0.0e+00 | 943 |

### ROBO1 (Q9Y6N7)

**Description:** Roundabout homolog 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_071524.1 | 94.51 | 1640 | 0.0e+00 | 2996 |
| NP_115289.1 | 52.73 | 1339 | 0.0e+00 | 1280 |
| NP_115289.1 | 29.29 | 198 | 4.6e-05 | 48 |
| NP_001101605.1 | 50.00 | 936 | 0.0e+00 | 908 |
| NP_001101605.1 | 26.61 | 218 | 2.1e-13 | 76 |
| XP_008774970.1 | 27.71 | 848 | 1.3e-58 | 223 |
| XP_008774970.1 | 27.95 | 762 | 4.0e-53 | 206 |
| XP_008774970.1 | 22.10 | 353 | 1.7e-04 | 47 |
| NP_001101611.1 | 26.58 | 839 | 2.0e-57 | 220 |
| NP_001101611.1 | 29.54 | 501 | 3.5e-33 | 141 |
| NP_001101611.1 | 27.29 | 480 | 8.2e-31 | 133 |
| NP_001101611.1 | 23.63 | 842 | 2.9e-24 | 112 |
| NP_001101611.1 | 22.25 | 863 | 9.9e-24 | 110 |
| NP_001101611.1 | 29.01 | 324 | 1.0e-21 | 103 |
| NP_001101611.1 | 24.84 | 310 | 3.7e-11 | 68 |
| NP_001101611.1 | 41.89 | 74 | 6.5e-06 | 51 |
| NP_598271.1 | 27.38 | 840 | 5.1e-54 | 209 |
| NP_598271.1 | 26.49 | 487 | 8.1e-27 | 120 |
| NP_598271.1 | 22.81 | 846 | 2.7e-25 | 115 |
| NP_598271.1 | 26.62 | 462 | 1.1e-22 | 106 |
| NP_598271.1 | 24.15 | 410 | 2.1e-21 | 102 |
| NP_598271.1 | 24.78 | 335 | 8.8e-14 | 77 |
| NP_598271.1 | 41.89 | 74 | 4.6e-07 | 55 |
| NP_446331.1 | 29.03 | 775 | 4.3e-52 | 201 |
| NP_446331.1 | 29.83 | 533 | 2.0e-49 | 192 |
| NP_446331.1 | 30.84 | 415 | 2.3e-36 | 150 |
| NP_001258190.1 | 35.50 | 400 | 8.7e-52 | 201 |
| NP_001258190.1 | 33.75 | 400 | 1.4e-48 | 191 |
| NP_001258190.1 | 32.90 | 304 | 4.5e-33 | 140 |
| XP_008773573.1 | 30.69 | 479 | 1.3e-51 | 202 |
| XP_008773573.1 | 30.12 | 488 | 7.9e-51 | 199 |
| XP_008773573.1 | 31.51 | 511 | 6.2e-48 | 190 |
| XP_008773573.1 | 30.26 | 532 | 8.7e-48 | 189 |
| XP_008773573.1 | 29.98 | 477 | 2.8e-47 | 187 |
| XP_008773573.1 | 29.31 | 505 | 3.8e-46 | 184 |
| XP_008773573.1 | 33.09 | 402 | 3.2e-43 | 174 |
| XP_008773573.1 | 30.36 | 504 | 9.1e-43 | 173 |
| XP_008773573.1 | 28.14 | 558 | 1.2e-42 | 172 |
| XP_008773573.1 | 30.33 | 488 | 2.6e-42 | 171 |
| XP_008773573.1 | 29.65 | 516 | 2.6e-42 | 171 |
| XP_008773573.1 | 29.34 | 518 | 1.2e-41 | 169 |
| XP_008773573.1 | 30.06 | 529 | 1.7e-41 | 169 |
| XP_008773573.1 | 29.38 | 497 | 2.8e-39 | 161 |
| XP_008773573.1 | 29.03 | 503 | 2.9e-37 | 155 |
| XP_008773573.1 | 27.93 | 494 | 2.8e-36 | 151 |
| XP_008773573.1 | 30.34 | 412 | 5.8e-35 | 147 |
| XP_008773573.1 | 28.47 | 548 | 6.2e-34 | 144 |
| XP_008773573.1 | 28.08 | 495 | 1.4e-33 | 142 |
| XP_008773573.1 | 32.91 | 319 | 3.3e-33 | 141 |
| XP_008773573.1 | 28.90 | 436 | 8.9e-31 | 133 |
| XP_008773573.1 | 26.55 | 501 | 5.8e-28 | 124 |
| XP_008773573.1 | 26.14 | 241 | 8.3e-13 | 74 |
| NP_001258221.1 | 29.77 | 571 | 1.3e-51 | 202 |
| NP_001258221.1 | 31.59 | 478 | 6.9e-51 | 199 |
| NP_001258221.1 | 30.61 | 490 | 4.3e-50 | 197 |
| NP_001258221.1 | 29.84 | 496 | 1.0e-47 | 189 |
| NP_001258221.1 | 30.58 | 484 | 1.7e-47 | 188 |
| NP_001258221.1 | 29.62 | 557 | 2.7e-47 | 187 |
| NP_001258221.1 | 30.64 | 532 | 5.2e-47 | 187 |
| NP_001258221.1 | 29.82 | 493 | 2.0e-46 | 185 |
| NP_001258221.1 | 29.65 | 489 | 1.3e-45 | 182 |
| NP_001258221.1 | 29.51 | 532 | 1.1e-44 | 179 |
| NP_001258221.1 | 30.93 | 472 | 4.5e-44 | 177 |
| NP_001258221.1 | 28.89 | 495 | 1.5e-43 | 175 |
| NP_001258221.1 | 29.48 | 502 | 1.7e-43 | 175 |
| NP_001258221.1 | 28.20 | 500 | 1.5e-41 | 169 |
| NP_001258221.1 | 29.92 | 488 | 4.3e-41 | 167 |
| NP_001258221.1 | 28.07 | 513 | 5.1e-41 | 167 |
| NP_001258221.1 | 28.93 | 484 | 9.7e-41 | 166 |
| NP_001258221.1 | 27.88 | 556 | 1.9e-40 | 165 |
| NP_001258221.1 | 29.11 | 505 | 1.8e-39 | 162 |
| NP_001258221.1 | 28.60 | 458 | 1.1e-38 | 159 |
| NP_001258221.1 | 28.95 | 449 | 1.2e-33 | 143 |
| NP_001258221.1 | 27.57 | 214 | 1.4e-14 | 80 |

### RPN1 (P04843)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_037199.2 | 95.76 | 589 | 0.0e+00 | 1179 |

### RPN2 (P04844)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_113886.1 | 93.03 | 631 | 0.0e+00 | 1147 |

### RPS27A (P62979)

**Description:** Ubiquitin-ribosomal protein eS31 fusion protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_112375.1 | 99.36 | 156 | 5.6e-114 | 319 |
| NP_001292372.1 | 99.36 | 156 | 5.6e-114 | 319 |
| XP_003754135.1 | 100.00 | 76 | 1.8e-50 | 156 |
| NP_113875.1 | 100.00 | 76 | 2.2e-49 | 155 |
| XP_002724552.1 | 97.37 | 76 | 2.2e-48 | 150 |
| NP_620250.2 | 97.44 | 78 | 7.1e-48 | 157 |
| NP_620250.2 | 97.44 | 78 | 7.1e-48 | 157 |
| NP_620250.2 | 100.00 | 76 | 9.5e-48 | 156 |
| NP_620250.2 | 96.15 | 78 | 1.8e-46 | 153 |
| XP_003752767.1 | 94.74 | 76 | 4.1e-47 | 147 |
| XP_006221493.1 | 97.30 | 74 | 4.1e-47 | 147 |
| NP_059010.1 | 97.44 | 78 | 4.5e-45 | 157 |
| NP_059010.1 | 97.44 | 78 | 4.5e-45 | 157 |
| NP_059010.1 | 97.44 | 78 | 4.5e-45 | 157 |
| NP_059010.1 | 97.44 | 78 | 4.5e-45 | 157 |
| NP_059010.1 | 97.44 | 78 | 4.5e-45 | 157 |
| NP_059010.1 | 97.44 | 78 | 4.5e-45 | 157 |
| NP_059010.1 | 97.44 | 78 | 4.5e-45 | 157 |
| NP_059010.1 | 97.44 | 78 | 4.5e-45 | 157 |
| NP_059010.1 | 97.44 | 78 | 4.5e-45 | 157 |
| NP_059010.1 | 97.44 | 78 | 4.5e-45 | 157 |
| XP_008773617.1 | 73.77 | 61 | 7.4e-24 | 90 |

### SEMA5B (Q9P283)

**Description:** Semaphorin-5B

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001100561.1 | 93.37 | 1041 | 0.0e+00 | 1981 |
| NP_001101129.1 | 59.35 | 1043 | 0.0e+00 | 1309 |
| NP_001101238.1 | 36.89 | 515 | 2.7e-88 | 307 |
| XP_006222635.1 | 35.37 | 492 | 6.7e-87 | 303 |
| XP_006222634.1 | 35.37 | 492 | 9.1e-87 | 303 |
| XP_008772349.1 | 35.37 | 492 | 1.1e-86 | 303 |
| XP_003753094.1 | 35.37 | 492 | 1.1e-86 | 303 |
| NP_059006.1 | 35.86 | 488 | 4.5e-86 | 296 |
| NP_445923.1 | 36.88 | 488 | 8.8e-84 | 292 |
| NP_059004.1 | 38.66 | 476 | 3.1e-81 | 286 |

### SIRT3 (Q9NTG7)

**Description:** NAD-dependent protein deacetylase sirtuin-3, mitochondrial

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001099783.1 | 85.94 | 256 | 1.7e-162 | 456 |
| NP_001008369.1 | 47.90 | 309 | 4.2e-96 | 291 |
| XP_017443543.1 | 40.30 | 263 | 3.4e-59 | 199 |
| XP_017443542.1 | 41.42 | 239 | 2.6e-52 | 182 |
| XP_008773172.1 | 40.11 | 187 | 6.2e-37 | 139 |
| NP_001100617.1 | 29.52 | 271 | 1.5e-18 | 85 |
| NP_001026819.1 | 28.67 | 300 | 9.5e-18 | 84 |
| NP_001004256.1 | 31.86 | 226 | 3.2e-17 | 82 |
| NP_001100543.1 | 29.06 | 203 | 7.7e-14 | 72 |

### SLIT2 (O94813)

**Description:** Slit homolog 2 protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_072154.2 | 96.09 | 1533 | 0.0e+00 | 3042 |
| NP_112611.1 | 66.82 | 1513 | 0.0e+00 | 2133 |
| NP_075242.1 | 65.83 | 1510 | 0.0e+00 | 2085 |
| NP_001100652.1 | 27.37 | 537 | 4.8e-44 | 176 |
| NP_001100652.1 | 35.52 | 259 | 5.4e-42 | 169 |
| NP_001100652.1 | 35.08 | 248 | 1.9e-33 | 141 |
| NP_001100652.1 | 34.86 | 218 | 7.6e-28 | 123 |
| NP_001100652.1 | 32.06 | 209 | 4.2e-22 | 104 |
| NP_001100652.1 | 34.57 | 162 | 6.2e-21 | 100 |
| NP_001100652.1 | 24.60 | 435 | 1.4e-19 | 96 |
| NP_001100652.1 | 29.20 | 113 | 3.2e-05 | 49 |
| NP_001100652.1 | 38.46 | 91 | 2.0e-04 | 46 |
| NP_001100652.1 | 28.33 | 120 | 8.9e-04 | 44 |
| NP_062020.1 | 35.32 | 269 | 6.5e-44 | 175 |
| NP_062020.1 | 32.71 | 266 | 1.8e-31 | 135 |
| NP_062020.1 | 32.06 | 262 | 1.5e-30 | 132 |
| NP_062020.1 | 28.89 | 270 | 8.6e-25 | 113 |
| NP_062020.1 | 30.93 | 236 | 1.4e-18 | 92 |
| NP_062020.1 | 32.84 | 134 | 2.4e-08 | 59 |
| NP_062020.1 | 30.38 | 158 | 6.6e-08 | 57 |
| NP_062020.1 | 32.43 | 185 | 2.9e-07 | 55 |
| NP_062020.1 | 30.36 | 112 | 1.4e-05 | 50 |
| NP_062020.1 | 29.28 | 181 | 1.8e-05 | 50 |
| NP_077334.2 | 38.32 | 274 | 1.7e-43 | 174 |
| NP_077334.2 | 37.74 | 265 | 6.9e-42 | 169 |
| NP_077334.2 | 37.54 | 285 | 7.2e-42 | 169 |
| NP_077334.2 | 38.09 | 252 | 6.1e-41 | 166 |
| NP_077334.2 | 35.46 | 282 | 1.0e-39 | 162 |
| NP_077334.2 | 36.21 | 290 | 4.3e-37 | 154 |
| NP_077334.2 | 26.48 | 623 | 9.2e-37 | 152 |
| NP_077334.2 | 32.37 | 278 | 2.2e-36 | 151 |
| NP_077334.2 | 31.02 | 332 | 6.5e-36 | 150 |
| NP_077334.2 | 31.82 | 286 | 4.9e-30 | 130 |
| NP_077334.2 | 36.48 | 233 | 1.5e-29 | 129 |
| NP_077334.2 | 32.92 | 240 | 7.1e-22 | 103 |
| NP_077334.2 | 33.54 | 158 | 1.9e-13 | 76 |
| NP_077334.2 | 34.19 | 117 | 3.6e-07 | 55 |
| NP_077334.2 | 35.00 | 120 | 9.6e-07 | 54 |
| NP_077334.2 | 33.90 | 118 | 2.8e-06 | 52 |
| NP_077334.2 | 34.69 | 98 | 4.8e-06 | 52 |
| NP_077334.2 | 30.67 | 150 | 1.7e-05 | 50 |
| NP_077334.2 | 31.71 | 123 | 1.4e-04 | 47 |
| NP_077334.2 | 30.17 | 116 | 2.9e-04 | 46 |
| NP_001099191.1 | 35.62 | 306 | 2.1e-43 | 174 |
| NP_001099191.1 | 28.14 | 622 | 2.6e-43 | 174 |
| NP_001099191.1 | 38.37 | 245 | 7.3e-40 | 163 |
| NP_001099191.1 | 37.41 | 262 | 8.7e-40 | 162 |
| NP_001099191.1 | 33.93 | 280 | 5.4e-38 | 157 |
| NP_001099191.1 | 38.02 | 242 | 7.2e-36 | 150 |
| NP_001099191.1 | 36.09 | 266 | 2.6e-34 | 144 |
| NP_001099191.1 | 33.56 | 295 | 4.3e-34 | 144 |
| NP_001099191.1 | 36.15 | 249 | 3.1e-33 | 141 |
| NP_001099191.1 | 32.17 | 286 | 1.1e-32 | 139 |
| NP_001099191.1 | 33.56 | 289 | 1.1e-30 | 132 |
| NP_001099191.1 | 30.92 | 304 | 2.0e-25 | 115 |
| NP_001099191.1 | 34.82 | 224 | 2.3e-23 | 108 |
| NP_001099191.1 | 22.70 | 467 | 6.5e-16 | 84 |
| NP_001099191.1 | 35.22 | 159 | 7.4e-16 | 84 |
| NP_001099191.1 | 31.82 | 132 | 4.2e-05 | 48 |
| NP_001099191.1 | 33.87 | 124 | 4.7e-05 | 48 |
| NP_001099191.1 | 30.08 | 123 | 2.1e-04 | 46 |
| NP_001099191.1 | 26.36 | 239 | 3.6e-04 | 45 |
| NP_001099191.1 | 24.88 | 213 | 4.7e-04 | 45 |
| XP_006225372.1 | 37.04 | 243 | 4.5e-41 | 167 |
| XP_006225372.1 | 37.00 | 200 | 2.1e-32 | 138 |
| XP_006225372.1 | 39.49 | 157 | 3.8e-26 | 118 |
| XP_006225372.1 | 35.87 | 184 | 5.2e-22 | 104 |
| XP_006225372.1 | 36.25 | 160 | 2.4e-21 | 102 |
| XP_006225372.1 | 30.50 | 141 | 1.1e-07 | 57 |
| XP_006225372.1 | 35.37 | 82 | 6.0e-07 | 55 |
| XP_006225372.1 | 31.31 | 99 | 3.9e-05 | 48 |
| NP_064472.2 | 39.83 | 236 | 4.6e-41 | 167 |
| NP_064472.2 | 38.02 | 242 | 4.0e-38 | 157 |
| NP_064472.2 | 37.98 | 237 | 7.0e-36 | 150 |
| NP_064472.2 | 34.27 | 248 | 8.0e-34 | 143 |
| NP_064472.2 | 33.97 | 312 | 1.2e-33 | 142 |
| NP_064472.2 | 34.01 | 247 | 1.0e-32 | 139 |
| NP_064472.2 | 36.33 | 245 | 1.8e-32 | 138 |
| NP_064472.2 | 29.11 | 316 | 1.8e-31 | 135 |
| NP_064472.2 | 31.30 | 278 | 5.4e-31 | 134 |
| NP_064472.2 | 36.78 | 242 | 1.7e-30 | 132 |
| NP_064472.2 | 28.71 | 303 | 3.3e-25 | 115 |
| NP_064472.2 | 32.84 | 201 | 4.3e-19 | 94 |
| NP_064472.2 | 28.92 | 249 | 2.9e-12 | 72 |
| NP_064472.2 | 31.45 | 124 | 1.8e-08 | 60 |
| NP_064472.2 | 36.29 | 124 | 8.5e-08 | 57 |
| NP_064472.2 | 32.77 | 119 | 2.4e-07 | 56 |
| NP_064472.2 | 30.87 | 149 | 2.6e-07 | 56 |
| NP_064472.2 | 36.04 | 111 | 1.7e-06 | 53 |
| NP_064472.2 | 34.34 | 99 | 4.5e-06 | 52 |
| NP_064472.2 | 34.23 | 111 | 5.5e-06 | 51 |
| NP_064472.2 | 31.50 | 127 | 2.2e-05 | 49 |
| NP_064472.2 | 32.20 | 118 | 3.0e-05 | 49 |
| NP_064472.2 | 29.60 | 125 | 9.5e-05 | 47 |
| NP_064472.2 | 34.55 | 110 | 1.1e-04 | 47 |
| NP_064472.2 | 31.37 | 102 | 1.1e-04 | 47 |
| NP_064472.2 | 32.08 | 106 | 3.1e-04 | 45 |
| NP_001100555.1 | 27.33 | 450 | 1.0e-40 | 160 |
| NP_001100555.1 | 25.00 | 464 | 3.0e-34 | 140 |
| NP_001100555.1 | 27.34 | 428 | 1.6e-33 | 138 |
| NP_001100555.1 | 33.17 | 205 | 2.7e-23 | 106 |

### SMAD2 (Q15796)

**Description:** Mothers against decapentaplegic homolog 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_062064.1 | 99.36 | 467 | 0.0e+00 | 970 |
| NP_001264379.1 | 99.36 | 467 | 0.0e+00 | 970 |
| NP_037227.1 | 83.94 | 467 | 0.0e+00 | 782 |
| NP_067724.1 | 55.88 | 510 | 0.0e+00 | 526 |
| NP_037262.2 | 55.38 | 511 | 0.0e+00 | 519 |
| XP_017451298.1 | 59.33 | 477 | 0.0e+00 | 518 |
| XP_017451297.1 | 59.33 | 477 | 0.0e+00 | 518 |
| NP_062148.1 | 41.32 | 242 | 8.7e-48 | 172 |
| NP_062148.1 | 46.53 | 144 | 3.3e-26 | 111 |
| NP_110485.1 | 27.14 | 398 | 1.6e-25 | 108 |
| NP_001102472.1 | 27.78 | 288 | 2.6e-22 | 99 |

### SMAD3 (P84022)

**Description:** Mothers against decapentaplegic homolog 3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_037227.1 | 100.00 | 425 | 0.0e+00 | 890 |
| NP_062064.1 | 83.51 | 467 | 0.0e+00 | 777 |
| NP_001264379.1 | 83.51 | 467 | 0.0e+00 | 777 |
| NP_067724.1 | 60.26 | 468 | 0.0e+00 | 542 |
| NP_037262.2 | 60.09 | 471 | 0.0e+00 | 534 |
| XP_017451298.1 | 63.53 | 436 | 0.0e+00 | 533 |
| XP_017451297.1 | 63.53 | 436 | 0.0e+00 | 533 |
| NP_062148.1 | 41.74 | 242 | 1.3e-48 | 174 |
| NP_062148.1 | 44.65 | 159 | 2.5e-31 | 125 |
| NP_110485.1 | 28.12 | 377 | 4.3e-28 | 115 |
| NP_001102472.1 | 28.40 | 338 | 3.3e-26 | 110 |

### SOD2 (P04179)

**Description:** Superoxide dismutase [Mn], mitochondrial

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_058747.1 | 87.84 | 222 | 2.7e-149 | 414 |

### SREBF1 (P36956)

**Description:** Sterol regulatory element-binding protein 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001263636.1 | 80.40 | 1148 | 0.0e+00 | 1654 |
| NP_001263637.1 | 80.36 | 1120 | 0.0e+00 | 1608 |
| NP_001028866.1 | 48.63 | 948 | 0.0e+00 | 756 |
| NP_001258156.1 | 35.05 | 97 | 1.5e-08 | 58 |
| NP_001178018.1 | 41.10 | 73 | 2.8e-07 | 54 |
| NP_113965.1 | 37.66 | 77 | 7.3e-07 | 52 |
| NP_001020878.1 | 37.21 | 86 | 9.2e-07 | 53 |
| NP_112401.1 | 28.07 | 114 | 2.0e-06 | 51 |
| NP_071546.1 | 31.07 | 103 | 2.1e-05 | 46 |

### SSPOP (A2VEC9)

**Description:** SCO-spondin

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001007017.1 | 75.45 | 5178 | 0.0e+00 | 7487 |
| XP_006223636.1 | 31.49 | 1178 | 5.6e-152 | 536 |
| XP_006223636.1 | 26.08 | 418 | 1.0e-22 | 108 |
| XP_006223636.1 | 25.40 | 374 | 2.4e-19 | 97 |
| XP_006223636.1 | 27.27 | 374 | 4.0e-19 | 97 |
| XP_006223636.1 | 25.40 | 378 | 4.4e-16 | 87 |
| XP_006223636.1 | 39.39 | 99 | 8.5e-09 | 63 |
| XP_006223636.1 | 37.62 | 101 | 1.3e-08 | 62 |
| XP_006223636.1 | 35.71 | 98 | 5.6e-06 | 53 |
| XP_006223636.1 | 26.73 | 217 | 1.9e-05 | 52 |
| XP_006223636.1 | 26.76 | 213 | 2.3e-05 | 51 |
| XP_006223636.1 | 41.18 | 68 | 5.0e-05 | 50 |
| XP_006223636.1 | 44.44 | 54 | 5.7e-05 | 50 |
| XP_006223636.1 | 42.00 | 50 | 2.6e-04 | 48 |
| XP_006223636.1 | 37.08 | 89 | 3.0e-04 | 47 |
| XP_006223636.1 | 33.95 | 109 | 8.9e-04 | 46 |
| NP_446341.1 | 31.06 | 1262 | 4.7e-151 | 531 |
| NP_446341.1 | 28.70 | 324 | 1.8e-27 | 124 |
| NP_446341.1 | 26.32 | 380 | 1.3e-22 | 108 |
| NP_446341.1 | 24.18 | 397 | 6.7e-11 | 70 |
| NP_446341.1 | 38.78 | 98 | 1.1e-07 | 59 |
| NP_446341.1 | 27.61 | 239 | 2.8e-07 | 57 |
| NP_446341.1 | 30.15 | 136 | 4.4e-06 | 53 |
| NP_446341.1 | 33.04 | 112 | 5.8e-06 | 53 |
| NP_446341.1 | 33.66 | 101 | 6.9e-06 | 53 |
| NP_446341.1 | 33.68 | 95 | 1.6e-05 | 52 |
| NP_446341.1 | 38.30 | 94 | 2.1e-05 | 51 |
| NP_446341.1 | 35.06 | 77 | 7.7e-05 | 49 |
| NP_446341.1 | 34.00 | 100 | 1.9e-04 | 48 |
| NP_446341.1 | 34.33 | 67 | 4.0e-04 | 47 |
| NP_446341.1 | 47.92 | 48 | 4.2e-04 | 47 |
| NP_446341.1 | 33.33 | 99 | 6.3e-04 | 47 |
| NP_446341.1 | 31.25 | 112 | 8.5e-04 | 46 |
| XP_001063331.3 | 30.91 | 1239 | 1.1e-146 | 516 |
| XP_001063331.3 | 26.36 | 387 | 1.0e-27 | 125 |
| XP_001063331.3 | 27.21 | 419 | 1.0e-25 | 119 |
| XP_001063331.3 | 27.67 | 430 | 2.2e-25 | 117 |
| XP_001063331.3 | 25.06 | 407 | 2.8e-18 | 94 |
| XP_001063331.3 | 37.86 | 103 | 1.1e-06 | 55 |
| XP_001063331.3 | 35.35 | 99 | 5.0e-06 | 53 |
| XP_001063331.3 | 27.35 | 223 | 1.7e-05 | 52 |
| XP_001063331.3 | 31.07 | 103 | 2.9e-04 | 48 |
| XP_001063331.3 | 39.39 | 66 | 3.1e-04 | 47 |
| XP_001063331.3 | 41.94 | 62 | 4.8e-04 | 47 |
| XP_001063331.3 | 33.91 | 115 | 7.6e-04 | 46 |
| XP_001063331.3 | 42.86 | 49 | 8.9e-04 | 46 |
| XP_017459733.1 | 30.06 | 1264 | 3.5e-140 | 493 |
| XP_017459733.1 | 30.27 | 413 | 5.1e-33 | 142 |
| XP_017459733.1 | 29.33 | 341 | 7.6e-33 | 142 |
| XP_017459733.1 | 29.79 | 376 | 6.3e-23 | 109 |
| XP_017459733.1 | 28.74 | 348 | 1.3e-15 | 85 |
| XP_017459733.1 | 27.43 | 226 | 8.5e-09 | 62 |
| XP_017459733.1 | 42.10 | 57 | 3.0e-05 | 51 |
| XP_017459733.1 | 38.09 | 84 | 9.0e-05 | 49 |
| NP_001099732.1 | 30.75 | 1135 | 1.2e-138 | 491 |
| NP_001099732.1 | 31.92 | 473 | 3.5e-57 | 223 |
| NP_001099732.1 | 32.60 | 411 | 3.8e-46 | 186 |
| NP_001099732.1 | 26.67 | 345 | 2.1e-14 | 81 |
| NP_001099732.1 | 23.88 | 356 | 6.4e-12 | 73 |
| NP_001099732.1 | 33.03 | 109 | 2.1e-09 | 65 |
| NP_001099732.1 | 38.81 | 67 | 5.8e-07 | 57 |
| NP_001099732.1 | 22.79 | 351 | 6.0e-07 | 57 |
| NP_001099732.1 | 41.82 | 55 | 4.8e-06 | 53 |
| NP_001099732.1 | 43.28 | 67 | 2.3e-05 | 51 |
| NP_001099732.1 | 32.98 | 94 | 3.2e-05 | 51 |
| NP_001099732.1 | 31.58 | 133 | 3.4e-05 | 51 |
| NP_001099732.1 | 34.31 | 102 | 4.9e-05 | 50 |
| NP_001099732.1 | 33.90 | 118 | 5.0e-05 | 50 |
| NP_001099732.1 | 32.77 | 119 | 9.5e-05 | 49 |
| NP_001099732.1 | 35.05 | 97 | 2.1e-04 | 48 |
| NP_001099732.1 | 39.02 | 82 | 6.4e-04 | 47 |
| NP_001099732.1 | 36.04 | 111 | 6.5e-04 | 47 |
| NP_001099732.1 | 41.94 | 62 | 6.8e-04 | 46 |
| XP_017458829.1 | 29.04 | 1126 | 1.0e-126 | 450 |
| XP_017458829.1 | 29.16 | 367 | 1.7e-43 | 177 |
| XP_017458829.1 | 20.21 | 287 | 5.3e-11 | 70 |
| XP_017458829.1 | 25.86 | 290 | 3.6e-07 | 57 |
| XP_017458829.1 | 25.91 | 247 | 3.7e-07 | 57 |
| XP_017458829.1 | 32.08 | 106 | 3.8e-07 | 57 |
| XP_017458829.1 | 33.33 | 81 | 1.2e-05 | 52 |
| XP_017458829.1 | 32.69 | 104 | 2.4e-05 | 51 |
| XP_017458829.1 | 31.31 | 99 | 3.6e-05 | 50 |
| XP_017458829.1 | 34.67 | 75 | 5.8e-05 | 50 |
| XP_017458829.1 | 21.36 | 323 | 2.0e-04 | 48 |
| XP_017458829.1 | 35.71 | 70 | 2.6e-04 | 48 |
| XP_017458829.1 | 38.18 | 55 | 4.3e-04 | 47 |
| XP_006223763.2 | 28.97 | 1191 | 1.7e-122 | 438 |
| XP_006223763.2 | 29.07 | 571 | 2.6e-40 | 167 |
| XP_006223763.2 | 26.49 | 419 | 2.9e-23 | 110 |
| XP_006223763.2 | 28.28 | 297 | 3.8e-15 | 84 |
| XP_006223763.2 | 38.18 | 110 | 2.1e-08 | 61 |
| XP_006223763.2 | 28.42 | 190 | 1.0e-07 | 59 |
| XP_006223763.2 | 35.71 | 98 | 7.2e-06 | 53 |
| XP_006223763.2 | 33.33 | 171 | 3.2e-05 | 51 |
| XP_006223763.2 | 35.19 | 108 | 8.1e-05 | 49 |
| XP_006223763.2 | 31.32 | 182 | 1.6e-04 | 48 |
| XP_006223763.2 | 39.06 | 64 | 2.4e-04 | 48 |
| XP_006223763.2 | 33.33 | 96 | 5.1e-04 | 47 |
| XP_006223763.2 | 42.62 | 61 | 7.1e-04 | 46 |
| NP_001296418.1 | 26.74 | 1264 | 2.8e-104 | 379 |
| NP_001296418.1 | 27.34 | 384 | 4.3e-24 | 113 |
| NP_001296418.1 | 42.86 | 49 | 4.6e-05 | 50 |
| NP_001296418.1 | 35.95 | 89 | 4.2e-04 | 47 |
| NP_001296418.1 | 42.10 | 57 | 5.1e-04 | 47 |
| NP_001296418.1 | 42.42 | 66 | 7.3e-04 | 46 |
| NP_001158129.2 | 28.36 | 1227 | 9.1e-79 | 293 |
| NP_001158129.2 | 27.06 | 1308 | 5.8e-78 | 291 |
| NP_001158129.2 | 25.09 | 1056 | 4.2e-50 | 199 |
| NP_001158129.2 | 27.71 | 783 | 5.6e-46 | 185 |
| NP_001158129.2 | 26.52 | 773 | 4.9e-39 | 162 |
| NP_001158129.2 | 29.21 | 469 | 2.0e-35 | 150 |
| NP_001158129.2 | 31.55 | 374 | 7.5e-33 | 142 |
| NP_001158129.2 | 40.28 | 72 | 1.2e-05 | 52 |
| NP_001158129.2 | 33.00 | 100 | 1.0e-04 | 49 |
| NP_001158129.2 | 35.00 | 100 | 2.0e-04 | 48 |
| NP_001158129.2 | 30.60 | 183 | 5.2e-04 | 47 |

### ST3GAL3 (Q11203)

**Description:** CMP-N-acetylneuraminate-beta-1,4-galactoside alpha-2,3-sialyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_113885.1 | 97.07 | 375 | 0.0e+00 | 751 |
| NP_112627.2 | 38.34 | 313 | 4.2e-74 | 235 |
| NP_997485.2 | 37.34 | 308 | 9.4e-71 | 224 |
| NP_976082.1 | 38.62 | 290 | 5.6e-62 | 202 |
| NP_001013237.1 | 28.32 | 286 | 1.8e-28 | 114 |
| NP_113883.2 | 29.29 | 280 | 4.4e-28 | 112 |
| NP_001026822.1 | 25.18 | 274 | 2.5e-15 | 76 |
| NP_001099329.1 | 37.04 | 108 | 8.9e-15 | 75 |
| NP_001094358.1 | 29.19 | 209 | 5.7e-13 | 70 |
| NP_476497.1 | 23.97 | 242 | 2.6e-11 | 64 |

### ST3GAL4 (Q11206)

**Description:** CMP-N-acetylneuraminate-beta-galactosamide-alpha-2,3-sialyltransferase 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_976082.1 | 90.09 | 333 | 0.0e+00 | 622 |
| NP_997485.2 | 43.77 | 249 | 1.2e-61 | 199 |
| NP_113885.1 | 39.46 | 294 | 1.3e-60 | 198 |
| NP_112627.2 | 34.43 | 305 | 6.7e-50 | 171 |
| NP_113883.2 | 32.08 | 212 | 3.0e-29 | 115 |
| NP_001013237.1 | 35.64 | 188 | 5.4e-27 | 108 |
| NP_001026822.1 | 24.72 | 271 | 3.2e-13 | 70 |
| NP_001106815.1 | 28.44 | 218 | 6.4e-13 | 69 |
| NP_001094358.1 | 26.48 | 219 | 8.9e-13 | 69 |
| NP_446366.1 | 28.00 | 225 | 1.5e-12 | 68 |
| NP_976082.1 | 90.09 | 333 | 0.0e+00 | 622 |
| NP_997485.2 | 43.77 | 249 | 1.2e-61 | 199 |
| NP_113885.1 | 39.46 | 294 | 1.3e-60 | 198 |
| NP_112627.2 | 34.43 | 305 | 6.7e-50 | 171 |
| NP_113883.2 | 32.08 | 212 | 3.0e-29 | 115 |
| NP_001013237.1 | 35.64 | 188 | 5.4e-27 | 108 |
| NP_001026822.1 | 24.72 | 271 | 3.2e-13 | 70 |
| NP_001106815.1 | 28.44 | 218 | 6.4e-13 | 69 |
| NP_001094358.1 | 26.48 | 219 | 8.9e-13 | 69 |
| NP_446366.1 | 28.00 | 225 | 1.5e-12 | 68 |

### ST3GAL4 (Q11206)

**Description:** CMP-N-acetylneuraminate-beta-galactosamide-alpha-2,3-sialyltransferase 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_976082.1 | 90.09 | 333 | 0.0e+00 | 622 |
| NP_997485.2 | 43.77 | 249 | 1.2e-61 | 199 |
| NP_113885.1 | 39.46 | 294 | 1.3e-60 | 198 |
| NP_112627.2 | 34.43 | 305 | 6.7e-50 | 171 |
| NP_113883.2 | 32.08 | 212 | 3.0e-29 | 115 |
| NP_001013237.1 | 35.64 | 188 | 5.4e-27 | 108 |
| NP_001026822.1 | 24.72 | 271 | 3.2e-13 | 70 |
| NP_001106815.1 | 28.44 | 218 | 6.4e-13 | 69 |
| NP_001094358.1 | 26.48 | 219 | 8.9e-13 | 69 |
| NP_446366.1 | 28.00 | 225 | 1.5e-12 | 68 |
| NP_976082.1 | 90.09 | 333 | 0.0e+00 | 622 |
| NP_997485.2 | 43.77 | 249 | 1.2e-61 | 199 |
| NP_113885.1 | 39.46 | 294 | 1.3e-60 | 198 |
| NP_112627.2 | 34.43 | 305 | 6.7e-50 | 171 |
| NP_113883.2 | 32.08 | 212 | 3.0e-29 | 115 |
| NP_001013237.1 | 35.64 | 188 | 5.4e-27 | 108 |
| NP_001026822.1 | 24.72 | 271 | 3.2e-13 | 70 |
| NP_001106815.1 | 28.44 | 218 | 6.4e-13 | 69 |
| NP_001094358.1 | 26.48 | 219 | 8.9e-13 | 69 |
| NP_446366.1 | 28.00 | 225 | 1.5e-12 | 68 |

### ST6GAL1 (P15907)

**Description:** Beta-galactoside alpha-2,6-sialyltransferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001106815.1 | 80.54 | 406 | 0.0e+00 | 687 |
| NP_671738.2 | 87.65 | 170 | 7.4e-111 | 324 |
| NP_001094358.1 | 46.85 | 318 | 1.3e-101 | 311 |
| NP_001258290.1 | 24.48 | 241 | 2.2e-14 | 73 |
| NP_113883.2 | 23.59 | 212 | 1.3e-13 | 71 |
| NP_976082.1 | 27.78 | 252 | 1.7e-12 | 68 |
| NP_112627.2 | 28.92 | 287 | 1.8e-12 | 68 |
| XP_008773544.2 | 26.82 | 220 | 3.6e-12 | 66 |
| NP_997485.2 | 24.54 | 216 | 5.6e-12 | 66 |
| NP_037161.1 | 29.02 | 193 | 1.0e-11 | 66 |
| NP_001106815.1 | 80.54 | 406 | 0.0e+00 | 687 |
| NP_671738.2 | 87.65 | 170 | 7.4e-111 | 324 |
| NP_001094358.1 | 46.85 | 318 | 1.3e-101 | 311 |
| NP_001258290.1 | 24.48 | 241 | 2.2e-14 | 73 |
| NP_113883.2 | 23.59 | 212 | 1.3e-13 | 71 |
| NP_976082.1 | 27.78 | 252 | 1.7e-12 | 68 |
| NP_112627.2 | 28.92 | 287 | 1.8e-12 | 68 |
| XP_008773544.2 | 26.82 | 220 | 3.6e-12 | 66 |
| NP_997485.2 | 24.54 | 216 | 5.6e-12 | 66 |
| NP_037161.1 | 29.02 | 193 | 1.0e-11 | 66 |

### ST6GAL1 (P15907)

**Description:** Beta-galactoside alpha-2,6-sialyltransferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001106815.1 | 80.54 | 406 | 0.0e+00 | 687 |
| NP_671738.2 | 87.65 | 170 | 7.4e-111 | 324 |
| NP_001094358.1 | 46.85 | 318 | 1.3e-101 | 311 |
| NP_001258290.1 | 24.48 | 241 | 2.2e-14 | 73 |
| NP_113883.2 | 23.59 | 212 | 1.3e-13 | 71 |
| NP_976082.1 | 27.78 | 252 | 1.7e-12 | 68 |
| NP_112627.2 | 28.92 | 287 | 1.8e-12 | 68 |
| XP_008773544.2 | 26.82 | 220 | 3.6e-12 | 66 |
| NP_997485.2 | 24.54 | 216 | 5.6e-12 | 66 |
| NP_037161.1 | 29.02 | 193 | 1.0e-11 | 66 |
| NP_001106815.1 | 80.54 | 406 | 0.0e+00 | 687 |
| NP_671738.2 | 87.65 | 170 | 7.4e-111 | 324 |
| NP_001094358.1 | 46.85 | 318 | 1.3e-101 | 311 |
| NP_001258290.1 | 24.48 | 241 | 2.2e-14 | 73 |
| NP_113883.2 | 23.59 | 212 | 1.3e-13 | 71 |
| NP_976082.1 | 27.78 | 252 | 1.7e-12 | 68 |
| NP_112627.2 | 28.92 | 287 | 1.8e-12 | 68 |
| XP_008773544.2 | 26.82 | 220 | 3.6e-12 | 66 |
| NP_997485.2 | 24.54 | 216 | 5.6e-12 | 66 |
| NP_037161.1 | 29.02 | 193 | 1.0e-11 | 66 |

### ST6GALNAC3 (Q8NDV1)

**Description:** Alpha-N-acetylgalactosaminide alpha-2,6-sialyltransferase 3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_061996.1 | 85.90 | 305 | 0.0e+00 | 572 |
| XP_017457464.1 | 53.31 | 242 | 4.7e-95 | 283 |
| XP_006224416.1 | 53.31 | 242 | 1.2e-94 | 284 |
| XP_008773544.2 | 54.63 | 227 | 1.4e-94 | 280 |
| NP_001258290.1 | 42.98 | 242 | 7.0e-71 | 223 |
| NP_001015036.1 | 44.50 | 200 | 1.6e-57 | 184 |
| NP_001094358.1 | 27.98 | 168 | 1.0e-13 | 71 |
| NP_113883.2 | 29.53 | 193 | 9.2e-13 | 68 |
| NP_037161.1 | 26.60 | 203 | 4.3e-11 | 63 |
| NP_976082.1 | 27.41 | 197 | 2.4e-09 | 57 |

### ST8SIA2 (Q92186)

**Description:** Alpha-2,8-sialyltransferase 8B

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_476497.1 | 97.60 | 375 | 0.0e+00 | 763 |
| NP_446366.1 | 56.76 | 370 | 4.9e-153 | 439 |
| NP_037161.1 | 38.46 | 312 | 1.4e-74 | 236 |
| NP_036945.2 | 31.79 | 346 | 7.0e-53 | 179 |
| NP_998789.1 | 31.02 | 303 | 7.8e-46 | 161 |
| NP_998793.1 | 32.27 | 251 | 9.7e-44 | 155 |
| NP_113883.2 | 24.90 | 249 | 5.3e-14 | 72 |
| NP_112627.2 | 27.02 | 248 | 7.3e-14 | 72 |
| NP_001258290.1 | 33.33 | 204 | 1.6e-12 | 68 |
| NP_113885.1 | 25.00 | 236 | 3.7e-12 | 67 |

### ST8SIA3 (O43173)

**Description:** Alpha-N-acetylneuraminate alpha-2,8-sialyltransferase ST8SIA3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_037161.1 | 96.84 | 380 | 0.0e+00 | 775 |
| NP_476497.1 | 39.27 | 303 | 1.7e-75 | 238 |
| NP_446366.1 | 39.52 | 291 | 1.1e-68 | 223 |
| NP_998789.1 | 36.52 | 293 | 1.0e-55 | 187 |
| NP_036945.2 | 31.82 | 308 | 1.1e-47 | 166 |
| NP_998793.1 | 34.14 | 249 | 1.1e-38 | 142 |
| NP_112627.2 | 27.97 | 236 | 9.8e-16 | 78 |
| NP_113883.2 | 28.77 | 212 | 4.1e-15 | 75 |
| NP_001013237.1 | 29.43 | 248 | 3.2e-14 | 73 |
| XP_017457464.1 | 28.78 | 205 | 4.7e-14 | 72 |

### ST8SIA6 (P61647)

**Description:** Alpha-2,8-sialyltransferase 8F

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_998789.1 | 81.41 | 398 | 0.0e+00 | 677 |
| NP_036945.2 | 43.36 | 286 | 1.6e-90 | 277 |
| NP_998793.1 | 44.95 | 287 | 2.2e-88 | 272 |
| NP_037161.1 | 35.43 | 302 | 1.1e-53 | 182 |
| NP_446366.1 | 29.27 | 287 | 4.0e-43 | 156 |
| NP_476497.1 | 29.87 | 298 | 3.5e-42 | 152 |
| NP_001013237.1 | 31.82 | 198 | 2.4e-12 | 67 |
| NP_001026822.1 | 30.57 | 229 | 1.2e-11 | 65 |
| NP_001099329.1 | 30.00 | 230 | 7.4e-11 | 63 |
| NP_113883.2 | 28.24 | 216 | 1.0e-10 | 62 |

### STK11 (Q15831)

**Description:** Serine/threonine-protein kinase STK11

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001101539.1 | 89.95 | 438 | 0.0e+00 | 750 |
| NP_062015.2 | 36.67 | 270 | 1.2e-49 | 177 |
| NP_076481.1 | 36.00 | 275 | 1.8e-47 | 171 |
| NP_001120809.1 | 34.98 | 283 | 2.4e-44 | 165 |
| NP_001102132.1 | 34.33 | 268 | 1.0e-40 | 149 |
| NP_067731.1 | 32.99 | 294 | 1.1e-40 | 154 |
| XP_017459734.1 | 33.71 | 264 | 2.4e-40 | 152 |
| XP_017458887.1 | 33.33 | 291 | 5.6e-40 | 149 |
| XP_017458886.1 | 33.33 | 291 | 5.6e-40 | 149 |
| XP_017458885.1 | 33.33 | 291 | 5.6e-40 | 149 |

### STT3A (P46977)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit STT3A

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001128221.1 | 99.43 | 705 | 0.0e+00 | 1452 |
| NP_001164010.1 | 59.75 | 713 | 0.0e+00 | 869 |

### THBS1 (P07996)

**Description:** Thrombospondin-1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001013080.1 | 95.30 | 1169 | 0.0e+00 | 2236 |
| NP_001162609.1 | 62.65 | 1154 | 0.0e+00 | 1486 |
| NP_001292387.1 | 52.05 | 682 | 0.0e+00 | 637 |
| NP_001292387.1 | 20.41 | 245 | 1.2e-04 | 46 |
| NP_036966.1 | 54.05 | 666 | 0.0e+00 | 621 |
| NP_058829.1 | 53.46 | 664 | 0.0e+00 | 614 |
| NP_001258221.1 | 46.86 | 175 | 1.7e-42 | 171 |
| NP_001258221.1 | 40.20 | 199 | 3.0e-40 | 163 |
| NP_001258221.1 | 40.70 | 199 | 1.6e-37 | 154 |
| NP_001258221.1 | 41.14 | 175 | 2.6e-37 | 154 |
| NP_001258221.1 | 48.28 | 116 | 7.6e-30 | 129 |
| NP_001258221.1 | 52.24 | 67 | 7.7e-14 | 77 |
| NP_001258221.1 | 34.43 | 122 | 1.2e-06 | 53 |
| NP_001164068.2 | 38.60 | 215 | 3.6e-34 | 143 |
| NP_001164068.2 | 34.16 | 243 | 4.7e-27 | 120 |
| NP_001164068.2 | 40.00 | 120 | 1.4e-18 | 92 |
| NP_001100227.1 | 41.21 | 182 | 5.7e-31 | 128 |
| NP_001100227.1 | 41.11 | 180 | 1.8e-29 | 124 |
| NP_001100227.1 | 27.98 | 168 | 7.4e-08 | 56 |
| NP_001100368.1 | 38.42 | 190 | 7.0e-31 | 132 |
| NP_001100368.1 | 37.87 | 169 | 9.2e-25 | 112 |
| NP_001100368.1 | 33.85 | 130 | 1.7e-16 | 85 |
| NP_001101384.1 | 38.38 | 198 | 4.7e-29 | 126 |
| NP_001101384.1 | 35.88 | 170 | 2.8e-24 | 111 |

### THSD1 (Q9NS62)

**Description:** Thrombospondin type-1 domain-containing protein 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001102348.1 | 77.66 | 855 | 0.0e+00 | 1347 |
| NP_001258221.1 | 41.51 | 53 | 1.8e-07 | 55 |
| NP_001258221.1 | 35.85 | 53 | 1.8e-04 | 45 |
| NP_001258221.1 | 31.67 | 60 | 8.2e-04 | 43 |
| NP_001100682.1 | 38.89 | 72 | 1.0e-06 | 52 |
| NP_001007017.1 | 32.98 | 94 | 4.5e-06 | 51 |
| NP_001007017.1 | 42.10 | 57 | 8.5e-06 | 50 |
| NP_001007017.1 | 32.71 | 107 | 2.1e-05 | 48 |
| NP_001100561.1 | 42.31 | 52 | 1.4e-05 | 49 |
| NP_001100561.1 | 37.93 | 58 | 8.1e-05 | 46 |
| NP_001101631.1 | 40.30 | 67 | 1.7e-05 | 48 |
| NP_001013080.1 | 33.72 | 86 | 2.4e-05 | 48 |
| NP_001013080.1 | 36.54 | 52 | 1.2e-04 | 46 |
| NP_001101129.1 | 34.38 | 64 | 2.6e-05 | 48 |
| NP_001101541.1 | 46.00 | 50 | 2.8e-05 | 47 |
| NP_001162609.1 | 38.33 | 60 | 3.0e-05 | 48 |
| NP_001162609.1 | 40.38 | 52 | 1.0e-04 | 46 |

### THSD4 (Q6ZMP0)

**Description:** Thrombospondin type-1 domain-containing protein 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| XP_017459138.1 | 87.83 | 1019 | 0.0e+00 | 1834 |
| XP_017459137.1 | 87.83 | 1019 | 0.0e+00 | 1834 |
| XP_017459136.1 | 87.83 | 1019 | 0.0e+00 | 1834 |
| XP_006226486.1 | 93.70 | 635 | 0.0e+00 | 1253 |
| NP_001029184.1 | 38.94 | 1022 | 0.0e+00 | 636 |
| XP_006222944.1 | 31.97 | 588 | 4.1e-79 | 279 |
| XP_006222944.1 | 28.93 | 363 | 4.9e-27 | 119 |
| XP_003753205.1 | 31.97 | 588 | 1.4e-78 | 280 |
| XP_003753205.1 | 28.93 | 363 | 1.2e-27 | 121 |
| NP_001178873.1 | 34.90 | 553 | 2.7e-77 | 276 |
| NP_001178873.1 | 27.98 | 411 | 3.3e-36 | 149 |
| NP_001101347.1 | 29.65 | 715 | 2.4e-73 | 266 |
| NP_001101347.1 | 33.42 | 371 | 8.6e-43 | 171 |
| NP_001101347.1 | 29.40 | 398 | 8.0e-39 | 158 |
| NP_001101347.1 | 30.82 | 331 | 4.3e-31 | 133 |
| NP_001101347.1 | 28.38 | 303 | 9.6e-26 | 115 |
| XP_008774516.1 | 33.65 | 523 | 2.1e-70 | 256 |
| XP_008774516.1 | 32.93 | 249 | 2.6e-26 | 117 |
| XP_008774516.1 | 31.17 | 247 | 2.2e-22 | 104 |
| XP_008774516.1 | 35.03 | 177 | 1.4e-19 | 95 |
| XP_008774516.1 | 51.11 | 45 | 2.6e-04 | 45 |

### TRIM63 (Q969Q1)

**Description:** E3 ubiquitin-protein ligase TRIM63

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_543179.1 | 92.36 | 327 | 0.0e+00 | 633 |
| NP_001013235.1 | 59.55 | 356 | 8.5e-151 | 429 |
| NP_001012218.1 | 60.92 | 325 | 4.6e-143 | 416 |
| NP_001101161.1 | 29.09 | 385 | 1.8e-39 | 147 |
| NP_075216.1 | 26.52 | 396 | 1.9e-36 | 139 |
| NP_001099617.1 | 28.61 | 360 | 8.7e-36 | 138 |
| NP_001178818.1 | 24.50 | 396 | 7.1e-25 | 106 |
| NP_001102415.1 | 27.24 | 257 | 1.0e-20 | 92 |
| NP_001012210.1 | 29.80 | 255 | 3.7e-20 | 90 |
| NP_001129187.1 | 28.29 | 205 | 1.6e-19 | 90 |
| NP_001129187.1 | 48.57 | 35 | 2.4e-04 | 43 |

### TUSC3 (Q13454)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit TUSC3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001004212.1 | 97.97 | 345 | 0.0e+00 | 704 |
| NP_446398.1 | 70.25 | 316 | 3.4e-178 | 497 |

### UAP1 (Q16222)

**Description:** UDP-N-acetylhexosamine pyrophosphorylase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001178859.1 | 93.30 | 522 | 0.0e+00 | 1019 |
| XP_006225874.1 | 91.75 | 521 | 0.0e+00 | 1004 |
| NP_001127988.1 | 56.16 | 511 | 0.0e+00 | 573 |
| NP_001019914.1 | 23.10 | 342 | 1.6e-07 | 53 |

### UBA52 (P62987)

**Description:** Ubiquitin-ribosomal protein eL40 fusion protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_113875.1 | 100.00 | 128 | 9.7e-93 | 263 |
| XP_003754135.1 | 100.00 | 76 | 8.1e-51 | 155 |
| NP_112375.1 | 100.00 | 76 | 2.4e-49 | 154 |
| NP_001292372.1 | 100.00 | 76 | 2.4e-49 | 154 |
| XP_002724552.1 | 97.37 | 76 | 7.4e-49 | 150 |
| NP_620250.2 | 98.70 | 77 | 2.4e-48 | 157 |
| NP_620250.2 | 98.70 | 77 | 2.4e-48 | 157 |
| NP_620250.2 | 100.00 | 76 | 3.7e-48 | 156 |
| NP_620250.2 | 97.40 | 77 | 7.3e-47 | 153 |
| XP_003752767.1 | 94.74 | 76 | 1.5e-47 | 147 |
| XP_006221493.1 | 97.30 | 74 | 1.6e-47 | 147 |
| NP_059010.1 | 98.70 | 77 | 1.6e-45 | 157 |
| NP_059010.1 | 98.70 | 77 | 1.6e-45 | 157 |
| NP_059010.1 | 98.70 | 77 | 1.6e-45 | 157 |
| NP_059010.1 | 98.70 | 77 | 1.6e-45 | 157 |
| NP_059010.1 | 98.70 | 77 | 1.6e-45 | 157 |
| NP_059010.1 | 98.70 | 77 | 1.6e-45 | 157 |
| NP_059010.1 | 98.70 | 77 | 1.6e-45 | 157 |
| NP_059010.1 | 98.70 | 77 | 1.6e-45 | 157 |
| NP_059010.1 | 98.70 | 77 | 1.6e-45 | 157 |
| NP_059010.1 | 98.70 | 77 | 1.6e-45 | 157 |
| NP_059010.1 | 81.48 | 27 | 9.2e-06 | 43 |
| XP_008773617.1 | 73.77 | 61 | 3.3e-24 | 90 |

### UBB (P0CG47)

**Description:** Polyubiquitin-B

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_620250.2 | 99.13 | 229 | 6.4e-162 | 449 |
| NP_620250.2 | 99.13 | 229 | 7.7e-162 | 449 |
| NP_059010.1 | 99.56 | 229 | 3.9e-156 | 452 |
| NP_059010.1 | 99.56 | 229 | 3.9e-156 | 452 |
| NP_059010.1 | 99.56 | 229 | 3.9e-156 | 452 |
| NP_059010.1 | 99.56 | 229 | 3.9e-156 | 452 |
| NP_059010.1 | 99.56 | 229 | 3.9e-156 | 452 |
| NP_059010.1 | 99.56 | 229 | 3.9e-156 | 452 |
| NP_059010.1 | 99.56 | 229 | 3.9e-156 | 452 |
| NP_059010.1 | 99.56 | 229 | 3.9e-156 | 452 |
| NP_059010.1 | 97.21 | 179 | 3.8e-113 | 342 |
| NP_113875.1 | 98.70 | 77 | 9.8e-49 | 156 |
| NP_113875.1 | 98.70 | 77 | 9.8e-49 | 156 |
| NP_113875.1 | 100.00 | 76 | 1.5e-48 | 155 |
| XP_003754135.1 | 100.00 | 76 | 2.0e-48 | 154 |
| XP_003754135.1 | 100.00 | 76 | 2.0e-48 | 154 |
| XP_003754135.1 | 100.00 | 76 | 2.0e-48 | 154 |
| NP_112375.1 | 97.44 | 78 | 2.5e-48 | 156 |
| NP_112375.1 | 97.44 | 78 | 2.5e-48 | 156 |
| NP_112375.1 | 100.00 | 76 | 3.3e-48 | 155 |
| NP_001292372.1 | 97.44 | 78 | 2.5e-48 | 156 |
| NP_001292372.1 | 97.44 | 78 | 2.5e-48 | 156 |
| NP_001292372.1 | 100.00 | 76 | 3.3e-48 | 155 |
| XP_002724552.1 | 97.37 | 76 | 1.5e-46 | 149 |
| XP_002724552.1 | 97.37 | 76 | 1.5e-46 | 149 |
| XP_002724552.1 | 97.37 | 76 | 1.5e-46 | 149 |
| XP_006221493.1 | 97.30 | 74 | 1.5e-45 | 146 |
| XP_006221493.1 | 97.30 | 74 | 1.5e-45 | 146 |
| XP_006221493.1 | 97.30 | 74 | 1.5e-45 | 146 |
| XP_003752767.1 | 94.74 | 76 | 4.0e-45 | 145 |
| XP_003752767.1 | 94.74 | 76 | 4.0e-45 | 145 |
| XP_003752767.1 | 94.74 | 76 | 4.0e-45 | 145 |
| XP_008773617.1 | 73.77 | 61 | 1.7e-22 | 89 |
| XP_008773617.1 | 73.77 | 61 | 1.7e-22 | 89 |
| XP_008773617.1 | 73.77 | 61 | 1.7e-22 | 89 |

### UBC (P0CG48)

**Description:** Polyubiquitin-C

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_059010.1 | 99.85 | 685 | 0.0e+00 | 1350 |
| NP_059010.1 | 99.85 | 685 | 0.0e+00 | 1350 |
| NP_059010.1 | 99.21 | 635 | 0.0e+00 | 1239 |
| NP_620250.2 | 99.67 | 304 | 0.0e+00 | 598 |
| NP_620250.2 | 99.67 | 304 | 0.0e+00 | 598 |
| NP_620250.2 | 99.67 | 304 | 0.0e+00 | 598 |
| NP_620250.2 | 99.67 | 304 | 0.0e+00 | 598 |
| NP_620250.2 | 99.67 | 304 | 0.0e+00 | 598 |
| NP_620250.2 | 99.67 | 304 | 0.0e+00 | 598 |
| NP_620250.2 | 99.13 | 229 | 1.1e-154 | 449 |
| NP_113875.1 | 98.70 | 77 | 3.0e-45 | 157 |
| NP_113875.1 | 98.70 | 77 | 5.2e-45 | 157 |
| NP_113875.1 | 98.70 | 77 | 5.2e-45 | 157 |
| NP_113875.1 | 98.70 | 77 | 5.2e-45 | 157 |
| NP_113875.1 | 98.70 | 77 | 5.2e-45 | 157 |
| NP_113875.1 | 98.70 | 77 | 5.2e-45 | 157 |
| NP_113875.1 | 98.70 | 77 | 5.2e-45 | 157 |
| NP_113875.1 | 98.70 | 77 | 5.2e-45 | 157 |
| NP_113875.1 | 98.70 | 77 | 5.2e-45 | 157 |
| XP_003754135.1 | 100.00 | 76 | 1.3e-44 | 154 |
| XP_003754135.1 | 100.00 | 76 | 1.3e-44 | 154 |
| XP_003754135.1 | 100.00 | 76 | 1.3e-44 | 154 |
| XP_003754135.1 | 100.00 | 76 | 1.3e-44 | 154 |
| XP_003754135.1 | 100.00 | 76 | 1.3e-44 | 154 |
| XP_003754135.1 | 100.00 | 76 | 1.3e-44 | 154 |
| XP_003754135.1 | 100.00 | 76 | 1.3e-44 | 154 |
| XP_003754135.1 | 100.00 | 76 | 1.3e-44 | 154 |
| XP_003754135.1 | 100.00 | 76 | 1.3e-44 | 154 |
| NP_112375.1 | 97.44 | 78 | 1.5e-44 | 156 |
| NP_112375.1 | 97.44 | 78 | 1.5e-44 | 156 |
| NP_112375.1 | 97.44 | 78 | 1.5e-44 | 156 |
| NP_112375.1 | 97.44 | 78 | 1.5e-44 | 156 |
| NP_112375.1 | 97.44 | 78 | 1.5e-44 | 156 |
| NP_112375.1 | 97.44 | 78 | 1.5e-44 | 156 |
| NP_112375.1 | 97.44 | 78 | 1.5e-44 | 156 |
| NP_112375.1 | 97.44 | 78 | 1.5e-44 | 156 |
| NP_112375.1 | 98.70 | 77 | 1.6e-44 | 156 |
| NP_001292372.1 | 97.44 | 78 | 1.5e-44 | 156 |
| NP_001292372.1 | 97.44 | 78 | 1.5e-44 | 156 |
| NP_001292372.1 | 97.44 | 78 | 1.5e-44 | 156 |
| NP_001292372.1 | 97.44 | 78 | 1.5e-44 | 156 |
| NP_001292372.1 | 97.44 | 78 | 1.5e-44 | 156 |
| NP_001292372.1 | 97.44 | 78 | 1.5e-44 | 156 |
| NP_001292372.1 | 97.44 | 78 | 1.5e-44 | 156 |
| NP_001292372.1 | 97.44 | 78 | 1.5e-44 | 156 |
| NP_001292372.1 | 98.70 | 77 | 1.6e-44 | 156 |
| XP_002724552.1 | 97.37 | 76 | 6.7e-43 | 149 |
| XP_002724552.1 | 97.37 | 76 | 6.7e-43 | 149 |
| XP_002724552.1 | 97.37 | 76 | 6.7e-43 | 149 |
| XP_002724552.1 | 97.37 | 76 | 6.7e-43 | 149 |
| XP_002724552.1 | 97.37 | 76 | 6.7e-43 | 149 |
| XP_002724552.1 | 97.37 | 76 | 6.7e-43 | 149 |
| XP_002724552.1 | 97.37 | 76 | 6.7e-43 | 149 |
| XP_002724552.1 | 97.37 | 76 | 6.7e-43 | 149 |
| XP_002724552.1 | 97.37 | 76 | 6.7e-43 | 149 |
| XP_006221493.1 | 97.30 | 74 | 6.9e-42 | 146 |
| XP_006221493.1 | 97.30 | 74 | 6.9e-42 | 146 |
| XP_006221493.1 | 97.30 | 74 | 6.9e-42 | 146 |
| XP_006221493.1 | 97.30 | 74 | 6.9e-42 | 146 |
| XP_006221493.1 | 97.30 | 74 | 6.9e-42 | 146 |
| XP_006221493.1 | 97.30 | 74 | 6.9e-42 | 146 |
| XP_006221493.1 | 97.30 | 74 | 6.9e-42 | 146 |
| XP_006221493.1 | 97.30 | 74 | 6.9e-42 | 146 |
| XP_006221493.1 | 97.30 | 74 | 6.9e-42 | 146 |
| XP_003752767.1 | 94.74 | 76 | 1.6e-41 | 145 |
| XP_003752767.1 | 94.74 | 76 | 1.6e-41 | 145 |
| XP_003752767.1 | 94.74 | 76 | 1.6e-41 | 145 |
| XP_003752767.1 | 94.74 | 76 | 1.6e-41 | 145 |
| XP_003752767.1 | 94.74 | 76 | 1.6e-41 | 145 |
| XP_003752767.1 | 94.74 | 76 | 1.6e-41 | 145 |
| XP_003752767.1 | 94.74 | 76 | 1.6e-41 | 145 |
| XP_003752767.1 | 94.74 | 76 | 1.6e-41 | 145 |
| XP_003752767.1 | 94.74 | 76 | 1.6e-41 | 145 |
| XP_008773617.1 | 73.77 | 61 | 5.5e-21 | 89 |
| XP_008773617.1 | 73.77 | 61 | 5.5e-21 | 89 |
| XP_008773617.1 | 73.77 | 61 | 5.5e-21 | 89 |
| XP_008773617.1 | 73.77 | 61 | 5.5e-21 | 89 |
| XP_008773617.1 | 73.77 | 61 | 5.5e-21 | 89 |
| XP_008773617.1 | 73.77 | 61 | 5.5e-21 | 89 |
| XP_008773617.1 | 73.77 | 61 | 5.5e-21 | 89 |
| XP_008773617.1 | 73.77 | 61 | 5.5e-21 | 89 |
| XP_008773617.1 | 73.77 | 61 | 5.5e-21 | 89 |

### UBXN1 (Q04323)

**Description:** UBX domain-containing protein 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001030001.1 | 91.58 | 297 | 7.2e-162 | 452 |
| NP_001012025.1 | 32.57 | 175 | 2.0e-17 | 82 |
| NP_001178721.1 | 50.82 | 61 | 5.1e-12 | 66 |
| NP_001100089.1 | 49.09 | 55 | 5.3e-11 | 63 |
| NP_001100089.1 | 36.73 | 49 | 1.0e-03 | 40 |
| NP_001007743.1 | 54.17 | 48 | 2.1e-08 | 55 |
| NP_001101135.1 | 43.14 | 51 | 9.7e-08 | 53 |
| XP_017443416.1 | 29.86 | 144 | 5.0e-06 | 48 |
| NP_001102282.1 | 38.82 | 85 | 2.2e-04 | 42 |
| NP_569090.1 | 28.12 | 192 | 7.6e-04 | 41 |

### UMOD (P07911)

**Description:** Uromodulin

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_058778.1 | 77.46 | 630 | 0.0e+00 | 1010 |
| NP_599245.2 | 53.86 | 492 | 3.6e-180 | 521 |
| NP_001100284.1 | 36.83 | 315 | 1.1e-59 | 217 |
| NP_001001507.1 | 27.93 | 537 | 6.8e-51 | 185 |
| XP_008773140.1 | 30.41 | 296 | 8.1e-22 | 101 |
| XP_001066690.3 | 28.52 | 291 | 8.7e-20 | 94 |
| NP_114014.1 | 45.08 | 122 | 7.2e-19 | 92 |
| NP_114014.1 | 46.43 | 112 | 6.5e-17 | 85 |
| NP_114014.1 | 32.55 | 212 | 1.0e-13 | 75 |
| NP_114014.1 | 32.10 | 190 | 4.1e-13 | 73 |
| NP_114014.1 | 36.80 | 125 | 7.1e-13 | 72 |
| NP_114014.1 | 42.99 | 107 | 1.0e-11 | 68 |
| NP_114014.1 | 46.81 | 94 | 2.7e-11 | 67 |
| NP_114014.1 | 36.05 | 147 | 1.4e-10 | 65 |
| NP_114014.1 | 36.29 | 124 | 3.7e-09 | 60 |
| NP_114014.1 | 41.51 | 106 | 4.2e-09 | 60 |
| NP_114014.1 | 41.03 | 117 | 4.7e-09 | 60 |
| NP_114014.1 | 37.07 | 116 | 6.7e-09 | 59 |
| NP_114014.1 | 36.52 | 115 | 7.6e-09 | 59 |
| NP_114014.1 | 43.16 | 95 | 1.4e-08 | 58 |
| NP_114014.1 | 36.00 | 125 | 1.0e-07 | 55 |
| NP_114014.1 | 33.09 | 139 | 1.5e-07 | 55 |
| NP_114014.1 | 38.78 | 98 | 2.4e-05 | 48 |
| NP_114014.1 | 41.54 | 65 | 1.3e-04 | 45 |
| NP_114014.1 | 36.23 | 138 | 2.5e-04 | 44 |
| XP_006225380.1 | 42.50 | 120 | 2.0e-18 | 89 |
| XP_006225379.1 | 42.50 | 120 | 2.2e-18 | 89 |
| XP_006225378.1 | 42.50 | 120 | 2.5e-18 | 89 |

### VCP (P55072)

**Description:** Transitional endoplasmic reticulum ATPase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_446316.1 | 99.88 | 806 | 0.0e+00 | 1645 |
| NP_001099450.1 | 43.91 | 624 | 4.6e-163 | 494 |
| NP_001099450.1 | 40.77 | 260 | 3.2e-53 | 199 |
| NP_001102019.1 | 52.93 | 444 | 2.5e-158 | 482 |
| NP_001102019.1 | 37.40 | 238 | 2.4e-45 | 175 |
| NP_001103117.1 | 35.68 | 695 | 2.5e-129 | 404 |
| NP_476466.1 | 35.74 | 526 | 7.0e-84 | 288 |
| NP_476466.1 | 40.89 | 269 | 3.9e-44 | 172 |
| NP_476466.1 | 26.32 | 228 | 4.9e-08 | 57 |
| NP_001102690.1 | 35.37 | 475 | 1.7e-80 | 282 |
| NP_001102690.1 | 45.61 | 228 | 2.9e-53 | 201 |
| NP_001128351.1 | 43.96 | 273 | 7.2e-64 | 233 |
| NP_001128351.1 | 41.20 | 233 | 2.0e-48 | 186 |
| NP_476464.1 | 47.14 | 227 | 9.6e-62 | 215 |
| NP_476464.1 | 35.83 | 307 | 2.2e-57 | 202 |
| NP_150239.1 | 37.18 | 277 | 4.1e-60 | 210 |
| NP_150239.1 | 42.17 | 230 | 1.4e-56 | 200 |
| NP_001093979.1 | 40.23 | 266 | 1.5e-59 | 207 |
| NP_001093979.1 | 41.50 | 253 | 7.9e-56 | 197 |

