# Cross-Species Pathway Analysis Report for Drosophila melanogaster

This report summarizes the BLAST search results for pathway enzymes.

## Run Parameters

- Report generated on: 2025-06-27 18:59:28
- Species: Drosophila melanogaster (Tax ID: 7227)
- Number of threads used for BLAST: 8
- Query File: input/human_glycosylation_pathway_proteins.fasta
- BLAST program: blastp
- BLAST output format: 6 std qlen slen (Query ID, Subject ID, % Identity, Alignment Length, Mismatches, Gap Opens, Query Start, Query End, Subject Start, Subject End, E-value, Bit Score, Query Length, Subject Length)

### BLAST Parameters

- E-value cutoff: 0.001
- Max target sequences: 10
- Word size: 3
- Scoring matrix: BLOSUM62
- Gap open penalty: 11
- Gap extension penalty: 1

## BLAST Results for Drosophila melanogaster (Tax ID: 7227)

### ABCA6 (Q8N139)

**Description:** ATP-binding cassette sub-family A member 6

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001259765.1 | 25.94 | 1546 | 1.8e-143 | 484 |
| NP_001259765.1 | 26.11 | 609 | 1.2e-45 | 182 |
| NP_001259764.1 | 26.20 | 1542 | 3.2e-143 | 484 |
| NP_001259764.1 | 26.11 | 609 | 1.5e-45 | 181 |
| NP_608445.2 | 25.92 | 1547 | 3.8e-143 | 483 |
| NP_608445.2 | 26.11 | 609 | 1.5e-45 | 181 |
| NP_730301.3 | 26.86 | 1221 | 1.0e-102 | 362 |
| NP_730301.3 | 20.78 | 664 | 1.1e-22 | 106 |
| NP_001036360.1 | 24.01 | 1391 | 1.5e-90 | 326 |
| NP_001036360.1 | 26.30 | 346 | 1.9e-30 | 132 |
| NP_611464.3 | 23.52 | 1399 | 1.0e-87 | 317 |
| NP_611464.3 | 29.81 | 312 | 4.7e-32 | 137 |
| NP_649002.3 | 24.11 | 1149 | 2.6e-75 | 278 |
| NP_649002.3 | 34.59 | 344 | 1.5e-50 | 198 |
| NP_649002.3 | 31.99 | 347 | 2.1e-46 | 184 |
| NP_649002.3 | 23.67 | 638 | 1.1e-35 | 149 |
| NP_732473.2 | 24.48 | 678 | 1.5e-44 | 178 |
| NP_732473.2 | 29.23 | 325 | 3.7e-38 | 157 |
| NP_732473.2 | 30.30 | 297 | 4.7e-37 | 154 |
| NP_732473.2 | 22.42 | 330 | 2.3e-16 | 86 |
| NP_001262752.1 | 24.48 | 678 | 1.5e-44 | 178 |
| NP_001262752.1 | 29.23 | 325 | 3.7e-38 | 157 |
| NP_001262752.1 | 30.30 | 297 | 4.7e-37 | 154 |
| NP_001262752.1 | 22.42 | 330 | 2.3e-16 | 86 |
| NP_608444.2 | 20.96 | 954 | 1.4e-40 | 165 |
| NP_608444.2 | 24.38 | 484 | 1.1e-18 | 93 |
| NP_608444.2 | 25.98 | 281 | 3.9e-17 | 88 |
| NP_608444.2 | 22.48 | 298 | 9.9e-13 | 74 |

### ABL1 (P00519)

**Description:** Tyrosine-protein kinase ABL1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_524843.2 | 75.32 | 470 | 0.0e+00 | 781 |
| NP_001287085.1 | 75.32 | 470 | 0.0e+00 | 781 |
| NP_001261964.1 | 75.32 | 470 | 0.0e+00 | 781 |
| NP_001261959.1 | 75.32 | 470 | 0.0e+00 | 781 |
| NP_001261963.1 | 75.32 | 470 | 0.0e+00 | 781 |
| NP_001261962.1 | 75.32 | 470 | 0.0e+00 | 781 |
| NP_001097623.1 | 75.32 | 470 | 0.0e+00 | 781 |
| NP_001261965.1 | 75.32 | 470 | 0.0e+00 | 780 |
| NP_001261961.1 | 75.32 | 470 | 0.0e+00 | 780 |
| NP_001261960.1 | 75.32 | 470 | 0.0e+00 | 780 |

### ABL2 (P42684)

**Description:** Tyrosine-protein kinase ABL2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_524843.2 | 72.13 | 470 | 0.0e+00 | 739 |
| NP_001287085.1 | 72.13 | 470 | 0.0e+00 | 739 |
| NP_001097623.1 | 72.13 | 470 | 0.0e+00 | 738 |
| NP_001261965.1 | 72.13 | 470 | 0.0e+00 | 738 |
| NP_001261961.1 | 72.13 | 470 | 0.0e+00 | 738 |
| NP_001261964.1 | 72.13 | 470 | 0.0e+00 | 738 |
| NP_001261962.1 | 72.13 | 470 | 0.0e+00 | 738 |
| NP_001261960.1 | 72.13 | 470 | 0.0e+00 | 738 |
| NP_001261959.1 | 72.13 | 470 | 0.0e+00 | 738 |
| NP_001261963.1 | 72.13 | 470 | 0.0e+00 | 737 |

### ADAMTS10 (Q9H324)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 10

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_572247.2 | 37.82 | 698 | 8.7e-132 | 426 |
| NP_572247.2 | 31.10 | 164 | 3.2e-07 | 55 |
| NP_732036.2 | 30.31 | 970 | 2.4e-108 | 371 |
| NP_732036.2 | 24.70 | 251 | 1.3e-12 | 73 |
| NP_001262604.1 | 30.31 | 970 | 3.4e-108 | 371 |
| NP_001262604.1 | 24.70 | 251 | 1.5e-12 | 72 |
| NP_996218.1 | 30.24 | 969 | 1.1e-107 | 371 |
| NP_996218.1 | 24.80 | 250 | 1.6e-12 | 72 |
| NP_001162678.1 | 40.83 | 480 | 1.1e-100 | 336 |
| NP_001162678.1 | 31.10 | 164 | 3.3e-07 | 55 |
| NP_001163761.1 | 35.65 | 533 | 1.8e-84 | 302 |
| NP_001163761.1 | 29.49 | 234 | 6.6e-25 | 113 |
| NP_788751.2 | 35.65 | 533 | 1.9e-84 | 302 |
| NP_788751.2 | 29.49 | 234 | 6.8e-25 | 113 |
| NP_788752.2 | 35.65 | 533 | 2.3e-84 | 302 |
| NP_788752.2 | 29.49 | 234 | 7.8e-25 | 113 |
| NP_001163760.1 | 35.65 | 533 | 2.7e-84 | 301 |
| NP_001163760.1 | 29.13 | 254 | 7.3e-25 | 113 |
| NP_001162939.2 | 24.45 | 593 | 7.2e-49 | 188 |
| NP_001162939.2 | 27.91 | 326 | 4.3e-20 | 97 |
| NP_001162939.2 | 31.03 | 116 | 3.5e-06 | 52 |

### ADAMTS12 (P58397)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 12

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_572247.2 | 40.39 | 718 | 1.5e-172 | 547 |
| NP_572247.2 | 26.58 | 158 | 1.8e-09 | 63 |
| NP_572247.2 | 32.77 | 119 | 6.7e-05 | 48 |
| NP_001162678.1 | 46.12 | 438 | 2.8e-122 | 402 |
| NP_001162678.1 | 26.58 | 158 | 2.9e-09 | 62 |
| NP_996218.1 | 29.75 | 911 | 4.9e-102 | 362 |
| NP_996218.1 | 24.11 | 253 | 1.9e-09 | 63 |
| NP_996218.1 | 28.93 | 197 | 1.5e-08 | 60 |
| NP_732036.2 | 29.75 | 911 | 5.2e-102 | 360 |
| NP_732036.2 | 24.11 | 253 | 2.1e-09 | 63 |
| NP_732036.2 | 28.93 | 197 | 1.5e-08 | 60 |
| NP_001262604.1 | 29.75 | 911 | 8.1e-102 | 360 |
| NP_001262604.1 | 24.11 | 253 | 2.2e-09 | 63 |
| NP_001262604.1 | 28.93 | 197 | 1.5e-08 | 60 |
| NP_001163760.1 | 33.46 | 511 | 5.4e-68 | 255 |
| NP_001163760.1 | 31.12 | 286 | 2.3e-16 | 86 |
| NP_001163760.1 | 34.21 | 152 | 3.9e-15 | 82 |
| NP_001163760.1 | 28.45 | 246 | 1.3e-11 | 70 |
| NP_001163760.1 | 25.63 | 238 | 2.0e-11 | 70 |
| NP_001163760.1 | 33.08 | 130 | 8.4e-09 | 61 |
| NP_788751.2 | 33.46 | 511 | 7.5e-68 | 254 |
| NP_788751.2 | 31.12 | 286 | 2.3e-16 | 86 |
| NP_788751.2 | 34.21 | 152 | 3.6e-15 | 82 |
| NP_788751.2 | 28.45 | 246 | 1.5e-11 | 70 |
| NP_788751.2 | 25.63 | 238 | 2.1e-11 | 70 |
| NP_788751.2 | 32.82 | 131 | 8.5e-09 | 61 |
| NP_788752.2 | 33.46 | 511 | 9.5e-68 | 254 |
| NP_788752.2 | 31.12 | 286 | 3.1e-16 | 85 |
| NP_788752.2 | 34.21 | 152 | 3.8e-15 | 82 |
| NP_788752.2 | 28.45 | 246 | 1.9e-11 | 70 |
| NP_788752.2 | 25.63 | 238 | 2.4e-11 | 69 |
| NP_788752.2 | 32.82 | 131 | 9.2e-09 | 61 |
| NP_001163761.1 | 33.46 | 511 | 1.2e-67 | 254 |
| NP_001163761.1 | 31.12 | 286 | 2.9e-16 | 85 |
| NP_001163761.1 | 34.21 | 152 | 3.9e-15 | 82 |
| NP_001163761.1 | 28.45 | 246 | 2.0e-11 | 70 |
| NP_001163761.1 | 25.63 | 238 | 2.5e-11 | 69 |
| NP_001163761.1 | 32.82 | 131 | 9.2e-09 | 61 |
| NP_609406.2 | 27.54 | 541 | 9.3e-46 | 180 |
| NP_609406.2 | 34.23 | 149 | 1.2e-15 | 83 |
| NP_609406.2 | 28.70 | 223 | 2.6e-14 | 79 |
| NP_609406.2 | 29.47 | 190 | 4.9e-12 | 71 |
| NP_609406.2 | 23.05 | 308 | 9.0e-08 | 57 |

### ADAMTS13 (Q76LX8)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 13

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_572247.2 | 31.82 | 704 | 2.5e-86 | 305 |
| NP_001162678.1 | 34.28 | 458 | 5.8e-68 | 246 |
| NP_996218.1 | 24.44 | 1109 | 3.0e-55 | 213 |
| NP_996218.1 | 28.31 | 166 | 8.8e-04 | 44 |
| NP_732036.2 | 24.44 | 1109 | 3.1e-55 | 212 |
| NP_732036.2 | 28.31 | 166 | 7.7e-04 | 44 |
| NP_001262604.1 | 24.44 | 1109 | 3.2e-55 | 212 |
| NP_001262604.1 | 28.31 | 166 | 8.2e-04 | 44 |
| NP_788751.2 | 26.89 | 688 | 4.3e-43 | 173 |
| NP_788751.2 | 26.45 | 363 | 4.7e-13 | 75 |
| NP_788751.2 | 30.27 | 185 | 3.5e-11 | 68 |
| NP_788751.2 | 24.11 | 336 | 3.2e-09 | 62 |
| NP_001163761.1 | 26.89 | 688 | 4.6e-43 | 173 |
| NP_001163761.1 | 26.45 | 363 | 4.4e-13 | 75 |
| NP_001163761.1 | 30.27 | 185 | 3.5e-11 | 68 |
| NP_001163761.1 | 24.11 | 336 | 3.2e-09 | 62 |
| NP_788752.2 | 26.89 | 688 | 7.0e-43 | 173 |
| NP_788752.2 | 26.45 | 363 | 5.4e-13 | 75 |
| NP_788752.2 | 30.27 | 185 | 3.9e-11 | 68 |
| NP_788752.2 | 24.11 | 336 | 3.5e-09 | 62 |
| NP_001163760.1 | 26.89 | 688 | 1.0e-42 | 172 |
| NP_001163760.1 | 26.45 | 363 | 1.6e-12 | 73 |
| NP_001163760.1 | 30.27 | 185 | 4.7e-11 | 68 |
| NP_001163760.1 | 24.59 | 305 | 6.2e-09 | 61 |
| NP_001246469.1 | 27.37 | 537 | 5.4e-38 | 155 |

### ADAMTS14 (Q8WXS8)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 14

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_572247.2 | 31.92 | 730 | 6.1e-98 | 336 |
| NP_572247.2 | 34.82 | 112 | 3.1e-08 | 58 |
| NP_572247.2 | 42.55 | 47 | 3.2e-04 | 45 |
| NP_996218.1 | 29.77 | 964 | 2.4e-93 | 331 |
| NP_732036.2 | 29.77 | 964 | 3.6e-93 | 329 |
| NP_001262604.1 | 29.77 | 964 | 3.8e-93 | 330 |
| NP_001162678.1 | 34.01 | 444 | 1.1e-67 | 244 |
| NP_001162678.1 | 34.82 | 112 | 2.9e-08 | 58 |
| NP_001163761.1 | 33.74 | 486 | 1.3e-60 | 229 |
| NP_001163761.1 | 29.89 | 184 | 9.1e-15 | 80 |
| NP_788751.2 | 33.74 | 486 | 1.3e-60 | 229 |
| NP_788751.2 | 29.89 | 184 | 9.3e-15 | 80 |
| NP_001163760.1 | 33.74 | 486 | 1.8e-60 | 229 |
| NP_001163760.1 | 29.89 | 184 | 7.3e-15 | 80 |
| NP_788752.2 | 33.74 | 486 | 1.9e-60 | 229 |
| NP_788752.2 | 29.89 | 184 | 1.0e-14 | 80 |
| NP_001162939.2 | 25.78 | 547 | 6.3e-42 | 167 |
| NP_001162939.2 | 24.71 | 255 | 1.7e-09 | 62 |

### ADAMTS16 (Q8TE57)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 16

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_572247.2 | 37.00 | 673 | 1.3e-130 | 426 |
| NP_572247.2 | 27.78 | 162 | 7.6e-09 | 60 |
| NP_001162678.1 | 41.27 | 441 | 2.2e-100 | 337 |
| NP_001162678.1 | 27.78 | 162 | 1.2e-08 | 60 |
| NP_732036.2 | 27.20 | 1180 | 7.1e-99 | 346 |
| NP_001262604.1 | 27.20 | 1180 | 1.1e-98 | 346 |
| NP_996218.1 | 27.20 | 1180 | 1.1e-98 | 347 |
| NP_001163760.1 | 32.74 | 672 | 2.0e-92 | 328 |
| NP_788751.2 | 32.74 | 672 | 2.7e-92 | 328 |
| NP_788752.2 | 32.74 | 672 | 3.8e-92 | 328 |
| NP_001163761.1 | 32.74 | 672 | 4.3e-92 | 328 |
| NP_609406.2 | 26.96 | 727 | 6.0e-60 | 223 |

### ADAMTS17 (Q8TE56)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 17

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_572247.2 | 33.24 | 707 | 2.7e-109 | 365 |
| NP_572247.2 | 27.22 | 169 | 1.1e-06 | 53 |
| NP_732036.2 | 28.22 | 1056 | 9.7e-88 | 311 |
| NP_732036.2 | 25.79 | 252 | 4.9e-10 | 64 |
| NP_001262604.1 | 28.22 | 1056 | 1.2e-87 | 311 |
| NP_001262604.1 | 25.79 | 252 | 4.4e-10 | 64 |
| NP_996218.1 | 28.06 | 1055 | 1.2e-87 | 311 |
| NP_996218.1 | 25.79 | 252 | 5.2e-10 | 64 |
| NP_001162678.1 | 35.05 | 465 | 9.1e-79 | 275 |
| NP_001162678.1 | 27.22 | 169 | 6.3e-07 | 54 |
| NP_788751.2 | 29.84 | 553 | 4.0e-52 | 201 |
| NP_788751.2 | 32.06 | 209 | 5.2e-17 | 87 |
| NP_788751.2 | 33.06 | 121 | 2.1e-07 | 56 |
| NP_788751.2 | 38.54 | 96 | 3.6e-07 | 55 |
| NP_788752.2 | 29.84 | 553 | 4.1e-52 | 201 |
| NP_788752.2 | 32.06 | 209 | 5.6e-17 | 87 |
| NP_788752.2 | 33.06 | 121 | 2.1e-07 | 56 |
| NP_788752.2 | 38.54 | 96 | 3.8e-07 | 55 |
| NP_001163761.1 | 29.84 | 553 | 5.5e-52 | 201 |
| NP_001163761.1 | 32.06 | 209 | 6.1e-17 | 87 |
| NP_001163761.1 | 33.06 | 121 | 2.1e-07 | 56 |
| NP_001163761.1 | 38.54 | 96 | 4.1e-07 | 55 |
| NP_001163760.1 | 29.66 | 553 | 9.5e-52 | 200 |
| NP_001163760.1 | 32.06 | 209 | 8.1e-17 | 87 |
| NP_001163760.1 | 33.06 | 121 | 2.2e-07 | 56 |
| NP_001163760.1 | 38.54 | 96 | 3.7e-07 | 55 |
| NP_001162939.2 | 26.77 | 594 | 2.2e-50 | 193 |
| NP_001162939.2 | 31.52 | 165 | 8.1e-12 | 70 |
| NP_001162939.2 | 25.83 | 271 | 2.1e-10 | 65 |

### ADAMTS18 (Q8TE60)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 18

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_572247.2 | 32.26 | 995 | 3.2e-156 | 494 |
| NP_572247.2 | 33.08 | 133 | 3.9e-06 | 52 |
| NP_001162678.1 | 33.85 | 715 | 4.8e-120 | 390 |
| NP_732036.2 | 27.85 | 1192 | 1.0e-110 | 381 |
| NP_001262604.1 | 27.85 | 1192 | 1.1e-110 | 381 |
| NP_996218.1 | 27.83 | 1193 | 2.6e-110 | 381 |
| NP_001163760.1 | 33.19 | 720 | 5.6e-98 | 345 |
| NP_788751.2 | 33.19 | 720 | 1.2e-97 | 345 |
| NP_001163761.1 | 33.19 | 720 | 1.4e-97 | 345 |
| NP_788752.2 | 33.19 | 720 | 1.7e-97 | 344 |
| NP_001162939.2 | 29.08 | 736 | 9.5e-76 | 270 |

### ADAMTS2 (O95450)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_572247.2 | 32.43 | 703 | 2.7e-96 | 332 |
| NP_572247.2 | 28.16 | 174 | 7.1e-06 | 51 |
| NP_572247.2 | 36.36 | 88 | 5.3e-04 | 45 |
| NP_001262604.1 | 28.96 | 1053 | 1.3e-92 | 328 |
| NP_001262604.1 | 28.36 | 208 | 5.7e-10 | 64 |
| NP_732036.2 | 28.96 | 1053 | 1.5e-92 | 327 |
| NP_732036.2 | 28.36 | 208 | 5.5e-10 | 64 |
| NP_996218.1 | 28.96 | 1053 | 2.2e-92 | 328 |
| NP_996218.1 | 28.36 | 208 | 6.2e-10 | 64 |
| NP_001163761.1 | 33.47 | 481 | 2.9e-64 | 241 |
| NP_001163761.1 | 31.52 | 165 | 3.9e-13 | 75 |
| NP_788751.2 | 33.47 | 481 | 3.2e-64 | 241 |
| NP_788751.2 | 31.52 | 165 | 3.9e-13 | 75 |
| NP_001163760.1 | 33.47 | 481 | 3.4e-64 | 241 |
| NP_001163760.1 | 31.52 | 165 | 3.3e-13 | 75 |
| NP_788752.2 | 33.47 | 481 | 3.8e-64 | 241 |
| NP_788752.2 | 31.52 | 165 | 4.5e-13 | 75 |
| NP_001162678.1 | 33.33 | 453 | 2.4e-62 | 228 |
| NP_001162678.1 | 28.16 | 174 | 6.2e-06 | 51 |
| NP_001162939.2 | 26.52 | 543 | 7.1e-39 | 157 |
| NP_001162939.2 | 26.14 | 241 | 1.2e-10 | 66 |
| NP_001162939.2 | 35.95 | 89 | 4.4e-06 | 51 |

### ADAMTS20 (P59510)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 20

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_732036.2 | 32.23 | 1238 | 3.2e-170 | 561 |
| NP_732036.2 | 25.38 | 654 | 1.2e-21 | 103 |
| NP_732036.2 | 28.17 | 284 | 1.9e-14 | 80 |
| NP_732036.2 | 22.85 | 372 | 2.0e-13 | 76 |
| NP_001262604.1 | 32.23 | 1238 | 9.3e-170 | 561 |
| NP_001262604.1 | 25.38 | 654 | 1.5e-21 | 103 |
| NP_001262604.1 | 28.17 | 284 | 2.2e-14 | 80 |
| NP_001262604.1 | 22.85 | 372 | 2.8e-13 | 76 |
| NP_996218.1 | 32.23 | 1238 | 8.9e-169 | 560 |
| NP_996218.1 | 25.38 | 654 | 1.1e-21 | 103 |
| NP_996218.1 | 28.17 | 284 | 2.4e-14 | 79 |
| NP_996218.1 | 22.85 | 372 | 2.8e-13 | 76 |
| NP_572247.2 | 36.06 | 635 | 2.1e-111 | 382 |
| NP_572247.2 | 34.71 | 121 | 2.1e-08 | 60 |
| NP_572247.2 | 32.50 | 120 | 1.0e-05 | 51 |
| NP_572247.2 | 24.26 | 202 | 1.3e-05 | 50 |
| NP_572247.2 | 28.93 | 121 | 1.2e-04 | 47 |
| NP_572247.2 | 28.15 | 135 | 3.8e-04 | 46 |
| NP_001162678.1 | 39.60 | 447 | 1.6e-92 | 319 |
| NP_001162678.1 | 24.26 | 202 | 1.3e-05 | 50 |
| NP_788751.2 | 30.17 | 643 | 4.2e-64 | 243 |
| NP_788751.2 | 26.68 | 431 | 4.5e-29 | 128 |
| NP_788751.2 | 32.92 | 243 | 4.1e-22 | 105 |
| NP_788751.2 | 31.32 | 281 | 8.1e-22 | 104 |
| NP_788751.2 | 25.79 | 380 | 6.3e-21 | 101 |
| NP_788751.2 | 25.34 | 363 | 1.8e-15 | 83 |
| NP_788751.2 | 29.24 | 171 | 2.7e-11 | 69 |
| NP_788751.2 | 24.31 | 362 | 7.6e-11 | 68 |
| NP_001163761.1 | 30.17 | 643 | 7.3e-64 | 243 |
| NP_001163761.1 | 26.68 | 431 | 5.5e-29 | 128 |
| NP_001163761.1 | 32.02 | 253 | 4.9e-22 | 105 |
| NP_001163761.1 | 31.32 | 281 | 1.1e-21 | 104 |
| NP_001163761.1 | 25.79 | 380 | 7.2e-21 | 101 |
| NP_001163761.1 | 25.34 | 363 | 2.3e-15 | 83 |
| NP_001163761.1 | 29.24 | 171 | 2.9e-11 | 69 |
| NP_001163761.1 | 24.31 | 362 | 7.7e-11 | 68 |
| NP_788752.2 | 30.17 | 643 | 7.5e-64 | 243 |
| NP_788752.2 | 26.68 | 431 | 5.9e-29 | 128 |
| NP_788752.2 | 32.92 | 243 | 4.9e-22 | 105 |
| NP_788752.2 | 31.32 | 281 | 1.1e-21 | 103 |
| NP_788752.2 | 25.79 | 380 | 6.9e-21 | 101 |
| NP_788752.2 | 25.34 | 363 | 2.2e-15 | 83 |
| NP_788752.2 | 29.24 | 171 | 2.9e-11 | 69 |
| NP_788752.2 | 24.31 | 362 | 8.4e-11 | 68 |
| NP_001163760.1 | 30.17 | 643 | 8.7e-64 | 242 |
| NP_001163760.1 | 26.68 | 431 | 4.8e-29 | 128 |
| NP_001163760.1 | 32.92 | 243 | 3.8e-22 | 105 |
| NP_001163760.1 | 31.32 | 281 | 1.2e-21 | 103 |
| NP_001163760.1 | 25.79 | 380 | 7.6e-21 | 101 |
| NP_001163760.1 | 25.34 | 363 | 2.8e-15 | 82 |
| NP_001163760.1 | 29.24 | 171 | 2.5e-11 | 70 |
| NP_001163760.1 | 25.08 | 315 | 1.0e-10 | 67 |
| NP_001162939.2 | 25.32 | 703 | 2.7e-44 | 176 |
| NP_001162939.2 | 25.95 | 343 | 9.4e-19 | 94 |
| NP_001162939.2 | 24.19 | 339 | 9.0e-15 | 80 |
| NP_001162939.2 | 28.41 | 183 | 1.8e-07 | 57 |
| NP_001162939.2 | 21.85 | 325 | 1.3e-04 | 47 |

### ADAMTS4 (O75173)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_572247.2 | 37.60 | 601 | 8.9e-111 | 363 |
| NP_001262604.1 | 33.16 | 781 | 6.8e-107 | 360 |
| NP_732036.2 | 33.16 | 781 | 1.1e-106 | 360 |
| NP_996218.1 | 33.16 | 781 | 1.6e-106 | 360 |
| NP_001162678.1 | 39.20 | 449 | 1.5e-88 | 297 |
| NP_788751.2 | 34.32 | 303 | 4.3e-40 | 161 |
| NP_001163761.1 | 34.32 | 303 | 4.8e-40 | 161 |
| NP_788752.2 | 34.32 | 303 | 5.1e-40 | 161 |
| NP_001163760.1 | 34.32 | 303 | 5.4e-40 | 160 |
| NP_001246469.1 | 25.60 | 539 | 3.3e-32 | 135 |

### ADAMTS6 (Q9UKP5)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 6

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_572247.2 | 38.96 | 729 | 4.5e-157 | 494 |
| NP_572247.2 | 33.53 | 167 | 3.1e-15 | 81 |
| NP_572247.2 | 28.46 | 130 | 4.9e-05 | 48 |
| NP_732036.2 | 29.85 | 1025 | 2.8e-118 | 400 |
| NP_732036.2 | 29.08 | 251 | 1.5e-15 | 82 |
| NP_001262604.1 | 29.85 | 1025 | 2.9e-118 | 400 |
| NP_001262604.1 | 29.08 | 251 | 1.5e-15 | 82 |
| NP_996218.1 | 29.92 | 1026 | 7.0e-118 | 400 |
| NP_996218.1 | 29.08 | 251 | 1.6e-15 | 82 |
| NP_001162678.1 | 42.35 | 477 | 5.3e-110 | 361 |
| NP_001162678.1 | 33.53 | 167 | 3.1e-15 | 81 |
| NP_001163760.1 | 38.65 | 502 | 2.9e-96 | 338 |
| NP_001163760.1 | 32.33 | 232 | 6.4e-23 | 107 |
| NP_001163760.1 | 40.45 | 89 | 1.6e-11 | 69 |
| NP_788751.2 | 38.65 | 502 | 3.3e-96 | 338 |
| NP_788751.2 | 32.31 | 229 | 4.6e-23 | 107 |
| NP_788751.2 | 40.45 | 89 | 1.8e-11 | 69 |
| NP_001163761.1 | 38.65 | 502 | 3.4e-96 | 338 |
| NP_001163761.1 | 32.31 | 229 | 4.1e-23 | 107 |
| NP_001163761.1 | 40.45 | 89 | 1.8e-11 | 69 |
| NP_788752.2 | 38.65 | 502 | 4.3e-96 | 338 |
| NP_788752.2 | 32.31 | 229 | 4.8e-23 | 107 |
| NP_788752.2 | 40.45 | 89 | 2.0e-11 | 69 |
| NP_001162939.2 | 25.67 | 705 | 4.5e-61 | 226 |

### ADAMTS9 (Q9P2N4)

**Description:** A disintegrin and metalloproteinase with thrombospondin motifs 9

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_732036.2 | 32.26 | 1330 | 1.1e-171 | 565 |
| NP_732036.2 | 30.69 | 202 | 2.4e-21 | 102 |
| NP_732036.2 | 25.88 | 371 | 5.8e-19 | 95 |
| NP_732036.2 | 26.54 | 309 | 7.6e-12 | 71 |
| NP_732036.2 | 23.38 | 402 | 3.3e-09 | 62 |
| NP_001262604.1 | 32.26 | 1330 | 2.7e-171 | 566 |
| NP_001262604.1 | 30.69 | 202 | 2.4e-21 | 102 |
| NP_001262604.1 | 25.88 | 371 | 6.5e-19 | 94 |
| NP_001262604.1 | 26.54 | 309 | 8.6e-12 | 71 |
| NP_001262604.1 | 23.38 | 402 | 3.4e-09 | 62 |
| NP_996218.1 | 32.26 | 1330 | 1.2e-170 | 566 |
| NP_996218.1 | 26.94 | 542 | 8.4e-32 | 137 |
| NP_996218.1 | 25.88 | 371 | 6.3e-19 | 94 |
| NP_996218.1 | 25.57 | 391 | 6.3e-16 | 85 |
| NP_996218.1 | 26.54 | 309 | 8.3e-12 | 71 |
| NP_996218.1 | 23.38 | 402 | 3.8e-09 | 62 |
| NP_572247.2 | 30.55 | 1031 | 1.6e-124 | 419 |
| NP_572247.2 | 32.52 | 123 | 2.5e-07 | 56 |
| NP_572247.2 | 33.33 | 120 | 2.6e-07 | 56 |
| NP_572247.2 | 35.83 | 120 | 3.1e-07 | 56 |
| NP_572247.2 | 30.71 | 127 | 6.6e-07 | 55 |
| NP_572247.2 | 35.04 | 117 | 1.2e-06 | 54 |
| NP_572247.2 | 31.20 | 125 | 3.8e-06 | 52 |
| NP_572247.2 | 32.90 | 76 | 2.5e-04 | 46 |
| NP_001162678.1 | 31.81 | 767 | 3.2e-102 | 348 |
| NP_788751.2 | 32.03 | 640 | 3.3e-72 | 270 |
| NP_788751.2 | 29.27 | 410 | 5.9e-32 | 137 |
| NP_788751.2 | 32.34 | 337 | 9.1e-32 | 137 |
| NP_788751.2 | 31.11 | 180 | 3.1e-17 | 89 |
| NP_001163760.1 | 32.03 | 640 | 3.7e-72 | 269 |
| NP_001163760.1 | 32.34 | 337 | 7.1e-32 | 137 |
| NP_001163760.1 | 29.27 | 410 | 7.9e-32 | 137 |
| NP_001163760.1 | 31.11 | 180 | 2.6e-17 | 89 |
| NP_001163761.1 | 32.03 | 640 | 3.8e-72 | 270 |
| NP_001163761.1 | 29.27 | 410 | 7.1e-32 | 137 |
| NP_001163761.1 | 32.34 | 337 | 1.0e-31 | 137 |
| NP_001163761.1 | 31.11 | 180 | 3.3e-17 | 89 |
| NP_788752.2 | 32.03 | 640 | 5.0e-72 | 269 |
| NP_788752.2 | 29.27 | 410 | 8.5e-32 | 137 |
| NP_788752.2 | 32.34 | 337 | 1.3e-31 | 136 |
| NP_788752.2 | 31.11 | 180 | 3.6e-17 | 89 |
| NP_609406.2 | 25.41 | 732 | 6.2e-47 | 184 |
| NP_609406.2 | 30.49 | 305 | 1.6e-24 | 112 |
| NP_609406.2 | 27.30 | 337 | 4.3e-18 | 91 |
| NP_609406.2 | 27.19 | 320 | 1.4e-17 | 90 |
| NP_609406.2 | 24.93 | 369 | 1.1e-15 | 84 |
| NP_609406.2 | 26.82 | 220 | 4.2e-09 | 62 |
| NP_609406.2 | 31.49 | 181 | 2.5e-08 | 59 |

### ADAMTSL1 (Q8N6G6)

**Description:** ADAMTS-like protein 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001163760.1 | 29.48 | 719 | 3.6e-70 | 262 |
| NP_001163760.1 | 28.54 | 403 | 1.4e-27 | 123 |
| NP_001163760.1 | 26.77 | 310 | 4.5e-22 | 105 |
| NP_001163760.1 | 25.44 | 401 | 1.4e-21 | 103 |
| NP_001163760.1 | 30.43 | 230 | 2.2e-17 | 89 |
| NP_001163760.1 | 30.81 | 198 | 1.3e-11 | 70 |
| NP_001163760.1 | 35.92 | 103 | 5.0e-11 | 68 |
| NP_001163760.1 | 28.80 | 191 | 2.4e-10 | 66 |
| NP_001163760.1 | 32.28 | 127 | 1.1e-07 | 57 |
| NP_001163760.1 | 22.45 | 196 | 2.0e-07 | 57 |
| NP_001163760.1 | 25.78 | 225 | 2.3e-07 | 56 |
| NP_001163760.1 | 28.24 | 131 | 3.7e-07 | 56 |
| NP_001163761.1 | 29.48 | 719 | 4.0e-70 | 263 |
| NP_001163761.1 | 28.54 | 403 | 2.4e-27 | 122 |
| NP_001163761.1 | 26.77 | 310 | 4.2e-22 | 105 |
| NP_001163761.1 | 25.50 | 400 | 1.3e-21 | 103 |
| NP_001163761.1 | 24.16 | 327 | 1.3e-18 | 94 |
| NP_001163761.1 | 30.70 | 228 | 4.0e-17 | 89 |
| NP_001163761.1 | 35.92 | 103 | 5.4e-11 | 68 |
| NP_001163761.1 | 28.80 | 191 | 4.3e-10 | 65 |
| NP_001163761.1 | 25.45 | 224 | 2.6e-07 | 56 |
| NP_001163761.1 | 32.28 | 127 | 2.7e-07 | 56 |
| NP_001163761.1 | 28.24 | 131 | 4.7e-07 | 55 |
| NP_788751.2 | 29.48 | 719 | 4.1e-70 | 263 |
| NP_788751.2 | 28.54 | 403 | 2.6e-27 | 122 |
| NP_788751.2 | 26.77 | 310 | 4.5e-22 | 105 |
| NP_788751.2 | 25.50 | 400 | 1.3e-21 | 103 |
| NP_788751.2 | 24.16 | 327 | 1.2e-18 | 94 |
| NP_788751.2 | 30.70 | 228 | 3.6e-17 | 89 |
| NP_788751.2 | 35.92 | 103 | 5.4e-11 | 68 |
| NP_788751.2 | 28.80 | 191 | 4.0e-10 | 65 |
| NP_788751.2 | 32.28 | 127 | 2.5e-07 | 56 |
| NP_788751.2 | 25.45 | 224 | 2.8e-07 | 56 |
| NP_788751.2 | 28.24 | 131 | 4.5e-07 | 55 |
| NP_788752.2 | 29.48 | 719 | 4.7e-70 | 262 |
| NP_788752.2 | 28.54 | 403 | 2.6e-27 | 122 |
| NP_788752.2 | 26.77 | 310 | 4.6e-22 | 105 |
| NP_788752.2 | 25.50 | 400 | 1.3e-21 | 103 |
| NP_788752.2 | 24.16 | 327 | 1.3e-18 | 94 |
| NP_788752.2 | 30.70 | 228 | 4.1e-17 | 89 |
| NP_788752.2 | 35.92 | 103 | 5.5e-11 | 68 |
| NP_788752.2 | 28.80 | 191 | 4.4e-10 | 65 |
| NP_788752.2 | 25.45 | 224 | 2.7e-07 | 56 |
| NP_788752.2 | 32.28 | 127 | 2.9e-07 | 56 |
| NP_788752.2 | 28.24 | 131 | 4.9e-07 | 55 |
| NP_724360.1 | 35.82 | 282 | 7.8e-52 | 200 |
| NP_724360.1 | 26.72 | 378 | 1.3e-32 | 139 |
| NP_724360.1 | 45.98 | 174 | 1.1e-28 | 126 |
| NP_724360.1 | 33.33 | 198 | 4.5e-23 | 107 |
| NP_724360.1 | 27.61 | 297 | 2.9e-21 | 102 |
| NP_724360.1 | 35.59 | 177 | 4.1e-21 | 101 |
| NP_724360.1 | 33.71 | 178 | 8.0e-20 | 97 |
| NP_724360.1 | 35.12 | 205 | 1.4e-19 | 96 |
| NP_724360.1 | 27.57 | 301 | 5.8e-17 | 87 |
| NP_724360.1 | 31.03 | 174 | 3.5e-16 | 85 |
| NP_724360.1 | 30.48 | 187 | 2.9e-14 | 79 |
| NP_724360.1 | 32.47 | 154 | 2.5e-13 | 76 |
| NP_724360.1 | 28.02 | 182 | 1.5e-10 | 67 |
| NP_724360.1 | 27.89 | 147 | 2.2e-07 | 56 |
| NP_724360.1 | 28.19 | 149 | 5.2e-07 | 55 |
| NP_724360.1 | 30.36 | 112 | 3.4e-04 | 46 |
| NP_001286126.1 | 35.82 | 282 | 1.4e-51 | 200 |
| NP_001286126.1 | 26.72 | 378 | 1.4e-32 | 139 |
| NP_001286126.1 | 45.98 | 174 | 8.1e-29 | 126 |
| NP_001286126.1 | 33.85 | 192 | 5.3e-23 | 107 |
| NP_001286126.1 | 27.61 | 297 | 2.4e-21 | 102 |
| NP_001286126.1 | 35.59 | 177 | 4.1e-21 | 101 |
| NP_001286126.1 | 33.71 | 178 | 9.4e-20 | 97 |
| NP_001286126.1 | 35.12 | 205 | 1.2e-19 | 97 |
| NP_001286126.1 | 27.57 | 301 | 6.2e-17 | 88 |
| NP_001286126.1 | 31.03 | 174 | 2.5e-16 | 85 |
| NP_001286126.1 | 30.48 | 187 | 2.2e-14 | 79 |
| NP_001286126.1 | 32.47 | 154 | 1.9e-13 | 76 |
| NP_001286126.1 | 28.02 | 182 | 1.9e-10 | 66 |
| NP_001286126.1 | 27.89 | 147 | 2.2e-07 | 56 |
| NP_001286126.1 | 28.19 | 149 | 3.8e-07 | 55 |
| NP_001286126.1 | 30.36 | 112 | 3.6e-04 | 46 |
| NP_001286127.1 | 35.82 | 282 | 3.4e-51 | 199 |
| NP_001286127.1 | 24.41 | 684 | 2.2e-40 | 164 |
| NP_001286127.1 | 45.98 | 174 | 1.7e-28 | 125 |
| NP_001286127.1 | 31.47 | 232 | 1.1e-26 | 120 |
| NP_001286127.1 | 35.12 | 205 | 2.1e-19 | 96 |
| NP_001286127.1 | 30.81 | 198 | 5.2e-17 | 88 |
| NP_001286127.1 | 27.57 | 301 | 8.9e-17 | 87 |
| NP_001286127.1 | 29.89 | 174 | 2.2e-14 | 79 |
| NP_001286127.1 | 30.48 | 187 | 5.0e-14 | 78 |
| NP_001286127.1 | 32.47 | 154 | 3.8e-13 | 75 |
| NP_001286127.1 | 31.10 | 164 | 8.3e-10 | 64 |
| NP_001286127.1 | 26.92 | 156 | 2.7e-07 | 56 |
| NP_001286127.1 | 25.00 | 236 | 3.2e-05 | 49 |
| NP_001286127.1 | 22.79 | 237 | 3.7e-04 | 46 |
| NP_724359.1 | 35.82 | 282 | 3.7e-51 | 199 |
| NP_724359.1 | 26.72 | 378 | 3.2e-32 | 138 |
| NP_724359.1 | 45.98 | 174 | 1.8e-28 | 125 |
| NP_724359.1 | 31.47 | 232 | 1.4e-26 | 119 |
| NP_724359.1 | 33.33 | 198 | 8.9e-23 | 107 |
| NP_724359.1 | 27.61 | 297 | 5.1e-21 | 101 |
| NP_724359.1 | 35.59 | 177 | 8.4e-21 | 100 |
| NP_724359.1 | 33.71 | 178 | 1.7e-19 | 96 |
| NP_724359.1 | 35.12 | 205 | 2.1e-19 | 96 |
| NP_724359.1 | 27.57 | 301 | 9.4e-17 | 87 |
| NP_724359.1 | 31.03 | 174 | 5.9e-16 | 84 |
| NP_724359.1 | 30.48 | 187 | 5.7e-14 | 78 |
| NP_724359.1 | 32.47 | 154 | 4.1e-13 | 75 |
| NP_724359.1 | 28.02 | 182 | 2.0e-10 | 66 |
| NP_724359.1 | 31.10 | 164 | 9.0e-10 | 64 |
| NP_724359.1 | 26.92 | 156 | 2.7e-07 | 56 |
| NP_724359.1 | 28.19 | 149 | 8.3e-07 | 54 |
| NP_724359.1 | 25.00 | 236 | 3.4e-05 | 49 |
| NP_724359.1 | 22.79 | 237 | 3.7e-04 | 46 |
| NP_724359.1 | 30.36 | 112 | 3.7e-04 | 46 |
| NP_001163040.1 | 35.82 | 282 | 3.7e-51 | 199 |
| NP_001163040.1 | 26.68 | 401 | 3.2e-32 | 138 |
| NP_001163040.1 | 45.98 | 174 | 1.6e-28 | 125 |
| NP_001163040.1 | 31.47 | 232 | 1.1e-26 | 120 |
| NP_001163040.1 | 33.33 | 198 | 7.2e-23 | 107 |
| NP_001163040.1 | 27.95 | 297 | 4.7e-21 | 101 |
| NP_001163040.1 | 35.59 | 177 | 8.1e-21 | 100 |
| NP_001163040.1 | 33.71 | 178 | 1.4e-19 | 96 |
| NP_001163040.1 | 35.12 | 205 | 1.8e-19 | 96 |
| NP_001163040.1 | 27.57 | 301 | 7.9e-17 | 87 |
| NP_001163040.1 | 31.03 | 174 | 5.9e-16 | 84 |
| NP_001163040.1 | 30.48 | 187 | 4.9e-14 | 78 |
| NP_001163040.1 | 32.47 | 154 | 3.8e-13 | 75 |
| NP_001163040.1 | 28.02 | 182 | 1.8e-10 | 67 |
| NP_001163040.1 | 31.10 | 164 | 7.9e-10 | 64 |
| NP_001163040.1 | 26.92 | 156 | 2.7e-07 | 56 |
| NP_001163040.1 | 28.19 | 149 | 7.5e-07 | 55 |
| NP_001163040.1 | 25.00 | 236 | 3.2e-05 | 49 |
| NP_001163040.1 | 30.36 | 112 | 3.9e-04 | 46 |
| NP_001163040.1 | 22.79 | 237 | 3.9e-04 | 46 |
| NP_001246128.1 | 35.82 | 282 | 3.9e-51 | 199 |
| NP_001246128.1 | 26.68 | 401 | 3.1e-32 | 138 |
| NP_001246128.1 | 45.98 | 174 | 1.5e-28 | 126 |
| NP_001246128.1 | 31.47 | 232 | 1.3e-26 | 119 |
| NP_001246128.1 | 33.33 | 198 | 6.7e-23 | 107 |
| NP_001246128.1 | 27.61 | 297 | 4.9e-21 | 101 |
| NP_001246128.1 | 35.59 | 177 | 8.0e-21 | 100 |
| NP_001246128.1 | 33.71 | 178 | 1.5e-19 | 96 |
| NP_001246128.1 | 35.12 | 205 | 1.9e-19 | 96 |
| NP_001246128.1 | 27.57 | 301 | 8.1e-17 | 87 |
| NP_001246128.1 | 31.03 | 174 | 5.9e-16 | 84 |
| NP_001246128.1 | 30.48 | 187 | 5.4e-14 | 78 |
| NP_001246128.1 | 32.47 | 154 | 4.1e-13 | 75 |
| NP_001246128.1 | 28.02 | 182 | 1.8e-10 | 67 |
| NP_001246128.1 | 31.10 | 164 | 9.0e-10 | 64 |
| NP_001246128.1 | 26.92 | 156 | 2.7e-07 | 56 |
| NP_001246128.1 | 28.19 | 149 | 8.2e-07 | 54 |
| NP_001246128.1 | 25.00 | 236 | 3.4e-05 | 49 |
| NP_001246128.1 | 30.36 | 112 | 3.8e-04 | 46 |
| NP_001246128.1 | 22.79 | 237 | 3.9e-04 | 46 |

### ADAMTSL3 (P82987)

**Description:** ADAMTS-like protein 3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001163760.1 | 30.22 | 718 | 2.7e-71 | 266 |
| NP_001163760.1 | 22.05 | 508 | 2.4e-16 | 86 |
| NP_001163760.1 | 28.29 | 251 | 2.5e-13 | 76 |
| NP_001163760.1 | 26.72 | 232 | 7.6e-10 | 64 |
| NP_001163760.1 | 30.11 | 176 | 1.6e-08 | 60 |
| NP_001163760.1 | 22.57 | 226 | 9.7e-08 | 57 |
| NP_001163760.1 | 27.32 | 216 | 3.0e-05 | 49 |
| NP_001163760.1 | 33.96 | 106 | 1.8e-04 | 47 |
| NP_788751.2 | 30.22 | 718 | 3.4e-71 | 266 |
| NP_788751.2 | 22.05 | 508 | 1.3e-16 | 87 |
| NP_788751.2 | 28.29 | 251 | 2.8e-13 | 76 |
| NP_788751.2 | 26.72 | 232 | 8.4e-10 | 64 |
| NP_788751.2 | 30.11 | 176 | 1.5e-08 | 60 |
| NP_788751.2 | 35.05 | 97 | 1.8e-08 | 60 |
| NP_788751.2 | 27.32 | 216 | 2.9e-05 | 49 |
| NP_788751.2 | 33.96 | 106 | 2.0e-04 | 47 |
| NP_788751.2 | 31.08 | 74 | 5.3e-04 | 45 |
| NP_788752.2 | 30.22 | 718 | 4.5e-71 | 265 |
| NP_788752.2 | 22.05 | 508 | 1.3e-16 | 87 |
| NP_788752.2 | 28.29 | 251 | 3.3e-13 | 75 |
| NP_788752.2 | 26.72 | 232 | 9.8e-10 | 64 |
| NP_788752.2 | 30.11 | 176 | 1.6e-08 | 60 |
| NP_788752.2 | 31.25 | 128 | 1.8e-08 | 60 |
| NP_788752.2 | 27.32 | 216 | 3.3e-05 | 49 |
| NP_788752.2 | 33.96 | 106 | 2.2e-04 | 47 |
| NP_788752.2 | 31.08 | 74 | 5.5e-04 | 45 |
| NP_001163761.1 | 30.22 | 718 | 5.0e-71 | 265 |
| NP_001163761.1 | 22.05 | 508 | 1.6e-16 | 87 |
| NP_001163761.1 | 28.29 | 251 | 3.6e-13 | 75 |
| NP_001163761.1 | 26.72 | 232 | 9.9e-10 | 64 |
| NP_001163761.1 | 30.11 | 176 | 1.8e-08 | 60 |
| NP_001163761.1 | 35.05 | 97 | 2.0e-08 | 60 |
| NP_001163761.1 | 27.32 | 216 | 3.7e-05 | 49 |
| NP_001163761.1 | 33.96 | 106 | 2.3e-04 | 47 |
| NP_001163761.1 | 31.08 | 74 | 5.9e-04 | 45 |
| NP_724360.1 | 26.49 | 706 | 7.8e-57 | 216 |
| NP_724360.1 | 45.98 | 174 | 1.2e-32 | 139 |
| NP_724360.1 | 36.59 | 164 | 1.4e-22 | 106 |
| NP_724360.1 | 32.56 | 172 | 6.6e-14 | 77 |
| NP_724360.1 | 33.08 | 133 | 8.7e-13 | 74 |
| NP_724360.1 | 36.09 | 133 | 6.7e-09 | 61 |
| NP_724360.1 | 28.11 | 249 | 8.4e-08 | 57 |
| NP_724360.1 | 34.19 | 117 | 2.4e-06 | 53 |
| NP_001286126.1 | 26.49 | 706 | 2.3e-56 | 216 |
| NP_001286126.1 | 45.98 | 174 | 9.7e-33 | 139 |
| NP_001286126.1 | 36.59 | 164 | 1.8e-22 | 105 |
| NP_001286126.1 | 32.56 | 172 | 5.4e-14 | 78 |
| NP_001286126.1 | 33.08 | 133 | 9.5e-13 | 74 |
| NP_001286126.1 | 36.09 | 133 | 8.8e-09 | 61 |
| NP_001286126.1 | 28.23 | 248 | 7.0e-08 | 58 |
| NP_001286126.1 | 34.19 | 117 | 2.5e-06 | 53 |
| NP_001286126.1 | 27.38 | 168 | 6.9e-04 | 45 |
| NP_724359.1 | 26.49 | 706 | 4.2e-56 | 215 |
| NP_724359.1 | 45.98 | 174 | 2.2e-32 | 138 |
| NP_724359.1 | 31.03 | 232 | 1.6e-22 | 106 |
| NP_724359.1 | 36.59 | 164 | 2.3e-22 | 105 |
| NP_724359.1 | 32.56 | 172 | 1.5e-13 | 77 |
| NP_724359.1 | 33.08 | 133 | 1.5e-12 | 73 |
| NP_724359.1 | 36.09 | 133 | 1.2e-08 | 60 |
| NP_724359.1 | 28.11 | 249 | 2.3e-07 | 56 |
| NP_724359.1 | 34.19 | 117 | 4.0e-06 | 52 |
| NP_724359.1 | 28.23 | 209 | 5.8e-06 | 52 |
| NP_724359.1 | 26.60 | 203 | 9.1e-05 | 48 |
| NP_724359.1 | 31.68 | 101 | 2.4e-04 | 46 |
| NP_001163040.1 | 26.49 | 706 | 4.3e-56 | 215 |
| NP_001163040.1 | 45.98 | 174 | 2.0e-32 | 138 |
| NP_001163040.1 | 31.03 | 232 | 1.4e-22 | 106 |
| NP_001163040.1 | 36.59 | 164 | 2.3e-22 | 105 |
| NP_001163040.1 | 32.56 | 172 | 1.4e-13 | 77 |
| NP_001163040.1 | 33.08 | 133 | 1.4e-12 | 73 |
| NP_001163040.1 | 36.09 | 133 | 1.2e-08 | 60 |
| NP_001163040.1 | 28.11 | 249 | 2.1e-07 | 56 |
| NP_001163040.1 | 34.19 | 117 | 3.7e-06 | 52 |
| NP_001163040.1 | 28.23 | 209 | 6.1e-06 | 52 |
| NP_001163040.1 | 26.60 | 203 | 9.8e-05 | 48 |
| NP_001163040.1 | 31.68 | 101 | 2.3e-04 | 46 |
| NP_001246128.1 | 26.49 | 706 | 4.3e-56 | 215 |
| NP_001246128.1 | 45.98 | 174 | 1.9e-32 | 138 |
| NP_001246128.1 | 31.03 | 232 | 1.2e-22 | 106 |
| NP_001246128.1 | 36.59 | 164 | 2.2e-22 | 105 |
| NP_001246128.1 | 32.56 | 172 | 1.3e-13 | 77 |
| NP_001246128.1 | 33.08 | 133 | 1.4e-12 | 73 |
| NP_001246128.1 | 36.09 | 133 | 1.2e-08 | 60 |
| NP_001246128.1 | 28.11 | 249 | 1.9e-07 | 57 |
| NP_001246128.1 | 34.19 | 117 | 3.7e-06 | 52 |
| NP_001246128.1 | 28.23 | 209 | 5.9e-06 | 52 |
| NP_001246128.1 | 26.60 | 203 | 9.6e-05 | 48 |
| NP_001246128.1 | 31.68 | 101 | 2.3e-04 | 46 |
| NP_001036374.1 | 26.49 | 706 | 4.3e-56 | 215 |
| NP_001036374.1 | 45.98 | 174 | 1.9e-32 | 138 |
| NP_001036374.1 | 31.03 | 232 | 1.2e-22 | 106 |
| NP_001036374.1 | 36.59 | 164 | 2.2e-22 | 105 |
| NP_001036374.1 | 32.56 | 172 | 1.3e-13 | 77 |
| NP_001036374.1 | 33.08 | 133 | 1.4e-12 | 73 |
| NP_001036374.1 | 36.09 | 133 | 1.2e-08 | 60 |
| NP_001036374.1 | 28.11 | 249 | 1.9e-07 | 57 |
| NP_001036374.1 | 34.19 | 117 | 3.7e-06 | 52 |
| NP_001036374.1 | 28.23 | 209 | 5.9e-06 | 52 |
| NP_001036374.1 | 26.60 | 203 | 9.6e-05 | 48 |
| NP_001036374.1 | 31.68 | 101 | 2.3e-04 | 46 |

### ADAMTSL4 (Q6UY14)

**Description:** ADAMTS-like protein 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001163761.1 | 31.04 | 683 | 1.8e-84 | 302 |
| NP_788751.2 | 31.04 | 683 | 1.9e-84 | 301 |
| NP_001163760.1 | 31.04 | 683 | 3.0e-84 | 301 |
| NP_788752.2 | 31.04 | 683 | 3.2e-84 | 301 |
| NP_609406.2 | 29.23 | 732 | 3.4e-78 | 275 |
| NP_001162939.2 | 29.14 | 731 | 1.7e-76 | 271 |
| NP_001188781.1 | 28.36 | 610 | 3.1e-56 | 205 |
| NP_001262604.1 | 24.72 | 712 | 2.4e-33 | 140 |
| NP_996218.1 | 24.72 | 712 | 2.7e-33 | 140 |
| NP_732036.2 | 24.72 | 712 | 3.1e-33 | 140 |

### ADAMTSL5 (Q6ZMM2)

**Description:** ADAMTS-like protein 5

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001163761.1 | 37.94 | 282 | 4.3e-52 | 190 |
| NP_788752.2 | 37.94 | 282 | 4.6e-52 | 190 |
| NP_788751.2 | 37.94 | 282 | 4.7e-52 | 190 |
| NP_001163760.1 | 37.94 | 282 | 7.2e-52 | 190 |
| NP_572247.2 | 36.69 | 278 | 2.2e-44 | 167 |
| NP_996218.1 | 30.41 | 319 | 2.9e-31 | 128 |
| NP_732036.2 | 30.43 | 322 | 1.1e-28 | 120 |
| NP_001262604.1 | 30.43 | 322 | 1.5e-28 | 120 |
| NP_609406.2 | 29.21 | 291 | 1.2e-25 | 111 |
| NP_001162939.2 | 29.21 | 291 | 1.3e-25 | 111 |

### ALG1 (Q9BT22)

**Description:** Chitobiosyldiphosphodolichol beta-mannosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_650662.1 | 45.77 | 437 | 8.0e-127 | 375 |

### ALG10 (Q5BKT4)

**Description:** Dol-P-Glc:Glc(2)Man(9)GlcNAc(2)-PP-Dol alpha-1,2-glucosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_729680.1 | 32.20 | 441 | 2.2e-61 | 207 |

### ALG10B (Q5I7T1)

**Description:** Dol-P-Glc:Glc(2)Man(9)GlcNAc(2)-PP-Dol alpha-1,2-glucosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_729680.1 | 32.95 | 440 | 9.4e-64 | 213 |

### ALG11 (Q2TAA5)

**Description:** GDP-Man:Man(3)GlcNAc(2)-PP-Dol alpha-1,2-mannosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_649340.1 | 51.33 | 452 | 4.9e-159 | 460 |
| NP_001262182.1 | 51.33 | 452 | 4.9e-159 | 460 |
| NP_647772.1 | 28.09 | 210 | 3.5e-12 | 68 |

### ALG12 (Q9BV10)

**Description:** Dol-P-Man:Man(7)GlcNAc(2)-PP-Dol alpha-1,6-mannosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_649939.1 | 51.06 | 423 | 2.3e-141 | 422 |
| NP_647832.1 | 26.24 | 263 | 7.0e-09 | 58 |
| NP_995991.1 | 26.24 | 263 | 9.3e-09 | 58 |
| NP_651353.1 | 22.18 | 266 | 7.5e-07 | 52 |

### ALG13 (Q9NP73)

**Description:** UDP-N-acetylglucosamine transferase subunit ALG13

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_996379.1 | 30.13 | 395 | 9.3e-49 | 188 |
| NP_511089.2 | 30.13 | 395 | 9.3e-49 | 188 |
| NP_727271.1 | 30.59 | 304 | 6.8e-39 | 157 |
| NP_001259333.1 | 30.59 | 304 | 6.8e-39 | 157 |
| NP_001259332.1 | 30.59 | 304 | 6.8e-39 | 157 |
| NP_608851.1 | 30.71 | 381 | 3.5e-38 | 150 |
| NP_651673.1 | 39.55 | 134 | 5.5e-25 | 103 |
| NP_001163417.1 | 26.96 | 115 | 1.8e-08 | 58 |
| NP_729723.2 | 26.96 | 115 | 2.1e-08 | 58 |
| NP_648482.3 | 26.96 | 115 | 2.1e-08 | 58 |

### ALG14 (Q96F25)

**Description:** UDP-N-acetylglucosamine transferase subunit ALG14

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_573031.1 | 42.33 | 189 | 2.7e-41 | 139 |

### ALG2 (Q9H553)

**Description:** Alpha-1,3/1,6-mannosyltransferase ALG2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_647772.1 | 51.09 | 413 | 3.2e-140 | 407 |
| NP_649340.1 | 28.74 | 261 | 1.1e-15 | 79 |
| NP_001262182.1 | 28.74 | 261 | 1.1e-15 | 79 |
| NP_573084.1 | 26.02 | 196 | 6.8e-06 | 48 |

### ALG3 (Q92685)

**Description:** Dol-P-Man:Man(5)GlcNAc(2)-PP-Dol alpha-1,3-mannosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_523829.2 | 51.80 | 305 | 1.2e-80 | 258 |
| NP_523829.2 | 60.00 | 75 | 2.3e-20 | 94 |

### ALG6 (Q9Y672)

**Description:** Dolichyl pyrophosphate Man9GlcNAc2 alpha-1,3-glucosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_609393.1 | 44.61 | 482 | 8.4e-136 | 401 |
| NP_001260338.1 | 44.61 | 482 | 8.4e-136 | 401 |
| NP_572355.1 | 30.81 | 396 | 2.6e-35 | 138 |

### ALG8 (Q9BVK2)

**Description:** Dolichyl pyrophosphate Glc1Man9GlcNAc2 alpha-1,3-glucosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_572355.1 | 48.55 | 519 | 4.8e-140 | 414 |
| NP_609393.1 | 28.67 | 436 | 2.1e-28 | 118 |
| NP_001260338.1 | 28.67 | 436 | 2.1e-28 | 118 |

### ALG9 (Q9H6U8)

**Description:** Alpha-1,2-mannosyltransferase ALG9

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_651353.1 | 45.05 | 555 | 3.7e-175 | 511 |
| NP_647832.1 | 24.26 | 404 | 6.5e-11 | 65 |
| NP_995991.1 | 24.26 | 404 | 6.8e-11 | 65 |
| NP_649939.1 | 21.90 | 274 | 6.6e-07 | 53 |

### AMDHD2 (Q9Y303)

**Description:** N-acetylglucosamine-6-phosphate deacetylase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_608380.1 | 59.49 | 395 | 4.9e-168 | 477 |
| NP_001285488.1 | 59.49 | 395 | 4.9e-168 | 477 |
| NP_001162805.1 | 57.60 | 408 | 8.3e-166 | 472 |

### AMFR (Q9UKV5)

**Description:** E3 ubiquitin-protein ligase AMFR

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_651894.3 | 30.68 | 326 | 3.0e-31 | 129 |
| NP_001263152.1 | 30.68 | 326 | 3.0e-31 | 129 |
| NP_733293.1 | 27.12 | 177 | 2.7e-09 | 60 |
| NP_733292.1 | 27.12 | 177 | 2.7e-09 | 60 |
| NP_001263070.1 | 27.12 | 177 | 2.7e-09 | 60 |
| NP_649859.1 | 44.64 | 56 | 7.2e-09 | 58 |
| NP_996303.1 | 42.86 | 63 | 8.7e-09 | 59 |
| NP_650729.1 | 32.39 | 71 | 1.8e-06 | 48 |
| NP_001287394.1 | 32.39 | 71 | 1.8e-06 | 48 |
| NP_001138076.1 | 32.39 | 71 | 1.8e-06 | 48 |

### ASGR1 (P07306)

**Description:** Asialoglycoprotein receptor 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_652641.1 | 25.13 | 187 | 1.7e-06 | 49 |
| NP_649461.2 | 32.10 | 81 | 2.8e-05 | 46 |
| NP_728587.1 | 29.06 | 117 | 1.5e-04 | 42 |
| NP_728586.1 | 29.06 | 117 | 1.5e-04 | 42 |
| NP_612091.1 | 29.06 | 117 | 1.5e-04 | 43 |
| NP_610208.1 | 24.80 | 125 | 1.6e-04 | 42 |

### ASGR2 (P07307)

**Description:** Asialoglycoprotein receptor 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_612091.1 | 31.93 | 119 | 3.6e-07 | 51 |
| NP_728587.1 | 31.93 | 119 | 4.9e-07 | 49 |
| NP_728586.1 | 31.93 | 119 | 4.9e-07 | 49 |
| NP_649461.2 | 40.38 | 52 | 1.1e-05 | 47 |
| NP_652641.1 | 20.82 | 245 | 3.9e-04 | 42 |

### ATXN3 (P54252)

**Description:** Ataxin-3

No significant BLAST hits found for this protein.

### B3GALNT2 (Q8NCR0)

**Description:** UDP-GalNAc:beta-1,3-N-acetylgalactosaminyltransferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_610399.1 | 32.12 | 137 | 2.4e-16 | 81 |
| NP_609184.1 | 27.45 | 153 | 1.9e-08 | 57 |
| NP_001285740.1 | 27.45 | 153 | 1.9e-08 | 57 |
| NP_609182.1 | 25.66 | 152 | 3.7e-07 | 52 |
| NP_647916.1 | 24.84 | 157 | 2.4e-04 | 43 |
| NP_001246626.1 | 24.84 | 157 | 2.4e-04 | 43 |
| NP_001097506.1 | 24.84 | 157 | 2.4e-04 | 43 |
| NP_001097505.1 | 24.84 | 157 | 2.4e-04 | 43 |

### B3GNT2 (Q9NY97)

**Description:** N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_476901.1 | 30.71 | 280 | 5.7e-32 | 123 |
| NP_725097.2 | 25.34 | 367 | 5.5e-25 | 105 |
| NP_788328.1 | 26.35 | 334 | 9.1e-25 | 105 |
| NP_725096.2 | 26.19 | 336 | 8.0e-24 | 102 |
| NP_609184.1 | 27.41 | 259 | 4.6e-23 | 101 |
| NP_001285740.1 | 27.41 | 259 | 4.6e-23 | 101 |
| NP_647916.1 | 27.34 | 278 | 4.6e-22 | 97 |
| NP_001246626.1 | 27.34 | 278 | 4.6e-22 | 97 |
| NP_001097506.1 | 27.34 | 278 | 4.6e-22 | 97 |
| NP_001097505.1 | 27.34 | 278 | 4.6e-22 | 97 |

### B3GNT4 (Q9C0J1)

**Description:** N-acetyllactosaminide beta-1,3-N-acetylglucosaminyltransferase 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_476901.1 | 35.25 | 244 | 7.9e-32 | 122 |
| NP_725096.2 | 26.35 | 334 | 5.4e-31 | 122 |
| NP_609184.1 | 28.18 | 252 | 6.5e-29 | 118 |
| NP_001285740.1 | 28.18 | 252 | 6.5e-29 | 118 |
| NP_725097.2 | 25.98 | 331 | 1.2e-27 | 113 |
| NP_788328.1 | 24.02 | 333 | 3.0e-27 | 112 |
| NP_609182.1 | 26.25 | 259 | 8.0e-25 | 105 |
| NP_647916.1 | 26.79 | 280 | 2.0e-23 | 101 |
| NP_001246626.1 | 26.79 | 280 | 2.0e-23 | 101 |
| NP_001097506.1 | 26.79 | 280 | 2.0e-23 | 101 |

### B3GNT6 (Q6ZMB0)

**Description:** Acetylgalactosaminyl-O-glycosyl-glycoprotein beta-1,3-N-acetylglucosaminyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_476901.1 | 35.91 | 259 | 7.0e-40 | 144 |
| NP_609182.1 | 26.12 | 268 | 8.5e-30 | 119 |
| NP_609184.1 | 27.14 | 280 | 1.1e-29 | 120 |
| NP_001285740.1 | 27.14 | 280 | 1.1e-29 | 120 |
| NP_647916.1 | 29.13 | 309 | 4.3e-29 | 117 |
| NP_001246626.1 | 29.13 | 309 | 4.3e-29 | 117 |
| NP_001097506.1 | 29.13 | 309 | 4.3e-29 | 117 |
| NP_001097505.1 | 29.13 | 309 | 4.3e-29 | 117 |
| NP_725096.2 | 24.31 | 325 | 1.2e-22 | 98 |
| NP_725097.2 | 26.79 | 321 | 4.1e-20 | 91 |

### B4GALNT2 (Q8NHY0)

**Description:** Beta-1,4 N-acetylgalactosaminyltransferase 2

No significant BLAST hits found for this protein.

### B4GALT1 (P15291)

**Description:** Beta-1,4-galactosyltransferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_610946.1 | 37.91 | 277 | 2.9e-72 | 232 |
| NP_651657.1 | 37.66 | 239 | 7.2e-53 | 179 |
| NP_651319.2 | 34.38 | 224 | 7.4e-28 | 112 |

### B4GALT2 (O60909)

**Description:** Beta-1,4-galactosyltransferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_610946.1 | 39.53 | 296 | 8.2e-74 | 235 |
| NP_651657.1 | 37.66 | 231 | 5.5e-56 | 186 |
| NP_651319.2 | 33.65 | 214 | 1.0e-24 | 103 |

### B4GALT3 (O60512)

**Description:** Beta-1,4-galactosyltransferase 3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_610946.1 | 41.09 | 275 | 8.3e-72 | 230 |
| NP_651657.1 | 40.83 | 240 | 1.4e-56 | 188 |
| NP_651319.2 | 39.49 | 157 | 1.8e-22 | 97 |

### B4GALT4 (O60513)

**Description:** Beta-1,4-galactosyltransferase 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_610946.1 | 40.14 | 284 | 2.3e-70 | 225 |
| NP_651657.1 | 39.74 | 229 | 4.2e-52 | 175 |
| NP_651319.2 | 35.44 | 158 | 7.3e-24 | 100 |

### B4GALT5 (O43286)

**Description:** Beta-1,4-galactosyltransferase 5

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_610946.1 | 38.54 | 288 | 7.4e-66 | 215 |
| NP_651657.1 | 38.16 | 228 | 3.8e-53 | 179 |
| NP_651319.2 | 33.63 | 226 | 3.0e-24 | 102 |

### B4GALT6 (Q9UBX8)

**Description:** Beta-1,4-galactosyltransferase 6

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_610946.1 | 37.76 | 286 | 1.7e-64 | 211 |
| NP_651657.1 | 37.76 | 241 | 4.1e-56 | 187 |
| NP_651319.2 | 33.63 | 226 | 1.9e-23 | 99 |
| NP_610946.1 | 37.76 | 286 | 1.7e-64 | 211 |
| NP_651657.1 | 37.76 | 241 | 4.1e-56 | 187 |
| NP_651319.2 | 33.63 | 226 | 1.9e-23 | 99 |

### B4GALT6 (Q9UBX8)

**Description:** Beta-1,4-galactosyltransferase 6

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_610946.1 | 37.76 | 286 | 1.7e-64 | 211 |
| NP_651657.1 | 37.76 | 241 | 4.1e-56 | 187 |
| NP_651319.2 | 33.63 | 226 | 1.9e-23 | 99 |
| NP_610946.1 | 37.76 | 286 | 1.7e-64 | 211 |
| NP_651657.1 | 37.76 | 241 | 4.1e-56 | 187 |
| NP_651319.2 | 33.63 | 226 | 1.9e-23 | 99 |

### BCL2L11 (O43521)

**Description:** Bcl-2-like protein 11

No significant BLAST hits found for this protein.

### BCL6 (P41182)

**Description:** B-cell lymphoma 6 protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001014498.1 | 38.17 | 262 | 1.5e-40 | 157 |
| NP_610137.1 | 37.69 | 260 | 3.2e-40 | 156 |
| NP_523867.1 | 50.75 | 134 | 6.4e-40 | 154 |
| NP_523867.1 | 42.86 | 112 | 3.9e-25 | 110 |
| NP_523867.1 | 44.19 | 86 | 7.0e-15 | 78 |
| NP_523867.1 | 39.58 | 48 | 2.4e-05 | 48 |
| NP_001261181.1 | 50.75 | 134 | 6.4e-40 | 154 |
| NP_001261181.1 | 42.86 | 112 | 3.9e-25 | 110 |
| NP_001261181.1 | 44.19 | 86 | 7.0e-15 | 78 |
| NP_001261181.1 | 39.58 | 48 | 2.4e-05 | 48 |
| NP_477466.1 | 42.26 | 168 | 1.5e-37 | 150 |
| NP_477466.1 | 39.26 | 163 | 1.3e-31 | 132 |
| NP_477466.1 | 42.14 | 140 | 8.0e-30 | 126 |
| NP_477466.1 | 43.97 | 116 | 3.4e-26 | 115 |
| NP_477466.1 | 36.62 | 142 | 3.0e-23 | 105 |
| NP_477466.1 | 29.77 | 178 | 1.5e-16 | 84 |
| NP_477467.1 | 42.26 | 168 | 2.4e-37 | 149 |
| NP_477467.1 | 39.26 | 163 | 2.1e-31 | 131 |
| NP_477467.1 | 42.14 | 140 | 1.4e-29 | 125 |
| NP_477467.1 | 43.97 | 116 | 6.0e-26 | 114 |
| NP_477467.1 | 36.62 | 142 | 4.1e-23 | 105 |
| NP_477467.1 | 29.77 | 178 | 2.6e-16 | 83 |
| NP_608631.1 | 36.45 | 203 | 1.3e-35 | 143 |
| NP_001259889.1 | 36.45 | 203 | 1.5e-35 | 142 |
| NP_001259891.2 | 36.45 | 203 | 1.8e-35 | 143 |
| NP_001259890.1 | 36.45 | 203 | 1.9e-35 | 143 |

### C1GALT1 (Q9NS00)

**Description:** Glycoprotein-N-acetylgalactosamine 3-beta-galactosyltransferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_723428.1 | 50.15 | 333 | 1.5e-116 | 343 |
| NP_723427.1 | 50.15 | 333 | 1.5e-116 | 343 |
| NP_609258.1 | 50.15 | 333 | 1.5e-116 | 343 |
| NP_724653.1 | 46.63 | 341 | 5.0e-108 | 324 |
| NP_610360.2 | 46.63 | 341 | 5.0e-108 | 324 |
| NP_001246201.1 | 46.63 | 341 | 5.3e-108 | 323 |
| NP_608719.1 | 44.92 | 256 | 3.9e-71 | 227 |
| NP_573349.4 | 40.71 | 280 | 5.0e-69 | 221 |
| NP_001285425.1 | 40.71 | 280 | 5.0e-69 | 221 |
| NP_001033979.1 | 42.54 | 268 | 4.8e-67 | 215 |

### CAP1 (Q01518)

**Description:** Adenylyl cyclase-associated protein 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001259856.1 | 47.48 | 436 | 4.4e-135 | 396 |
| NP_001033869.1 | 47.48 | 436 | 4.4e-135 | 396 |
| NP_001033870.1 | 47.25 | 436 | 8.9e-130 | 394 |

### CAP2 (P40123)

**Description:** Adenylyl cyclase-associated protein 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001259856.1 | 48.48 | 427 | 1.6e-135 | 397 |
| NP_001033869.1 | 48.48 | 427 | 1.6e-135 | 397 |
| NP_001033870.1 | 48.48 | 427 | 4.9e-130 | 395 |

### CAV1 (Q03135)

**Description:** Caveolin-1

No significant BLAST hits found for this protein.

### CCNG2 (Q16589)

**Description:** Cyclin-G2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_733436.1 | 30.26 | 271 | 9.0e-27 | 111 |
| NP_733435.1 | 30.26 | 271 | 9.0e-27 | 111 |
| NP_733434.1 | 30.26 | 271 | 9.0e-27 | 111 |
| NP_524609.2 | 30.26 | 271 | 9.0e-27 | 111 |
| NP_001263146.1 | 30.26 | 271 | 9.0e-27 | 111 |
| NP_476960.1 | 29.08 | 141 | 5.6e-09 | 57 |
| NP_001246037.1 | 29.08 | 141 | 1.2e-08 | 57 |
| NP_723926.1 | 29.08 | 141 | 1.2e-08 | 57 |
| NP_723925.1 | 29.08 | 141 | 1.2e-08 | 57 |
| NP_723924.1 | 29.08 | 141 | 1.2e-08 | 57 |

### CGA (P01215)

**Description:** Glycoprotein hormones alpha chain

No significant BLAST hits found for this protein.

### CHST10 (O43529)

**Description:** Carbohydrate sulfotransferase 10

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_609820.1 | 29.81 | 312 | 3.7e-34 | 129 |
| NP_001286000.1 | 29.81 | 312 | 3.7e-34 | 129 |
| NP_647673.2 | 24.73 | 279 | 1.9e-19 | 88 |
| NP_001261298.1 | 24.73 | 279 | 1.9e-19 | 88 |
| NP_001163325.1 | 24.73 | 279 | 1.9e-19 | 88 |
| NP_001261297.1 | 24.73 | 279 | 3.7e-19 | 88 |
| NP_728689.2 | 24.73 | 279 | 3.8e-19 | 88 |
| NP_728688.2 | 24.73 | 279 | 3.8e-19 | 88 |
| NP_608920.1 | 24.12 | 257 | 1.6e-13 | 71 |
| NP_001260094.1 | 24.12 | 257 | 1.6e-13 | 71 |

### CHST8 (Q9H2A9)

**Description:** Carbohydrate sulfotransferase 8

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_609820.1 | 32.01 | 278 | 2.2e-37 | 139 |
| NP_001286000.1 | 32.01 | 278 | 2.2e-37 | 139 |
| NP_647673.2 | 30.32 | 277 | 1.0e-25 | 107 |
| NP_001261298.1 | 30.32 | 277 | 1.0e-25 | 107 |
| NP_001163325.1 | 30.32 | 277 | 1.0e-25 | 107 |
| NP_728689.2 | 30.32 | 277 | 1.4e-25 | 108 |
| NP_728688.2 | 30.32 | 277 | 1.4e-25 | 108 |
| NP_001261297.1 | 30.32 | 277 | 1.7e-25 | 108 |
| NP_608920.1 | 26.95 | 282 | 5.4e-20 | 92 |
| NP_001260094.1 | 26.95 | 282 | 5.4e-20 | 92 |

### CITED2 (Q99967)

**Description:** Cbp/p300-interacting transactivator 2

No significant BLAST hits found for this protein.

### CLASP1 (Q7Z460)

**Description:** CLIP-associating protein 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_730597.1 | 33.10 | 1139 | 1.2e-170 | 553 |
| NP_730597.1 | 33.84 | 198 | 5.9e-25 | 114 |
| NP_730596.1 | 33.10 | 1139 | 1.2e-170 | 553 |
| NP_730596.1 | 33.84 | 198 | 5.9e-25 | 114 |
| NP_524651.2 | 33.10 | 1139 | 1.2e-170 | 553 |
| NP_524651.2 | 33.84 | 198 | 5.9e-25 | 114 |
| NP_001247130.1 | 26.21 | 145 | 8.8e-04 | 44 |
| NP_001247129.1 | 26.21 | 145 | 8.9e-04 | 44 |
| NP_732105.2 | 26.21 | 145 | 1.0e-03 | 44 |
| NP_001247131.1 | 26.21 | 145 | 1.0e-03 | 44 |
| NP_001163627.1 | 26.21 | 145 | 1.0e-03 | 44 |

### CLASP2 (O75122)

**Description:** CLIP-associating protein 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_730597.1 | 33.30 | 868 | 1.3e-115 | 395 |
| NP_730597.1 | 33.17 | 202 | 3.5e-25 | 114 |
| NP_730596.1 | 33.30 | 868 | 1.3e-115 | 395 |
| NP_730596.1 | 33.17 | 202 | 3.5e-25 | 114 |
| NP_524651.2 | 33.30 | 868 | 1.3e-115 | 395 |
| NP_524651.2 | 33.17 | 202 | 3.5e-25 | 114 |

### CREBBP (Q92793)

**Description:** CREB-binding protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001259387.1 | 69.81 | 805 | 0.0e+00 | 1232 |
| NP_001259387.1 | 75.90 | 83 | 3.7e-32 | 139 |
| NP_001259387.1 | 47.22 | 72 | 2.2e-16 | 87 |
| NP_524642.2 | 68.96 | 815 | 0.0e+00 | 1224 |
| NP_524642.2 | 75.90 | 83 | 3.7e-32 | 139 |
| NP_524642.2 | 47.22 | 72 | 2.1e-16 | 87 |
| NP_001188576.1 | 68.96 | 815 | 0.0e+00 | 1224 |
| NP_001188576.1 | 75.90 | 83 | 3.7e-32 | 139 |
| NP_001188576.1 | 47.22 | 72 | 2.1e-16 | 87 |
| NP_001188575.1 | 68.96 | 815 | 0.0e+00 | 1223 |
| NP_001188575.1 | 75.90 | 83 | 3.7e-32 | 139 |
| NP_001188575.1 | 47.22 | 72 | 2.1e-16 | 87 |
| NP_511078.2 | 39.42 | 104 | 2.1e-15 | 84 |
| NP_511078.2 | 37.37 | 99 | 7.1e-15 | 82 |
| NP_001259321.1 | 39.42 | 104 | 2.1e-15 | 84 |
| NP_001259321.1 | 37.37 | 99 | 6.9e-15 | 82 |
| NP_001259322.1 | 44.30 | 79 | 8.2e-15 | 81 |
| NP_001259322.1 | 37.37 | 99 | 2.5e-14 | 80 |
| NP_996370.1 | 44.30 | 79 | 8.6e-15 | 81 |
| NP_996370.1 | 37.37 | 99 | 2.5e-14 | 80 |
| NP_996369.1 | 44.30 | 79 | 8.6e-15 | 81 |
| NP_996369.1 | 37.37 | 99 | 2.5e-14 | 80 |
| NP_996368.1 | 44.30 | 79 | 8.6e-15 | 81 |
| NP_996368.1 | 37.37 | 99 | 2.5e-14 | 80 |

### CYP1A2 (P05177)

**Description:** Cytochrome P450 1A2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_728191.1 | 30.64 | 457 | 1.3e-69 | 232 |
| NP_523403.2 | 30.64 | 457 | 1.3e-69 | 232 |
| NP_649151.1 | 26.34 | 486 | 4.1e-46 | 168 |
| NP_573319.1 | 26.08 | 510 | 3.4e-45 | 167 |
| NP_001285414.1 | 26.08 | 510 | 3.4e-45 | 167 |
| NP_731751.1 | 28.66 | 506 | 5.3e-45 | 166 |
| NP_652070.1 | 27.34 | 501 | 3.9e-44 | 163 |
| NP_001285977.1 | 27.34 | 501 | 3.9e-44 | 163 |
| NP_001104460.2 | 25.60 | 539 | 3.0e-37 | 144 |
| NP_647975.2 | 26.19 | 527 | 2.3e-33 | 133 |

### DAD1 (P61803)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit DAD1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_609222.1 | 71.96 | 107 | 5.2e-53 | 162 |

### DDIT3 (P35638)

**Description:** DNA damage-inducible transcript 3 protein

No significant BLAST hits found for this protein.

### DDOST (P39656)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase 48 kDa subunit

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_511096.2 | 60.18 | 437 | 0.0e+00 | 541 |

### DERL1 (Q9BUN8)

**Description:** Derlin-1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_608632.1 | 42.91 | 247 | 8.6e-64 | 199 |
| NP_001259892.1 | 42.91 | 247 | 8.6e-64 | 199 |
| NP_650553.1 | 33.49 | 209 | 1.4e-30 | 114 |

### DPAGT1 (Q9H3H5)

**Description:** UDP-N-acetylglucosamine--dolichyl-phosphate N-acetylglucosaminephosphotransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_609608.1 | 55.47 | 402 | 4.0e-153 | 439 |

### ENGASE (Q8NFI3)

**Description:** Cytosolic endo-beta-N-acetylglucosaminidase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_788923.1 | 34.92 | 524 | 2.5e-74 | 252 |
| NP_573215.1 | 38.26 | 413 | 5.3e-74 | 253 |

### FASLG (P48023)

**Description:** Tumor necrosis factor ligand superfamily member 6

No significant BLAST hits found for this protein.

### FBXO32 (Q969P5)

**Description:** F-box only protein 32

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_996061.1 | 27.62 | 344 | 3.2e-35 | 132 |
| NP_996060.1 | 27.62 | 344 | 3.2e-35 | 132 |
| NP_996059.1 | 27.62 | 344 | 3.2e-35 | 132 |
| NP_996058.1 | 27.62 | 344 | 3.2e-35 | 132 |
| NP_648498.1 | 27.62 | 344 | 3.2e-35 | 132 |

### FOXO3 (O43524)

**Description:** Forkhead box protein O3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_650330.3 | 61.24 | 209 | 1.4e-63 | 222 |
| NP_001262557.1 | 61.24 | 209 | 1.4e-63 | 222 |
| NP_996205.1 | 61.24 | 209 | 1.5e-63 | 222 |
| NP_996204.1 | 61.24 | 209 | 1.5e-63 | 222 |
| NP_001303426.1 | 61.24 | 209 | 2.3e-63 | 221 |
| NP_729672.1 | 40.00 | 145 | 5.8e-19 | 92 |
| NP_001261703.1 | 40.00 | 145 | 6.0e-19 | 92 |
| NP_001261702.1 | 40.00 | 145 | 6.5e-19 | 91 |
| NP_729675.2 | 40.00 | 145 | 6.7e-19 | 91 |
| NP_729674.2 | 40.00 | 145 | 6.7e-19 | 91 |

### FOXO4 (P98177)

**Description:** Forkhead box protein O4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_650330.3 | 48.54 | 274 | 4.7e-64 | 219 |
| NP_001262557.1 | 48.54 | 274 | 4.7e-64 | 219 |
| NP_996205.1 | 48.54 | 274 | 5.0e-64 | 219 |
| NP_996204.1 | 48.54 | 274 | 5.0e-64 | 219 |
| NP_001303426.1 | 48.54 | 274 | 8.8e-64 | 218 |
| NP_729675.2 | 41.52 | 118 | 2.5e-18 | 89 |
| NP_729674.2 | 41.52 | 118 | 2.5e-18 | 89 |
| NP_729673.2 | 41.52 | 118 | 2.5e-18 | 89 |
| NP_648440.2 | 41.52 | 118 | 2.5e-18 | 89 |
| NP_001261703.1 | 41.52 | 118 | 2.8e-18 | 88 |

### FUCA1 (P04066)

**Description:** Tissue alpha-L-fucosidase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_648472.1 | 45.85 | 482 | 2.9e-152 | 442 |
| NP_001303363.1 | 45.85 | 482 | 2.9e-152 | 442 |

### FUT3 (P21217)

**Description:** 3-galactosyl-N-acetylglucosaminide 4-alpha-L-fucosyltransferase FUT3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_648754.2 | 35.11 | 282 | 1.0e-37 | 142 |
| NP_001261872.1 | 35.11 | 282 | 1.0e-37 | 142 |
| NP_001036320.3 | 34.33 | 300 | 2.9e-33 | 128 |
| NP_609288.4 | 29.12 | 261 | 3.6e-16 | 79 |
| NP_001285779.1 | 29.12 | 261 | 3.6e-16 | 79 |
| NP_612107.1 | 26.55 | 226 | 2.4e-11 | 65 |
| NP_001285780.1 | 40.26 | 77 | 1.9e-06 | 48 |

### FUT8 (Q9BYC5)

**Description:** Alpha-(1,6)-fucosyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_572740.1 | 45.90 | 610 | 0.0e+00 | 564 |

### G6PC1 (P35575)

**Description:** Glucose-6-phosphatase catalytic subunit 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001097063.1 | 25.58 | 258 | 7.5e-17 | 81 |

### GADD45A (P24522)

**Description:** Growth arrest and DNA damage-inducible protein GADD45 alpha

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_610264.1 | 32.14 | 84 | 3.7e-10 | 55 |
| NP_729866.1 | 32.93 | 82 | 1.7e-04 | 40 |

### GALNT1 (Q10472)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001036338.1 | 57.69 | 605 | 0.0e+00 | 721 |
| NP_608906.2 | 57.35 | 605 | 0.0e+00 | 718 |
| NP_725603.2 | 50.48 | 515 | 7.1e-172 | 501 |
| NP_001097341.1 | 50.48 | 515 | 7.1e-172 | 501 |
| NP_725602.1 | 50.19 | 518 | 3.8e-169 | 494 |
| NP_001097342.1 | 50.19 | 518 | 3.8e-169 | 494 |
| NP_610256.1 | 55.44 | 386 | 6.1e-158 | 466 |
| NP_995625.2 | 48.72 | 470 | 1.9e-150 | 446 |
| NP_608773.2 | 48.72 | 470 | 1.9e-150 | 446 |
| NP_728178.1 | 40.58 | 515 | 1.5e-124 | 378 |

### GALNT10 (Q86SR1)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 10

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_728779.1 | 49.46 | 554 | 5.8e-171 | 501 |
| NP_647749.2 | 49.46 | 554 | 5.8e-171 | 501 |
| NP_001261342.1 | 49.46 | 554 | 5.8e-171 | 501 |
| NP_722910.2 | 44.51 | 546 | 3.1e-150 | 447 |
| NP_001137779.1 | 44.51 | 546 | 3.1e-150 | 447 |
| NP_725603.2 | 46.64 | 521 | 2.1e-146 | 438 |
| NP_001097341.1 | 46.64 | 521 | 2.1e-146 | 438 |
| NP_725602.1 | 46.40 | 528 | 2.7e-144 | 432 |
| NP_001097342.1 | 46.40 | 528 | 2.7e-144 | 432 |
| NP_001036338.1 | 41.86 | 571 | 3.9e-140 | 421 |

### GALNT11 (Q8NCW6)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 11

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_652069.2 | 47.25 | 527 | 1.1e-163 | 482 |
| NP_001036338.1 | 43.96 | 505 | 7.0e-143 | 428 |
| NP_608906.2 | 43.37 | 505 | 2.5e-140 | 422 |
| NP_725603.2 | 43.59 | 546 | 9.1e-140 | 421 |
| NP_001097341.1 | 43.59 | 546 | 9.1e-140 | 421 |
| NP_725602.1 | 43.77 | 546 | 5.3e-138 | 417 |
| NP_001097342.1 | 43.77 | 546 | 5.3e-138 | 417 |
| NP_995625.2 | 45.72 | 479 | 3.1e-129 | 394 |
| NP_608773.2 | 45.72 | 479 | 3.1e-129 | 394 |
| NP_725472.1 | 42.34 | 496 | 5.0e-120 | 369 |

### GALNT12 (Q8IXK2)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 12

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_725603.2 | 43.90 | 549 | 1.4e-143 | 430 |
| NP_001097341.1 | 43.90 | 549 | 1.4e-143 | 430 |
| NP_725602.1 | 43.80 | 548 | 4.5e-142 | 426 |
| NP_001097342.1 | 43.80 | 548 | 4.5e-142 | 426 |
| NP_608906.2 | 43.46 | 520 | 3.3e-140 | 421 |
| NP_001036338.1 | 43.27 | 520 | 6.9e-138 | 415 |
| NP_995625.2 | 43.89 | 499 | 1.1e-129 | 394 |
| NP_608773.2 | 43.89 | 499 | 1.1e-129 | 394 |
| NP_610256.1 | 48.59 | 391 | 1.5e-109 | 343 |
| NP_725472.1 | 42.95 | 440 | 1.4e-105 | 330 |

### GALNT13 (Q8IUC8)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 13

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001036338.1 | 67.07 | 495 | 0.0e+00 | 708 |
| NP_608906.2 | 66.87 | 495 | 0.0e+00 | 706 |
| NP_725603.2 | 50.89 | 507 | 1.0e-169 | 496 |
| NP_001097341.1 | 50.89 | 507 | 1.0e-169 | 496 |
| NP_725602.1 | 50.39 | 510 | 1.1e-166 | 488 |
| NP_001097342.1 | 50.39 | 510 | 1.1e-166 | 488 |
| NP_610256.1 | 56.02 | 382 | 3.3e-153 | 454 |
| NP_995625.2 | 48.72 | 468 | 2.3e-152 | 451 |
| NP_608773.2 | 48.72 | 468 | 2.3e-152 | 451 |
| NP_722910.2 | 41.43 | 519 | 5.3e-124 | 379 |

### GALNT14 (Q96FL9)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 14

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_995625.2 | 53.64 | 494 | 0.0e+00 | 540 |
| NP_608773.2 | 53.64 | 494 | 0.0e+00 | 540 |
| NP_001036338.1 | 44.08 | 490 | 7.7e-129 | 390 |
| NP_725603.2 | 43.67 | 490 | 1.2e-128 | 390 |
| NP_001097341.1 | 43.67 | 490 | 1.2e-128 | 390 |
| NP_725602.1 | 43.29 | 492 | 1.6e-128 | 390 |
| NP_001097342.1 | 43.29 | 492 | 1.6e-128 | 390 |
| NP_608906.2 | 43.67 | 490 | 4.6e-128 | 388 |
| NP_652069.2 | 40.43 | 554 | 1.1e-120 | 369 |
| NP_725472.1 | 40.12 | 511 | 6.1e-114 | 351 |

### GALNT16 (Q8N428)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 16

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_995625.2 | 52.21 | 498 | 0.0e+00 | 528 |
| NP_608773.2 | 52.21 | 498 | 0.0e+00 | 528 |
| NP_001036338.1 | 43.84 | 463 | 5.3e-126 | 383 |
| NP_608906.2 | 43.20 | 463 | 1.1e-124 | 380 |
| NP_725602.1 | 42.00 | 500 | 2.2e-120 | 369 |
| NP_001097342.1 | 42.00 | 500 | 2.2e-120 | 369 |
| NP_725603.2 | 42.17 | 498 | 2.3e-120 | 369 |
| NP_001097341.1 | 42.17 | 498 | 2.3e-120 | 369 |
| NP_652069.2 | 39.96 | 483 | 1.2e-115 | 357 |
| NP_725472.1 | 39.29 | 509 | 3.0e-109 | 339 |

### GALNT18 (Q6P9A2)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 18

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_608906.2 | 39.82 | 447 | 8.9e-94 | 301 |
| NP_001036338.1 | 39.60 | 447 | 2.3e-92 | 297 |
| NP_725603.2 | 33.40 | 512 | 1.1e-89 | 291 |
| NP_001097341.1 | 33.40 | 512 | 1.1e-89 | 291 |
| NP_995625.2 | 35.23 | 528 | 3.8e-88 | 286 |
| NP_608773.2 | 35.23 | 528 | 3.8e-88 | 286 |
| NP_725602.1 | 32.88 | 514 | 2.9e-86 | 282 |
| NP_001097342.1 | 32.88 | 514 | 2.9e-86 | 282 |
| NP_652069.2 | 38.46 | 390 | 3.3e-83 | 273 |
| NP_610256.1 | 39.78 | 367 | 1.1e-79 | 265 |

### GALNT2 (Q10471)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_995625.2 | 66.05 | 486 | 0.0e+00 | 685 |
| NP_608773.2 | 66.05 | 486 | 0.0e+00 | 685 |
| NP_608906.2 | 45.07 | 477 | 7.8e-142 | 424 |
| NP_001036338.1 | 45.28 | 477 | 8.3e-142 | 424 |
| NP_725603.2 | 44.06 | 488 | 2.4e-132 | 400 |
| NP_001097341.1 | 44.06 | 488 | 2.4e-132 | 400 |
| NP_725602.1 | 43.79 | 491 | 3.4e-132 | 400 |
| NP_001097342.1 | 43.79 | 491 | 3.4e-132 | 400 |
| NP_652069.2 | 35.81 | 592 | 7.4e-115 | 355 |
| NP_610256.1 | 38.54 | 506 | 2.3e-109 | 342 |

### GALNT3 (Q14435)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_725603.2 | 43.21 | 530 | 2.9e-138 | 418 |
| NP_001097341.1 | 43.21 | 530 | 2.9e-138 | 418 |
| NP_608906.2 | 48.11 | 476 | 2.0e-135 | 410 |
| NP_001036338.1 | 47.90 | 476 | 9.5e-133 | 403 |
| NP_725602.1 | 42.88 | 534 | 1.5e-132 | 404 |
| NP_001097342.1 | 42.88 | 534 | 1.5e-132 | 404 |
| NP_995625.2 | 42.71 | 487 | 1.8e-116 | 361 |
| NP_608773.2 | 42.71 | 487 | 1.8e-116 | 361 |
| NP_610256.1 | 49.47 | 376 | 1.7e-110 | 347 |
| NP_725472.1 | 39.69 | 509 | 1.4e-106 | 335 |

### GALNT4 (Q8N4A0)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_725603.2 | 44.55 | 532 | 2.4e-143 | 429 |
| NP_001097341.1 | 44.55 | 532 | 2.4e-143 | 429 |
| NP_725602.1 | 44.47 | 533 | 6.3e-143 | 428 |
| NP_001097342.1 | 44.47 | 533 | 6.3e-143 | 428 |
| NP_608906.2 | 45.70 | 512 | 5.6e-141 | 422 |
| NP_001036338.1 | 45.70 | 512 | 1.9e-138 | 416 |
| NP_995625.2 | 45.09 | 499 | 2.0e-127 | 388 |
| NP_608773.2 | 45.09 | 499 | 2.0e-127 | 388 |
| NP_725472.1 | 38.64 | 603 | 2.9e-118 | 363 |
| NP_611043.1 | 38.64 | 603 | 2.9e-118 | 363 |

### GALNT9 (Q9HCQ5)

**Description:** Polypeptide N-acetylgalactosaminyltransferase 9

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_608906.2 | 37.24 | 521 | 2.0e-104 | 329 |
| NP_001036338.1 | 37.59 | 524 | 8.2e-103 | 325 |
| NP_995625.2 | 38.51 | 470 | 1.2e-98 | 314 |
| NP_608773.2 | 38.51 | 470 | 1.2e-98 | 314 |
| NP_725602.1 | 35.71 | 532 | 1.3e-97 | 311 |
| NP_001097342.1 | 35.71 | 532 | 1.3e-97 | 311 |
| NP_725603.2 | 35.41 | 531 | 2.8e-97 | 311 |
| NP_001097341.1 | 35.41 | 531 | 2.8e-97 | 311 |
| NP_728178.1 | 39.15 | 493 | 5.1e-95 | 303 |
| NP_573301.2 | 39.15 | 493 | 5.1e-95 | 303 |

### GALNTL5 (Q7Z4T8)

**Description:** Inactive polypeptide N-acetylgalactosaminyltransferase-like protein 5

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_652069.2 | 48.56 | 348 | 3.0e-109 | 336 |
| NP_001036338.1 | 48.14 | 349 | 7.7e-109 | 335 |
| NP_608906.2 | 46.42 | 349 | 5.7e-105 | 325 |
| NP_725602.1 | 45.19 | 343 | 3.0e-99 | 310 |
| NP_001097342.1 | 45.19 | 343 | 3.0e-99 | 310 |
| NP_725603.2 | 45.19 | 343 | 3.1e-99 | 310 |
| NP_001097341.1 | 45.19 | 343 | 3.1e-99 | 310 |
| NP_995625.2 | 45.35 | 344 | 1.7e-96 | 303 |
| NP_608773.2 | 45.35 | 344 | 1.7e-96 | 303 |
| NP_728178.1 | 42.54 | 362 | 2.5e-95 | 298 |

### GALNTL6 (Q49A17)

**Description:** Polypeptide N-acetylgalactosaminyltransferase-like 6

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_728779.1 | 47.82 | 550 | 8.2e-168 | 493 |
| NP_647749.2 | 47.82 | 550 | 8.2e-168 | 493 |
| NP_001261342.1 | 47.82 | 550 | 8.2e-168 | 493 |
| NP_722910.2 | 43.98 | 548 | 8.1e-147 | 439 |
| NP_001137779.1 | 43.98 | 548 | 8.1e-147 | 439 |
| NP_001036338.1 | 39.96 | 558 | 4.5e-134 | 405 |
| NP_725603.2 | 42.99 | 521 | 7.0e-133 | 403 |
| NP_001097341.1 | 42.99 | 521 | 7.0e-133 | 403 |
| NP_728178.1 | 42.72 | 522 | 1.3e-132 | 400 |
| NP_573301.2 | 42.72 | 522 | 1.3e-132 | 400 |

### GANAB (Q14697)

**Description:** Neutral alpha-glucosidase AB

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_728437.1 | 46.74 | 935 | 0.0e+00 | 881 |
| NP_728436.1 | 46.74 | 935 | 0.0e+00 | 881 |
| NP_728435.1 | 46.74 | 935 | 0.0e+00 | 881 |
| NP_728434.1 | 46.74 | 935 | 0.0e+00 | 881 |
| NP_652145.1 | 46.74 | 935 | 0.0e+00 | 881 |
| NP_651391.1 | 24.27 | 412 | 7.5e-15 | 79 |
| NP_788858.1 | 22.89 | 380 | 2.0e-12 | 72 |
| NP_001284904.1 | 22.89 | 380 | 2.4e-12 | 71 |
| NP_788859.1 | 22.89 | 380 | 4.0e-12 | 70 |
| NP_001284905.1 | 22.89 | 380 | 4.0e-12 | 70 |

### GCK (P35557)

**Description:** Hexokinase-4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_524848.1 | 46.47 | 467 | 5.7e-146 | 428 |
| NP_727350.1 | 46.49 | 456 | 2.8e-145 | 423 |
| NP_001259384.1 | 46.49 | 456 | 2.8e-145 | 423 |
| NP_733151.2 | 41.58 | 457 | 1.2e-131 | 389 |
| NP_524674.1 | 43.82 | 461 | 1.2e-122 | 365 |
| NP_788744.1 | 31.10 | 447 | 1.9e-80 | 257 |

### GFPT1 (Q06210)

**Description:** Glutamine--fructose-6-phosphate aminotransferase [isomerizing] 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_651617.1 | 65.29 | 703 | 0.0e+00 | 947 |
| NP_001015158.1 | 63.29 | 700 | 0.0e+00 | 923 |
| NP_001015160.2 | 62.59 | 711 | 0.0e+00 | 922 |
| NP_001104465.1 | 62.57 | 708 | 0.0e+00 | 919 |
| NP_001188369.1 | 62.61 | 706 | 0.0e+00 | 917 |
| NP_001015161.2 | 63.14 | 700 | 0.0e+00 | 917 |
| NP_001015157.2 | 62.61 | 706 | 0.0e+00 | 917 |
| NP_001188368.1 | 63.43 | 648 | 0.0e+00 | 853 |
| NP_001015159.2 | 72.07 | 444 | 0.0e+00 | 683 |
| NP_001015162.1 | 72.81 | 434 | 0.0e+00 | 677 |

### GFPT2 (O94808)

**Description:** Glutamine--fructose-6-phosphate aminotransferase [isomerizing] 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_651617.1 | 66.03 | 686 | 0.0e+00 | 951 |
| NP_001015160.2 | 64.39 | 688 | 0.0e+00 | 937 |
| NP_001015158.1 | 64.57 | 683 | 0.0e+00 | 932 |
| NP_001015161.2 | 64.57 | 683 | 0.0e+00 | 932 |
| NP_001104465.1 | 63.97 | 691 | 0.0e+00 | 931 |
| NP_001188369.1 | 63.43 | 700 | 0.0e+00 | 931 |
| NP_001015157.2 | 63.43 | 700 | 0.0e+00 | 931 |
| NP_001188368.1 | 64.19 | 645 | 0.0e+00 | 866 |
| NP_001015159.2 | 71.62 | 444 | 0.0e+00 | 687 |
| NP_001015162.1 | 73.49 | 430 | 0.0e+00 | 686 |

### GNPNAT1 (Q96EK6)

**Description:** Glucosamine 6-phosphate N-acetyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_733301.1 | 47.25 | 182 | 2.3e-57 | 179 |
| NP_001263080.1 | 47.25 | 182 | 2.3e-57 | 179 |
| NP_001263079.1 | 47.25 | 182 | 2.3e-57 | 179 |
| NP_001263081.1 | 47.25 | 182 | 2.5e-57 | 180 |
| NP_651719.1 | 47.25 | 182 | 2.6e-57 | 179 |
| NP_001247360.1 | 47.25 | 182 | 3.4e-57 | 179 |
| NP_001247361.1 | 47.25 | 182 | 5.9e-57 | 179 |

### HDAC2 (Q92769)

**Description:** Histone deacetylase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_647918.2 | 78.08 | 470 | 0.0e+00 | 780 |
| NP_651978.2 | 57.75 | 426 | 0.0e+00 | 548 |
| NP_001245681.1 | 26.97 | 330 | 6.9e-30 | 124 |
| NP_001245681.1 | 28.30 | 265 | 8.8e-23 | 102 |
| NP_001162760.1 | 26.97 | 330 | 8.7e-30 | 124 |
| NP_001162760.1 | 28.20 | 266 | 1.1e-22 | 102 |
| NP_573017.2 | 26.97 | 330 | 1.2e-29 | 124 |
| NP_573017.2 | 28.20 | 266 | 9.9e-23 | 102 |
| NP_727843.1 | 26.97 | 330 | 1.2e-29 | 123 |
| NP_727843.1 | 28.20 | 266 | 1.0e-22 | 102 |
| NP_001245680.1 | 26.97 | 330 | 1.3e-29 | 123 |
| NP_001245680.1 | 28.20 | 266 | 1.1e-22 | 102 |
| NP_727842.1 | 26.97 | 330 | 1.4e-29 | 123 |
| NP_727842.1 | 28.20 | 266 | 1.2e-22 | 102 |
| NP_001259569.1 | 26.97 | 330 | 2.2e-29 | 123 |
| NP_001259569.1 | 28.20 | 266 | 1.2e-22 | 102 |
| NP_727682.1 | 24.61 | 317 | 2.4e-21 | 98 |

### INS (P01308)

**Description:** Insulin

No significant BLAST hits found for this protein.

### KLF4 (O43474)

**Description:** Krueppel-like factor 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_995811.1 | 65.74 | 108 | 5.2e-43 | 161 |
| NP_001286303.1 | 65.74 | 108 | 5.2e-43 | 161 |
| NP_001260879.1 | 65.74 | 108 | 5.2e-43 | 161 |
| NP_611747.2 | 73.86 | 88 | 3.3e-41 | 152 |
| NP_001097493.1 | 65.66 | 99 | 6.8e-39 | 151 |
| NP_651232.1 | 55.21 | 96 | 4.9e-32 | 131 |
| NP_001262902.1 | 53.47 | 101 | 5.4e-32 | 131 |
| NP_722636.1 | 56.32 | 87 | 6.7e-31 | 124 |
| NP_722636.1 | 43.33 | 60 | 4.6e-09 | 58 |
| NP_608529.2 | 56.32 | 87 | 7.8e-31 | 123 |
| NP_608529.2 | 43.33 | 60 | 8.2e-09 | 57 |
| NP_001259832.1 | 56.32 | 87 | 7.8e-31 | 123 |
| NP_001259832.1 | 43.33 | 60 | 8.2e-09 | 57 |

### LARGE1 (O95461)

**Description:** Xylosyl- and glucuronyltransferase LARGE1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_611886.1 | 25.00 | 304 | 1.7e-21 | 97 |
| NP_001246488.1 | 25.00 | 304 | 1.7e-21 | 97 |
| NP_611871.2 | 25.89 | 282 | 3.0e-15 | 79 |
| NP_001097911.1 | 32.91 | 158 | 1.6e-13 | 73 |
| NP_609592.1 | 28.80 | 184 | 5.2e-12 | 68 |
| NP_723117.1 | 27.72 | 184 | 1.1e-08 | 58 |
| NP_608958.2 | 27.72 | 184 | 1.1e-08 | 58 |
| NP_001260106.1 | 27.72 | 184 | 1.1e-08 | 58 |
| NP_001260105.1 | 27.72 | 184 | 1.1e-08 | 58 |
| NP_001162891.1 | 27.72 | 184 | 1.1e-08 | 58 |

### LHB (P01229)

**Description:** Lutropin subunit beta

No significant BLAST hits found for this protein.

### MAGT1 (Q9H0U3)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit MAGT1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_609204.2 | 52.16 | 324 | 4.9e-126 | 364 |

### MAN2A1 (Q16706)

**Description:** Alpha-mannosidase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001262413.1 | 40.38 | 1097 | 0.0e+00 | 775 |
| NP_524291.2 | 40.38 | 1097 | 0.0e+00 | 774 |
| NP_650494.2 | 33.27 | 1118 | 0.0e+00 | 568 |
| NP_001287341.1 | 33.27 | 1118 | 0.0e+00 | 568 |
| NP_001262614.1 | 33.27 | 1118 | 0.0e+00 | 568 |
| NP_609408.1 | 28.72 | 846 | 3.3e-82 | 291 |
| NP_723591.1 | 28.72 | 846 | 3.7e-82 | 291 |
| NP_609407.1 | 27.85 | 851 | 2.8e-76 | 272 |
| NP_001260350.1 | 27.85 | 851 | 2.8e-76 | 272 |
| NP_609253.1 | 27.81 | 863 | 1.5e-75 | 271 |

### MAN2A2 (P49641)

**Description:** Alpha-mannosidase 2x

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_524291.2 | 38.84 | 1169 | 0.0e+00 | 755 |
| NP_001262413.1 | 38.67 | 1169 | 0.0e+00 | 754 |
| NP_650494.2 | 34.78 | 1127 | 0.0e+00 | 595 |
| NP_001287341.1 | 34.78 | 1127 | 0.0e+00 | 595 |
| NP_001262614.1 | 34.78 | 1127 | 0.0e+00 | 595 |
| NP_609408.1 | 28.87 | 852 | 1.2e-81 | 290 |
| NP_723591.1 | 28.40 | 852 | 4.3e-79 | 282 |
| NP_609407.1 | 28.66 | 848 | 6.6e-79 | 280 |
| NP_001260350.1 | 28.66 | 848 | 6.6e-79 | 280 |
| NP_609253.1 | 27.46 | 863 | 8.5e-78 | 277 |

### MANEA (Q5SRI9)

**Description:** Glycoprotein endo-alpha-1,2-mannosidase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_608937.1 | 27.02 | 433 | 4.1e-39 | 147 |

### MGAT1 (P26572)

**Description:** Alpha-1,3-mannosyl-glycoprotein 2-beta-N-acetylglucosaminyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_525117.2 | 54.15 | 410 | 5.7e-158 | 454 |

### MGAT2 (Q10469)

**Description:** Alpha-1,6-mannosyl-glycoprotein 2-beta-N-acetylglucosaminyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001014684.1 | 42.57 | 397 | 1.3e-117 | 353 |
| NP_001263096.1 | 40.43 | 418 | 1.0e-112 | 341 |
| NP_651763.4 | 40.89 | 291 | 2.4e-70 | 234 |
| NP_651763.4 | 40.69 | 145 | 3.1e-34 | 135 |

### MGAT3 (Q09327)

**Description:** Beta-1,4-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_723787.1 | 24.23 | 355 | 3.3e-19 | 90 |

### MGAT4A (Q9UM21)

**Description:** Alpha-1,3-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase A

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_648720.3 | 43.82 | 445 | 1.8e-137 | 410 |
| NP_001261847.1 | 43.82 | 445 | 1.8e-137 | 410 |
| NP_648721.1 | 37.11 | 318 | 3.8e-67 | 226 |

### MGAT4B (Q9UQ53)

**Description:** Alpha-1,3-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase B

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_648720.3 | 46.05 | 456 | 2.7e-144 | 428 |
| NP_001261847.1 | 46.05 | 456 | 2.7e-144 | 428 |
| NP_648721.1 | 36.53 | 334 | 3.1e-68 | 229 |

### MGAT4C (Q9UBM8)

**Description:** Alpha-1,3-mannosyl-glycoprotein 4-beta-N-acetylglucosaminyltransferase C

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_648720.3 | 29.98 | 417 | 1.5e-57 | 200 |
| NP_001261847.1 | 29.98 | 417 | 1.5e-57 | 200 |
| NP_648721.1 | 31.08 | 251 | 2.8e-33 | 132 |

### MGAT5 (Q09328)

**Description:** Alpha-1,6-mannosylglycoprotein 6-beta-N-acetylglucosaminyltransferase A

No significant BLAST hits found for this protein.

### MLEC (Q14165)

**Description:** Malectin

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_610087.2 | 39.87 | 301 | 1.5e-70 | 222 |

### MOGS (Q13724)

**Description:** Mannosyl-oligosaccharide glucosidase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_572707.4 | 43.02 | 781 | 0.0e+00 | 590 |
| NP_001245621.1 | 43.02 | 781 | 0.0e+00 | 590 |

### MPDU1 (O75352)

**Description:** Mannose-P-dolichol utilization defect 1 protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_608889.1 | 49.13 | 230 | 5.7e-77 | 233 |
| NP_647878.1 | 24.59 | 183 | 9.4e-10 | 57 |

### MUC1 (P15941)

**Description:** Mucin-1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001368944.1 | 31.08 | 1123 | 2.5e-36 | 151 |
| NP_001368944.1 | 31.31 | 1073 | 1.1e-35 | 149 |
| NP_001368944.1 | 30.88 | 1088 | 1.1e-33 | 142 |
| NP_001368944.1 | 31.32 | 1057 | 2.9e-32 | 137 |
| NP_001368944.1 | 31.11 | 1077 | 3.0e-32 | 137 |
| NP_001368944.1 | 31.11 | 1077 | 3.2e-32 | 137 |
| NP_001368944.1 | 31.02 | 1067 | 7.8e-32 | 136 |
| NP_001368944.1 | 30.97 | 1059 | 2.9e-31 | 134 |
| NP_001368944.1 | 31.26 | 1059 | 7.1e-31 | 133 |
| NP_001368944.1 | 30.78 | 1046 | 4.3e-29 | 127 |
| NP_996064.1 | 23.91 | 1079 | 1.4e-13 | 76 |
| NP_996064.1 | 23.67 | 997 | 6.6e-13 | 74 |
| NP_001285150.1 | 40.23 | 875 | 2.0e-11 | 69 |
| NP_001285150.1 | 40.30 | 603 | 5.0e-05 | 48 |

### MUC12 (Q9UKN1)

**Description:** Mucin-12

No significant BLAST hits found for this protein.

### MUC15 (Q8N387)

**Description:** Mucin-15

No significant BLAST hits found for this protein.

### MUC17 (Q685J3)

**Description:** Mucin-17

No significant BLAST hits found for this protein.

### MUC19 (Q7Z5P9)

**Description:** Mucin-19

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_524060.2 | 30.81 | 1032 | 3.0e-87 | 323 |
| NP_524060.2 | 28.37 | 356 | 6.8e-10 | 67 |
| NP_001261809.1 | 30.81 | 1032 | 1.1e-86 | 322 |
| NP_001261809.1 | 28.37 | 356 | 1.1e-09 | 67 |
| NP_524809.2 | 26.02 | 342 | 5.8e-19 | 96 |
| NP_524809.2 | 23.89 | 427 | 6.1e-10 | 67 |
| NP_524809.2 | 24.37 | 279 | 2.1e-06 | 55 |
| NP_524809.2 | 28.32 | 113 | 1.4e-04 | 49 |
| NP_611280.2 | 34.78 | 322 | 5.3e-08 | 60 |
| NP_611280.2 | 34.47 | 322 | 8.0e-08 | 60 |
| NP_611280.2 | 34.89 | 321 | 8.9e-08 | 60 |
| NP_611280.2 | 34.16 | 322 | 1.7e-07 | 58 |
| NP_611280.2 | 33.96 | 318 | 4.9e-07 | 57 |
| NP_611280.2 | 34.82 | 313 | 6.0e-07 | 57 |
| NP_611280.2 | 32.70 | 370 | 8.5e-07 | 56 |
| NP_611280.2 | 31.93 | 404 | 1.6e-06 | 55 |
| NP_611280.2 | 32.27 | 375 | 2.0e-06 | 55 |
| NP_611280.2 | 32.70 | 367 | 6.2e-06 | 53 |
| NP_611280.2 | 31.58 | 361 | 7.2e-05 | 50 |
| NP_611280.2 | 32.15 | 367 | 1.3e-04 | 49 |
| NP_611280.2 | 31.20 | 375 | 2.1e-04 | 48 |

### MUC20 (Q8N307)

**Description:** Mucin-20

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001137657.1 | 24.00 | 225 | 2.7e-05 | 48 |
| NP_725336.1 | 24.00 | 225 | 2.7e-05 | 48 |
| NP_725337.1 | 24.00 | 225 | 3.6e-05 | 48 |
| NP_523733.2 | 24.00 | 225 | 4.0e-05 | 47 |
| NP_001137660.1 | 24.00 | 225 | 4.0e-05 | 47 |
| NP_001137658.1 | 24.29 | 210 | 5.1e-05 | 47 |
| NP_001246310.1 | 24.29 | 210 | 5.5e-05 | 47 |
| NP_001163148.1 | 24.29 | 210 | 5.7e-05 | 47 |
| NP_001246309.1 | 24.29 | 210 | 5.7e-05 | 47 |
| NP_001246308.1 | 24.00 | 225 | 6.1e-05 | 47 |

### MUC21 (Q5SSG8)

**Description:** Mucin-21

No significant BLAST hits found for this protein.

### MUC3A (Q02505)

**Description:** Mucin-3A

No significant BLAST hits found for this protein.

### MUC4 (Q99102)

**Description:** Mucin-4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001287618.1 | 28.65 | 370 | 5.6e-28 | 126 |
| NP_001163781.1 | 28.65 | 370 | 5.6e-28 | 126 |
| NP_733419.1 | 28.65 | 370 | 1.3e-25 | 118 |
| NP_733418.1 | 28.65 | 370 | 1.3e-25 | 118 |
| NP_001163782.1 | 28.65 | 370 | 2.6e-25 | 117 |
| NP_001261809.1 | 25.10 | 259 | 1.3e-05 | 52 |
| NP_524060.2 | 25.10 | 259 | 1.4e-05 | 52 |

### NAGK (Q9UJ70)

**Description:** N-acetyl-D-glucosamine kinase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_650448.1 | 44.97 | 338 | 2.1e-100 | 300 |
| NP_001262598.1 | 44.97 | 338 | 2.1e-100 | 300 |

### NFYA (P23511)

**Description:** Nuclear transcription factor Y subunit alpha

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_648313.1 | 79.69 | 64 | 3.0e-27 | 111 |

### NFYC (Q13952)

**Description:** Nuclear transcription factor Y subunit gamma

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_572354.1 | 60.18 | 113 | 6.7e-44 | 162 |
| NP_611669.1 | 42.10 | 76 | 1.0e-14 | 72 |
| NP_001163253.1 | 42.10 | 76 | 1.0e-14 | 72 |
| NP_572776.1 | 31.25 | 80 | 2.3e-05 | 44 |
| NP_001285148.1 | 31.25 | 80 | 2.3e-05 | 44 |
| NP_611601.2 | 28.95 | 76 | 4.3e-05 | 45 |

### NGLY1 (Q96IV0)

**Description:** Peptide-N(4)-(N-acetyl-beta-glucosaminyl)asparagine amidase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_724444.3 | 34.27 | 639 | 3.5e-110 | 345 |
| NP_610192.1 | 34.27 | 639 | 3.5e-110 | 345 |
| NP_725813.1 | 36.05 | 86 | 5.5e-05 | 46 |
| NP_725812.1 | 36.05 | 86 | 5.5e-05 | 46 |
| NP_611356.1 | 36.05 | 86 | 5.5e-05 | 46 |

### NR3C1 (P04150)

**Description:** Glucocorticoid receptor

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_729340.1 | 31.83 | 289 | 1.7e-32 | 132 |
| NP_648183.3 | 31.83 | 289 | 1.8e-32 | 132 |
| NP_724458.1 | 26.46 | 461 | 6.2e-27 | 118 |
| NP_724457.1 | 26.46 | 461 | 6.2e-27 | 118 |
| NP_724456.1 | 26.46 | 461 | 6.2e-27 | 118 |
| NP_724460.1 | 25.99 | 454 | 1.6e-26 | 116 |
| NP_001163061.1 | 25.99 | 454 | 1.6e-26 | 116 |
| NP_724459.1 | 26.41 | 409 | 3.0e-26 | 115 |
| NP_723414.2 | 24.68 | 316 | 2.6e-24 | 109 |
| NP_476887.2 | 24.68 | 316 | 2.6e-24 | 109 |

### PCBP4 (P57723)

**Description:** Poly(rC)-binding protein 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_996145.1 | 49.85 | 331 | 1.7e-95 | 290 |
| NP_001036619.1 | 49.40 | 332 | 2.6e-95 | 295 |
| NP_001262198.1 | 49.25 | 335 | 8.7e-95 | 288 |
| NP_730700.2 | 49.09 | 330 | 6.1e-94 | 286 |
| NP_001262197.1 | 49.09 | 330 | 6.1e-94 | 286 |
| NP_001246874.1 | 48.52 | 338 | 1.7e-92 | 288 |
| NP_524208.1 | 47.02 | 353 | 1.7e-92 | 283 |
| NP_001246875.1 | 45.08 | 366 | 4.9e-92 | 288 |
| NP_001246873.1 | 46.35 | 356 | 3.2e-90 | 283 |
| NP_001163493.1 | 46.35 | 356 | 3.2e-90 | 283 |

### PCK1 (P35558)

**Description:** Phosphoenolpyruvate carboxykinase, cytosolic [GTP]

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_523784.2 | 64.60 | 613 | 0.0e+00 | 837 |
| NP_611341.1 | 63.42 | 626 | 0.0e+00 | 832 |
| NP_001097367.1 | 66.43 | 566 | 0.0e+00 | 791 |

### PGM3 (O95394)

**Description:** Phosphoacetylglucosamine mutase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_648588.1 | 53.88 | 529 | 0.0e+00 | 547 |

### PINK1 (Q9BXM7)

**Description:** Serine/threonine-protein kinase PINK1, mitochondrial

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_727110.1 | 42.99 | 314 | 1.4e-74 | 252 |
| NP_572340.2 | 42.99 | 314 | 1.4e-74 | 252 |
| NP_001284952.1 | 42.99 | 314 | 1.4e-74 | 252 |
| NP_001027054.1 | 42.99 | 314 | 1.4e-74 | 252 |
| NP_001027053.1 | 42.99 | 314 | 1.4e-74 | 252 |
| NP_001027052.1 | 42.99 | 314 | 1.4e-74 | 252 |
| NP_001027051.1 | 42.99 | 314 | 1.4e-74 | 252 |
| NP_001027050.1 | 42.99 | 314 | 1.4e-74 | 252 |
| NP_001027049.1 | 42.99 | 314 | 1.4e-74 | 252 |
| NP_726570.1 | 25.67 | 261 | 8.8e-13 | 70 |

### PLXNA4 (Q9HCM2)

**Description:** Plexin-A4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_524637.2 | 42.18 | 1956 | 0.0e+00 | 1378 |
| NP_001245428.1 | 41.83 | 1953 | 0.0e+00 | 1367 |
| NP_726628.1 | 44.59 | 1534 | 0.0e+00 | 1180 |
| NP_726627.1 | 44.59 | 1534 | 0.0e+00 | 1180 |
| NP_524616.2 | 44.35 | 832 | 0.0e+00 | 634 |
| NP_524616.2 | 29.69 | 1125 | 4.0e-130 | 453 |
| NP_001245400.1 | 44.35 | 832 | 0.0e+00 | 634 |
| NP_001245400.1 | 29.69 | 1125 | 4.0e-130 | 453 |
| NP_001163430.1 | 22.29 | 516 | 2.8e-13 | 76 |
| NP_611293.3 | 22.29 | 516 | 2.8e-13 | 76 |
| NP_001137680.1 | 21.34 | 539 | 6.6e-11 | 68 |
| NP_001261024.1 | 21.34 | 539 | 7.6e-11 | 67 |

### POFUT2 (Q9Y2G5)

**Description:** GDP-fucose protein O-fucosyltransferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_569916.1 | 39.13 | 414 | 4.6e-84 | 266 |

### POMGNT1 (Q8WZA1)

**Description:** Protein O-linked-mannose beta-1,2-N-acetylglucosaminyltransferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_525117.2 | 41.51 | 159 | 1.1e-30 | 126 |

### POMT1 (Q9Y6A1)

**Description:** Protein O-mannosyl-transferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_524025.2 | 42.17 | 747 | 0.0e+00 | 619 |
| NP_569858.1 | 29.07 | 774 | 2.9e-82 | 278 |
| NP_001259102.1 | 29.07 | 774 | 2.9e-82 | 278 |

### POMT2 (Q9UKY4)

**Description:** Protein O-mannosyl-transferase 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_569858.1 | 50.39 | 770 | 0.0e+00 | 737 |
| NP_001259102.1 | 50.39 | 770 | 0.0e+00 | 737 |
| NP_524025.2 | 30.94 | 766 | 7.0e-98 | 323 |
| NP_649527.1 | 34.64 | 153 | 8.8e-15 | 74 |

### PPARGC1A (Q9UBK2)

**Description:** Peroxisome proliferator-activated receptor gamma coactivator 1-alpha

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_730836.3 | 41.24 | 97 | 1.9e-18 | 91 |
| NP_730835.2 | 41.24 | 97 | 2.0e-18 | 91 |
| NP_649469.2 | 41.24 | 97 | 2.0e-18 | 91 |

### PRKCSH (P14314)

**Description:** Glucosidase 2 subunit beta

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_609844.1 | 37.85 | 531 | 1.8e-110 | 339 |
| NP_001246063.1 | 37.85 | 531 | 1.8e-110 | 339 |
| NP_650724.1 | 48.67 | 113 | 4.9e-25 | 103 |
| NP_001262710.1 | 48.67 | 113 | 5.6e-25 | 103 |
| NP_572563.3 | 46.48 | 71 | 1.8e-05 | 48 |
| NP_001285054.1 | 46.48 | 71 | 1.8e-05 | 48 |
| NP_001285053.1 | 46.48 | 71 | 1.8e-05 | 48 |
| NP_001096924.2 | 46.48 | 71 | 1.8e-05 | 48 |

### PSMC1 (P62191)

**Description:** 26S proteasome regulatory subunit 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_524469.2 | 91.36 | 440 | 0.0e+00 | 798 |
| NP_001287493.1 | 91.36 | 440 | 0.0e+00 | 798 |
| NP_572686.1 | 50.41 | 369 | 4.4e-138 | 402 |
| NP_001285108.1 | 50.41 | 369 | 4.4e-138 | 402 |
| NP_731401.3 | 49.16 | 358 | 1.1e-130 | 383 |
| NP_649938.2 | 49.16 | 358 | 1.1e-130 | 383 |
| NP_524464.1 | 46.70 | 379 | 1.0e-119 | 356 |
| NP_608447.1 | 49.86 | 353 | 2.9e-119 | 354 |
| NP_651811.1 | 51.42 | 317 | 1.1e-112 | 337 |
| NP_477473.1 | 47.62 | 336 | 1.7e-107 | 325 |

### RAD23B (P54727)

**Description:** UV excision repair protein RAD23 homolog B

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_651918.2 | 37.14 | 455 | 2.0e-84 | 264 |
| NP_001259052.1 | 37.14 | 455 | 2.0e-84 | 264 |
| NP_726561.1 | 40.07 | 297 | 2.2e-59 | 197 |
| NP_651212.1 | 31.56 | 263 | 6.9e-26 | 106 |
| NP_651212.1 | 29.60 | 125 | 5.1e-08 | 54 |
| NP_609919.1 | 37.50 | 64 | 1.1e-07 | 49 |
| NP_001286070.1 | 37.50 | 64 | 1.1e-07 | 49 |
| NP_476776.1 | 35.29 | 68 | 1.4e-07 | 50 |
| NP_001260018.1 | 35.29 | 68 | 1.4e-07 | 50 |
| NP_572306.1 | 39.06 | 64 | 2.3e-06 | 49 |
| NP_572306.1 | 34.38 | 64 | 2.0e-04 | 43 |
| NP_572306.1 | 29.73 | 74 | 5.1e-04 | 42 |
| NP_572306.1 | 29.73 | 74 | 5.1e-04 | 42 |
| NP_476778.1 | 37.50 | 64 | 4.1e-06 | 47 |

### RBL2 (Q08999)

**Description:** Retinoblastoma-like protein 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_525036.2 | 30.00 | 600 | 1.1e-81 | 285 |
| NP_525036.2 | 28.88 | 277 | 6.1e-23 | 106 |
| NP_524372.1 | 23.16 | 626 | 4.7e-28 | 122 |
| NP_524372.1 | 40.00 | 95 | 1.4e-07 | 56 |

### RENBP (P51606)

**Description:** N-acylglucosamine 2-epimerase

No significant BLAST hits found for this protein.

### RETN (Q9HD89)

**Description:** Resistin

No significant BLAST hits found for this protein.

### RFT1 (Q96AA3)

**Description:** Man(5)GlcNAc(2)-PP-dolichol translocation protein RFT1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_572246.1 | 39.17 | 480 | 3.5e-90 | 288 |

### ROBO1 (Q9Y6N7)

**Description:** Roundabout homolog 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_726224.3 | 38.67 | 825 | 4.0e-173 | 560 |
| NP_726224.3 | 28.04 | 296 | 4.3e-23 | 108 |
| NP_726224.3 | 26.42 | 246 | 5.2e-11 | 68 |
| NP_476899.1 | 38.67 | 825 | 4.0e-173 | 560 |
| NP_476899.1 | 28.04 | 296 | 4.3e-23 | 108 |
| NP_476899.1 | 26.42 | 246 | 5.2e-11 | 68 |
| NP_608592.2 | 34.85 | 924 | 6.4e-162 | 528 |
| NP_001368960.1 | 34.85 | 924 | 2.3e-161 | 527 |
| NP_536792.2 | 39.31 | 669 | 1.6e-140 | 472 |
| NP_536792.2 | 31.42 | 296 | 8.7e-28 | 123 |
| NP_536792.2 | 28.79 | 198 | 1.3e-16 | 87 |
| NP_536792.2 | 27.83 | 212 | 1.3e-08 | 60 |
| NP_001259868.1 | 39.31 | 669 | 3.8e-140 | 472 |
| NP_001259868.1 | 31.42 | 296 | 8.7e-28 | 123 |
| NP_001259868.1 | 28.79 | 198 | 1.3e-16 | 87 |
| NP_001259868.1 | 27.83 | 212 | 1.5e-08 | 60 |
| NP_001036493.1 | 27.01 | 833 | 2.8e-61 | 233 |
| NP_001036493.1 | 28.63 | 517 | 1.2e-33 | 143 |
| NP_001036493.1 | 27.87 | 183 | 1.6e-05 | 50 |
| NP_001036493.1 | 23.08 | 195 | 1.9e-05 | 50 |
| NP_001036493.1 | 27.81 | 187 | 1.2e-04 | 47 |
| NP_001036493.1 | 22.86 | 175 | 4.4e-04 | 45 |
| NP_001036495.1 | 27.01 | 833 | 2.9e-61 | 233 |
| NP_001036495.1 | 27.61 | 518 | 2.0e-33 | 142 |
| NP_001036495.1 | 27.83 | 575 | 4.2e-33 | 141 |
| NP_001036495.1 | 27.87 | 183 | 1.4e-05 | 50 |
| NP_001036495.1 | 23.08 | 195 | 1.9e-05 | 50 |
| NP_001036495.1 | 27.81 | 187 | 1.3e-04 | 47 |
| NP_001036495.1 | 22.86 | 175 | 4.3e-04 | 45 |
| NP_001036505.1 | 26.35 | 835 | 1.4e-57 | 221 |
| NP_001036505.1 | 28.16 | 515 | 9.7e-30 | 130 |
| NP_001036505.1 | 27.87 | 183 | 1.4e-05 | 50 |
| NP_001036505.1 | 23.08 | 195 | 2.0e-05 | 50 |
| NP_001036505.1 | 27.81 | 187 | 1.1e-04 | 47 |
| NP_001036505.1 | 22.86 | 175 | 4.4e-04 | 45 |
| NP_001036478.1 | 26.11 | 835 | 2.1e-57 | 220 |
| NP_001036478.1 | 27.77 | 515 | 6.6e-31 | 134 |
| NP_001036478.1 | 27.87 | 183 | 1.4e-05 | 50 |
| NP_001036478.1 | 23.08 | 195 | 1.7e-05 | 50 |
| NP_001036478.1 | 27.81 | 187 | 1.1e-04 | 47 |
| NP_001036478.1 | 22.86 | 175 | 3.7e-04 | 46 |

### RPN1 (P04843)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_995680.1 | 46.03 | 441 | 3.7e-140 | 415 |
| NP_001260335.1 | 46.03 | 441 | 3.7e-140 | 415 |

### RPN2 (P04844)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001286549.1 | 33.49 | 645 | 9.0e-95 | 305 |
| NP_611265.1 | 33.49 | 645 | 1.0e-94 | 305 |

### RPS27A (P62979)

**Description:** Ubiquitin-ribosomal protein eS31 fusion protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_476778.1 | 91.03 | 156 | 7.6e-91 | 261 |
| NP_476776.1 | 100.00 | 76 | 3.2e-49 | 155 |
| NP_001260018.1 | 100.00 | 76 | 3.2e-49 | 155 |
| NP_727078.1 | 100.00 | 77 | 1.5e-46 | 159 |
| NP_727078.1 | 97.44 | 78 | 6.1e-46 | 157 |
| NP_727078.1 | 97.44 | 78 | 6.1e-46 | 157 |
| NP_727078.1 | 97.44 | 78 | 6.1e-46 | 157 |
| NP_727078.1 | 97.44 | 78 | 6.1e-46 | 157 |
| NP_727078.1 | 97.44 | 78 | 6.1e-46 | 157 |
| NP_727078.1 | 97.44 | 78 | 6.1e-46 | 157 |
| NP_001284937.1 | 100.00 | 77 | 1.5e-46 | 159 |
| NP_001284937.1 | 97.44 | 78 | 6.1e-46 | 157 |
| NP_001284937.1 | 97.44 | 78 | 6.1e-46 | 157 |
| NP_001284937.1 | 97.44 | 78 | 6.1e-46 | 157 |
| NP_001284937.1 | 97.44 | 78 | 6.1e-46 | 157 |
| NP_001284937.1 | 97.44 | 78 | 6.1e-46 | 157 |
| NP_001284937.1 | 97.44 | 78 | 6.1e-46 | 157 |
| NP_572306.1 | 93.59 | 78 | 3.6e-46 | 152 |
| NP_572306.1 | 85.90 | 78 | 2.8e-40 | 137 |
| NP_572306.1 | 85.90 | 78 | 2.8e-40 | 137 |
| NP_572306.1 | 91.18 | 68 | 2.0e-37 | 130 |
| NP_995994.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_995994.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_995994.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_995994.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_995994.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_995994.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_995994.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_995994.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_995994.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_995994.1 | 97.44 | 78 | 5.1e-45 | 157 |
| NP_728908.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_728908.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_728908.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_728908.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_728908.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_728908.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_728908.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_728908.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_728908.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_728908.1 | 97.44 | 78 | 5.1e-45 | 157 |
| NP_523909.2 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_523909.2 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_523909.2 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_523909.2 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_523909.2 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_523909.2 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_523909.2 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_523909.2 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_523909.2 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_523909.2 | 97.44 | 78 | 5.1e-45 | 157 |
| NP_001261383.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_001261383.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_001261383.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_001261383.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_001261383.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_001261383.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_001261383.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_001261383.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_001261383.1 | 97.44 | 78 | 4.6e-45 | 157 |
| NP_001261383.1 | 97.44 | 78 | 5.1e-45 | 157 |

### SEMA5B (Q9P283)

**Description:** Semaphorin-5B

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_611293.3 | 38.58 | 959 | 0.0e+00 | 629 |
| NP_001163430.1 | 38.58 | 959 | 0.0e+00 | 629 |
| NP_001260267.1 | 40.16 | 513 | 1.3e-98 | 332 |
| NP_001188732.1 | 40.16 | 513 | 1.4e-98 | 332 |
| NP_001260266.1 | 40.16 | 513 | 1.8e-98 | 332 |
| NP_609243.3 | 40.16 | 513 | 6.9e-98 | 332 |
| NP_001188731.1 | 40.16 | 513 | 6.9e-98 | 332 |
| NP_723407.2 | 40.16 | 513 | 9.9e-98 | 332 |
| NP_001260265.1 | 40.16 | 513 | 1.6e-96 | 333 |
| NP_995879.1 | 34.77 | 578 | 2.0e-83 | 289 |

### SIRT3 (Q9NTG7)

**Description:** NAD-dependent protein deacetylase sirtuin-3, mitochondrial

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_650880.3 | 50.32 | 314 | 2.2e-97 | 296 |
| NP_001287422.2 | 50.32 | 314 | 2.2e-97 | 296 |
| NP_477351.1 | 39.41 | 269 | 1.1e-58 | 204 |
| NP_572241.2 | 27.92 | 197 | 2.7e-14 | 73 |
| NP_649990.2 | 28.70 | 216 | 2.7e-14 | 73 |
| NP_651664.2 | 29.13 | 230 | 1.4e-12 | 70 |
| NP_727014.2 | 26.83 | 123 | 3.0e-06 | 48 |
| NP_727013.2 | 26.83 | 123 | 3.0e-06 | 48 |

### SLIT2 (O94813)

**Description:** Slit homolog 2 protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001261017.1 | 44.39 | 1514 | 0.0e+00 | 1294 |
| NP_476727.1 | 44.39 | 1514 | 0.0e+00 | 1286 |
| NP_001097334.1 | 44.39 | 1514 | 0.0e+00 | 1286 |
| NP_476728.1 | 43.66 | 1514 | 0.0e+00 | 1253 |
| NP_001097333.1 | 43.66 | 1514 | 0.0e+00 | 1253 |
| NP_476729.1 | 43.40 | 1514 | 0.0e+00 | 1248 |
| NP_476859.2 | 45.23 | 241 | 8.1e-48 | 189 |
| NP_476859.2 | 29.36 | 613 | 8.1e-47 | 186 |
| NP_476859.2 | 31.78 | 343 | 1.4e-42 | 172 |
| NP_476859.2 | 38.52 | 244 | 2.4e-41 | 168 |
| NP_476859.2 | 38.84 | 242 | 1.9e-38 | 158 |
| NP_476859.2 | 37.10 | 248 | 6.7e-36 | 150 |
| NP_476859.2 | 31.88 | 298 | 1.0e-35 | 149 |
| NP_476859.2 | 37.34 | 241 | 2.8e-34 | 145 |
| NP_476859.2 | 38.09 | 231 | 4.7e-34 | 144 |
| NP_476859.2 | 31.08 | 296 | 1.1e-29 | 129 |
| NP_476859.2 | 31.09 | 283 | 6.5e-27 | 120 |
| NP_476859.2 | 29.30 | 256 | 1.2e-19 | 97 |
| NP_476859.2 | 36.84 | 114 | 3.4e-12 | 72 |
| NP_476859.2 | 30.83 | 120 | 4.3e-06 | 52 |
| NP_476859.2 | 26.43 | 193 | 4.4e-06 | 52 |
| NP_476859.2 | 32.52 | 123 | 6.9e-06 | 51 |
| NP_476859.2 | 33.03 | 109 | 2.5e-05 | 49 |
| NP_476859.2 | 30.70 | 114 | 4.6e-05 | 48 |
| NP_476859.2 | 29.60 | 125 | 4.8e-05 | 48 |
| NP_476859.2 | 28.99 | 169 | 3.9e-04 | 45 |
| NP_476859.2 | 30.64 | 124 | 5.2e-04 | 45 |
| NP_476859.2 | 28.20 | 117 | 5.5e-04 | 45 |
| NP_001245510.1 | 45.23 | 241 | 8.1e-48 | 189 |
| NP_001245510.1 | 29.36 | 613 | 8.1e-47 | 186 |
| NP_001245510.1 | 31.78 | 343 | 1.4e-42 | 172 |
| NP_001245510.1 | 38.52 | 244 | 2.4e-41 | 168 |
| NP_001245510.1 | 38.84 | 242 | 1.9e-38 | 158 |
| NP_001245510.1 | 37.10 | 248 | 6.7e-36 | 150 |
| NP_001245510.1 | 31.88 | 298 | 1.0e-35 | 149 |
| NP_001245510.1 | 37.34 | 241 | 2.8e-34 | 145 |
| NP_001245510.1 | 38.09 | 231 | 4.7e-34 | 144 |
| NP_001245510.1 | 31.08 | 296 | 1.1e-29 | 129 |
| NP_001245510.1 | 31.09 | 283 | 6.5e-27 | 120 |
| NP_001245510.1 | 29.30 | 256 | 1.2e-19 | 97 |
| NP_001245510.1 | 36.84 | 114 | 3.4e-12 | 72 |
| NP_001245510.1 | 30.83 | 120 | 4.3e-06 | 52 |
| NP_001245510.1 | 26.43 | 193 | 4.4e-06 | 52 |
| NP_001245510.1 | 32.52 | 123 | 6.9e-06 | 51 |
| NP_001245510.1 | 33.03 | 109 | 2.5e-05 | 49 |
| NP_001245510.1 | 30.70 | 114 | 4.6e-05 | 48 |
| NP_001245510.1 | 29.60 | 125 | 4.8e-05 | 48 |
| NP_001245510.1 | 28.99 | 169 | 3.9e-04 | 45 |
| NP_001245510.1 | 30.64 | 124 | 5.2e-04 | 45 |
| NP_001245510.1 | 28.20 | 117 | 5.5e-04 | 45 |
| NP_524081.1 | 22.92 | 890 | 2.3e-44 | 177 |
| NP_524081.1 | 22.91 | 873 | 3.4e-38 | 157 |
| NP_524081.1 | 25.75 | 664 | 4.9e-35 | 147 |
| NP_524081.1 | 22.85 | 569 | 4.1e-18 | 91 |
| NP_524081.1 | 24.02 | 562 | 6.9e-18 | 90 |
| NP_524081.1 | 28.64 | 206 | 5.3e-16 | 84 |
| NP_001246766.1 | 22.92 | 890 | 2.3e-44 | 177 |
| NP_001246766.1 | 22.91 | 873 | 3.4e-38 | 157 |
| NP_001246766.1 | 25.75 | 664 | 4.9e-35 | 147 |
| NP_001246766.1 | 22.85 | 569 | 4.1e-18 | 91 |
| NP_001246766.1 | 24.02 | 562 | 6.9e-18 | 90 |
| NP_001246766.1 | 28.64 | 206 | 5.3e-16 | 84 |

### SMAD2 (Q15796)

**Description:** Mothers against decapentaplegic homolog 2

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001259992.1 | 56.94 | 497 | 0.0e+00 | 522 |
| NP_477017.1 | 56.94 | 497 | 0.0e+00 | 521 |
| NP_511079.1 | 81.45 | 248 | 3.0e-145 | 424 |
| NP_511079.1 | 47.34 | 207 | 5.6e-54 | 188 |
| NP_001285006.1 | 81.45 | 248 | 3.0e-145 | 424 |
| NP_001285006.1 | 47.34 | 207 | 5.6e-54 | 188 |
| NP_733438.1 | 40.45 | 220 | 4.9e-44 | 164 |
| NP_733438.1 | 47.22 | 144 | 1.0e-30 | 125 |
| NP_733439.2 | 40.45 | 220 | 7.3e-44 | 164 |
| NP_733439.2 | 47.22 | 144 | 8.9e-31 | 126 |
| NP_524610.1 | 40.45 | 220 | 7.3e-44 | 164 |
| NP_524610.1 | 47.22 | 144 | 8.9e-31 | 126 |
| NP_732196.1 | 25.93 | 405 | 1.3e-22 | 101 |
| NP_477260.1 | 25.93 | 405 | 1.3e-22 | 101 |
| NP_001014629.1 | 25.93 | 405 | 1.3e-22 | 101 |

### SMAD3 (P84022)

**Description:** Mothers against decapentaplegic homolog 3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001259992.1 | 61.18 | 456 | 0.0e+00 | 540 |
| NP_477017.1 | 61.18 | 456 | 0.0e+00 | 539 |
| NP_511079.1 | 81.50 | 254 | 6.0e-145 | 422 |
| NP_511079.1 | 51.34 | 187 | 2.4e-58 | 198 |
| NP_001285006.1 | 81.50 | 254 | 6.0e-145 | 422 |
| NP_001285006.1 | 51.34 | 187 | 2.4e-58 | 198 |
| NP_733438.1 | 40.00 | 220 | 9.6e-44 | 162 |
| NP_733438.1 | 57.90 | 114 | 6.4e-36 | 140 |
| NP_733439.2 | 40.00 | 220 | 1.2e-43 | 163 |
| NP_733439.2 | 57.90 | 114 | 7.5e-36 | 140 |
| NP_524610.1 | 40.00 | 220 | 1.2e-43 | 163 |
| NP_524610.1 | 57.90 | 114 | 7.5e-36 | 140 |
| NP_732196.1 | 27.70 | 379 | 3.7e-24 | 105 |
| NP_477260.1 | 27.70 | 379 | 3.7e-24 | 105 |
| NP_001014629.1 | 27.70 | 379 | 3.7e-24 | 105 |

### SOD2 (P04179)

**Description:** Superoxide dismutase [Mn], mitochondrial

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_476925.1 | 62.07 | 203 | 7.0e-95 | 276 |
| NP_001286503.1 | 62.07 | 203 | 7.0e-95 | 276 |

### SREBF1 (P36956)

**Description:** Sterol regulatory element-binding protein 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_730450.1 | 34.86 | 872 | 1.5e-124 | 410 |
| NP_730449.1 | 34.86 | 872 | 1.5e-124 | 410 |
| NP_524166.2 | 34.86 | 872 | 1.5e-124 | 410 |
| NP_001262064.1 | 34.86 | 872 | 1.5e-124 | 410 |
| NP_001245436.1 | 33.33 | 84 | 1.0e-06 | 53 |
| NP_001245437.1 | 33.33 | 84 | 1.1e-06 | 53 |
| NP_001033808.1 | 33.33 | 84 | 1.2e-06 | 53 |
| NP_001033807.1 | 33.33 | 84 | 1.2e-06 | 53 |
| NP_001260034.1 | 26.39 | 216 | 2.5e-04 | 46 |
| NP_001260034.1 | 26.28 | 156 | 2.7e-04 | 46 |
| NP_001260034.1 | 31.13 | 151 | 3.7e-04 | 45 |
| NP_001260034.1 | 26.13 | 199 | 7.2e-04 | 44 |
| NP_001260040.1 | 26.39 | 216 | 2.5e-04 | 46 |
| NP_001260040.1 | 26.28 | 156 | 2.8e-04 | 46 |
| NP_001260040.1 | 31.13 | 151 | 3.8e-04 | 45 |
| NP_001260040.1 | 26.13 | 199 | 7.2e-04 | 44 |

### SSPOP (A2VEC9)

**Description:** SCO-spondin

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_524060.2 | 30.44 | 1347 | 3.0e-148 | 524 |
| NP_524060.2 | 23.68 | 794 | 2.2e-36 | 154 |
| NP_524060.2 | 28.07 | 342 | 5.7e-23 | 110 |
| NP_524060.2 | 26.88 | 480 | 4.6e-20 | 100 |
| NP_524060.2 | 24.94 | 437 | 2.1e-14 | 81 |
| NP_524060.2 | 26.87 | 227 | 3.9e-07 | 57 |
| NP_524060.2 | 36.77 | 68 | 1.2e-05 | 52 |
| NP_524060.2 | 25.78 | 287 | 2.9e-05 | 51 |
| NP_001261809.1 | 30.44 | 1347 | 3.9e-148 | 523 |
| NP_001261809.1 | 23.68 | 794 | 2.8e-36 | 154 |
| NP_001261809.1 | 28.07 | 342 | 6.3e-23 | 110 |
| NP_001261809.1 | 26.88 | 480 | 5.2e-20 | 100 |
| NP_001261809.1 | 24.94 | 437 | 2.3e-14 | 81 |
| NP_001261809.1 | 26.87 | 227 | 4.2e-07 | 57 |
| NP_001261809.1 | 36.77 | 68 | 1.2e-05 | 52 |
| NP_001261809.1 | 25.78 | 287 | 2.8e-05 | 51 |
| NP_572563.3 | 30.32 | 465 | 8.5e-35 | 149 |
| NP_572563.3 | 33.03 | 330 | 5.7e-34 | 146 |
| NP_572563.3 | 30.23 | 387 | 5.1e-33 | 143 |
| NP_572563.3 | 33.21 | 271 | 6.0e-27 | 123 |
| NP_572563.3 | 32.03 | 306 | 4.4e-24 | 114 |
| NP_572563.3 | 36.47 | 170 | 4.7e-20 | 100 |
| NP_572563.3 | 28.99 | 238 | 2.4e-16 | 88 |
| NP_572563.3 | 37.21 | 129 | 3.4e-16 | 87 |
| NP_572563.3 | 31.86 | 248 | 1.4e-14 | 82 |
| NP_572563.3 | 27.89 | 337 | 4.8e-14 | 80 |
| NP_572563.3 | 35.83 | 120 | 7.1e-13 | 76 |
| NP_572563.3 | 31.53 | 203 | 9.6e-12 | 73 |
| NP_572563.3 | 30.51 | 118 | 6.2e-04 | 47 |
| NP_001285054.1 | 30.32 | 465 | 8.5e-35 | 149 |
| NP_001285054.1 | 33.03 | 330 | 5.7e-34 | 146 |
| NP_001285054.1 | 30.23 | 387 | 5.1e-33 | 143 |
| NP_001285054.1 | 33.21 | 271 | 6.0e-27 | 123 |
| NP_001285054.1 | 32.03 | 306 | 4.4e-24 | 114 |
| NP_001285054.1 | 36.47 | 170 | 4.7e-20 | 100 |
| NP_001285054.1 | 28.99 | 238 | 2.4e-16 | 88 |
| NP_001285054.1 | 37.21 | 129 | 3.4e-16 | 87 |
| NP_001285054.1 | 31.86 | 248 | 1.4e-14 | 82 |
| NP_001285054.1 | 27.89 | 337 | 4.8e-14 | 80 |
| NP_001285054.1 | 35.83 | 120 | 7.1e-13 | 76 |
| NP_001285054.1 | 31.53 | 203 | 9.6e-12 | 73 |
| NP_001285054.1 | 30.51 | 118 | 6.2e-04 | 47 |
| NP_001285053.1 | 30.32 | 465 | 8.5e-35 | 149 |
| NP_001285053.1 | 33.03 | 330 | 5.7e-34 | 146 |
| NP_001285053.1 | 30.23 | 387 | 5.1e-33 | 143 |
| NP_001285053.1 | 33.21 | 271 | 6.0e-27 | 123 |
| NP_001285053.1 | 32.03 | 306 | 4.4e-24 | 114 |
| NP_001285053.1 | 36.47 | 170 | 4.7e-20 | 100 |
| NP_001285053.1 | 28.99 | 238 | 2.4e-16 | 88 |
| NP_001285053.1 | 37.21 | 129 | 3.4e-16 | 87 |
| NP_001285053.1 | 31.86 | 248 | 1.4e-14 | 82 |
| NP_001285053.1 | 27.89 | 337 | 4.8e-14 | 80 |
| NP_001285053.1 | 35.83 | 120 | 7.1e-13 | 76 |
| NP_001285053.1 | 31.53 | 203 | 9.6e-12 | 73 |
| NP_001285053.1 | 30.51 | 118 | 6.2e-04 | 47 |
| NP_001096924.2 | 30.32 | 465 | 8.5e-35 | 149 |
| NP_001096924.2 | 33.03 | 330 | 5.7e-34 | 146 |
| NP_001096924.2 | 30.23 | 387 | 5.1e-33 | 143 |
| NP_001096924.2 | 33.21 | 271 | 6.0e-27 | 123 |
| NP_001096924.2 | 32.03 | 306 | 4.4e-24 | 114 |
| NP_001096924.2 | 36.47 | 170 | 4.7e-20 | 100 |
| NP_001096924.2 | 28.99 | 238 | 2.4e-16 | 88 |
| NP_001096924.2 | 37.21 | 129 | 3.4e-16 | 87 |
| NP_001096924.2 | 31.86 | 248 | 1.4e-14 | 82 |
| NP_001096924.2 | 27.89 | 337 | 4.8e-14 | 80 |
| NP_001096924.2 | 35.83 | 120 | 7.1e-13 | 76 |
| NP_001096924.2 | 31.53 | 203 | 9.6e-12 | 73 |
| NP_001096924.2 | 30.51 | 118 | 6.2e-04 | 47 |
| NP_788284.3 | 32.11 | 327 | 9.5e-32 | 139 |
| NP_788284.3 | 32.06 | 418 | 1.5e-28 | 128 |
| NP_788284.3 | 33.81 | 278 | 1.5e-28 | 128 |
| NP_788284.3 | 32.22 | 360 | 7.7e-28 | 126 |
| NP_788284.3 | 34.85 | 330 | 1.3e-27 | 125 |
| NP_788284.3 | 32.13 | 277 | 4.7e-24 | 113 |
| NP_788284.3 | 28.33 | 360 | 4.7e-22 | 107 |
| NP_788284.3 | 32.28 | 316 | 1.6e-18 | 95 |
| NP_788284.3 | 30.06 | 326 | 7.4e-18 | 93 |
| NP_788284.3 | 30.60 | 232 | 6.7e-15 | 83 |
| NP_788284.3 | 38.39 | 112 | 4.2e-08 | 60 |
| NP_788284.3 | 38.38 | 99 | 7.3e-07 | 57 |
| NP_788284.3 | 36.29 | 124 | 8.7e-07 | 56 |
| NP_788284.3 | 26.82 | 261 | 1.7e-06 | 55 |
| NP_788284.3 | 30.00 | 130 | 3.4e-04 | 48 |
| NP_001260800.2 | 32.11 | 327 | 1.2e-31 | 139 |
| NP_001260800.2 | 32.06 | 418 | 1.6e-28 | 128 |
| NP_001260800.2 | 33.81 | 278 | 1.7e-28 | 128 |
| NP_001260800.2 | 32.22 | 360 | 8.1e-28 | 126 |
| NP_001260800.2 | 34.85 | 330 | 1.5e-27 | 125 |
| NP_001260800.2 | 32.13 | 277 | 4.7e-24 | 113 |
| NP_001260800.2 | 28.33 | 360 | 5.4e-22 | 107 |
| NP_001260800.2 | 26.37 | 292 | 6.8e-20 | 100 |
| NP_001260800.2 | 32.28 | 316 | 1.7e-18 | 95 |
| NP_001260800.2 | 30.60 | 232 | 6.7e-15 | 83 |
| NP_001260800.2 | 32.17 | 143 | 3.0e-11 | 71 |
| NP_001260800.2 | 38.38 | 99 | 7.5e-07 | 57 |
| NP_001260800.2 | 36.29 | 124 | 9.2e-07 | 56 |
| NP_001260800.2 | 26.82 | 261 | 1.9e-06 | 55 |
| NP_001260800.2 | 37.27 | 110 | 2.6e-06 | 55 |
| NP_001260800.2 | 30.00 | 130 | 3.3e-04 | 48 |
| NP_001097934.2 | 30.90 | 343 | 1.4e-31 | 137 |
| NP_001097934.2 | 31.94 | 360 | 2.2e-31 | 137 |
| NP_001097934.2 | 31.36 | 169 | 2.6e-11 | 71 |
| NP_001097934.2 | 30.16 | 126 | 5.2e-07 | 57 |
| NP_733119.2 | 32.28 | 316 | 1.2e-28 | 127 |
| NP_733119.2 | 32.09 | 296 | 3.8e-13 | 77 |
| NP_733119.2 | 37.41 | 147 | 1.4e-12 | 75 |
| NP_733119.2 | 25.90 | 278 | 4.7e-12 | 73 |

### ST3GAL3 (Q11203)

**Description:** CMP-N-acetylneuraminate-beta-1,4-galactoside alpha-2,3-sialyltransferase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_726474.1 | 44.29 | 70 | 1.4e-09 | 59 |
| NP_523853.1 | 44.29 | 70 | 1.6e-09 | 59 |

### ST3GAL4 (Q11206)

**Description:** CMP-N-acetylneuraminate-beta-galactosamide-alpha-2,3-sialyltransferase 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_523853.1 | 23.64 | 220 | 6.5e-08 | 54 |
| NP_726474.1 | 23.64 | 220 | 8.0e-08 | 53 |
| NP_523853.1 | 23.64 | 220 | 6.5e-08 | 54 |
| NP_726474.1 | 23.64 | 220 | 8.0e-08 | 53 |

### ST3GAL4 (Q11206)

**Description:** CMP-N-acetylneuraminate-beta-galactosamide-alpha-2,3-sialyltransferase 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_523853.1 | 23.64 | 220 | 6.5e-08 | 54 |
| NP_726474.1 | 23.64 | 220 | 8.0e-08 | 53 |
| NP_523853.1 | 23.64 | 220 | 6.5e-08 | 54 |
| NP_726474.1 | 23.64 | 220 | 8.0e-08 | 53 |

### ST6GAL1 (P15907)

**Description:** Beta-galactoside alpha-2,6-sialyltransferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_726474.1 | 38.16 | 283 | 8.6e-60 | 201 |
| NP_523853.1 | 38.16 | 283 | 1.1e-59 | 201 |
| NP_726474.1 | 38.16 | 283 | 8.6e-60 | 201 |
| NP_523853.1 | 38.16 | 283 | 1.1e-59 | 201 |

### ST6GAL1 (P15907)

**Description:** Beta-galactoside alpha-2,6-sialyltransferase 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_726474.1 | 38.16 | 283 | 8.6e-60 | 201 |
| NP_523853.1 | 38.16 | 283 | 1.1e-59 | 201 |
| NP_726474.1 | 38.16 | 283 | 8.6e-60 | 201 |
| NP_523853.1 | 38.16 | 283 | 1.1e-59 | 201 |

### ST6GALNAC3 (Q8NDV1)

**Description:** Alpha-N-acetylgalactosaminide alpha-2,6-sialyltransferase 3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_726474.1 | 40.70 | 86 | 2.6e-10 | 61 |
| NP_523853.1 | 40.70 | 86 | 3.1e-10 | 61 |

### ST8SIA2 (Q92186)

**Description:** Alpha-2,8-sialyltransferase 8B

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_726474.1 | 40.00 | 80 | 1.9e-08 | 56 |
| NP_523853.1 | 40.00 | 80 | 1.9e-08 | 56 |

### ST8SIA3 (O43173)

**Description:** Alpha-N-acetylneuraminate alpha-2,8-sialyltransferase ST8SIA3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_726474.1 | 40.98 | 61 | 1.7e-07 | 53 |
| NP_523853.1 | 40.98 | 61 | 1.9e-07 | 53 |

### ST8SIA6 (P61647)

**Description:** Alpha-2,8-sialyltransferase 8F

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_726474.1 | 35.95 | 89 | 2.7e-05 | 46 |
| NP_523853.1 | 35.95 | 89 | 2.7e-05 | 46 |

### STK11 (Q15831)

**Description:** Serine/threonine-protein kinase STK11

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_731846.1 | 61.00 | 341 | 5.6e-154 | 448 |
| NP_731845.1 | 61.00 | 341 | 5.6e-154 | 448 |
| NP_731844.1 | 61.00 | 341 | 5.6e-154 | 448 |
| NP_731843.1 | 61.00 | 341 | 5.6e-154 | 448 |
| NP_731842.1 | 61.00 | 341 | 5.6e-154 | 448 |
| NP_731841.1 | 61.00 | 341 | 5.6e-154 | 448 |
| NP_731840.1 | 61.00 | 341 | 5.6e-154 | 448 |
| NP_650302.1 | 61.00 | 341 | 5.6e-154 | 448 |
| NP_001163606.1 | 61.00 | 341 | 5.6e-154 | 448 |
| NP_996327.1 | 35.42 | 288 | 3.7e-48 | 173 |

### STT3A (P46977)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit STT3A

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_608425.1 | 77.17 | 701 | 0.0e+00 | 1153 |
| NP_001259751.2 | 77.17 | 701 | 0.0e+00 | 1153 |
| NP_524494.1 | 57.87 | 712 | 0.0e+00 | 843 |

### THBS1 (P07996)

**Description:** Thrombospondin-1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_523495.2 | 48.04 | 662 | 3.8e-178 | 552 |
| NP_001137802.1 | 48.04 | 662 | 3.8e-178 | 552 |
| NP_001260148.1 | 48.50 | 633 | 7.5e-175 | 530 |
| NP_001137803.1 | 48.50 | 633 | 7.5e-175 | 530 |
| NP_611293.3 | 39.01 | 182 | 2.6e-27 | 120 |
| NP_611293.3 | 34.04 | 188 | 1.1e-15 | 83 |
| NP_611293.3 | 35.66 | 129 | 4.3e-14 | 77 |
| NP_611293.3 | 32.39 | 176 | 1.0e-08 | 60 |
| NP_611293.3 | 39.13 | 69 | 8.1e-06 | 50 |
| NP_001163430.1 | 39.01 | 182 | 2.8e-27 | 120 |
| NP_001163430.1 | 34.04 | 188 | 1.2e-15 | 82 |
| NP_001163430.1 | 35.66 | 129 | 4.6e-14 | 77 |
| NP_001163430.1 | 32.39 | 176 | 1.2e-08 | 60 |
| NP_001163430.1 | 39.13 | 69 | 8.1e-06 | 50 |
| NP_001162678.1 | 34.82 | 112 | 9.7e-14 | 76 |
| NP_001162678.1 | 48.21 | 56 | 1.2e-08 | 60 |
| NP_732036.2 | 54.39 | 57 | 1.3e-13 | 76 |
| NP_996218.1 | 54.39 | 57 | 1.3e-13 | 76 |
| NP_001262604.1 | 54.39 | 57 | 1.5e-13 | 76 |

### THSD1 (Q9NS62)

**Description:** Thrombospondin type-1 domain-containing protein 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_611293.3 | 44.23 | 52 | 2.0e-05 | 48 |
| NP_611293.3 | 32.99 | 97 | 6.9e-05 | 47 |
| NP_001163430.1 | 44.23 | 52 | 2.0e-05 | 48 |
| NP_001163430.1 | 32.99 | 97 | 7.1e-05 | 47 |
| NP_649200.2 | 24.12 | 170 | 1.5e-04 | 46 |
| NP_001162678.1 | 30.00 | 90 | 1.5e-04 | 45 |
| NP_572247.2 | 30.00 | 90 | 1.8e-04 | 45 |
| NP_732036.2 | 30.86 | 81 | 1.0e-03 | 43 |

### THSD4 (Q6ZMP0)

**Description:** Thrombospondin type-1 domain-containing protein 4

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_788751.2 | 32.89 | 605 | 2.9e-71 | 260 |
| NP_788751.2 | 35.26 | 363 | 6.6e-45 | 178 |
| NP_788752.2 | 32.89 | 605 | 4.3e-71 | 260 |
| NP_788752.2 | 35.07 | 365 | 8.4e-45 | 177 |
| NP_001163761.1 | 32.89 | 605 | 4.5e-71 | 260 |
| NP_001163761.1 | 35.07 | 365 | 6.8e-45 | 178 |
| NP_001162939.2 | 27.44 | 747 | 5.2e-71 | 254 |
| NP_609406.2 | 27.41 | 748 | 6.8e-71 | 254 |
| NP_001163760.1 | 32.89 | 605 | 7.9e-71 | 259 |
| NP_001163760.1 | 35.26 | 363 | 9.1e-45 | 177 |
| NP_001188781.1 | 25.88 | 622 | 4.3e-47 | 178 |
| NP_996218.1 | 24.50 | 751 | 8.5e-36 | 148 |
| NP_001262604.1 | 24.50 | 751 | 1.2e-35 | 147 |
| NP_732036.2 | 24.50 | 751 | 1.2e-35 | 147 |

### TRIM63 (Q969Q1)

**Description:** E3 ubiquitin-protein ligase TRIM63

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001188786.1 | 21.65 | 328 | 9.0e-11 | 63 |
| NP_723600.2 | 21.65 | 328 | 1.2e-10 | 63 |
| NP_725843.1 | 19.30 | 259 | 1.3e-06 | 50 |
| NP_611390.2 | 19.30 | 259 | 1.3e-06 | 50 |
| NP_001286596.1 | 19.30 | 259 | 1.5e-06 | 50 |
| NP_001286597.1 | 19.30 | 259 | 1.7e-06 | 50 |
| NP_001137707.1 | 19.30 | 259 | 1.8e-06 | 50 |
| NP_523776.1 | 31.18 | 93 | 1.4e-04 | 44 |
| NP_477509.1 | 26.74 | 86 | 5.7e-04 | 42 |

### TUSC3 (Q13454)

**Description:** Dolichyl-diphosphooligosaccharide--protein glycosyltransferase subunit TUSC3

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_609204.2 | 57.05 | 319 | 1.7e-139 | 399 |

### UAP1 (Q16222)

**Description:** UDP-N-acetylhexosamine pyrophosphorylase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_723183.1 | 50.00 | 508 | 5.1e-178 | 509 |
| NP_001285673.1 | 50.00 | 508 | 5.1e-178 | 509 |
| NP_609032.1 | 50.00 | 508 | 6.2e-178 | 511 |

### UBA52 (P62987)

**Description:** Ubiquitin-ribosomal protein eL40 fusion protein

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_476776.1 | 97.66 | 128 | 3.1e-90 | 257 |
| NP_001260018.1 | 97.66 | 128 | 3.1e-90 | 257 |
| NP_476778.1 | 100.00 | 76 | 5.0e-49 | 154 |
| NP_572306.1 | 94.81 | 77 | 1.4e-46 | 152 |
| NP_572306.1 | 87.01 | 77 | 8.7e-41 | 137 |
| NP_572306.1 | 87.01 | 77 | 8.7e-41 | 137 |
| NP_572306.1 | 91.18 | 68 | 5.7e-38 | 130 |
| NP_727078.1 | 98.72 | 78 | 1.4e-46 | 157 |
| NP_727078.1 | 98.70 | 77 | 2.5e-46 | 157 |
| NP_727078.1 | 98.70 | 77 | 2.5e-46 | 157 |
| NP_727078.1 | 98.70 | 77 | 2.5e-46 | 157 |
| NP_727078.1 | 98.70 | 77 | 2.5e-46 | 157 |
| NP_727078.1 | 98.70 | 77 | 2.5e-46 | 157 |
| NP_727078.1 | 98.70 | 77 | 2.5e-46 | 157 |
| NP_001284937.1 | 98.72 | 78 | 1.4e-46 | 157 |
| NP_001284937.1 | 98.70 | 77 | 2.5e-46 | 157 |
| NP_001284937.1 | 98.70 | 77 | 2.5e-46 | 157 |
| NP_001284937.1 | 98.70 | 77 | 2.5e-46 | 157 |
| NP_001284937.1 | 98.70 | 77 | 2.5e-46 | 157 |
| NP_001284937.1 | 98.70 | 77 | 2.5e-46 | 157 |
| NP_001284937.1 | 98.70 | 77 | 2.5e-46 | 157 |
| NP_995994.1 | 100.00 | 77 | 6.2e-46 | 158 |
| NP_995994.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_995994.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_995994.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_995994.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_995994.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_995994.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_995994.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_995994.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_995994.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_728908.1 | 100.00 | 77 | 6.2e-46 | 158 |
| NP_728908.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_728908.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_728908.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_728908.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_728908.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_728908.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_728908.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_728908.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_728908.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_523909.2 | 100.00 | 77 | 6.2e-46 | 158 |
| NP_523909.2 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_523909.2 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_523909.2 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_523909.2 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_523909.2 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_523909.2 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_523909.2 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_523909.2 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_523909.2 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_001261383.1 | 100.00 | 77 | 6.2e-46 | 158 |
| NP_001261383.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_001261383.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_001261383.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_001261383.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_001261383.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_001261383.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_001261383.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_001261383.1 | 98.70 | 77 | 1.7e-45 | 157 |
| NP_001261383.1 | 98.70 | 77 | 1.7e-45 | 157 |

### UBB (P0CG47)

**Description:** Polyubiquitin-B

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_727078.1 | 99.56 | 229 | 4.6e-160 | 454 |
| NP_727078.1 | 99.56 | 229 | 7.5e-160 | 453 |
| NP_727078.1 | 99.56 | 229 | 7.5e-160 | 453 |
| NP_727078.1 | 99.56 | 229 | 7.5e-160 | 453 |
| NP_727078.1 | 99.56 | 229 | 7.5e-160 | 453 |
| NP_001284937.1 | 99.56 | 229 | 4.6e-160 | 454 |
| NP_001284937.1 | 99.56 | 229 | 7.5e-160 | 453 |
| NP_001284937.1 | 99.56 | 229 | 7.5e-160 | 453 |
| NP_001284937.1 | 99.56 | 229 | 7.5e-160 | 453 |
| NP_001284937.1 | 99.56 | 229 | 7.5e-160 | 453 |
| NP_995994.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_995994.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_995994.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_995994.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_995994.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_995994.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_995994.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_995994.1 | 99.56 | 229 | 9.8e-157 | 453 |
| NP_995994.1 | 99.35 | 154 | 1.7e-99 | 305 |
| NP_728908.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_728908.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_728908.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_728908.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_728908.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_728908.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_728908.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_728908.1 | 99.56 | 229 | 9.8e-157 | 453 |
| NP_728908.1 | 99.35 | 154 | 1.7e-99 | 305 |
| NP_523909.2 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_523909.2 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_523909.2 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_523909.2 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_523909.2 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_523909.2 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_523909.2 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_523909.2 | 99.56 | 229 | 9.8e-157 | 453 |
| NP_523909.2 | 99.35 | 154 | 1.7e-99 | 305 |
| NP_001261383.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_001261383.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_001261383.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_001261383.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_001261383.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_001261383.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_001261383.1 | 99.56 | 229 | 9.7e-157 | 453 |
| NP_001261383.1 | 99.56 | 229 | 9.8e-157 | 453 |
| NP_001261383.1 | 99.35 | 154 | 1.7e-99 | 305 |
| NP_572306.1 | 90.39 | 229 | 7.2e-148 | 414 |
| NP_572306.1 | 91.82 | 220 | 3.9e-145 | 407 |
| NP_476776.1 | 98.70 | 77 | 1.4e-48 | 156 |
| NP_476776.1 | 98.70 | 77 | 1.4e-48 | 156 |
| NP_476776.1 | 100.00 | 76 | 2.1e-48 | 155 |
| NP_001260018.1 | 98.70 | 77 | 1.4e-48 | 156 |
| NP_001260018.1 | 98.70 | 77 | 1.4e-48 | 156 |
| NP_001260018.1 | 100.00 | 76 | 2.1e-48 | 155 |
| NP_476778.1 | 97.44 | 78 | 3.7e-48 | 155 |
| NP_476778.1 | 97.44 | 78 | 3.7e-48 | 155 |
| NP_476778.1 | 100.00 | 76 | 4.9e-48 | 155 |

### UBC (P0CG48)

**Description:** Polyubiquitin-C

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_995994.1 | 99.85 | 685 | 0.0e+00 | 1349 |
| NP_995994.1 | 99.85 | 685 | 0.0e+00 | 1348 |
| NP_995994.1 | 99.84 | 610 | 0.0e+00 | 1201 |
| NP_728908.1 | 99.85 | 685 | 0.0e+00 | 1349 |
| NP_728908.1 | 99.85 | 685 | 0.0e+00 | 1348 |
| NP_728908.1 | 99.84 | 610 | 0.0e+00 | 1201 |
| NP_523909.2 | 99.85 | 685 | 0.0e+00 | 1349 |
| NP_523909.2 | 99.85 | 685 | 0.0e+00 | 1348 |
| NP_523909.2 | 99.84 | 610 | 0.0e+00 | 1201 |
| NP_001261383.1 | 99.85 | 685 | 0.0e+00 | 1349 |
| NP_001261383.1 | 99.85 | 685 | 0.0e+00 | 1348 |
| NP_001261383.1 | 99.84 | 610 | 0.0e+00 | 1201 |
| NP_727078.1 | 99.81 | 533 | 0.0e+00 | 1050 |
| NP_727078.1 | 100.00 | 532 | 0.0e+00 | 1050 |
| NP_727078.1 | 100.00 | 532 | 0.0e+00 | 1050 |
| NP_001284937.1 | 99.81 | 533 | 0.0e+00 | 1050 |
| NP_001284937.1 | 100.00 | 532 | 0.0e+00 | 1050 |
| NP_001284937.1 | 100.00 | 532 | 0.0e+00 | 1050 |
| NP_572306.1 | 90.88 | 296 | 0.0e+00 | 539 |
| NP_572306.1 | 90.88 | 296 | 0.0e+00 | 539 |
| NP_572306.1 | 90.88 | 296 | 0.0e+00 | 539 |
| NP_572306.1 | 90.88 | 296 | 0.0e+00 | 539 |
| NP_572306.1 | 90.88 | 296 | 0.0e+00 | 539 |
| NP_572306.1 | 90.88 | 296 | 0.0e+00 | 539 |
| NP_572306.1 | 90.39 | 229 | 6.4e-141 | 414 |
| NP_476776.1 | 98.70 | 77 | 3.7e-45 | 157 |
| NP_476776.1 | 98.70 | 77 | 6.5e-45 | 157 |
| NP_476776.1 | 98.70 | 77 | 6.5e-45 | 157 |
| NP_476776.1 | 98.70 | 77 | 6.5e-45 | 157 |
| NP_476776.1 | 98.70 | 77 | 6.5e-45 | 157 |
| NP_476776.1 | 98.70 | 77 | 6.5e-45 | 157 |
| NP_476776.1 | 98.70 | 77 | 6.5e-45 | 157 |
| NP_476776.1 | 98.70 | 77 | 6.5e-45 | 157 |
| NP_476776.1 | 98.70 | 77 | 6.5e-45 | 157 |
| NP_001260018.1 | 98.70 | 77 | 3.7e-45 | 157 |
| NP_001260018.1 | 98.70 | 77 | 6.5e-45 | 157 |
| NP_001260018.1 | 98.70 | 77 | 6.5e-45 | 157 |
| NP_001260018.1 | 98.70 | 77 | 6.5e-45 | 157 |
| NP_001260018.1 | 98.70 | 77 | 6.5e-45 | 157 |
| NP_001260018.1 | 98.70 | 77 | 6.5e-45 | 157 |
| NP_001260018.1 | 98.70 | 77 | 6.5e-45 | 157 |
| NP_001260018.1 | 98.70 | 77 | 6.5e-45 | 157 |
| NP_001260018.1 | 98.70 | 77 | 6.5e-45 | 157 |
| NP_476778.1 | 97.44 | 78 | 1.8e-44 | 156 |
| NP_476778.1 | 97.44 | 78 | 1.8e-44 | 156 |
| NP_476778.1 | 97.44 | 78 | 1.8e-44 | 156 |
| NP_476778.1 | 97.44 | 78 | 1.8e-44 | 156 |
| NP_476778.1 | 97.44 | 78 | 1.8e-44 | 156 |
| NP_476778.1 | 97.44 | 78 | 1.8e-44 | 156 |
| NP_476778.1 | 97.44 | 78 | 1.8e-44 | 156 |
| NP_476778.1 | 97.44 | 78 | 1.8e-44 | 156 |
| NP_476778.1 | 98.70 | 77 | 1.9e-44 | 156 |

### UBXN1 (Q04323)

**Description:** UBX domain-containing protein 1

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_648167.1 | 36.91 | 336 | 5.2e-53 | 176 |
| NP_648165.1 | 36.49 | 211 | 3.7e-27 | 111 |
| NP_001262894.1 | 50.79 | 63 | 6.5e-12 | 66 |
| NP_651202.1 | 50.79 | 63 | 7.1e-12 | 66 |
| NP_001097898.1 | 50.79 | 63 | 8.0e-12 | 66 |
| NP_647773.1 | 42.59 | 54 | 6.2e-07 | 51 |
| NP_001334708.1 | 42.59 | 54 | 6.2e-07 | 51 |
| NP_001246589.1 | 42.59 | 54 | 6.2e-07 | 51 |

### UMOD (P07911)

**Description:** Uromodulin

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_001245876.1 | 35.98 | 189 | 9.3e-19 | 92 |
| NP_001245876.1 | 43.33 | 120 | 3.9e-16 | 83 |
| NP_001245876.1 | 42.10 | 133 | 8.0e-16 | 82 |
| NP_001245876.1 | 31.90 | 232 | 5.9e-14 | 76 |
| NP_001245876.1 | 40.22 | 92 | 4.3e-13 | 73 |
| NP_001245876.1 | 40.40 | 99 | 2.9e-11 | 67 |
| NP_001245876.1 | 33.14 | 169 | 1.4e-10 | 65 |
| NP_001245876.1 | 29.44 | 180 | 1.9e-10 | 65 |
| NP_001245876.1 | 30.25 | 119 | 1.8e-07 | 55 |
| NP_001245876.1 | 50.91 | 55 | 4.1e-07 | 54 |
| NP_001245876.1 | 31.14 | 167 | 1.0e-06 | 52 |
| NP_001245876.1 | 26.90 | 316 | 1.2e-06 | 52 |
| NP_001245876.1 | 29.80 | 151 | 2.0e-06 | 52 |
| NP_001245876.1 | 30.56 | 144 | 1.8e-05 | 48 |
| NP_001245876.1 | 27.27 | 187 | 2.2e-05 | 48 |
| NP_001245876.1 | 31.37 | 153 | 1.2e-04 | 46 |
| NP_001260039.1 | 35.98 | 189 | 9.4e-19 | 92 |
| NP_001260039.1 | 43.33 | 120 | 4.0e-16 | 83 |
| NP_001260039.1 | 42.10 | 133 | 8.3e-16 | 82 |
| NP_001260039.1 | 31.90 | 232 | 6.6e-14 | 76 |
| NP_001260039.1 | 40.22 | 92 | 4.4e-13 | 73 |
| NP_001260039.1 | 40.40 | 99 | 2.9e-11 | 67 |
| NP_001260039.1 | 33.14 | 169 | 1.3e-10 | 65 |
| NP_001260039.1 | 29.44 | 180 | 2.0e-10 | 65 |
| NP_001260039.1 | 30.25 | 119 | 1.7e-07 | 55 |
| NP_001260039.1 | 50.91 | 55 | 4.1e-07 | 54 |
| NP_001260039.1 | 31.14 | 167 | 1.0e-06 | 52 |
| NP_001260039.1 | 26.90 | 316 | 1.3e-06 | 52 |
| NP_001260039.1 | 29.80 | 151 | 2.0e-06 | 52 |
| NP_001260039.1 | 30.56 | 144 | 1.8e-05 | 48 |
| NP_001260039.1 | 27.27 | 187 | 2.2e-05 | 48 |
| NP_001260039.1 | 31.37 | 153 | 1.2e-04 | 46 |
| NP_001245875.1 | 35.98 | 189 | 9.5e-19 | 92 |
| NP_001245875.1 | 43.33 | 120 | 4.2e-16 | 83 |
| NP_001245875.1 | 42.10 | 133 | 8.7e-16 | 82 |
| NP_001245875.1 | 31.90 | 232 | 6.8e-14 | 76 |
| NP_001245875.1 | 40.22 | 92 | 4.7e-13 | 73 |
| NP_001245875.1 | 40.40 | 99 | 3.1e-11 | 67 |
| NP_001245875.1 | 33.14 | 169 | 1.4e-10 | 65 |
| NP_001245875.1 | 29.44 | 180 | 2.1e-10 | 65 |
| NP_001245875.1 | 30.25 | 119 | 1.8e-07 | 55 |
| NP_001245875.1 | 50.91 | 55 | 4.3e-07 | 54 |
| NP_001245875.1 | 31.14 | 167 | 1.1e-06 | 52 |
| NP_001245875.1 | 26.90 | 316 | 1.4e-06 | 52 |
| NP_001245875.1 | 29.80 | 151 | 2.2e-06 | 52 |
| NP_001245875.1 | 30.56 | 144 | 1.9e-05 | 48 |
| NP_001245875.1 | 27.27 | 187 | 2.5e-05 | 48 |
| NP_001245875.1 | 31.37 | 153 | 1.4e-04 | 45 |
| NP_001260031.1 | 35.98 | 189 | 1.4e-18 | 91 |
| NP_001260031.1 | 43.33 | 120 | 3.9e-16 | 83 |
| NP_001260031.1 | 42.10 | 133 | 1.1e-15 | 82 |
| NP_001260031.1 | 33.17 | 208 | 8.3e-14 | 75 |
| NP_001260031.1 | 40.22 | 92 | 5.8e-13 | 73 |
| NP_001260031.1 | 40.40 | 99 | 4.2e-11 | 67 |
| NP_001260031.1 | 33.14 | 169 | 1.7e-10 | 65 |
| NP_001260031.1 | 29.44 | 180 | 2.9e-10 | 64 |
| NP_001260031.1 | 30.25 | 119 | 1.6e-07 | 55 |
| NP_001260031.1 | 50.91 | 55 | 5.8e-07 | 53 |
| NP_001260031.1 | 29.63 | 162 | 9.7e-07 | 53 |
| NP_001260031.1 | 26.90 | 316 | 1.6e-06 | 52 |
| NP_001260031.1 | 29.80 | 151 | 2.1e-06 | 52 |
| NP_001260031.1 | 30.56 | 144 | 1.9e-05 | 48 |
| NP_001260031.1 | 27.27 | 187 | 2.9e-05 | 48 |
| NP_001260031.1 | 31.37 | 153 | 1.7e-04 | 45 |
| NP_001260035.1 | 35.98 | 189 | 1.6e-18 | 91 |
| NP_001260035.1 | 43.33 | 120 | 4.6e-16 | 83 |
| NP_001260035.1 | 42.10 | 133 | 1.4e-15 | 81 |
| NP_001260035.1 | 33.49 | 206 | 1.1e-13 | 75 |
| NP_001260035.1 | 40.22 | 92 | 6.7e-13 | 73 |
| NP_001260035.1 | 40.40 | 99 | 4.7e-11 | 67 |
| NP_001260035.1 | 33.14 | 169 | 1.9e-10 | 65 |
| NP_001260035.1 | 29.44 | 180 | 4.0e-10 | 63 |
| NP_001260035.1 | 30.25 | 119 | 2.0e-07 | 55 |
| NP_001260035.1 | 50.91 | 55 | 6.3e-07 | 53 |
| NP_001260035.1 | 29.63 | 162 | 1.1e-06 | 52 |
| NP_001260035.1 | 26.90 | 316 | 2.1e-06 | 52 |
| NP_001260035.1 | 29.80 | 151 | 2.6e-06 | 51 |
| NP_001260035.1 | 30.56 | 144 | 2.2e-05 | 48 |
| NP_001260035.1 | 27.27 | 187 | 3.8e-05 | 47 |
| NP_001260035.1 | 31.37 | 153 | 2.1e-04 | 45 |
| NP_001260035.1 | 29.24 | 171 | 7.8e-04 | 43 |
| NP_001260038.1 | 35.98 | 189 | 1.7e-18 | 91 |
| NP_001260038.1 | 43.33 | 120 | 5.6e-16 | 83 |
| NP_001260038.1 | 42.10 | 133 | 1.5e-15 | 81 |
| NP_001260038.1 | 31.90 | 232 | 1.3e-13 | 75 |
| NP_001260038.1 | 40.22 | 92 | 6.7e-13 | 73 |
| NP_001260038.1 | 40.40 | 99 | 4.6e-11 | 67 |
| NP_001260038.1 | 33.14 | 169 | 2.0e-10 | 65 |
| NP_001260038.1 | 29.44 | 180 | 4.0e-10 | 63 |
| NP_001260038.1 | 30.25 | 119 | 2.5e-07 | 55 |
| NP_001260038.1 | 50.91 | 55 | 5.7e-07 | 53 |
| NP_001260038.1 | 31.10 | 164 | 1.4e-06 | 52 |
| NP_001260038.1 | 26.90 | 316 | 2.4e-06 | 51 |
| NP_001260038.1 | 29.80 | 151 | 3.1e-06 | 51 |
| NP_001260038.1 | 30.56 | 144 | 2.6e-05 | 48 |
| NP_001260038.1 | 27.27 | 187 | 4.3e-05 | 47 |
| NP_001260038.1 | 31.37 | 153 | 2.2e-04 | 45 |
| NP_001260038.1 | 29.24 | 171 | 1.0e-03 | 43 |
| NP_001260042.1 | 35.98 | 189 | 1.8e-18 | 91 |
| NP_001260042.1 | 43.33 | 120 | 6.0e-16 | 82 |
| NP_001260042.1 | 42.10 | 133 | 1.5e-15 | 81 |
| NP_001260042.1 | 31.90 | 232 | 1.4e-13 | 75 |
| NP_001260042.1 | 40.22 | 92 | 7.1e-13 | 73 |
| NP_001260042.1 | 40.40 | 99 | 5.1e-11 | 67 |
| NP_001260042.1 | 33.14 | 169 | 2.1e-10 | 65 |
| NP_001260042.1 | 29.44 | 180 | 4.4e-10 | 63 |
| NP_001260042.1 | 30.25 | 119 | 2.5e-07 | 55 |
| NP_001260042.1 | 50.91 | 55 | 6.2e-07 | 53 |
| NP_001260042.1 | 31.10 | 164 | 1.5e-06 | 52 |
| NP_001260042.1 | 26.90 | 316 | 2.5e-06 | 51 |
| NP_001260042.1 | 29.80 | 151 | 3.2e-06 | 51 |
| NP_001260042.1 | 30.56 | 144 | 2.6e-05 | 48 |
| NP_001260042.1 | 27.27 | 187 | 4.5e-05 | 47 |
| NP_001260042.1 | 31.37 | 153 | 2.5e-04 | 45 |
| NP_001260042.1 | 29.24 | 171 | 1.0e-03 | 43 |
| NP_001260036.1 | 35.98 | 189 | 1.8e-18 | 91 |
| NP_001260036.1 | 43.33 | 120 | 4.7e-16 | 83 |
| NP_001260036.1 | 42.10 | 133 | 1.4e-15 | 81 |
| NP_001260036.1 | 33.49 | 206 | 1.1e-13 | 75 |
| NP_001260036.1 | 40.22 | 92 | 6.7e-13 | 73 |
| NP_001260036.1 | 40.40 | 99 | 4.9e-11 | 67 |
| NP_001260036.1 | 33.14 | 169 | 2.0e-10 | 65 |
| NP_001260036.1 | 29.44 | 180 | 3.9e-10 | 63 |
| NP_001260036.1 | 30.25 | 119 | 2.0e-07 | 55 |
| NP_001260036.1 | 50.91 | 55 | 6.3e-07 | 53 |
| NP_001260036.1 | 29.63 | 162 | 1.1e-06 | 52 |
| NP_001260036.1 | 26.90 | 316 | 2.2e-06 | 52 |
| NP_001260036.1 | 29.80 | 151 | 2.6e-06 | 51 |
| NP_001260036.1 | 30.56 | 144 | 2.3e-05 | 48 |
| NP_001260036.1 | 27.27 | 187 | 3.9e-05 | 47 |
| NP_001260036.1 | 31.37 | 153 | 2.2e-04 | 45 |
| NP_001260036.1 | 29.24 | 171 | 8.0e-04 | 43 |
| NP_001260030.1 | 35.98 | 189 | 1.9e-18 | 90 |
| NP_001260030.1 | 43.33 | 120 | 4.8e-16 | 83 |
| NP_001260030.1 | 42.10 | 133 | 1.5e-15 | 81 |
| NP_001260030.1 | 33.49 | 206 | 1.2e-13 | 75 |
| NP_001260030.1 | 40.22 | 92 | 7.7e-13 | 72 |
| NP_001260030.1 | 40.40 | 99 | 5.1e-11 | 67 |
| NP_001260030.1 | 33.14 | 169 | 2.1e-10 | 65 |
| NP_001260030.1 | 29.44 | 180 | 4.5e-10 | 63 |
| NP_001260030.1 | 30.25 | 119 | 2.1e-07 | 55 |
| NP_001260030.1 | 50.91 | 55 | 6.7e-07 | 53 |
| NP_001260030.1 | 29.63 | 162 | 1.1e-06 | 52 |
| NP_001260030.1 | 26.90 | 316 | 2.4e-06 | 51 |
| NP_001260030.1 | 29.80 | 151 | 2.8e-06 | 51 |
| NP_001260030.1 | 29.73 | 148 | 2.5e-05 | 48 |
| NP_001260030.1 | 27.27 | 187 | 4.4e-05 | 47 |
| NP_001260030.1 | 31.37 | 153 | 2.4e-04 | 45 |
| NP_001260030.1 | 29.24 | 171 | 1.0e-03 | 43 |
| NP_001260032.1 | 35.98 | 189 | 2.0e-18 | 90 |
| NP_001260032.1 | 43.33 | 120 | 4.9e-16 | 83 |
| NP_001260032.1 | 42.10 | 133 | 1.6e-15 | 81 |
| NP_001260032.1 | 33.49 | 206 | 1.3e-13 | 75 |
| NP_001260032.1 | 40.22 | 92 | 7.9e-13 | 72 |
| NP_001260032.1 | 40.40 | 99 | 5.5e-11 | 67 |
| NP_001260032.1 | 33.14 | 169 | 2.1e-10 | 65 |
| NP_001260032.1 | 29.44 | 180 | 4.6e-10 | 63 |
| NP_001260032.1 | 30.25 | 119 | 2.2e-07 | 55 |
| NP_001260032.1 | 50.91 | 55 | 6.8e-07 | 53 |
| NP_001260032.1 | 29.63 | 162 | 1.2e-06 | 52 |
| NP_001260032.1 | 26.90 | 316 | 2.6e-06 | 51 |
| NP_001260032.1 | 29.80 | 151 | 2.8e-06 | 51 |
| NP_001260032.1 | 30.56 | 144 | 2.5e-05 | 48 |
| NP_001260032.1 | 27.27 | 187 | 4.6e-05 | 47 |
| NP_001260032.1 | 31.37 | 153 | 2.5e-04 | 45 |
| NP_001260032.1 | 29.24 | 171 | 1.0e-03 | 43 |

### VCP (P55072)

**Description:** Transitional endoplasmic reticulum ATPase

| Subject ID | % Identity | Alignment Length | E-value | Bit Score |
|:-----------|:-----------|:-----------------|:--------|:----------|
| NP_477369.1 | 83.19 | 803 | 0.0e+00 | 1385 |
| NP_001097249.1 | 83.00 | 800 | 0.0e+00 | 1377 |
| NP_001286261.1 | 82.73 | 799 | 0.0e+00 | 1373 |
| NP_001097250.1 | 83.33 | 762 | 0.0e+00 | 1321 |
| NP_609585.1 | 37.21 | 559 | 2.4e-120 | 382 |
| NP_609585.1 | 42.86 | 238 | 1.4e-54 | 202 |
| NP_001260420.1 | 37.21 | 559 | 2.4e-120 | 382 |
| NP_001260420.1 | 42.86 | 238 | 1.4e-54 | 202 |
| NP_996009.2 | 46.52 | 316 | 2.7e-83 | 285 |
| NP_996009.2 | 41.70 | 271 | 1.8e-57 | 212 |
| NP_996009.2 | 39.13 | 253 | 9.2e-47 | 180 |
| NP_996009.2 | 33.33 | 267 | 4.6e-43 | 169 |
| NP_001261512.1 | 46.52 | 316 | 2.7e-83 | 285 |
| NP_001261512.1 | 41.70 | 271 | 1.8e-57 | 212 |
| NP_001261512.1 | 39.13 | 253 | 9.2e-47 | 180 |
| NP_001261512.1 | 33.33 | 267 | 4.6e-43 | 169 |
| NP_001261513.1 | 46.52 | 316 | 4.3e-83 | 285 |
| NP_001261513.1 | 41.70 | 271 | 2.8e-57 | 212 |
| NP_001261513.1 | 39.13 | 253 | 1.2e-46 | 180 |
| NP_001261513.1 | 33.33 | 267 | 4.4e-43 | 169 |
| NP_001163371.1 | 46.52 | 316 | 5.3e-83 | 285 |
| NP_001163371.1 | 41.70 | 271 | 3.7e-57 | 211 |
| NP_001163371.1 | 39.13 | 253 | 1.3e-46 | 180 |
| NP_001163371.1 | 33.33 | 267 | 4.8e-43 | 169 |

